---
output: html_document
editor_options: 
  chunk_output_type: console
---
# Methods
```{r libs, echo=FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R")
```

## Study participants, protocols and training interventions

Study I was designed to examine effects of low- and moderate-volume on responses to acute exercise and long-term training within participants. Forty-one healthy individuals were recruited and 34 of these completed at least 85% of the prescribed sessions and were thus included in subsequent data analyses. Reasons for not completing the trial included injury not related to the study (*n =* 1), pain or discomfort during exercises (*n = *5) and non-adherence to the study protocol. There were no differences in characteristics between participants included in or excluded from data analysis in Study I. Study II was designed to study the effects of resistance training *per se* and effects of variable volume on selected markers related to ribosome biogenesis. Participants were therefore recruited to a training group (*n =* 11) and a non-training control group (*n =* 8). Eligible for participation were young (Study I 18-40; Study II 18-35), non-smoking men and women. Exclusion criteria included a training history of more than one weekly session during the last 12 (Study I) or six (Study II) months leading up to the study. Participants were also screened for intolerance to local anesthetic, current or previous injuries affecting their ability to perform resistance training, self-reported symptoms or history of disease, intake of medication or supplements with known effects on adaptations to training. Participant characteristics for both studies are shown in Table \@ref(tab:characteristics-table). 

```{r characteristics-table, results = "asis", message=FALSE, warning=FALSE, echo=FALSE}

### Read data on included/excluded, body composition and cybex data
characteristics <- read_excel("./data/study-1/body-composition/bodycomp_DXA.xlsx") 
participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";")
load("./data/study-1/strength/cybex.rda")

library(knitr); library(kableExtra)

# Study 1 characteristics
cybex.temp <- data.frame(cybex) %>%
  filter(timepoint %in% c("pre", "session1"), type=="Isom")%>%
  group_by(subject)%>%
  summarise(trq=max(torque.max))

studyI <- participants %>%
  inner_join(characteristics) %>%
  filter(subject!="FP10") %>% # removes participant excluded prior to comecing the study
  filter(timepoint=="pre") %>%
  inner_join(cybex.temp) %>%
  dplyr::select(subject, sex, include, age, height, weight, fat.whole, lean.whole,BMC.whole, trq)%>%
  mutate(fat=((fat.whole/1000)/weight)*100,
         trq.lean=trq/(lean.whole/1000)) %>%
  inner_join(participants) %>%
        mutate(group = if_else(include == "incl", "Included", "Excluded"), 
               sex = factor(sex, levels = c("female", "male"), 
                            labels = c("Female", "Male")), 
               study = "Study I", 
               bmi = weight/(height/100)^2) %>%
        dplyr::select(participant = subject, 
                      study,
                      sex,group, age, height, weight,bmi,
                      fat.whole, 
                      lean.whole,
                      BMC.whole) 

# 
# age.m1 <- lme(age ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# weight <- lme(weight ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# height <- lme(height ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# fat    <- lme(fat ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# mvc    <- lme(trq.lean ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# 
# summary(age.m1)

# summary(weight)

# summary(height)

# summary(fat)

# summary(mvc)
  
### No differences between included not included.

# Study 2 characteristics 



dxa_whole <- read_excel("./data/study-2/body-composition/tr010_dxa_standarddefinitions.xlsx") %>%
  dplyr::select(Etternavn, Kjønn,Målingsdato,Alder, Vekt:`HeleVev%fett`) %>%
  pivot_longer(names_to = "variable", values_to = "value", cols = Vekt:`HeleVev%fett`) %>%
  mutate(value = gsub("kg", "", value), 
         value = gsub("[[:space:]]", "", value, fixed = FALSE), 
         value = as.numeric(value)) %>%

  separate(Etternavn, into = c("Study", "participant")) %>%
  dplyr::select(participant, Date = Målingsdato, variable, value) %>%
  filter(!(participant %in% c("P08", "P20", "P17"))) %>%
  group_by(participant) %>%
  mutate(time = if_else(Date == min(Date), "S0", 
                        if_else(Date == max(Date), "post1w", 
                                "S12")), 
         leg = if_else(variable == "BeinhøyreMagermasse", "R", "L")) %>%
  filter(variable %in% c("HeleBeinmasse", 
                         "HeleFettmasse", 
                         "HeleVevmasse", 
                         "HeleMagermasse" ,
                         "Vekt", 
                         "HeleTotalmasse")) %>%
  pivot_wider(names_from = variable, values_from = value) %>%
  mutate(fat_rel = 100*(HeleFettmasse/HeleTotalmasse)) %>%
          filter(time == "S0") 




studyII <- read_excel("./data/study-2/body-composition/tr010_age_participant.xlsx") %>%
  inner_join(dxa_whole) %>%
  mutate(age = as.numeric(difftime(recruited, dob)/365.2425), 
         bmi = weight/(height/100)^2, 
         study = "Study II") %>%
          dplyr::select(participant, 
                        study,
                        sex,
                        group,
                        age,
                        height, 
                weight = Vekt,
                bmi,

                fat.whole = HeleFettmasse,
                lean.whole = HeleMagermasse,
                BMC.whole = HeleBeinmasse) %>%
        mutate(sex = factor(sex, levels = c("F", "M"), 
                            labels = c("Female", "Male"))) 
        

# Combine both studies and make table
rbind(studyI, studyII) %>%

        mutate(rel_lean = ((lean.whole/1000) / weight) * 100, 
               rel_fat =  ((fat.whole/1000) / weight) * 100) %>%
        dplyr::select(participant:bmi, rel_lean:rel_fat) %>%
        pivot_longer(names_to = "variable", values_to = "values", cols = age:rel_fat ) %>%

        
        group_by(study, group, sex, variable) %>%
        summarise(m = mean(values), 
                  sd = sd(values)) %>% 
        ungroup() %>%
        mutate(m.sd = if_else(variable == "age", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                              if_else(variable == "bmi", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                      if_else(variable == "height", paste0(sprintf("%.0f", m), " (", sprintf("%.0f", sd), ")"), 
                                              if_else(variable == "rel_fat", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                      if_else(variable == "rel_lean", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                              paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")")))))),
               group = factor(group, levels = c("Included", "Excluded", "experiment", "control"), 
                              labels = c("Included", "Excluded", "Training", "Control"))) %>%
        dplyr::select(study:variable, m.sd) %>%
        pivot_wider(names_from = variable, values_from = m.sd) %>%
        arrange(study, group) %>%
        dplyr::select(study, group, sex, age, height, weight, rel_fat, rel_lean) %>%

  kable(format = "latex", 
        booktabs = TRUE, 
        caption = "Participant characteristics", 
        escape = FALSE, 
        col.names = c(" ", " ", "Sex", "Age (years)", "Stature\n(cm)", "Mass (kg)", "Fat mass (\\%)", "Lean mass (\\%)")) %>%
        
        kable_styling(font_size = 7) %>%
        collapse_rows(1:2) %>%

  footnote(general = "Data are means and (SD)",
           general_title = "", 
           footnote_as_chunk = TRUE)


```



Each training session started with a light standardized warm-up (5 min ergometer cycling and 10 repetitions each of push-ups, sit-ups, back-extensions and squats). Before each exercise in the main program, one set of 10 repetitions were performed in the specific exercise with approximately 50% of 1RM. 

Both studies were fully or partially performed as within-participant studies as each participant had their legs assigned to different training conditions (not including the control group in Study II). Allocation was performed after enrollment where each participant had their legs randomized to either low- or moderate volume (Study I), or variable or constant volume (Study II). 

In Study I, the low-volume protocol consisted of a single set of each exercise and the moderate-volume consisted of three sets per exercise. Three unilateral leg exercises were used (leg press, leg curl and knee extension). The moderate volume-leg commenced all sessions and the low volume-leg performed a single set of each exercise in the rest between second and third set of the moderate volume training protocol.  

In Study II, only unilateral knee-extension was performed in an effort to concentrate the stimulus to the quadriceps muscles.<!-- Check this --> The constant-volume leg performed six sets of 10RM throughout the study and variable leg performed six sets in session one to four, three sets in session five to eight and nine sets in session nine to twelve with same intensity (10RM).      


### Ethical considerations
Both studies were approved by the local ethics committee Lillehammer University College/Inland Norway University of Applied Sciences and the Norwegian Centre for Research Data. In accordance with the *Declaration of Helsinki*[@RN2548] the studies were pre-registered in publicly accessible databases (Study I, ClinicalTrials.gov Identifier: NCT02179307; Study II, https://osf.io/wa96y). Participants were informed of the study design, potential risks and sources of discomfort prior to giving their informed consent.    


## Measures of muscle mass
<!-- 
```{r body-comp-measures, fig.cap="Long caption.", fig.scap="Short caption", out.extra='',fig.height=4, fig.width=4, echo=FALSE, cache=TRUE}

mr_dxa_fig <- readRDS("./figures/methods/mr-dxa.RDS")

mr_dxa_fig


```
-->
In Study I muscle mass was measured by magnetic resonance imaging (MRI) and dual energy X-ray absorptiometry (DXA) prior to and after the intervention. Both MRI and DXA measurements were completed during the same visit to the laboratory. Participants were instructed to refrain from strenuous physical activity during the last 48 h leading up to the measurements. The post-training measurements were completed at least 48 h after the last strength testing session. Participants were asked to refrain from food consumption during 2 h leading up to the measurements. 

MRI images were obtained from the mid-thigh and analyzed by the same investigator blinded for time (pre- and post-training) and condition (low- and moderate-volume). Multiple images were used to estimate the cross-sectional area of the extensor muscles at the same distance from the knee-joint. 

See figure <!-- \@ref(fig:body-comp-measures)-->

Dallin et al. recently estimated the [@RN2541]
<!-- MRI has shown better agreement with cadaver... see ref in Dallin -->

## Muscle strength assessments

Muscle strength was with  


## Blood variables

## Muscle tissue sampling and preparations for downstream analyses
Muscle samples were obtained under local anesthesia (Study I, Xylocaine, \SI{10}{\mg\per\ml} with adrenalin \SI{5}{\micro\gram\per\ml}, AstraZeneca, Oslo, Norway; Study II, Lidocaine Mylan, \SI{10}{\mg\per\ml}, Mylan Ireland Ltd, Ireland) with a fine needle (12-14 gauge; Universal-plus, Medax, Italy) operated with a spring-loaded instrument (Bard Magnum, Bard Norway AS, Norway). Sampling was performed as previously described
[@RN824],<!-- Hayot 2005 relible micriboiopsy technique -->
with modifications. Anestheisa was injected in the subcutanous tissue with care taken not to inject anesthesia into the muscle itself. Following a short period (5 min) the effect of the anesthesia was confirmed using an injection needle. Following pilot experiments we decided not to use an insertion cannula as described in [@RN824] as the biopsy needle itself could be used to puncture the skin and muscle fascia. This also resulted in less discomfort. Several passes through the same skin puncture was made to obtain sufficient material for downstream analyses. A smaller needle (14 vs. 12 gauge) was used to further minimized discomfort in Study II where more biopsies were sampled over a shorter time span, with exception from when material was used for immunhistochemistry. The first biopsy was sampled at one third of the distance between the patella to the *anterior superior iliac spinae* with subsequent biopsies sampled $\sim$\SI{2}{cm} proximal to previous samples. In Study II samples obtained more than one week apart were sampled with closer proximity and distally from previous samples but never at previous sampling sites. 

The microbiopsy technique produces smaller samples compared to other biopsy techniques
[@RN2549]<!--Ekblom describes the Bergström and conchotome -->, and thus requires several passes to produce sufficient material for multiple downstream experiments. However, reports confirms that the microbiopsy technique is comparable to the traditionally used Bergström technique in several measures of muscle characteristics at the same time as being well tolerated
[@RN824; 
@RN2553]. 
Any reported differences in fiber type distributions between sampling techniques have been suggested relating to differences in  sampling depth 
[@RN2553;
@RN2552]. <!-- Hayot -->    
For determination of fiber type distributions, a threshold of 200-300 fibers has been suggested as a suitable sample size per specimen as more fibers does not reduce the variation between dupliacte samples
[@RN874].<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
In Study I one or several pieces of muscle were chosen for analysis of fiber type distributions (described in detail below). The total number of fibers were counted from these specimens 

```{r immuno-methods, fig.cap="Number of fibres in immunohistochemistry analyses", fig.scap="Fiber counts", warning=FALSE, echo = FALSE, warning=FALSE, fig.width = 4, fig.heigh = 6, fig.align='center'}


source("./R/libraries.R")
source("./R/themes.R")

# Sample set up (included samples)

legs <- read_csv2("./data/study-1/body-composition/oneThreeSetLeg.csv") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


# Total number of fibers per sample

low_fiber_counts <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total = type1 + type2a + type2ax + type2x) %>%
        filter(total < 200)
        
total_histogram <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total = type1 + type2a + type2ax + type2x) %>%        
        
        ggplot(aes(total)) + geom_histogram(binwidth = 30, 
                                            fill = "blue", alpha = 0.4) + 
        dissertation_theme() + 
        scale_x_continuous(limits = c(0, 1600),
                           expand = c(0,0),
                           breaks = c(0, 200, 400, 600, 800, 1000, 1200, 1400, 1600), 
                           labels = c(0, "", 400, "", 800, "", 1200, "", 1600)) + 
        scale_y_continuous(limits = c(0, 20), 
                           breaks = c(0, 5, 10, 15, 20), 
                           expand = c(0,0)) +
        
        labs(x = "Total number of fibers per sample", 
             y = "Counts") + 
        theme(axis.text.x = element_text(hjust = c(0, rep(0.5, 7), 1)))
        
### Comparison between "duplicate samples"

corr_plot_fiber_types <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total  = type1 + type2a + type2ax + type2x, 
               type1  =(type1 / total)*100, 
               type2a =(type2a / total)*100, 
               type2ax=(type2ax / total)*100, 
               type2x =(type2x / total)*100) %>%  
        filter(timepoint %in% c(2, 3)) %>%
        dplyr::select(subject, timepoint, leg,total, type1:type2x) %>%
        pivot_longer(names_to = "type", values_to = "percentage", cols = type1:type2x) %>%
        mutate(timepoint = paste0("t", timepoint)) %>%
        pivot_wider(names_from = timepoint, values_from = c(percentage, total)) %>%

        mutate(type = factor(type, levels = c("type1", 
                                              "type2a", 
                                              "type2ax", 
                                              "type2x"), 
                             labels = c("Type I", 
                                        "Type IIA", 
                                        "Type IIAX", 
                                        "Type IIX"))) %>%
        
        
        ggplot(aes(percentage_t2, percentage_t3)) +  geom_point(alpha = 0.4, 
                                                                 color = "blue") + 
        facet_wrap(~type, ncol = 2, strip.position = "top")  +

        scale_x_continuous(limits = c(0, 100)) + 
        scale_y_continuous(limits = c(0, 100)) + 
        
        geom_abline(intercept = 0, slope = 1) + 
        dissertation_theme() + 
        theme(strip.background = element_blank(), 
              panel.border = element_rect(color = "black", fill = NA))
        


plot_grid(total_histogram, corr_plot_fiber_types, ncol = 1, rel_heights = c(1, 2)) + 
        draw_plot_label(label=c("A", "B"),
                        x = c(0.02, 0.02), 
                        y = c(0.98, 0.6),
                        hjust=.5, vjust=.5, 
                        size = label.size)




```



## Gene expression analysis

## Determination of protein abundance

## Statistics and data analysis






TO DO:

- For methods discussion, compare product length, efficiencies and ct values in relation to RQI-values. See Fleige 2006 for reference.


## Gene expression analysis 

### Normalization



- An external reference gene was added at a constant amount in Trizol preps
- A normalization factor was used to express relative target gene abundance per-weight tissue.
- In qPCR the linearised expression (effectivety ^cq) was used to express the fraction of external reference per total RNA. 
- In RNA-seq the external reference gene was sequenced and counts were used to express external RNA as a fraction of total RNA.
- In both cases the normalization factor was calculated as mw * counts.

A simulation to see that this is equivalent to tissue used in prep when no measurement errors exists.

```{r}

library(tidyverse)

expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 250, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) 



expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 50, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) %>%
         ggplot(aes(rna.mg, mg.inprep,  color = mg)) + geom_point(size = 2)




```















## Training protocols

A full body protocol was used in study I including 