---
output: html_document
bibliography: bib/complete-bib.bib
editor_options: 
  chunk_output_type: console
  markdown: 
    wrap: 72
---

# Methods

```{r libs, echo=FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R")
source("./R/themes.R")
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

## Study overview and participants

Study I was designed to examine effects of low- (single-set per exercise) and moderate-volume (three sets per exercise, multiple-sets) on
responses to acute exercise and long-term training within participants.
A schematic overview of Study I can be seen in Figure \@ref(fig:study1-overview).
Prior to the training intervention, participants reported to the laboratory for bilateral baseline muscle biopsies (*m. vastus lateralis*).
Bilateral muscle biopsies were additionally sampled prior to, and 60 minutes after the fifth training session as well as after the intervention (Figure \@ref(fig:study1-overview)).
Assessments of muscle strength (unilateral isokinetic and isometric knee extension torque as well as unilateral knee extension and leg press one repetition maximum) were performed twice at baseline and further measured in weeks 3, 5, 9 and after the intervention (Figure \@ref(fig:study1-overview)).
Body composition measurements (magnetic resonance imaging and dual-energy X-ray absorptiometry) were performed at baseline and after the intervention.

Study II was designed to study the effects of resistance training *per se* and effects of variable compared to constant inter-session volume on selected markers related to ribosome biogenesis.
Participants were therefore recruited to an experimental group (*n =* 11) and a non-training control group (*n =* 8).
An overview of Study II can be seen in Figure \@ref(fig:study2-overview).
Baseline muscle strength (unilateral isokinetic and isometric knee-extension torque) was assessed during three initial visits to the laboratory, with the last baseline-assessment performed at least seven days prior to the first biopsy sampling. Follow-up measures of muscle strength in the experimental group were completed three and nine days after the last training session. 
Muscle thickness (*m. vastus lateralis*) was measured bilaterally prior to the study as well as two and eight days after the last training session in the experimental group.
Muscle biopsies were sampled bilaterally prior to any training, 48 hours after the first, fourth, fifth, eighth, ninth and twelfth session as well as eight days after the twelfth session (Figure \@ref(fig:study2-overview)).
The control group in Study II performed the same initial assessments as the experimental group. Muscle thickness was again assessed after a control period of 2-4 weeks.
Follow-up muscle biopsies were sampled from one leg 48 hours after the first biopsy as well as after the control period. Follow-up strength assessments were performed 24 hours after the last biopsy.


```{r study1-overview, fig.cap="Schematic representation of Study I, see text for details.  ", fig.scap="Study I, schematic overview", fig.height=2.8, fig.width=4.72, fig.align = "center"}


source("./R/themes.R")


data.frame(x = c(-8, 13), 
           y = c(-0.1, 0.6)) %>%
        ggplot(aes(x, y)) + 
        geom_blank() + 
        
        
        
        ## Horizontal text
        annotate("richtext", x = -1.5, 
                 y = c(0.52,
                       0.47, 
                       0.42,
                       0.3, 
                       0.15, 
                       0.06),
                 label = c("Biopsy sample", 
                           "Muscle mass assessment", 
                           "Strength assessment", 
                           "Training frequency<br>sessions week<sup>-1</sup>", 
                           "Training intensity", 
                           "Within-participant<br>training volume allocation"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) + 
        
        # Measurements
        annotate("richtext", 
                 y = c(rep(0.52, 4), 
                       rep(0.47, 2), 
                       rep(0.42, 6)), 
                 x = c(0, 1.9, 2, 12, 
                       -0.2, 12.5, 
                       -0.2, 0, 2.5, 4.5, 8.5, 12.5), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 4), 
                           rep("&Dagger;", 2), 
                           rep("*", 6))) +
        
        
        
        
        # Training frequency
        annotate("rect", 
                 xmin = seq(0, 11), 
                 xmax = seq(0.8, 11.8), 
                 ymin = rep(0.2, 12), 
                 ymax = c(0.3, 0.35, 0.3, 0.35, 0.3, rep(0.35, 3), 0.3, 0.35, 0.35, 0.3)) +
        
        # Training frequency y-axis
        
        annotate("segment", 
                 x =    c(-0.5, -0.5, -0.5, -0.5, -0.5), 
                 xend = c(-0.5, -0.7, -0.7, -0.7, -0.7), 
                 y =    c(0.2,   0.35,  0.3,  0.25,  0.2), 
                 yend = c(0.35,   0.35,  0.3,  0.25,  0.2)) +
        annotate("text", 
                 x = rep(-0.9, 4), 
                 y = c(0.2, 0.25, 0.3, 0.35), 
                 label = c(0, 1, 2, 3), 
                 size = text.size) +
        
        # Training intensity
        annotate("rect", 
                 fill = c("gray40", "gray60", "gray70"),
                 xmin = c(0, 2, 5), 
                 xmax = c(2, 5, 12), 
                 ymin = 0.12, 
                 ymax = 0.16) +
        
        annotate("text", 
                 x = c(0.1, 2.1, 5.1), 
                 hjust = 0,
                 y = 0.14, 
                 size = text.size,
                 label = c("10RM", "8RM", "7RM")) +
        
        # Training volume 
        
        
        annotate("rect", 
                 fill = c(group.study.color[1], group.study.color[2]),
                 xmin = 0, 
                 xmax = 12, 
                 ymin = c(0.06, 0.01), 
                 ymax = c(0.1, 0.05)) +
        
        annotate("text", 
                 x = 1, 
                 hjust = 0,
                 y = c(0.085, 0.035), 
                 size = text.size,
                 label = c("Moderate volume: Three sets per exercise", 
                           "Low volume: One set per exercise")) +
        
        # Time frame
        
        annotate("segment", 
                 x =    c(0, seq(0, 12)), 
                 xend = c(12, seq(0, 12)), 
                 y =    c(-0.025, rep(-0.025, 13)),
                 yend = c(-0.025, rep(-0.05, 13))) +        
        annotate("text", 
                 x = c(0, 2, 4, 6, 8, 10, 12, 6), 
                 y = c(rep(-0.07, 7), -0.1),  
                 label = c(0, 2, 4, 6, 8, 10, 12, "Week"), 
                 size = text.size) +
        
        
        theme_void()



```




```{r study2-overview, fig.cap="Schematic representation of Study II, see text for details.  ", fig.scap="Study II, schematic overview", fig.height=3.2, fig.width=4.72, fig.align = "center"}


data.frame(x = c(-8, 16), 
           y = c(-0.1, 0.6)) %>%
        ggplot(aes(x, y)) + 
        geom_blank() + 
        
        
     ## Group labels
        
        annotate("richtext", 
                 x = -2, 
                 y = c(0.6, 0.0125), 
                 label = c("<b>Experimental group</b>", 
                           "<b>Control group</b>"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size + 0.5, 
                 hjust = 1) +
        
        
           
        ## Horizontal text
        annotate("richtext", x = -1.5, 
                 y = c(0.52,
                       0.47, 
                       0.42,
                       0.3, 
                       0.15),
                 label = c("Biopsy sample", 
                           "Muscle thickness assessment", 
                           "Strength assessment", 
                           "Training volume<br>sets session<sup>-1</sup>", 
                           "Training intensity"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) + 
        
        # Measurements
        annotate("richtext", 
                 y = c(rep(0.52, 8), 
                       rep(0.47, 3), 
                       rep(0.42, 3)), 
                 x = c(0, 1.5, 4.5, 5.5, 8.5, 9.5, 12.5, 14.5, 
                       -0.2, 12.5, 14.5, 
                       -0.5, 12.9, 14.9), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 8), 
                           rep("&Dagger;", 3), 
                           rep("*", 3))) +
        
        
        # Training volume
        annotate("rect", 
                 fill = group.study.color[4],
                 xmin = seq(0.8, 11.8), 
                 xmax = seq(0.95, 11.95), 
                 ymin = rep(0.2, 12), 
                 ymax = rep(0.3, 12)) +
        
        annotate("rect", 
                 fill = group.study.color[5],
                 xmin = seq(1.05, 12.05), 
                 xmax = seq(1.2, 12.2), 
                 ymin = rep(0.2, 12), 
                 ymax = c(rep(0.3, 4), rep(0.25, 4), rep(0.35, 4))) +
        
        
        # Training frequency y-axis
        
        annotate("segment", 
                 x =    c(-0.5, -0.5, -0.5, -0.5, -0.5), 
                 xend = c(-0.5, -0.7, -0.7, -0.7, -0.7), 
                 y =    c(0.2,   0.35,  0.3,  0.25,  0.2), 
                 yend = c(0.35,   0.35,  0.3,  0.25,  0.2)) +
        
                annotate("text", 
                 x = rep(-0.9, 4), 
                 y = c(0.2, 0.25, 0.3, 0.35), 
                 label = c(0, 3, 6, 9), 
                 size = text.size) +
        
        # Training intensity
        annotate("rect", 
                 fill = c("gray40"),
                 xmin = c(0.8), 
                 xmax = c(12.2), 
                 ymin = 0.12, 
                 ymax = 0.16) +
        
        annotate("text", 
                 x = c(1.1), 
                 hjust = 0,
                 y = 0.14, 
                 size = text.size,
                 label = c("10RM")) +
        

        # Time frame
        
        annotate("segment", 
                 x =    c(1, seq(1, 12)), 
                 xend = c(12, seq(1, 12)), 
                 y =    c(0.1, rep(0.1, 12)),
                 yend = c(0.1, rep(0.09, 12))) +        
        annotate("text", 
                 x = c(1, 6, 12, 6), 
                 y = c(rep(0.07, 3), 0.025),  
                 label = c(1, 6, 12, "Session"), 
                 size = text.size) +
        
        
        # Control group
        
        
  annotate("richtext", x = -1.5, 
                 y = c(-0.075,
                       -0.125, 
                       - .175),
                 label = c("Biopsy sample", 
                           "Muscle thickness assessment", 
                           "Strength assessment"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) +
  # Annotations
     annotate("richtext", 
                 y = c(rep(-0.075, 3), 
                       rep(-0.125, 2), 
                       rep(-0.175, 2)), 
                 x = c(0, 1.5,  12.5,
                       -0.2, 12.5, 
                       -0.5, 12.9), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 3), 
                           rep("&Dagger;", 2), 
                           rep("\U2051", 2))) +
        
        annotate("segment", 
                 x =    c(1.5, 1.5, 12), 
                 xend = c(12, 1.5, 12), 
                 y =    c(-0.25, -0.25, -0.25),
                 yend = c(-0.25, -0.26, -0.26)) +        
        annotate("text", 
                 x = c(6.25), 
                 y = c(-0.34),  
                 label = c("2-4 weeks"), 
                 size = text.size) +
        
        
        
        theme_void()




```


```{r pwr_calc, echo=FALSE, warning=FALSE, message=FALSE}
library(pwr)
# Approximate power calculations that reproduce power calculations made in GPower

# To be able to detect a training-induced 5-unit change (...) given a within-group standard 
# deviation of 12, an alpha-level of 0.05 and a desired power of 0.8, 40 subjects needs to be
# recruited to the experiment in study B.


# Ronnestad 2007
# Isokinetic testing 
sd.iso <- 4 * sqrt(10)
diff.isok <- 6

d.ronnestad.isokinetic <- diff.isok/sd.iso

# pwr.t.test(d = d.ronnestad.isokinetic, power = 0.8, type = "paired")

### Knee extensor CSA
sd.csa <- 2 * sqrt(10)
diff.csa <- 3
d.ronnestad.csa <- diff.csa/sd.csa 

# pwr.t.test(d = d.ronnestad.csa, power = 0.8, type = "paired")

# Mitchell MVC
# This comparison is between 80%, 1-set and 80%1RM 3-set in MVC 
single.pre <- 192
single.post <- 248
  
multiple.pre <- 188
multiple.post <- 255

diff.mvc.mitchell <- (((multiple.post/multiple.pre)-1) - ((single.post / single.pre)-1))*100
d.mitchell.mvc <- diff.mvc.mitchell/sd.iso # Standard deviations borrowed from Rønnestad
# pwr.t.test(d = d.mitchell.mvc, power = 0.8, type = "paired")

# Mitchell CSA

diff.csa <- 3.5
sd.csa <- 2 * sqrt(12)
d.mitchell.csa <- diff.csa/sd.csa
# pwr.t.test(d = 0.47, power = 0.8, type = "paired")


es.min <- round(min(c(d.ronnestad.isokinetic, d.ronnestad.csa, d.mitchell.mvc, d.mitchell.csa)),2)
es.max <- round(max(c(d.ronnestad.isokinetic, d.ronnestad.csa, d.mitchell.mvc, d.mitchell.csa)),2)
```

Recruitment to both studies was done through advertising and word of mouth, primarily at Lillehammer University College/Inland University of Applied Sciences. Potential participants were interviewed and matched against inclusion and exclusion criteria. During initial interviews, participants were informed of the study design and time requirements. Participants were also informed about potential risks and sources of discomfort associated with the study prior to giving their informed consent.

To be eligible for participation in both studies, participants had to be young (Study I age 18-40; Study II 18-35) and non-smoking. Both men and women were considered for participation. Exclusion criteria included a training history of more than one weekly session during the last 12 (Study I) or six (Study II) months leading up to the study. Participants were also screened for intolerance to local anesthetic, current or previous injuries affecting their ability to perform resistance training, self-reported symptoms or history of disease, intake of medications or supplements with known effects on adaptations to training.
Participant characteristics for both studies are shown in Table
\@ref(tab:characteristics-table).

Forty-one healthy individuals were recruited and 34 of these completed at least 85% of the prescribed sessions and were thus included in subsequent data analyses. Reasons for not completing the trial included injury not related to the study (*n =* 1), pain or discomfort during exercises (*n =* 5) and non-adherence to the study protocol. There were no systematic differences in characteristics between participants included in or excluded from data analysis in Study I.

Prior to the study, all participants reported that they previously had been engaged in sporting activities. At enrollment, twenty participants reported regular physical activities ranging from once every other week to four times per week. Ten participants reported performing resistance-type exercise at the time of enrollment limited to no more than once a week.  


```{r characteristics-table, results = "asis", message=FALSE, warning=FALSE, echo=FALSE}

### Read data on included/excluded, body composition and cybex data
characteristics <- read_excel("./data/study-1/body-composition/bodycomp_DXA.xlsx") 
participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";")
load("./data/study-1/strength/cybex.rda")

library(knitr); library(kableExtra)

# Study 1 characteristics
cybex.temp <- data.frame(cybex) %>%
  filter(timepoint %in% c("pre", "session1"), type=="Isom")%>%
  group_by(subject)%>%
  summarise(trq=max(torque.max))

studyI <- participants %>%
  inner_join(characteristics) %>%
  filter(subject!="FP10") %>% # removes participant excluded prior to comecing the study
  filter(timepoint=="pre") %>%
  inner_join(cybex.temp) %>%
  dplyr::select(subject, sex, include, age, height, weight, fat.whole, lean.whole,BMC.whole, trq)%>%
  mutate(fat=((fat.whole/1000)/weight)*100,
         trq.lean=trq/(lean.whole/1000)) %>%
  inner_join(participants) %>%
        mutate(group = if_else(include == "incl", "Included", "Excluded"), 
               sex = factor(sex, levels = c("female", "male"), 
                            labels = c("Female", "Male")), 
               study = "Study I", 
               bmi = weight/(height/100)^2) %>%
        dplyr::select(participant = subject, 
                      study,
                      sex,group, age, height, weight,bmi,
                      fat.whole, 
                      lean.whole,
                      BMC.whole) 

# 
# age.m1 <- lme(age ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# weight <- lme(weight ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# height <- lme(height ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# fat    <- lme(fat ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# mvc    <- lme(trq.lean ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# 
# summary(age.m1)

# summary(weight)

# summary(height)

# summary(fat)

# summary(mvc)
  
### No differences between included not included.

# Study 2 characteristics 



dxa_whole <- read_excel("./data/study-2/body-composition/tr010_dxa_standarddefinitions.xlsx") %>%
  dplyr::select(Etternavn, Kjønn,Målingsdato,Alder, Vekt:`HeleVev%fett`) %>%
  pivot_longer(names_to = "variable", values_to = "value", cols = Vekt:`HeleVev%fett`) %>%
  mutate(value = gsub("kg", "", value), 
         value = gsub("[[:space:]]", "", value, fixed = FALSE), 
         value = as.numeric(value)) %>%

  separate(Etternavn, into = c("Study", "participant")) %>%
  dplyr::select(participant, Date = Målingsdato, variable, value) %>%
  filter(!(participant %in% c("P08", "P20", "P17"))) %>%
  group_by(participant) %>%
  mutate(time = if_else(Date == min(Date), "S0", 
                        if_else(Date == max(Date), "post1w", 
                                "S12")), 
         leg = if_else(variable == "BeinhøyreMagermasse", "R", "L")) %>%
  filter(variable %in% c("HeleBeinmasse", 
                         "HeleFettmasse", 
                         "HeleVevmasse", 
                         "HeleMagermasse" ,
                         "Vekt", 
                         "HeleTotalmasse")) %>%
  pivot_wider(names_from = variable, values_from = value) %>%
  mutate(fat_rel = 100*(HeleFettmasse/HeleTotalmasse)) %>%
          filter(time == "S0") 




studyII <- read_excel("./data/study-2/body-composition/tr010_age_participant.xlsx") %>%
  inner_join(dxa_whole) %>%
  mutate(age = as.numeric(difftime(recruited, dob)/365.2425), 
         bmi = weight/(height/100)^2, 
         study = "Study II") %>%
          dplyr::select(participant, 
                        study,
                        sex,
                        group,
                        age,
                        height, 
                weight = Vekt,
                bmi,

                fat.whole = HeleFettmasse,
                lean.whole = HeleMagermasse,
                BMC.whole = HeleBeinmasse) %>%
        mutate(sex = factor(sex, levels = c("F", "M"), 
                            labels = c("Female", "Male"))) 
        

# Combine both studies and make table
rbind(studyI, studyII) %>%

        mutate(rel_lean = ((lean.whole/1000) / weight) * 100, 
               rel_fat =  ((fat.whole/1000) / weight) * 100) %>%
        dplyr::select(participant:bmi, rel_lean:rel_fat) %>%
        pivot_longer(names_to = "variable", values_to = "values", cols = age:rel_fat ) %>%

        
        group_by(study, group, sex, variable) %>%
        summarise(m = mean(values), 
                  sd = sd(values)) %>% 
        ungroup() %>%
        mutate(m.sd = if_else(variable == "age", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                              if_else(variable == "bmi", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                      if_else(variable == "height", paste0(sprintf("%.0f", m), " (", sprintf("%.0f", sd), ")"), 
                                              if_else(variable == "rel_fat", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                      if_else(variable == "rel_lean", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                              paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")")))))),
               group = factor(group, levels = c("Included", "Excluded", "experiment", "control"), 
                              labels = c("Included", "Excluded", "Training", "Control"))) %>%
        dplyr::select(study:variable, m.sd) %>%
        pivot_wider(names_from = variable, values_from = m.sd) %>%
        arrange(study, group) %>%
        dplyr::select(study, group, sex, age, height, weight, rel_fat, rel_lean) %>%

  kable(format = "latex", 
        booktabs = TRUE, 
        caption = "Participant characteristics", 
        escape = FALSE, 
        col.names = c(" ", " ", "Sex", "Age (years)", "Stature\n(cm)", "Mass (kg)", "Fat mass (\\%)", "Lean mass (\\%)")) %>%
        
        kable_styling(font_size = 7) %>%
        collapse_rows(1:2) %>%

  footnote(general = "Data are means and (SD)",
           general_title = "", 
           footnote_as_chunk = TRUE)


```



## Resistance training interventions

Studies were fully or partially performed as within-participant studies
as each participant had their legs assigned to different training
conditions (not including the control group in Study II). Allocation was
performed after enrollment where each participant had their legs
randomized to either low- or moderate volume (Study I, see Figure \@ref(fig:study1-overview)), or variable or constant volume (Study II).

Each training session started with a light, standardized warm-up (5 min
ergometer cycling and 10 repetitions each of push-ups, sit-ups,
back-extensions and squats). Before each exercise in the main program,
one set of 10 repetitions were performed in the specific exercise with
approximately 50% of 1RM.

In Study I, the low-volume protocol consisted of a single set of each
exercise and the moderate-volume consisted of three sets per exercise.
Three unilateral leg exercises were used (leg press, leg curl and knee
extension). The moderate volume-leg commenced all sessions and the low
volume-leg performed a single set of each exercise in the rest between
second and third set of the moderate volume training protocol.

In Study II, only unilateral knee-extension was performed. The constant-volume leg performed six sets of 10RM throughout the study and variable leg performed six sets in session one to four, three sets in session five to eight and nine sets in session nine to twelve with same relative intensity (see Figure \@ref(fig:study2-overview)).


## Muscle strength assessments

### Isokinetic and isometric maximal torque
Maximal isokinetic and isometric unilateral knee-extension strength was determined by use of a dynamometer (Study I: Cybex 6000, Cybex International, Medway USA; Study II: Humac Norm, CSMi, Stoughton, MA, USA).
After a brief warm-up (5 min ergometer cycling, RPE 12-14), participants were secured at the hip and shoulders in the dynamometer with the knee joint aligned to its rotation axis.
Individual settings were recorded and used in subsequent measurements.
Participants were familiarized with the test protocol by performing three sub-maximal efforts at each angular speed. 
In Study I, three angular velocities were used to determine isokinetic torque (60$^{\circ}$, 120$^{\circ}$ and 240$^{\circ} ~\times$ sec$^{-1}$), in Study II an angular velocity of 90$^{\circ} ~\times$ sec$^{-1}$ was used for this purpose. 
In Study I, participants performed two attempts at 60$^{\circ} ~\times$ sec$^{-1}$ and three attempts at 120 and 240$^{\circ}~\times$ sec$^{-1}$.
In Study II, three attempts were made at the designated angular velocity.
In both studies, attempts within each angular speed were performed in immediate succession.
After isokinetic testing, the lever was fixed at 30$^{\circ}$ (full extension $=90^{\circ}$), participants were instructed push with maximal effort for 5 seconds and the maximal isometric torque was recorded.
Two attempts were made for maximal isometric torque in Study I and a single attempt was made in Study II.
Sixty seconds of restitution was given between each measurement in both studies with the exception of between isometric contractions in Study I where a 30 second restitution period was used.     

In subsequent assessment sessions, the first measurement was performed on alternate legs.
In Study II, the dynamometer allowed for participants to remain seated for assessments of both legs. This was not possible in Study I as the measurement system required participants to be re-seated for assessment of the contra-lateral leg.

### One-repetition maximum
One repetition-maximum (1RM) was assessed in unilateral leg-press and knee-extension in Study I.
Each exercise was assessed after a specific warm-up (ten, six and three repetitions at $50$, 75 and $85\%$ of the anticipated maximum). Attempts were made with increasing resistance (four to six attempts) and one repetition maximum was defined as the highest resistance successfully lifted through the full range of motion.

### Strength assessment frequency (and statistics)

In Study I, maximal values from each assessment and time-point was used in statistical analyses including two separate assessments at baseline, separated by at least four days. Strength was determined 


In Study II, the maximal value from all successful attempts were used in statistical analyses with unsuccessful attempts identified based on obvious misinterpretation of instructions or failure to reach maximal subjective effort.  
Strength assessments were separated by at least 48 hours from preceding training sessions. 

At baseline, 1RM, isokinetic and isometric strength assessments were performed twice, separated by at least four days. The maximum value achieved for each of the tests was used in subsequent analysis. Strength tests were separated by at least 48 hours from preceeding training sessions. A combined measure of muscle strength was calculated as the average of all tests (1RM, isometric and isokinetic), wherein each test modality was given equal weight. A subset of the participants (n=18) performed strength assessment during the course of the study (at week two, five and nine). For the remaining participants, ordinary training sessions were prioritised when participants missed training or testing due to illness or scheduling difficulties. 

<!-- From TR010 -->

Muscle strength was assessed as maximal voluntary isokinetic (90&deg; sec<sup>-1</sup>) and isometric (60&deg; angle, fully extended leg 0&deg;) knee extension torque. After a brief warm-up (5-min cycling, RPE 12-14), participants were seated and secured in the individually adjusted dynamometer. Participants were instructed to gradually increase their effort during three warm-up repetitions (50, 60 and, 70% of subjective maximal effort). After a 30-sec restitution period participants were instructed to perform three repetitions with maximal effort in the concentric phase. Sixty seconds after the isokinetic test the lever automatically moved to a 60&deg; angle and participants were instructed to push against the lever enough to see feedback from the visual feedback system. After an additional 15-sec restitution period, participants were instructed to push against the lever with maximal effort. Within the same assessment session, participants remained seated in the dynamometer for measurement perfomed on both legs. The first measurement was alternated between legs every other session. For statistical treatment of the data, all successful attempts were used. The last strength assessment at baseline was performed at least seven days prior to the first biopsy sampling. At least one of the baseline strength tests was performed on separate day with two sessions allowed to be perform on the same day with a short rest between assessments. Post training assessments were performed 48 hours and eight days after the last session.


## Measures of muscle mass

In Study I muscle mass was measured by magnetic resonance imaging (MRI)
and dual energy X-ray absorptiometry (DXA) prior to and after the
intervention. Both MRI and DXA measurements were completed during the
same visit to the laboratory. Participants were instructed to refrain
from strenuous physical activity during the last 48 h leading up to the
measurements. The post-training measurements were completed at least 48
h after the last strength testing session. Participants were asked to
refrain from food consumption during 2 h leading up to the measurements.

MRI images were obtained from the mid-thigh and analyzed by the same
investigator blinded for time (pre- and post-training) and condition
(low- and moderate-volume). Multiple images were used to estimate the
cross-sectional area of the extensor muscles at the same distance from
the knee-joint.

See figure <!-- @ref(fig:body-comp-measures)-->

<!--Dallin et al. recently estimated the [@RN2541]
     MRI has shown better agreement with cadaver... see ref in Dallin 

    ## Muscle strength assessments

    Muscle strength was with  -->

In Study II, m. vastus lateralis muscle thickness was measured by B-mode ultrasonography (SmartUs EXT-1M, Telemed, Vilnius, Lithuania) using a 39 mm 12 MHz, linear array probe. At least three images were captured for each leg per time-point. Between each image acquisition the probe was relocated to the same position where on the skin and subsequently marked on a soft transparent plastic sheet superimposed on the thigh.  

Muscle thickness of m. vastus lateralis and m. rectus femoris were measured using  Transverse images were obtained $\sim$ 60 % distally from the trochanter major towards the femoral lateral epicondyle. Landmarks such as moles and scars were also marked on the plastic sheet for relocation of the scanned areas during post-training measurements. During analysis, pre and post images from the same participant were analyzed consecutively using the Fiji software,66 and by two independent researchers. The average muscle thickness of the three images captured per muscle was used for further analyses.

<!-- From tr010 -->
For determination of body composition participants were scanned using DXA (Lunar Prodigy densitometer, GE Healthcare, Madison, WI, USA) with the standard scanning mode (13-25 cm). Participants were lying supine within the scanning bed reference lines, with a strap secured around the ankles to ensure a standardized body position in each scan. The scans were conducted with participants in a fasted state between 07.00-10.00 AM, with empty bladder and wearing only under-wear. Prior to each scan, a phantom scan was run to prevent baseline drifting from affecting analyses. The same technician was used at each time point. Analyses was performed using GE enCORE version 17.0 software (GE Healthcare). Region of interest was customized for covering upper thigh, marked with a sqaure from pubic symphysis to lateral part of tuberculum major, and distal to art. genu.

Muscle thickness (MT) was measured using a B-mode ultra sound unit (SmartUS EXT-1M, Telemed, Vilnius, Lithuania). Participants lay supine in a relaxed position for 20 min before assessments, with their feet strapped in a standardized position. A mark was set on the line 60% of the distance between Spinia Iliac Anterior Superior and the lateral femur condyle. MT of *m.vastus lateralis* was measured applying a water-soluble transmission gel (Aquasonic 100 Ultrasound Transmission Gel; Parker Laboratories Inc., Fairfield, NJ, USA), and a 39 mm 12 MHz ultrasound probe was placed perpendicular to the site of interest without pressing the skin. When the quality of the image was satisfactory, evident as distinct upper and lower muscle fascia, three images were captured, where the probe was relocated to the same position between each image. Position of the probe was marked on the skin and subsequently marked on a transparent paper to ensure similar probe placement for both the right and left *m.vastus lateralis* at subsequent assessments. Analyses were done in ImageJ Fiji [@RN2561] with images cropped and coded to ensure blinding of the assessor.



## Muscle tissue sampling and preparations for downstream analyses

Muscle samples were obtained under local anesthesia (Study I, Xylocaine,
\SI{10}{\mg\per\ml} with adrenalin \SI{5}{\micro\gram\per\ml},
AstraZeneca, Oslo, Norway; Study II, Lidocaine Mylan,
\SI{10}{\mg\per\ml}, Mylan Ireland Ltd, Ireland) with a fine needle
(12-14 gauge; Universal-plus, Medax, Italy) operated with a
spring-loaded instrument (Bard Magnum, Bard Norway AS, Norway). Sampling
was performed as previously described
[@RN824],<!-- Hayot 2005 relible micriboiopsy technique --> with
modifications. Anesthesia was injected in the subcutaneous tissue with
care taken not to inject anesthesia into the muscle itself. Following
pilot experiments we decided not to use an insertion cannula as
described in [@RN824] as the biopsy needle itself could be used to
puncture the skin and muscle fascia. This also resulted in less
discomfort. Several passes through the same skin puncture was made to
obtain sufficient material for downstream analyses. A smaller needle (14
vs. 12 gauge) was used to further minimize discomfort in Study II where
more biopsies were sampled over a shorter time span, with exception from
when material was used for immunohistochemistry. The first biopsy was
sampled at one third of the distance between the patella to the
*anterior superior iliac spinae* with subsequent biopsies sampled
$\sim$\SI{2}{cm} proximal to previous samples. In Study II, samples
obtained more than one week apart were sampled with closer proximity and
distally from previous samples but never at previous sampling sites.

The micro biopsy technique produces smaller samples compared to other
biopsy techniques
[@RN2549]<!--Ekblom describes the Bergström and conchotome -->, and thus
requires several passes to produce sufficient material for multiple
downstream experiments. However, reports confirms that the micro biopsy
technique is comparable to the traditionally used Bergström technique in
several measures of muscle characteristics at the same time as being
well tolerated [@RN824; @RN2553]. Any reported differences in fiber type
distributions between sampling techniques have been suggested relating
to differences in sampling depth [@RN2553; @RN2552].<!-- Hayot -->

For determination of fiber type distributions, a threshold of 200-300
fibers has been suggested as a suitable sample size per specimen as more
fibers does not reduce the variation between dupliacte samples
[@RN874].<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
In Study I, one or several pieces of muscle (total weight
$\sim$\SI{15}{mg}) were chosen per sampling for analysis of fiber type
distributions (described in detail below). The total number of fibers
were counted from these specimens (Figure ref fig). Using an average of
fibers from the first sampling time point the between leg coefficient of
variation was determined to 14% for Type I fibers and 11.3 for type II
fibers. The between leg variation in Type I fibers is similar to what
has been previously reported...

<!-- Latest paper on variability between samples from the same leg and due to number of fibers counted
Appl Physiol Nutr Metab
. 2020 Apr;45(4):368-375. doi: 10.1139/apnm-2019-0263. Epub 2020 Mar 24.

The within-leg was similar to between leg comparison in our samples. This might highlight sampling depth variation in microbiopsy technique in immunohistochemistry ?

To calculate variation in proportions

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3875499/
-->




#### Total RNA extraction
```{r rna-muscle-weight}

# Descriptive data on total RNA extractions from both studies

study1_sample <- read_excel("./data/study-1/sample_weight.xlsx") %>%
  mutate(rna = (conc1.5 * 5) * elution.volume, 
         sample = paste(subject, leg, timepoint, sep = "_")) %>%
  dplyr::select(sample, weight, rna) %>%
    mutate(study = "Study1") 


complete_sample <- readRDS("./data/derivedData/study-2/tot-rna.RDS") %>%
  filter(outlier != "out") %>%
  mutate(sample = paste(participant, leg, time, sep = "_")) %>%
  group_by(sample) %>%
  summarise(weight = mean(tissue_weight, na.rm = TRUE), 
            rna = mean(rna, na.rm = TRUE)) %>%
  mutate(study = "Study2") %>%
  rbind(study1_sample) 

## Calculate mean weights and rna per weight ##

desc_to_rna <- complete_sample %>%
  group_by(study) %>%
  summarise(m.weight = mean(weight, na.rm = TRUE), 
            s.weight = sd(weight, na.rm = TRUE), 
            m.rna_weight = mean(rna/weight, na.rm = TRUE), 
            s.rna_weight = sd(rna/weight, na.rm = TRUE)) 




## Figure on RNA descriptives

rna_fig_a <- complete_sample %>%
  ggplot(aes(weight, rna, fill = study)) + 
  geom_point(shape = 21, alpha = 0.2) +
    scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
      scale_y_continuous(limits = c(0, 25000), 
                     expand = c(0,0), 
                     breaks = c(0, 5000, 10000, 15000, 20000, 25000)) +
  dissertation_theme() + 
  theme(axis.title.x = element_blank(), 
        axis.text.x = element_blank(), 
        axis.line.x = element_blank(), 
        axis.ticks.x = element_blank(), 
        legend.position = "none") + 
  labs(y = "RNA (ng)")


rna_fig_b <- complete_sample %>%
  ggplot(aes(weight, fill = study)) + geom_histogram() +
  facet_grid(study ~ .) +
  dissertation_theme() + 
  scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
  theme(legend.position = "none", 
        strip.background = element_blank(),
        strip.text = element_blank()) +

  labs(y = "Counts", 
       x = "Weight (mg)")

# plot_grid(rna_fig_a, rna_fig_b,
#           ncol = 1,
#          rel_heights = c(1, 0.75), 
#          align = "v", 
#          axis = "trbl")
# 


### Calculate RNA per tissue at baseline
# 
# complete_sample %>%
#   separate(sample, into = c("subject", "leg", "timepoint")) %>%
#   filter(timepoint %in% c("S0", "w0")) %>%
#   mutate(rna_corrected = if_else(study == "Study1", rna * 1.125, rna)) %>%
#   lmer(rna ~ study + weight + (1|subject),
#       
#       data = .) %>%
#   summary()
#   
#   ggplot(aes(rna_corrected / weight, fill = study)) + geom_histogram()
#   
#   
#   group_by(study) %>%
#   summarise(rna_weight = mean(rna/weight), 
#             rna_corrected = mean(rna_corrected/weight)) %>%
#   print()
# 
# 
# 

```

Total-RNA was extracted from frozen muscle samples with weights measured at collection using a protocol modified from
[@RN2672]<!-- Chomczynski and Sacchi -->
using Trizol reagent (Life Technologies).
Muscle tissue was homogenized in \SI{300}{ul} Trizol with mechanical disruption achieved by Zirconium Oxide Beads (0.5 mm, Next Advance, Inc., New York, USA) and a bead mill (Bullet blender, Next Advance). External, non-mammalian RNA (Lambda PolyA External Standard Kit, Takara Bio Europe, Saint-Germain-en-Laye, France) was added with the initial volume of Trizol to enable per-weight normalization in subsequent analyses. After homogenization, additional Trizol was added to a total volume of \SI{1}{ml}. Phase separation was achieved by centrifugation after addition of chloroform (\SI{200}{ul}). The upper phase (\SI{400}{ul} in Study I; \SI{450}{ul} in Study II) was transferred to a fresh tube and RNA was precipitated using isopropanol (\SI{500}{ul}). After incubation (10 min, room-temperature) and centrifugation (12000 g, 10 min at 4$^{\circ}$C) the resulting RNA pellet was washed three times in chilled 75% ethanol.

As previously mentioned, to minimize discomfort with a larger number of biopsies sampled over a short time, a smaller needle was used for most biopsies in Study II. This generally led to a less tissue used for RNA extractions (Figure XX).  


#### Protein extraction immunoblotting 

<!-- TRaining response -->

Aliquots of muscle-tissue (approximately 25 mg wet weight) were homogenised using a plastic pestle in ice-cold lysis buffer (2 mM HEPES pH 7.4, 1 mM EDTA, 5 mM EGTA, 10 mM MgCl~2~, 1$\%$ Triton X-100) spiked with protease and phosphatase inhibitors (Halt, Thermo Fischer Scientific, Life Technologies AS, Oslo Norway), incubated at $4^{\circ}$ for 1 hr and centrifuged for 10 min at 10 000 g and 4$^{\circ}$C, after which the supernatants were collected.
Total protein concentrations were determined on a 1:10 dilution (Pierce Detergent Compatible Bradford Assay Reagent, Thermo Fischer Scientific). The remaining supernatant was diluted to 1.5 $\mu g  \times \mu l^{-1}$ total protein in lysis buffer and 4X Laemmli sample buffer (Bio-Rad Laboratories AB, Oslo Norway) containing 2-Mercaptoethanol.
Samples were heated to 95$^{\circ}$C for 5 min and stored at -20$^{\circ}$C until further processing.
During analyses, protein samples (20 $\mu g$ of total protein) were separated at 300 V for 30 min using 4-20% gels (Criterion TGX, Bio-Rad), followed by wet transfer to PVDF membranes (0.2 $\mu m$ Immun-Blot, Bio-Rad) at 300 mA for 3 h.
Gel electrophoresis and protein transfer were performed at 4$^{\circ}$C.
Membranes were then stained using a reversible total protein stain (Pierce Reversible Protein Stain, ThermoFischer Scientific) to ensure appropriate protein transfer.
Primary antibodies were purchased from Cell Signaling Technology (Leiden, The Netherlands): mTOR (mTOR^Ser2448^: #5536; pan: #4517), S6 kinase 1 (p85 S6K1^Thr412^: #9206; p70 S6K1^Thr389^: #9234; pan: #2708), ribosomal protein S6 (rpS6^Ser235/236^: #4858; pan: #2317).
Membranes were blocked for 2 h in tris-buffered saline (TBS, 20 mM Tris, 150 mM NaCl) containing $3\%$ bovine serum albumin and $0.1\%$ Tween-20, followed by over-night incubation with primary antibodies targeting either the phosphorylated or non-phosphorylated epitope diluted in blocking buffer, followed by 2 h incubation with secondary horseradish peroxidase-conjugated antibodies diluted in TBS containing 0.1% Tween-20 and $5\%$ skimmed milk. Membranes were washed in TBS containing $0.1\%$ Tween-20 for 6 $\times$ 5 min after incubation with primary antibody, and for 8 $\times$ 5 min after incubation with secondary antibodies. For rpS6 and mTOR antibodies, following chemiluminescent detection (SuperSignal™ West Femto Maximum Sensitivity Substrate, ThermoFischer Scientific), membranes were incubated with hydrogen peroxide (15 min, 37$^{\circ}$C) to inactivate the horseradish peroxidase (HRP), as described by Sennepin *et al.* [-@RN1930], followed by over-night incubation with primary or secondary antibodies as described above. If the phosphorylated epitope was targeted during the first incubation, antibodies for the non-phosphorylated epitope were used in the second and vice versa. HRP inactivation did not affect the phosphospecific to non-phosphorylated signal ratios. Importantly, as this technique did not involve removing the first primary antibody, antibodies from different hosts (mouse or rabbit) were used for phosphorylated and non-phosphorylated epitopes respectively. As the antibody targeting p70 S6K1^Thr389^ had the same host as the pan-antibody, total-protein was used to normalise chemiluminescent signals.  All incubation and washing steps were performed at $4^{\circ}$C using an automated membrane processor (BlotCycler, Precision Biosystems, Mansfield, MA, USA), except for p70 S6K1 experiments, which were performed by hand at room temperature with incubations at $4^{\circ}$C. For mTOR and rpS6, total-protein and chemiluminescence quantification was calculated as the mean value of two separate experiments. S6K1 was quantified once for each phospho-specific antibody. Total-protein content was quantified using ImageJ [@RN2259], and was defined as the mean grey value of the whole well with between-well values subtracted as background. Chemiluminescence signals were quantified using Image Studio Lite (LI-COR Biotechnology, Lincoln, Nebraska USA). 

<!-- Tr010 -->

Protein was extracted from Trizol preparations according to manufacturers instructions and [@RN2050]<!-- Kopec trizol protocol --> with modifications. The remaining aqueous phase was removed and DNA was precipitated by the addition of \SI{300}{ul} of absolute ethanol followed by gentle centrifugation (2000 g, 5 min at room temperature). An aliquot of the phenol-ethanol phase, corresponding to ~1.75 mg of tissue, was transferred to to a fresh tube. After addition of at least two volumes of isopropanol and incubation (10 min at room temperature), samples were centrifuged (7500 g, 10 min 4$^{\circ}$C) and a pellet formed. The pellet was washed three times in 95% ethanol with each wash separated by centrifugation (5000g, 5 min at room temperature). After the last wash all liquid was removed and \SI{45}{ul} of Kopec buffer [@RN2050] was added (5% SDS, 10 mM Tris, 140 mM NaCl and 20 mM EDTA, pH 8; containing protease and phosphatase inhibitors). Pellets were incubated at 50$^{\circ}$C for three hours after which the majority of samples were dissolved. Any undissolved material was sedimented by centrifugation (10000 g, 10 min at room temperature). Protein concentrations were measured (Pierce Detergent Compatible Bradford Assay, ThermoFisher Scientific). Sample were normalized in 4X Laemmli buffer (Bio-Rad Norway AS, Oslo, Norway), boiled (95$^{\circ}$C, 5 min) and stored at -20$^{\circ}$C before later use.

## RNA analysis

<!-- TR010 -->
Complementary DNA (cDNA) was synthesized in technical duplicates from 500 ng of total RNA using random hexemer and anchored Oligo-dT primers (Thermo Fisher Scientific) together with Superscript IV (Thermo Fisher Scientific) according to manufacturer's instruction. qPCR reactions were performed with diluted cDNA (2 ul, 1:25 dilution), a SYBR-green based commercial master mix (PowerUp™ SYBR™ Green Master Mix, Thermo Fisher) and, target-specific primers (500 nM) in 10ul reaction volumes using a real-time detection system (QuantStudio 5 Real-Time PCR System, Thermo Fisher Scientific). Fast cycling was used (1 sec denaturing, 30 sec annealing) after UNG (2 min, 50$^{\circ}$C) and polymerase  (2 min, 95&deg;C) activation. Melt curves were collected from all reactions to confirm single product amplification. Primers were further evaluated by agarose gel electrophoresis which confirmed primer sizes and non-template control experiments confirming no amplification without template. Primer sequences and their respective average performances are shown in Table 2.

<!-- Training response -->

Approximately 25 $mg$ of wet muscle-tissue was homogenised in a total volume of 1 ml of TRIzol reagent (Invitrogen, Life technologies AS, Oslo, Norway) using 0.5 mm RNase-free Zirconium Oxide beads and a bead homogeniser (Bullet Blender, Next Advanced, Averill Park, NY, USA) according to the manufacturer’s instructions.
In order to enable analysis of target gene-expression per-unit tissue weight, an exogenous RNA control ($\lambda$ polyA External Standard Kit, Takara Bio Inc, Shiga, Japan) was added at a fixed amount ($0.04  ~ng  \times ml^{-1}$ of Trizol reagent) per extraction prior to homogenisation, as previously described
[@RN1162;<!-- Ellefsen Improved normalization using an external RNA control -->
@RN1512]<!-- Ellefsen Irisin and FNDC5 -->.
Following phase-separation, 400 $\mu l$ of the upper phase was transferred to a fresh tube and RNA was precipitated using isopropanol.
The resulting RNA pellet was washed three times with 70% EtOH and finally eluted in TE buffer.
RNA quantity and purity was evaluated using a spectrophotometer, all samples had a 260/280 $nm$ ratio > 1.95. RNA was stored at -80$^{\circ}$C until further processing. In the analysis of total RNA content per-unit tissue weight, one sample was excluded prior to analysis due to negative deviation from the expected value based on the relationship between sample weight and RNA content suggesting sample loss in washing steps. 
RNA integrity was assessed by capillary electrophoresis (Experion Automated Electrophoresis Station using RNA StdSens Assay, Bio-Rad) with average integrity scores (RQI) 
Five-hundred nanograms of RNA were reverse transcribed using anchored Oligo-dT, random hexamer primers (Thermo Scientific) and SuperScript IV Reverse Transcriptase (Invitrogen) according to manufacturer's instructions.
All samples were reverse transcribed in duplicates and diluted 1:50 prior to real-time polymerase chain reaction (qPCR).
qPCR reactions were run on a fast-cycling real-time detection system (Applied Biosystems 7500 fast Real-Time PCR Systems, Life technologies AS), with a total volume of 10 $\mu l$, containing 2 $\mu l$ of cDNA, specific primers (0.5 $\mu M$ final concentration) and a commercial master mix (2X SYBR Select Master Mix, Applied Biosystems, Life technologies AS). qPCR reactions consisted of 40 cycles (three seconds 95$^{\circ}$C denaturing and 30 seconds 60$^{\circ}$C annealing).
Melt-curve analyses were performed for all reactions to verify single-product amplification. Gene-specific primers were designed for all targets using Primer-BLAST [@RN1815] and Primer3Plus [@RN1816] and ordered from Thermo Scientific, except for the external RNA control, for which primers were supplied with the kit.
Raw fluorescence data was exported from the platform specific software and amplification curves were modelled with a best-fit sigmoidal model using the qpcR-package [@RN1768] written for R [@R-base]. Threshold cycles (Ct) were estimated from the models by the second-derivate maximum method with technical duplicates modelled independently.
Amplification efficiencies were estimated for every reaction [as described by @RN1934; implemented in @RN1768]. For every primer pair, mean amplification efficiencies ($E$) were utilised to transform data to the linear scale using $E^{-Ct}$. Primer sequences and primer characteristics (i.e. average primer efficiencies and Ct values) are presented in Table 2. Gene expression data was log-transformed prior to statistical analysis. As Ct-values, but not efficiencies are related to RNA integrity [@RN2248], RQI scores were used in the statistical treatment of qPCR data to control for potential degradation effects on a by target basis (see below). 




### Hormonal measurements
Hormone analyses were performed on blood samples collected at five time points: alongside muscle biopsies (Figure 1, four sampling events) and 10 minutes after completion of the fifth training session. Samples were drawn from the antecubital vein into serum-separating tubes and kept at room temperature for 30 min before centrifugation (1500 g, 10 min). Serum was immediately aliquoted and stored at -80$^{\circ}$C until further processing. Serum concentrations of total testosterone, cortisol, growth hormone and insulin-like growth-factor 1 (IGF-1) were measured on an Immulite 1000 analyzer, using kits from the Immulite Immunoassay System menu (Siemens Medical Solutions Diagnostics, NY, USA), performed according to manufacturer’s protocols. Serum Vitamin D (S-25-OH-D) levels were measured in samples collected before and after the intervention using a electrochemiluminescence immunoassay (Roche Cobas Vitamin D total assay, Roche Diagnostics GmbH., Mannheim, Germany) using automated instrumentation (Roche Cobas 6000’s module e601, Roche Diagnostics GmbH., Mannheim, Germany). 




## Statistics and data analysis



In Study I, an *a priori* sample-size calculations ($\beta=20\%$, $\alpha=5\%$) indicated that 40 participants was sufficient to detect $\sim$ 3\% and 5\%-point differences between volume conditions in primary outcomes, training induced changes in muscle cross-sectional area and maximal voluntary strength respectively. Sample-size calculations were based on assumed differences between volume condition corresponding to effect sizes of `r es.min`-`r es.max`, as estimated from previous studies [@RN834; @RN776].

In Study II, an initial sample size calculation was made based on a best case scenario. Data from Brook et al. [@RN1642] indicated that a within-participant differences in RNA per tissue weight of $15\%$ could be detected with seven participants in the experimental group (e.g. 2.79 (0.65) to 3.21 (0.74) \SI{}{\micro\gram\per\micro\litre}, $\alpha<0.05$, $\beta<0.2$, effect size=1.2). The control group was included in the study as a negative control primarily for the sake of systems validation of experimental procedures regarding measures of e.g. ribosomal biogenesis. With a balanced design, accounting for drop-outs eight participants were required in each group. After a preliminary analysis of the experiment (experimental group *n = *7, control group *n = *7)[@RN2822], additional participants (experimental group *n =* 4, control group *n = *1) were recruited to the study to increase the precision of estimates, primarily in analyses within the experimental group.




TO DO:

-   For methods discussion, compare product length, efficiencies and ct
    values in relation to RQI-values. See Fleige 2006 for reference.


### Normalization

-   An external reference gene was added at a constant amount in Trizol
    preps
-   A normalization factor was used to express relative target gene
    abundance per-weight tissue.
-   In qPCR the linearised expression (effectivety \^cq) was used to
    express the fraction of external reference per total RNA.
-   In RNA-seq the external reference gene was sequenced and counts were
    used to express external RNA as a fraction of total RNA.
-   In both cases the normalization factor was calculated as mw \*
    counts.

A simulation to see that this is equivalent to tissue used in prep when
no measurement errors exists.

```{r}

library(tidyverse)

expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 250, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) 



expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 50, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) %>%
         ggplot(aes(rna.mg, mg.inprep,  color = mg)) + geom_point(size = 2)




```

## Meta-analysis of within-session training volume

### Literature search, inclusion criteria and coding of studies

A first set of studies were coded based previously published
meta-analyses [@RN1767; @RN789]. For more recent studies, PubMed, Google
Scholar and SportDiscuss searches were made with search terms being
"training volume", "resistance training", "strength training", "set",
"muscle strength", "muscle hypertrophy" used in different combinations.
Studies examining the effect of within-session training volume on muscle
strength and mass, with all other training variables kept constant
between study groups were considered for inclusion in the meta-analysis.
Studies were further assessed for inclusion based on criteria being; (i)
participants described as healthy without medications affecting muscle
metabolism, (ii) interventions lasting at least 6 weeks and (iii) RT
performed without additional stimuli (e.g. blood flow restriction) at
intensities above 65% of 1RM or 20RM.

All available outcome measures of muscle mass and strength gains in
response to RT were extracted from each study with exception of outcomes
reported both as summaries and individual measures (e.g. muscle
thickness reported as individual muscles and summarized for the whole
muscle group). In such cases the summary was used as outcome. Weekly
training volume was calculated as product of weekly sessions, number of
sets and exercises for each muscle group assessed for muscle hypertrophy
or strength gains. An intervention average of weekly sessions was used
when the number of sessions per week differed over the course of the
intervention. An exercise was assumed to influence an outcome when it
targeted prime movers also assessed for strength or muscle hypertrophy
measures. Participant characteristics were coded based on sex (male,
female or mixed when a study failed to discern between male and
females), age (young, middle-aged, old or mixed), body-mass index (BMI,
calculated from average body mass and height when BMI values were not
available), training status (trained, \> 1 session per week during the
last 6 months leading up to the intervention; untrained \< 1 session).
Study groups were considered independent also in studies utilizing
within-participant models.

### Calculations of effect sizes and statistical analysis

Group-wise effect sizes were calculated for each outcome measure based
on the within-group change score pre- to post-training divided by the
pre-training standard deviation (SD). Pre-training SD's were calculated
as a pooled SD within outcome and study. Variances of the effect size
were calculated using an average effect size across all outcomes within
muscle strength or mass, and correlations specific to each measurement
type (isokinetic-, isometric- or repetition maximum strength tests;
muscle thickness, magnetic resonance imaging, dual energy X‐ray
absorptiometry) estimated from previous studies.\
A correction factor () was applied to both effect sizes and their
variances.

Mixed-effects meta-regression models were used to model the effect of
weekly number of sets on RT-induced muscle mass and strength gains.
Models were fitted in a Bayesian framework using the brms-package
[@RN2562].
