---
output: html_document
editor_options: 
  chunk_output_type: console
---
# Methods

## Study protocols, participants and training interventions

For this thesis two studies were completed consisting of traini


## Measures of muscle mass

```{r body-comp-measures, fig.cap="Long caption.", fig.scap="Short caption", out.extra='',fig.height=4, fig.width=4, echo=FALSE}

mr_dxa_fig <- readRDS("./figures/methods/mr-dxa.RDS")

mr_dxa_fig


```

In Study I muscle mass was measured by magnetic resonance imaging (MRI) and dual energy X-ray absorptiometry (DXA) prior to and after the intervention. Both MRI and DXA measurements were completed during the same visit to the laboratory. Participants were instructed to refrain from strenuous physical activity during the last 48 h leading up to the measurements. The post-training measurements were completed at least 48 h after the last strength testing session. Participants were asked to refrain from food consumption during 2 h leading up to the measurements. 

MRI images were obtained from the mid-thigh and analyzed by the same investigator blinded for time (pre- and post-training) and condition (low- and moderate-volume). Multiple images were used to estimate the cross-sectional area of the extensor muscles at the same distance from the knee-joint. 

See figure \@ref(fig:body-comp-measures)

Dallin et al. recently estimated the [@RN2541]
<!-- MRI has shown better agreement with cadaver... see ref in Dallin -->

## Muscle strength assessments

## Blood variables

## Muscle tissue sampling and preparations for downstream analyses

## Gene expression analysis

## Determination of protein abundance

## Statistics and data analysis






TO DO:

- For methods discussion, compare product length, efficiencies and ct values in relation to RQI-values. See Fleige 2006 for reference.


## Gene expression analysis 

### Normalization



- An external reference gene was added at a constant amount in Trizol preps
- A normalization factor was used to express relative target gene abundance per-weight tissue.
- In qPCR the linearised expression (effectivety ^cq) was used to express the fraction of external reference per total RNA. 
- In RNA-seq the external reference gene was sequenced and counts were used to express external RNA as a fraction of total RNA.
- In both cases the normalization factor was calculated as mw * counts.

A simulation to see that this is equivalent to tissue used in prep when no measurement errors exists.

```{r}

library(tidyverse)

expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 250, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) 



expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 50, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) %>%
         ggplot(aes(rna.mg, mg.inprep,  color = mg)) + geom_point(size = 2)




```















## Training protocols

A full body protocol was used in study I including 