# Results and Discussion

```{r settings, include=FALSE}

library(nlme); library(kableExtra)


```


## Within-session training volume affects training-induced changes in muscle mass and strength and molecular determinants of muscle hypertrophy {#muscle-mass-growth}

```{r comparing-avg-strength-csa, echo=FALSE, warning=FALSE, message=FALSE}


source("./R/study-1/mr-and-dxa-data.R")


participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


### Overall % increase in CSA (not corrected for baseline values)
over.all.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(subject) %>%
  summarise(mean.csa = mean(change, na.rm = TRUE)) %>%
  ungroup() %>%
  summarise(m.csa = mean(mean.csa),
            s.csa = sd(mean.csa))

sets.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(sets) %>%
  summarise(m.csa = mean(change),
            s.csa = sd(change))



### Baseline corrected % increase
csa.corrected.data <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>% # Calculates % change
  group_by(sex) %>%
  mutate(pre = pre-mean(pre)) %>% # Mean centered baseline 
  # (no need for sex-diff in model)
  group_by(subject, sex) %>% # Keeps sex for test in model
  summarise(mean.csa = mean(change, na.rm = TRUE),
            mean.baseline = mean(pre)) # Average over both conditions 



## Anocova with baseline values as co-variate
csa.corr.m1 <- lm(mean.csa ~  mean.baseline,
                  data = csa.corrected.data)

csa.m1 <- lm(mean.csa ~  1,
                  data = csa.corrected.data)



csa.corr.confint  <- round(confint(csa.corr.m1)[1,],1)
csa.corr.estimate <- round(summary(csa.corr.m1)$coef[1,1],1)




csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  group_by(timepoint, sets, sex) %>%
  summarise(m=sprintf("%.1f", round(mean(CSA.avg),1)),
            s=sprintf("%.1f", round(sd(CSA.avg),1)))


csa.r <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject, sex, sets, csa.change) %>%
  spread(sets, csa.change)

csa.r.model <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject,leg, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject,leg, sex, sets, pre, csa.change) %>%
  group_by(sex) %>%
  mutate(pre.m = pre-mean(pre, na.rm = TRUE), 
         sets = factor(sets, levels = c("single", "multiple"))) 

#### Change to ancova ###


csa.m0 <-nlme::lme(csa.change ~ sets +  pre.m,
                random = list(subject = ~1),
                na.action = na.omit, 
                data = csa.r.model, method = "ML", 
                control=list(msMaxIter=120,
                             opt = "nloptwrap", msVerbose = FALSE))




csa.estimate <- sprintf("%.2f",round(coef(summary(csa.m0))[2,1], 2))
csa.confint <- intervals(csa.m0)$fixed[2, c(1,3)]

### t-test Does not change the conclusion 
# csa.ttest <- with(csa.r, t.test(multiple,single, paired=T))
# csa.estimate <- sprintf("%.2f",round(csa.ttest$estimate, 2))
# csa.confint <- csa.ttest$conf.int


csa.corr <- cor.test(csa.r$multiple, csa.r$single)


female.t <- with(csa.r[csa.r$sex=="female",], t.test(multiple,single, paired=T))
female.estimate <- sprintf("%.2f",round(female.t$estimate, 2))
female.confint <- female.t$conf.int


male.t <- with(csa.r[csa.r$sex=="male",], t.test(multiple,single, paired=T))
male.estimate <- sprintf("%.2f",round(male.t$estimate, 2))
male.confint <- male.t$conf.int

#### Strength data for overall stats



rm_str <- read_excel("./data/study-1/strength/strengthTests.xlsx") %>%
        filter(!(exercise %in% c("benchpress", "legcurl"))) %>%
        inner_join(participants) %>%
        filter(include == "incl") %>%

  dplyr::select(subject, sex, sets, exercise, timepoint, load)%>%
  mutate(load = as.numeric(load),
         sets = factor(sets, levels = c("single", "multiple"))) %>%
  spread(timepoint, load) %>%
  mutate(baseline = pmax(pre, session1)) %>%
  dplyr::select(subject, sex, sets, exercise, baseline, week2, week5, week9, post) %>%
  gather(week, load, baseline:post) %>%
  mutate(week = factor(week, levels = c("baseline", "week2", "week5", "week9", "week12", "post"))) %>%
  group_by(exercise) %>%
  mutate(str = load / max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) 

# load cybex data 
load("./data/study-1/strength/cybex.rda")


cybex <- data.frame(cybex) %>%
        mutate(leg = if_else(leg == "Left", "L", "R")) %>%
        inner_join(participants) %>%
        filter(include == "incl", 
               timepoint %in% c("pre","session1", "post")) %>%

        filter(!(subject=="FP6" & timepoint == "post" & speed == 240))  # This removes one record, exclusion due to error in measurement, see protocol FP6


# Clean cybex data set and combine with 1RM 
total_strength <- cybex %>%
  ungroup() %>%
  dplyr::select(subject, timepoint, sets,speed, sex, torque.max) %>%
  spread(timepoint, torque.max) %>%
  rowwise()%>%
  mutate(baseline = max(pre, session1, na.rm=T)) %>%
  ungroup() %>%
  dplyr::select(subject, sex, sets, exercise = speed, baseline,  post) %>%
  gather(week, load, baseline:post) %>%
  group_by(exercise) %>%
  mutate(str = load/max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) %>%
  rbind(rm_str) %>% # This includes all strength tests
  mutate(ex_fam = if_else(exercise %in% c("legext", "legpress"), 
                          "rm", if_else(exercise == 0, "isom", "isok"))) %>%
  group_by(subject, sex, sets, week, ex_fam) %>%
  summarise(str = mean(str, na.rm = TRUE)) %>%
  group_by(subject, sex, sets, week) %>%
  summarise(str = mean(str, na.rm = TRUE)) 


strength.stats <- total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  group_by(sex) %>%
  mutate(BASELINE = baseline - mean(baseline)) %>%
    group_by(sex, subject) %>%
    summarise(increase = mean(increase, na.rm = TRUE),
              BASELINE = mean(BASELINE, na.rm = TRUE)) 




## ANCOVA for the strength 1RM
strength.m1 <- lm(increase ~ sex + BASELINE,
                  data = strength.stats)

strength.corr.confint  <- round(confint(strength.m1)[1,],0)
strength.corr.estimate <- round(summary(strength.m1)$coef[1,1],0)



### Correlation in strength gains ###

str.corr.data <-  total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject, sex, sets, increase) %>%
  spread(sets, increase) 

str.corr <- cor.test(str.corr.data$single,
                          str.corr.data$multiple)

#### Correlation strength and csa increases

# Average increases

csa.str.corr.data <- total_strength %>%
  spread(week, str) %>%
  mutate(str.rel.change = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject,  sets, sex, sets,baseline, str.rel.change) %>%

  inner_join(mr.results %>%
                mutate(condition=leg) %>%
                inner_join(participants) %>%
                filter(include=="incl") %>%
                dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
                spread(timepoint, CSA.avg) %>%
                mutate(csa.rel.change = 100 * ((post/pre)-1)) %>%
                dplyr::select(subject, sets, csa.rel.change)) %>%
        ungroup() 





# The code below could possibly be used to calculate the relative contribution of 
# csa increase to muscle strength increases


# m <- lme4::lmer(str.rel.change ~ csa.rel.change + sets + (1|subject), 
#                 data = csa.str.corr.data)
# 
# 
# 
# mx.null <- lme4::lmer(str.rel.change ~ 1 + (1|subject), data = csa.str.corr.data)
# mx.csa <- lme4::lmer(str.rel.change ~ csa.rel.change  + (1|subject), data = csa.str.corr.data)
# mx.sets <- lme4::lmer(str.rel.change ~ sets + (1|subject), data = csa.str.corr.data)
# 
# 
# 
# boot.fun <- function(x) {
#         data.frame(nlme::VarCorr(x))[2, 4]
# }
# 
# 
# null.boot <- lme4::bootMer(mx.null, boot.fun, nsim = 500, parallel = "snow", cl = 12)
# csa.boot <- lme4::bootMer(mx.csa, boot.fun, nsim = 500)
# sets.boot <- lme4::bootMer(mx.sets, boot.fun, nsim = 500)
# 
# 
# 
# qt.null <- quantile(null.boot$t, c(0.025, 0.975))
# qt.csa <- quantile(csa.boot$t, c(0.025, 0.975))
# qt.sets <- quantile(sets.boot$t, c(0.025, 0.975))
# 
# 
# results <- data.frame(variable = c("sets", "csa"),
#                            m = c(1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4],
#                                  1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4]),
#                            lwr = c(1- qt.sets[1] / qt.null[1],
#                                    1- qt.csa[1] / qt.null[1]),
#                            upr = c(1- qt.sets[2] / qt.null[2],
#                                    1- qt.csa[2] / qt.null[2]))
# 
# 
# 
# 
# 
# csa <- 1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# sets <- 1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# exp <- 1 - data.frame(nlme::VarCorr(mx.exp))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# 

# Average csa and strength gains

avg.str.csa <- csa.str.corr.data %>%
        group_by(subject) %>%
        summarise(str = mean(str.rel.change), 
                  csa = mean(csa.rel.change))


csa.str_corr <- cor.test(avg.str.csa$csa, avg.str.csa$str)

## Comparisons with other published studies on per session gains in csa and strength

### Per session ##
per.session <- read_excel("./data/study-1/trainingdata.xlsx", sheet = 1) %>%
  filter(!is.na(date)) %>%
  group_by(subject) %>%
  summarise(sessions = n()) %>%
        
inner_join(csa.str.corr.data) %>%

  mutate(str.change.day = str.rel.change/((sessions/31) * 12*7),
         csa.change.day = csa.rel.change/((sessions/31) * 12*7), 
         str.session = str.rel.change/sessions,
          csa.session = csa.rel.change/sessions ) %>%
   dplyr::select(-baseline) %>%
  ungroup() %>%
        pivot_longer(names_to = "variable", values_to = "gain", cols = str.rel.change:csa.session) 
       

sum.stats <- cbind(per.session %>%
        group_by(sets, variable, sex) %>%
        summarise(mean = mean(gain), 
                  sd = sd(gain)) %>%
        arrange(variable) %>%
        data.frame(), 
        data.frame(reference.mean = c(0.11, 0.11, 0.11, 0.11,
                              NA, NA, NA, NA, 
                              0.0825, 0.1375,0.0825, 0.1375, 
                              NA, NA, NA, NA, 
                               NA, NA, NA, NA,
                              0.6675, 0.475, 0.6675, 0.475), 
           reference.range = c("[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "", "", "", "",
                               "(0.22)b", "(0.14)b",  "(0.22)b", "(0.14)b", 
                               "", "", "", "",
                              "", "", "", "",
                               "(0.35)b", "(0.22)b","(0.35)b", "(0.22)b" ))) %>%
        mutate(m.sd = if_else(mean > 6, paste0(sprintf("%.1f",mean), " (", sprintf("%.1f",sd), ")"), 
                              paste0(sprintf("%.2f", mean), " (", sprintf("%.2f",sd), ")")), 
               reference = if_else(is.na(reference.mean), "",
                                   paste0(sprintf("%.2f",reference.mean), " ", reference.range))) %>%
        filter(!(variable %in% c("str.change.day") )) %>%
                       dplyr::select(variable, sex, sets, m.sd, reference) %>%
        
        mutate(variable = factor(variable, levels = c("csa.rel.change", 
                                                      "csa.change.day", 
                                                      "csa.session", 
                                                      "str.rel.change", 
                                                      "str.session"), 
                                 labels = c("CSA %-change", 
                                            "CSA %-change per day", 
                                            "CSA %-change per session", 
                                            "Average strength %-change", 
                                            "Average strength %-change per session")), 
               sex = factor(sex, levels = c("female", 
                                            "male"), 
                            labels = c("Female", "Male")), 
               sets = factor(sets, levels = c("single", "multiple"), 
                             labels = c("LOW", "MOD"))) 

## Ahtiaianen on average  20 weeks with 2 sessions per week = 40 sessions

## Muscle hypertrophy
## In the younger cohorts, male average 5.5% (5.6) = 5.5/40 ~ 0.1375 per session
# in females 3.3 (8.6)% = 0.825
# 8.6/3.3 * 3.3/40
# 5.6/5.5 * 5.5/40
# Muscle strength in younger cohorts
#  male    19% (8.6)/40 sessions ~  0.5 % per session
#  female  26.7% (13.9)/40 ~ 
# 8.6/19 * (19/40)
# 13.9/26.7 * 26.7/40

# Wernbom increase per day in quadriceps CSA per day was 0.06%







```

In Study I, the average increases in muscle strength and mass in each volume condition corresponded to what could be expected based on previous research in young, healthy participants (Table \@ref(tab:csa-str-tab)) 
[@RN2007;<!-- Wernbom 2007 -->
@RN1741],<!-- Ahtiainen  -->
indicating the general efficacy of the training program.


(ref:wernbom-citation) @RN2007

(ref:ahtiainen-citation) @RN1741

```{r csa-str-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

sum.stats %>%
        arrange(variable, sex, sets) %>%
  dplyr::select(-variable) %>%
        kable(format = "latex", booktabs = TRUE, linesep = "",
              col.names = c("Sex", "Condition", "Mean (SD)", "Reference"), 
              caption = "Training induced changes in muscle CSA and average strength in Study I")  %>%
        collapse_rows(c(1, 4))  %>%
        kable_styling(font_size = 7) %>%
          
          pack_rows("CSA %-change", 1, 4) %>%
        pack_rows("CSA %-change per day", 5, 8) %>%
          pack_rows("CSA %-change per session", 9, 12) %>%
          pack_rows("Average strength %-change", 13, 16) %>%
          pack_rows("Average strength %-change per session", 17, 20) %>%
  
        add_footnote(label = c(paste0("Estimates from Wernbom et al. ", "(ref:wernbom-citation)"),
                               paste0("Estimates from Ahtiainen et al. ", "(ref:ahtiainen-citation)")), 
                     notation = "alphabet")

```

The moderate-volume condition consistently showed favorable adaptations when compared to the low-volume condition in measures of muscle hypertrophy and strength gains (Figure \@ref(fig:comb-fig-s1)a).
This was in general agreement with previous meta-analyses indicating that an increased training volume will lead to more favorable outcomes, both in terms of muscle mass
[@RN1767;<!-- Schenfeld  2016--> 
@RN789]<!-- Krieger 2010 -->
and strength
[@RN2492;<!-- Ralston --> 
@RN793].<!-- Krieger 2009 -->

```{r meta-analyses-descriptive, warning=FALSE, message=FALSE, echo=FALSE}

summary_table_meta <- readRDS( "./data/derivedData/meta/naive_models.RDS")


summary_table_meta <- summary_table_meta %>%
  mutate(stat = paste0(sprintf("%.3f", median), 
                       " [",
                       sprintf("%.3f", cri.lwr), 
                       ", ",
                       sprintf("%.3f", cri.upr), 
                       "]"))


```

Since publication of the most recent meta-analyses [@RN1767;@RN2492],<!-- Schenfeld  2016 / Ralston 2107 --> 
additional studies have been published investigating the effect of resistance training volume on muscle mass and strength. Based on studies included in the meta-analysis performed by Schoenfeld *et al.* [@RN1767], additional results were coded based on a systematic literature search among newly published studies (from January 2015 to June 2020). Compared to Schoenfeld *et al.* [@RN1767], an additional ten studies were included in the present analysis.
Subsequently, effect sizes were coded from 25 studies, both for muscle mass and strength and meta-regression models were used to investigate the effect of weekly number of sets on training-induced changes in these outcomes.
In "naive" models, with the weekly number of sets as the only explanatory variable, both muscle mass and strength were shown to be robustly affected by an increased training volume.
For every additional weekly set, the effect size measure (Hedges' *g*, [95% credible intervals] (CrI)), increased by
`r summary_table_meta[1, 5]` and `r summary_table_meta[2, 5]`
for muscle mass (Figure \@ref(fig:comb-fig-s1)b)
and strength (Figure \@ref(fig:comb-fig-s1)c),
respectively.
Together these data underlines the training volume dose-dependence of both muscle hypertrophy and strength gains.

\pagebreak

```{r comb-fig-s1, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences in training-induced relative changes in muscle-mass and strength (a). Estimates are derived from ANCOVA models controling for baseline values. Estimates from meta-regression models examining the effect of weekly number of sets per muscle group on muscle hypertrophy (b) and strength gains (c). Estimates are effect size measures (Hedges g) with 95\\% credible intervals (CrI).", fig.scap = "Differences in training induced changes to muscle mass and strength measures between volume conditions in Study I and weekly training volume meta-regression.", fig.height=6, fig.width=4.72, fig.align='center'}

library(ggplot2); library(ggtext)

# The figure and estimates are from ./R/study-1/muscle-strength-size.R
muscle_strength_fig <- readRDS("./data/derivedData/study1_muscle-strength-size/muscle-strength-fig.RDS")

muscle_strength_fig

```

\pagebreak

In an attempt to explain differences in training outcomes between volume-conditions, selected molecular markers, known to influence adaptations to resistance training were investigated for volume-dependency.
First, acute activation of signaling along the mechanosensitive mTORC1-pathway was measured in before and after the fifth training session in Study I.
A commonly used readout of mTORC1-signaling is the phosphorylation of S6-kinase (S6K1) at Thr\textsuperscript{389}/Thr\textsuperscript{412} which in turn precedes phosphorylation of ribosomal protein S6 (rpS6, see Figure \@ref(fig:mtor-fig)a).
In the present study, exercise-induced S6K1\textsuperscript{Thr\textsuperscript{389}/Thr\textsuperscript{412}} phosphorylation was indeed shown to be volume dependent along with phosphorylation of rpS6\textsuperscript{Ser\textsuperscript{235/236}} and mTOR\textsuperscript{Ser\textsuperscript{2448}} (Figure \@ref(fig:mtor-fig) b).
Together these observations indicates larger perturbations along the mTORC1 signaling axis which confirms previous studies showing exercise-volume dependency of mTORC1-related signaling in the acute phase (< 5 hours) after exercise, with the low-volume condition doing at least three sets activating the muscle under study
[@RN2716;<!-- Kumar 2012 -->
@RN1837;<!-- Ahtiainen 5 vs 10 sets p70 and s6 different -->
@RN784;<!-- Terzis volume affect activation -->
@RN791;
@RN834].
It is recognized that phosphorylation of mTOR itself at Ser\textsuperscript{2448} primarily should be regarded as indicative for negative feedback as this site is phosphorylated due to S6K1 activity
[@RN2309] (Figure \@ref(fig:mtor-fig) a).<!-- Figueriedo mTOR and ser2448 signalling review -->
It is also recognized that the phosphorylation status of rpS6 at Ser\textsuperscript{235/236} is not solely due to mTORC1 signalling as both mTORC1 and extracellular signal-regulated kinases (ERK)-signalling converges here
[@RN2311].<!-- Roux S6 phopshorylation at ser 235 236 promoted by ras/erk -->
Interestingly, Terzis *et al.* and Ahtiainen *et al.* did not report any clear volume dependency in exercise-induced activation of ERK 1/2
[@RN785;<!--Terzis et al., 2008-->
@RN1837], 
suggesting that any volume-dependent regulation of this pathway was outside the time-frame of these studies. However, it might also suggest that volume selectively modulates specific pathways in the acute phase after resistance exercise.
It should however be noted that the present, and previous studies are limited in their temporal resolution and different patterns over time in related to exercise-volume cannot be ruled out.

```{r p70-csa-correlation-data, warning=FALSE, message=FALSE, echo=FALSE}

### Check association with 

p70_csa <- readRDS("./data/derivedData/study1-western-blot/csa_p70.RDS")

### Calculate raw correlations
cor_est_p70 <- p70_csa %>%
  group_by(sets) %>%
   summarise(p70.fc.cor = cor.test(fc, change, method = "spearman")$estimate, 
            p85.fc.cor = cor.test(fc.p85, change, method = "spearman")$estimate, 
            p70.fc.cor.p = cor.test(fc, change, method = "spearman")$p.value, 
            p85.fc.cor.p = cor.test(fc.p85, change, method = "spearman")$p.value)  %>%
  data.frame()

p70.m <- lme(change ~ w2pre + scale(fc) + sets, 
             random = list(subject = ~ 1), 
             weights = varExp(),
             data = p70_csa)

p85.m <- lme(change ~  w2pre.p85 + scale(fc.p85)  + sets , 
             random = list(subject = ~ 1),
             weights = varExp(),
             data = p70_csa)

# plot(p70.m)
# plot(p85.m)


p70.mod <- paste0(sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 3]), 
                  "]")

p85.mod <- paste0(sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 3]), 
                  "]")

corr_est_p70 <- paste0(sprintf("%.3f",min(c(cor_est_p70[,2],cor_est_p70[,3]))), 
       " - ", 
       sprintf("%.3f",max(c(cor_est_p70[,2],cor_est_p70[,3])))) 

```

The importance of mTORC1 signalling for protein synthesis in the acute phase after resistance exercise is well established.
In humans, administration of rapamycin, a selective inhibitor of mTORC1, prior to resistance exercise leads to delayed or blunted activation of mTORC1 effectors such as
S6K1 and rpS6 as well as unchanged levels of phopshorylation of 4E-BP1 and eEF2 concomitantly with abolished exercise-induced increase in protein synthesis
[@RN780].<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
Volume dependent increase in mTORC1 signalling also coincide with larger protein synthesis rates
[@RN791;<!-- Burd in single session single and multiple sets phosphorylation p70   -->
@RN2716]<!-- Kumar volume affects protein synthesis -->
As such, volume dependence of mTORC1-related signalling suggests that higher within-session volume can be regarded as leading to an increased potential for protein synthesis in the acute-phase after exercise.
Previous studies has indirectly linked mTORC1-related signaling to muscle growth as acute perturbation along this pathway has been correlated with muscle growth
[@RN785;<!--Terzis et al., 2008-->
@RN788].<!-- Mitchell et al., 2013-->
However, this is not a consistent findings in the literature 
[@RN834;<!-- Mitchell et al., 2012-->
@RN2171],<!-- Phillips et al., 2017-->
nor is it consistent with results in the present study as acute activation of S6K1 in response to the fifth session was not associated with training-induced changes in muscle mass.^[Raw Spearman's correlations between exercise induced change in S6K1 phosphorylation and change in muscle mass (measured with MRI) indicated weak (insignificant) associations ($\rho=$ `r corr_est_p70`) depedning on the isoform measured (p70 or p85) and number of sets. More elaborate modeling confirmed weak relationships between degree of phsophorylation and muscle mass change. When controlling for baseline values in degree of phsophorylation (as this influence the calculation of fold-change) and number of sets, standardized estimates for the effect of phosphorylation were (increase in %-muscle mass change for every 1SD change in fold-change phsophorylation, with [95% CI])  $\beta=$ `r p70.mod` for p70 S6K1-p70\textsuperscript{Thr389} and $\beta=$ `r p85.mod` for S6K1-p85\textsuperscript{Thr412}.].

```{r mtor-fig, warning=FALSE, message=FALSE, echo=FALSE, fig.cap="Measured phosphorylation sites in context (a) and differences between volume conditions in phosphorylation status of S6K1 at Thr \\textsuperscript{389} (p70) Thr \\textsuperscript{412} (p85), rpS6 at Ser \\textsuperscript{235 236} and mTOR at Ser \\textsuperscript{2448} induced by acute exercise and expressed as fold-changes (b). Estimates are derived from ANCOVA models controling for baseline values. Values in b are point estimates with 95\\% confidence intervals.", fig.scap = "Differences between volume conditions in exercise induced phosphorylation of proteins related to mTORC1 signaling", fig.height=3.5, fig.width=4.72, fig.align='center'}

western_fig <- readRDS("./figures/results/study1-western-blot.RDS")
western_fig

```

Although mTORC1 signaling most certainly contributes to protein synthesis in the acute recovery phase after resistance exercise [@RN780],<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
its use as a predictor of training induced muscle hypertrophy is complicated by a number of reasons.
First, measuring its activity is not straight forward for biological reasons including pathway cross-talk and negative feedback as well as technical aspects
[@RN1949;<!-- Figueriedo, mTOR feedback -->
@RN2311;<!-- Cross talk with erk -->
@RN1942].<!--  Bass technical considerations in western blotting -->
Secondly, mTORC1 signaling is not a stable phenomenon either between or within individuals as short- and long-term resistance training leads to diminished activity along its pathway
[@RN1809;<!-- mTOR signal lost after three weeks -->
@RN1871].
Thirdly, mTORC1 readily integrate multiple signals but also convey these to multiple downstream processes
[@RN2320].
As such, mTORC1 activity represent an early, transient response to resistance exercise with its signaling contributing to accumulative responses.
One such downstream target of mTORC1 is ribosomal biogenesis
[@RN1632;<!-- Nader mTOR regulates ribo biogenthrough cell cycle regulators -->
@RN1810;<!-- von Walden 2012 mTOR regulates myotube... -->  
@RN1828;<!-- von Walden 2016 -->
@RN2321],<!-- Chauvin S6K controls RiBi transcriptional program -->
ultimately leading to accumulation of ribosomal RNA, a response typically seen in connection with to resistance training
[@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].

```{r ribo-markers}

descriptive_ribo_markers <- readRDS("./data/derivedData/study1-ribo-biogenesis/descriptive_ribo_markers.RDS")

rrna <- descriptive_ribo_markers %>%
  filter(target != "totalrna") %>%
  summarise(min.w2pre = round(min(w2pre),0), 
            max.w2pre = round(max(w2pre),0), 
            min.w12 = round(min(w12),0), 
            max.w12 = round(max(w12),0)) 
```

As ribosomal RNA is the most abundant constituent of muscle RNA it provides an estimate of muscle ribosomal abundance when expressed per unit tissue weight
[@RN2054;<!-- Zak 1967 in heart muscle 85% of RNA is ribosomal RNA -->
@RN2145].<!-- Young -->
From baseline to prior to the fifth training session (Week 2), total RNA per mg tissue increased by $\sim$ `r round(descriptive_ribo_markers[9,2], 0)` and `r round(descriptive_ribo_markers[10,2], 0)`% in the single-set and multiple-sets condition, respectively. Total RNA levels were still elevated above baseline after the intervention (Week 12, single-set $\sim$ `r round(descriptive_ribo_markers[9,3], 0)`; multiple-sets $\sim$ `r round(descriptive_ribo_markers[10,3], 0)`%). Similar patterns were seen in target analysis of ribosomal RNA using qPCR (single-set increase from baseline to Week 2 $\sim$ `r rrna[1,2]`-`r rrna[1,3]`% and Week 12 $\sim$ `r rrna[1,4]`-`r rrna[1,5]`%; multiple-sets $\sim$ `r rrna[2,2]`-`r rrna[2,3]`% and Week 12 $\sim$ `r rrna[2,4]`-`r rrna[2,5]`%). Comparing volume conditions revealed higher levels of total RNA and mature ribosomal RNA species in the multiple-sets condition at Week 2 (Figure \@ref(fig:rrna-fig)a). At Week 12 differences between volume conditions were in total RNA less clear and ribosomal RNA 28S showing higher levels in the single-set leg (Figure \@ref(fig:rrna-fig)a).

```{r rrna-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences between volume conditions in total RNA and ribosomal RNA-species (pre-rRNA 45S, rRNA 18S, rRNA 5.8S and rRNA 28S) measured at rest during the course of Study I (a). Acute changes in abundance of c-Myc mRNA and pre-rRNA 45S in response to acute exercise in Week 2 and differences between volume conditions (b). Errorbars represents 95\\% confidence intervals, transparent points and errorbars signifies that the confidence interval contain 1.", fig.scap = "Differences between volume conditions total RNA and ribosomal RNA", fig.height=2.5, fig.width=4.72, fig.align='center'}

ribo_markers_fig <- readRDS("./figures/results/study1-ribo-markers.RDS")

ribo_markers_fig

```

Analysis of c-Myc mRNA abundance in response to the fifth training session also showed volume-dependent regulation with exercise-induced increases being $\sim$ 1.5-fold higher in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:rrna-fig)b). c-Myc represents a rapamycin-insesitive signaling pathway, paralell to mTORC1, known to also stimulating ribosomal biogenesis 
[@RN1754;<!-- West -->
@RN1834;
@RN1832].<!-- cMyc driver of ribosomal biogenesis -->

Ribosomal biogenesis is an early adaptation to resistance training with initial increases in total RNA evident before any hypertrophy can be detected
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113].<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions-->
The increase in ribosomal RNA represents an accumulation of ribosomes and an increased translational capacity. This improved translational capacity contributes to protein accreetion resulting in muscle hypertrophy, reflected in associations between measures of ribosome abundance (typically total RNA) and muscle hypertrophy
[@RN1644;<!-- Figueriedo 2015 correlation between delta csa and fold change of total RNA -->
@RN1755;<!-- Stec 2016 only high responders have increased total RNA and rRNA abundance after short training -->
@RN2055;<!-- Mobley weak correlation r = 0.28 p = 0.026  and no cluster * time interaction effect but increases in total RNA 34% in high responders vs 8% in low -->
@RN1897].<!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->
Due to the sequential nature of these events (increased translational capacity before muscle hypertrophy), early increases in total RNA would possible better reflect training-induced changes in muscle hypertrophy.
Any increase in muscle mass would also contribute to a dilution of the RNA pool
[@RN2142]<!-- figueriedo description of dilution effect -->
possibly masking their causal relationship.
Comparing the relationships between RNA abundance and muscle hypertrophy at different time-points supported this view as the amount of RNA at Week 2 showed the strongest relationship with the resulting muscle growth (Figure \@ref(fig:rrna-csa-fig), \@table(rna-csa-tab)). This indicates that translational capacity, affected by exercise, early in the training program is determining training-induced changes in muscle mass.  

```{r rna-csa-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

rna_csa_change_models <- readRDS("./data/derivedData/study1-rna-csa-cor/regression.RDS")

rna_csa_change_models %>%
  mutate(Estimate = round(Estimate, 2), 
         SE = round(SE, 2), 
         df = df, 
         t.value = round(t.value, 2), 
         ci = paste0("[", sprintf('%#.2f', lower), ", ", sprintf('%#.2f', upper), "]"), 
         beta = paste0(sprintf('%#.2f',scaled.est), 
                       " [", 
                       sprintf('%#.2f', scaled.lower), 
                       ", ", 
                       round(scaled.upper, 2), "]"),
         beta = if_else(term %in% c("(Intercept)", "setsmultiple"), "", beta)) %>%
  dplyr::select(term, Estimate, SE, df, t.value, ci, beta) %>%
  mutate(term = factor(term, 
                       levels = c("(Intercept)",
                                  "setsmultiple", 
                                        "w0", 
                                        "w2pre", 
                                        "w12"), 
                       labels = c("(Intercept)", 
                                  "Moderate- vs. low-volume",
                                  "Week 0", 
                                  "Week 2", 
                                  "Week 12"))) %>%
  kable(format = "latex", 
        booktabs = TRUE, 
        escape = FALSE,
        caption = "Influence of RNA abundance on training-induced muscle growth measured with MRI", 
        col.names = c("", "Estimate", "SE", "df", "\\textit{t}-value", "95\\% CI", "Estimate [95\\% CI]")) %>%
  add_header_above(c(" " = 6, "Standardized coefficients" = 1), 
                   escape = FALSE) %>%
  pack_rows("RNA abundance (ng mg\\textsuperscript{-1})", 3, 5, escape = FALSE) %>%
  kable_styling(font_size = 8) %>%
  add_footnote("Standardized coefficients scaled by its SD")


```



```{r rrna-csa-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Relationship between total RNA and training induced change in thigh cross sectional area (CSA) as measured by MRI. Dashed line represent naive relationship, solid line represent adjusted reliationship as seen in \\@ref(table:rna-csa-tab).", fig.scap = "Relationship between total RNA and training induced muscle growth", fig.height=4.72/3, fig.width=4.72, fig.align='center'}

rna_csa_fig <- readRDS("./figures/results/study1-rna-csa.RDS")


rna_csa_fig

```

## Volume-dependent remodeling of muscle fiber-type composition

```{r myhc-stats, warning=FALSE,message=FALSE, echo=FALSE}

mx.summary <- readRDS("./data/derivedData/study1-myhc-composition/myhc2x_stats.RDS")

or.week2 <- paste0(sprintf('%#.2f', exp(mx.summary[5, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[5, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[5, 6])), 
                   "]")

or.week12 <- paste0(sprintf('%#.2f', exp(mx.summary[6, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[6, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[6, 6])), 
                   "]")


```

Resistance training readily leads to remodeling of the muscle, this includes qualitative changes. These include fiber type transitions through changes in the myosin heavy-chain protein composition
[@RN2220;<!-- Adams Down regulation of MyHc 2x content in training group both myhc isoform sds page and atpase staining -->
@RN2799;<!-- Williamsson less IIx dominant hybrid fibers with training conversion towards IIa -->
@RN819;<!-- Schiaffino review, other protein families also different between fiber types-->
@RN1489].<!-- Ellefsen 2014, training induced switch from 2A to 2X -->
Resistance-training induced changes in fiber-types identified by myosin heavy-chain expression occurs primarily as a IIX $\rightarrow$ IIA transition which results in a more fatigue-resistant phenotype
[@RN819;<!-- Schiaffino review, other protein families also different between fiber types-->
@RN2056;<!-- Andersen, primarily transitions fiber in 2x to 2a direction minimal 2 to 1 trans -->
@RN2801]<!-- Karatzaferi 2001 Drop in ATP most pronounced in type 2x fibers in response to acute exercise -->
and thus represent an important aspect of the adaptive response to resistance training by reversing effects of inactivity and disuse
[@RN2057;<!-- Andersen 2x overshoot after de training -->
@RN2108].<!-- Bed rest increases type IIx -->
In Study I, muscle fiber-type compositions were determined based on immunostaining of myosin heavy-chain protein in muscle cross-sections. The staining protocol allowed for the identification of Type I (red), Type IIA (brown), Type IIX (unstained), and Type IIA/IIX hybrids (light brown, Figure \@ref(fig:myhc-fig)a). Throughout the study, a general reduction in Type IIX fibers was seen as pure Type IIX was replaced with Type IIA/IIX hybrids at Week 2, and these, in turn, were measured at lower proportions at Week 12 (Figure \@ref(fig:myhc-fig)b).

When comparing volume conditions, moderate training volume led to more pronounced down regulation of the *MYH1* gene, coding for Type IIX myosin heavy-chain protein, at all time-points during the intervention (Figure \@ref(fig:myhc-fig)c). This presumably led to the accumalative effect seen after twelve weeks of training where the proportion of Type IIX fibers was lower in the moderate- compared to the low-volume condition (indicative as an odd-ratio (OR) < 1, OR [95% CI]: `r or.week12`, combined measure of IIX/IIA hybrids and pure IIX fibers, Figure \@ref(fig:myhc-fig)d).  
This underlines that training volume is an important factor for fiber-type composition remodeling. Although not surprising, similar observations are lacking in the litterateur as previous studies have not compared this transition directly between volume protocols.
However, when comparing non-exhaustive high-load resistance training to load-matched training to volatile failure, Pareja-Blanco *et al.* observed a blunted IIX $\rightarrow$ IIA transitions in response to the non-exhaustive protocol 
[@RN2217].
Together these data indicates that increased metabolic stress and/or dosage of neuromuscular activity are plausible candidates for regulation of IIX $\rightarrow$ IIA reprogramming in humans in response to resistance training.
This as opposed to a tensile stimuli.
Based on observations in animal models, neural input might however be considered the primary candidate.
For example, in a rat model of resistance training, force did not affect fast-to-slow fiber-type transitions when performed with the same electrical stimulation pattern, indicating that neural activity governs fiber-type transitions
[@RN2104].<!-- Eftestol -->

At Week 2, a more pronounced reduction in *MYH1* gene abundance in the moderat-volume condition did not translate to the protein level. Instead a greater number of fibers were classified as IIX in the moderate-volume condition compared to the low-volume condition (OR: `r or.week2`).
The moderate-volume condition thus showed an even greater mismatch between gene and protein levels.
Such mismatch is seen after resistance-exercise and is indicative for early transition of muscle fibers
[@RN2056].<!-- Andersen mismatch between  -->
Interestingly, the difference between volume conditions was driven by less Type IIX expression in the low-volume condition.
This indicates that remodeling occurred faster in response to the low-volume condition despite a greater "transition signal" in the moderate-volume leg.
A plausible explanation to this observation could be that the greater damage caused by a greater training stimuli lead to a delayed fiber-type transition. 
This hypothesis is supported by observations indicating that a proportion of the protein synthesis seen early in a resistance-training period can be attributed to tissue repair rather than myofibrilar synthesis
[@RN1521;<!--Kim 2005 -->
@RN2144].<!-- Damas 2016 -->
Regardless of causality, the more rapid early adaptation seen in response to a lower training volume suggests that optimized resistance training should include a progressive volume component.



\pagebreak

```{r myhc-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Representative muscle cross-sections stained for myosin-heavy chain isoforms are shown in (a). Using antibodies for Type I (MyHC Slow) and all but Type IIX (BF-35) enabeled separating Type I fibres from other fibres (red stain, a lower panel). Type IIX fibers were indentified as unstained, and weak brown staining was analysed as Type IIX/IIA hybrids (a, upper panel). Resistance training led to a reduction in Type IIX and IIX/IIA hybrids, (b) shows participant-average Type IIX and IIX/IIA percentages. Training led to reduction in \\textit{MYH1} gene expression, and more so in the moderate-volume condition, evident from pairwise comparisons in (fold-differences with 95\\% CI, c). Differeces in protein expression of Type IIX fiber (counted as pure IIX + 0.5 $\\times$ IIX/IIA hybrids) reversed throught the study as the moderate-volume condition showed greater proprtions of IIX fibers at Week 2 but lesser expression at Week 12 (d).", fig.scap = "Fiber-type composition in Study I", fig.height=6, fig.width=4.72, fig.align='center'}

# load fiber typing figure
img <- ggdraw() + draw_image("./data/study-1/immuno/immuno_control_image2.png")

# load figures
myhc_fig_comps <- readRDS( "./data/derivedData/study1-myhc-composition/myhc_fig.RDS")


## Combine figure directly in manuscript as saving the png takes some space 

myhc_fig <- plot_grid(plot_grid(NULL, img, NULL, ncol = 3, rel_widths = c(0.2, 1, 0.2)), 
                      plot_grid(myhc_fig_comps$overall_type2x, 
                                NULL,
                                plot_grid(myhc_fig_comps$myhc2x_mrna_means, 
                                          myhc_fig_comps$diff_2x_mRNA, 
                                                    ncol = 1, 
                                                    align = "v",
                                                    rel_heights = c(0.5, 0.5)),
                                          ncol = 3, rel_widths = c(0.4,0.05, 0.5)),
                  
                       myhc_fig_comps$fiber_type_insets, 
          ncol = 1, 
          rel_heights = c(0.6, 0.6, 0.6)) + 
  draw_plot_label(x = c(0.25,
                   0.005, 0.45, 
                   0.005), 
             y = c(0.98, 
                   0.67, 0.67, 
                   0.36), 
             label = c("a", "b", "c", "d"))

myhc_fig

```

\pagebreak

## Volume-dependent effects on transcriptome characteristics

As already [discussed](#muscle-mass-growth), resistance training in Study I led to an increase in total RNA per unit tissue weight. Furthermore, the magnitude of this increase was different between volume conditions (see Figure \@ref(fig:rrna-fig)).
Studies of gene expression, using techniques such as qPCR or RNA sequencing, comes with the assumption that an equal amount of biological material is used when evaluating abundance of a single transcript.
It is therefore common that a normalized amount of total RNA is used to satisfy such an assumption
[@RN2298;<!-- Damas 2018 -->
@RN2400; <!-- Robinson 2017-->
@RN2401].<!-- Laker 2017.-->
However, if the ratio of RNA to tissue mass has changed throughout a study, analyses are performed on a different amount of tissue. This may be the case after a resistance-training intervention, or between study conditions, as in Study I.
Indeed, by calculating the amount of tissue used when preparing samples for RNA sequencing it is obvious that the increased amount of total RNA per-unit-muscle weight results in a situation where samples are prepared from a lesser amount of tissue.
This effect was more pronounced in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:lib-size-fig)a).
Total library sizes were quantified from RNA sequencing as the total number of quantified gene counts, this revealed a general increase in library sizes in response to resistance training, however, the increase was more apparent in the low-volume condition
(Figure \@ref(fig:lib-size-fig)b).
When library sizes were normalized to the amount of tissue used in sample preparation, differences between conditions were diminished
(Figure \@ref(fig:lib-size-fig)c).
This suggests that resistance-training leads to an increased global mRNA expression per-unit-muscle weight (43-53\% in the present study). However, this increase was not different between volume-conditions, suggesting that a volume-dependent regulation of global RNA transcription primarily relates to transcription of RNA-fractions other than mRNA (presumably rRNA).

```{r, lib-size-fig, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Less muscle tissue was used for cDNA synthesis in the moderat- compared to low-volume condition as a result of total RNA increases per tissue weight (a). Library sizes tended to be greater in the low-volume condition (b), after normalizing library sizes to muscle weight differences were diminished (c). Lower panels shows differences between volume conditions", fig.scap = "Muscle weight and RNA-seq library size", fig.height=3, fig.width=4.72, fig.align='center'}

libsize.fig <- readRDS("./data/derivedData/study1-transcriptome/muscle-weight-lib-size.RDS")


libsize.fig +
  draw_plot_label(x = c(0, 0.33, 0.67), 
             y = c(1,1,1), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b", "c"))
```

The general increase in mRNA  

[@RN2616]<!-- Kirby nascent transcript in smaller fibers during hypertrophy (SA), global promoter activation -->

[@RN1707]<!-- Habets 1999 in rat higher 28S RNA and total rna in slow compared to fast fibers, unchanged myhc mRNA/RNA between fibers   -->



```{r, week2-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Genes identified as differentially expressed (log2 fold-change > |0.5|, FDR < 0.05) are highlighted in a.  Overlap in differentially expressed genes between models are shown in (b)", fig.scap = "General patterns of differentially expressed genes at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

week2_grid <- readRDS("./data/derivedData/study1-transcriptome/week2_volcano_upset.RDS")

week2_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

To account for differences in biological input between volume conditions, three different normalization strategies were used to characterize volume-dependence in transcriptome characteristics.
By implementing each normalization strategy as variations of generalized linear mixed models (GLMM), we could compare outcomes related to conceptually different normalization techniques using the same statistical framework.
The use of GLMM also had the benefit of accounting for the study design which included correlated data (within participants)
[@RN2366].<!-- Cui et al what if we ignore random effects -->
The model with most parameters included an offset term expressing gene counts per tissue weight. This was combined with library size included as a fixed effect in the model together with study conditions, time and volume-conditions (tissue offset-model).
In a second model the offset term was removed, thus only the library size was used to normalize the gene counts, as suggested by Cui *et al.*
[@RN2366] (effective library-size model).<!-- Cui et al what if we ignore random effects -->
For comparison, a naive model containing no normalization-parameters was also included in analyses.
At Week 2, tissue-offset normalization (modeling mRNA counts per-mg-muscle weight) revealed a general shift of mRNA counts towards the moderate-volume condition (Figure \@ref(fig:week2-upset-volcano)a).
This was in contrast to patterns found when using library-size and naive models resulting in a pattern where the "conventional" normalization strategy, accounting only for library sizes more resembled the non-normalized naive model in as there was overlap of genes identified as differentially expressed between the models (Figure \@ref(fig:week2-upset-volcano)b).
Gene-set enrichment analyzes revealed that genes related to extracellular matrix remodeling and collagen synthesis was driving the difference in transcriptome profiles between volume-conditions at Week 2
(Figure \@ref(fig:week2-go-analysis)a).
This was apparent using both the tissue offset model and the library-size normalized model, however volume-dependent effects were more pronounced in the tissue-offset model
(Figure \@ref(fig:week2-go-analysis)b).

```{r week2-go-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Gene-set enrichment analysis based on gene-ontology gene sets and estimates from the tissue-offset model (a). Adjusted \\textit{P}-values and Normalized enrichment scores are retrieved from analysis of directional rank-based tests (GSEA), labeling of points indicate if gene-sets were also identified from differentially expressed genes (Log2 fold-change >|0.5|, FDR < 0.05). In (b) comparison between the tissue-offset and library size normalization models are shown in rank-based (non-directional, cerno-test) analysis of gene ontology gene sets.", fig.scap = "Gene-set enrichment analysis at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

go_analysis_week2 <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/go_week2.RDS")


go_analysis_week2 +
  draw_plot_label(x = c(0.02, 0.55), 
             y = c(1, 0.85), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))



```

The differences in normalization models led to globals shifts of estimates of differences between volume conditions. Figure \@ref(fig:gene-set-rug) shows this global shift as density curves representing all fold-changes between volume conditions. Accounting for the amount of muscle mass in sample preparations shifted also specific gene-sets towards gretaer differences between volume conditions, this is exemplified in Figure \@ref(fig:gene-set-rug) by the gene set "Collagen containing extracellular matrix".


```{r gene-set-rug, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Differences between volume conditions at Week 2 as seen using different normalization models. Shifts in the full distribution of Log2-fold change values between volume conditions were seen between on normalization models (density curves). As an example, genes associated with the \\textit{Collagen containing extracellular matrix} gene set are highlighted (black bars), differentially expressed genes are identified with gene symbols in each method.", fig.scap = "Global shifts in volume-dependent fold change as an effect of normalization methods at Week 2", fig.height=5, fig.width=3.72, fig.align='center'}

gene_set_rug <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/gene-set-rug.RDS")

gene_set_rug

```

At week 12, differences seen at Week 2 between volume conditions derived from the tissue-offset model were abolished (Figure \@ref(week12-upset-volcano)a). Volume dependent differences were seen in the library-size normalized model, however, these overlapped with the naive model indicating that in-balance in muscle tissue between conditions may have contributed to global shifts in favor of the low-volume condition.  

```{r week12-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Overlap in differentially expressed genes between models are shown in (b)", fig.scap = "General patterns of differentially expressed genes at Week 12", fig.height=5, fig.width=4.72, fig.align='center'}

week12_grid<- readRDS("./data/derivedData/study1-transcriptome/week12_volcano_upset.RDS")

week12_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```
<!-- rewrite! -->
Using the tissue-offset model, we were able to identify genes relating to ECM functions as volume-sensitive during the early stages of resistance training. This may indicate a role for ECM remodeling in the beneficial effects of higher training volumes on muscular adaptations and strength. As such, previous research has shown that ECM remodeling is induced by exercise training, both acutely [@RN2451; <!-- Wessner acute exercise gives different regulation of metalloproteinase -->
@RN2453;<!-- Mackey  repeated bout effect 30 days -->
@RN2454;<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load --> 
@RN2457]<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->,
and after prolonged endurance and resistance training [@RN774; <!-- Raue, upregulation of ECM after long term exercise --> 
@RN2298;<!-- Damas long term training ECM up gene-->
@RN2448; <!-- Makhnovskii cycling exercise, mRNA and protein scales for ECM -->
@RN2459]<!-- Hjorth 2015 cycling exercise upregulates -->,
at both mRNA and protein levels [@RN774; <!-- Raue, upregulation of ECM after long term exercise --> 
@RN2298;<!-- Damas long term training ECM up gene-->
@RN2448; <!-- Makhnovskii cycling exercise, mRNA and protein scales for ECM -->
@RN2459]<!-- Hjorth 2015 cycling exercise upregulates -->.
However, none of these previous studies have found ECM remodeling to be differentially affected by different exercise modalities. Rather, different resistance training modalities such as low- and high-load training have been associated with similar responses, measured as collagen synthesis [@RN2454]<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load -->.
Importantly, there seems to be a close association between training-induced changes in abundances of ECM-related mRNAs and their respective proteins, including collagen-organization proteins [@RN2448]<!-- Makhnovskii cycling exercise, mRNA and protein scales for ECM -->.
This suggests that ECM remodeling is primarily controlled at the transcriptional level, which arguably increases the biological relevance of the herein presented transcriptome analyses.
However, this relationship seems to involve a complex time course dependency. For example, transcriptional regulation of COL1A2 shows a considerable lag from stimuli to transcription, as shown in fibroblasts [@RN2449].<!-- Schwarz 2015 Type I collagen review -->
In line with this, our data on the effects of acute resistance exercise on transcriptome profiles, suggests a counterintuitive reduction in the expression of e.g., collagen mRNA immediately after exercise, as has also been found by others [@RN774]<!-- Raue, upregulation of ECM after long term exercise -->.
Indeed, enrichment analyses confirmed these negative changes in ECM-related transcripts in response to acute exercise, contrasting the effects of chronic resistance training [@RN774; <!-- Raue, upregulation of ECM after long term exercise --> 
@RN2298]<!-- Damas long term training ECM up gene-->. 

The physical properties of ECM are distorted by disuse and aging, resulting in increased stiffness and potentially decreased force transmission and muscle efficiency [@RN2460;<!-- Caspo review -->
@RN2450<!-- Azizi collagen deposition leads to reduced muscle efficiency -->].
Training-induced ECM remodeling may thus constitute an effective measure to reverse these adversities [@RN2451; <!-- Wessner acute exercise gives different regulation of metalloproteinase -->
@RN2453;<!-- Mackey  repeated bout effect 30 days -->
@RN2454;<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load --> 
@RN2457]<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->,
and has been suggested to exert a protective role against injury [@RN2453]<!-- Mackey  repeated bout effect 30 days -->.
However, available studies are ambiguous in their conclusions [@RN2454;<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load --> 
@RN2457]<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->,
and the link between observed changes in ECM-related gene expression and muscle biology and functionality remain uncertain. Adding to this, ECM remodeling-responses to training seems to be age-dependent[@RN2423;<!-- Sorensen acute adamging exercise leads to less fibronectin protein induction --> @RN2451]<!-- Wessner acute exercise gives different regulation of metalloproteinase -->,
and also shows a clear dependency of time. Indeed, in the present study, the volume-associated differences in ECM-related transcriptome profiles disappeared entirely after twelve weeks of resistance training, whereby no genes were identified as showing volume-dependent responses.
From a general perspective, this indicates that after prolonged training, the biological state of the muscle may have reached a new equilibrium, with low- and moderate-volume training having led to similar muscle phenotypic traits.
This would imply that the benefits of higher training volume are restricted to augmented increases in muscle mass, perhaps facilitated by increases in ribosomal biogenesis [@RN2358].
Notably, this is likely to be an oversimplification. Indeed, at the protein level, in the current study, higher training volume led to a more robust phenotypic switch from type IIX $\rightarrow$ type IIA fiber also after twelve weeks of training [@RN2358]. Taken together, our data provide valuable directions for future research. It suggests that ECM remodeling is volume-dependent, at least during the initial part of a training program. This needs to be confirmed by studies in other populations, and its biological and functional significance needs clarification. Such studies should take advantage of the increased biological resolution of within-subject contralateral models. 

## Determinants of moderate- over low-volume training benefit

```{r csa-str-sets-corr, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="", fig.scap = "", fig.height=2.4, fig.width=4.72, fig.align='center'}

csa_str_cor <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor.RDS")


corr <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor_df.RDS")

csa_str_cor

```


```{r univariate-benefit-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="", fig.scap = "", fig.height=3.72, fig.width=4.72, fig.align='center'}

readRDS("./data/derivedData/study-1-benefit-analysis/univariate_regression_results_fig.Rds")

univariate_regression


```



```{r model-reduction-plot, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="", fig.scap = "", fig.height=3.72, fig.width=4.72, fig.align='center'}

model_reduction_plot <- readRDS("./data/derivedData/study-1-determinants/model_reduction_plot.RDS")                    

model_reduction_plot

```



