# Results and Discussion

```{r settings, include=FALSE}

library(nlme); library(kableExtra)


```

## Training volume affects training-induced changes in muscle mass and strength as well as molecular determinants of muscle hypertrophy {#muscle-mass-growth}

```{r comparing-avg-strength-csa, echo=FALSE, warning=FALSE, message=FALSE}


source("./R/study-1/mr-and-dxa-data.R")


participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


### Overall % increase in CSA (not corrected for baseline values)
over.all.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(subject) %>%
  summarise(mean.csa = mean(change, na.rm = TRUE)) %>%
  ungroup() %>%
  summarise(m.csa = mean(mean.csa),
            s.csa = sd(mean.csa))

sets.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(sets) %>%
  summarise(m.csa = mean(change),
            s.csa = sd(change))



### Baseline corrected % increase
csa.corrected.data <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>% # Calculates % change
  group_by(sex) %>%
  mutate(pre = pre-mean(pre)) %>% # Mean centered baseline 
  # (no need for sex-diff in model)
  group_by(subject, sex) %>% # Keeps sex for test in model
  summarise(mean.csa = mean(change, na.rm = TRUE),
            mean.baseline = mean(pre)) # Average over both conditions 



## Anocova with baseline values as co-variate
csa.corr.m1 <- lm(mean.csa ~  mean.baseline,
                  data = csa.corrected.data)

csa.m1 <- lm(mean.csa ~  1,
                  data = csa.corrected.data)



csa.corr.confint  <- round(confint(csa.corr.m1)[1,],1)
csa.corr.estimate <- round(summary(csa.corr.m1)$coef[1,1],1)




csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  group_by(timepoint, sets, sex) %>%
  summarise(m=sprintf("%.1f", round(mean(CSA.avg),1)),
            s=sprintf("%.1f", round(sd(CSA.avg),1)))


csa.r <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject, sex, sets, csa.change) %>%
  spread(sets, csa.change)

csa.r.model <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject,leg, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject,leg, sex, sets, pre, csa.change) %>%
  group_by(sex) %>%
  mutate(pre.m = pre-mean(pre, na.rm = TRUE), 
         sets = factor(sets, levels = c("single", "multiple"))) 

#### Change to ancova ###


csa.m0 <-nlme::lme(csa.change ~ sets +  pre.m,
                random = list(subject = ~1),
                na.action = na.omit, 
                data = csa.r.model, method = "ML", 
                control=list(msMaxIter=120,
                             opt = "nloptwrap", msVerbose = FALSE))




csa.estimate <- sprintf("%.2f",round(coef(summary(csa.m0))[2,1], 2))
csa.confint <- intervals(csa.m0)$fixed[2, c(1,3)]

### t-test Does not change the conclusion 
# csa.ttest <- with(csa.r, t.test(multiple,single, paired=T))
# csa.estimate <- sprintf("%.2f",round(csa.ttest$estimate, 2))
# csa.confint <- csa.ttest$conf.int


csa.corr <- cor.test(csa.r$multiple, csa.r$single)


female.t <- with(csa.r[csa.r$sex=="female",], t.test(multiple,single, paired=T))
female.estimate <- sprintf("%.2f",round(female.t$estimate, 2))
female.confint <- female.t$conf.int


male.t <- with(csa.r[csa.r$sex=="male",], t.test(multiple,single, paired=T))
male.estimate <- sprintf("%.2f",round(male.t$estimate, 2))
male.confint <- male.t$conf.int

#### Strength data for overall stats



rm_str <- read_excel("./data/study-1/strength/strengthTests.xlsx") %>%
        filter(!(exercise %in% c("benchpress", "legcurl"))) %>%
        inner_join(participants) %>%
        filter(include == "incl") %>%

  dplyr::select(subject, sex, sets, exercise, timepoint, load)%>%
  mutate(load = as.numeric(load),
         sets = factor(sets, levels = c("single", "multiple"))) %>%
  spread(timepoint, load) %>%
  mutate(baseline = pmax(pre, session1)) %>%
  dplyr::select(subject, sex, sets, exercise, baseline, week2, week5, week9, post) %>%
  gather(week, load, baseline:post) %>%
  mutate(week = factor(week, levels = c("baseline", "week2", "week5", "week9", "week12", "post"))) %>%
  group_by(exercise) %>%
  mutate(str = load / max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) 

# load cybex data 
load("./data/study-1/strength/cybex.rda")


cybex <- data.frame(cybex) %>%
        mutate(leg = if_else(leg == "Left", "L", "R")) %>%
        inner_join(participants) %>%
        filter(include == "incl", 
               timepoint %in% c("pre","session1", "post")) %>%

        filter(!(subject=="FP6" & timepoint == "post" & speed == 240))  # This removes one record, exclusion due to error in measurement, see protocol FP6


# Clean cybex data set and combine with 1RM 
total_strength <- cybex %>%
  ungroup() %>%
  dplyr::select(subject, timepoint, sets,speed, sex, torque.max) %>%
  spread(timepoint, torque.max) %>%
  rowwise()%>%
  mutate(baseline = max(pre, session1, na.rm=T)) %>%
  ungroup() %>%
  dplyr::select(subject, sex, sets, exercise = speed, baseline,  post) %>%
  gather(week, load, baseline:post) %>%
  group_by(exercise) %>%
  mutate(str = load/max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) %>%
  rbind(rm_str) %>% # This includes all strength tests
  mutate(ex_fam = if_else(exercise %in% c("legext", "legpress"), 
                          "rm", if_else(exercise == 0, "isom", "isok"))) %>%
  group_by(subject, sex, sets, week, ex_fam) %>%
  summarise(str = mean(str, na.rm = TRUE)) %>%
  group_by(subject, sex, sets, week) %>%
  summarise(str = mean(str, na.rm = TRUE)) 


strength.stats <- total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  group_by(sex) %>%
  mutate(BASELINE = baseline - mean(baseline)) %>%
    group_by(sex, subject) %>%
    summarise(increase = mean(increase, na.rm = TRUE),
              BASELINE = mean(BASELINE, na.rm = TRUE)) 




## ANCOVA for the strength 1RM
strength.m1 <- lm(increase ~ sex + BASELINE,
                  data = strength.stats)

strength.corr.confint  <- round(confint(strength.m1)[1,],0)
strength.corr.estimate <- round(summary(strength.m1)$coef[1,1],0)



### Correlation in strength gains ###

str.corr.data <-  total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject, sex, sets, increase) %>%
  spread(sets, increase) 

str.corr <- cor.test(str.corr.data$single,
                          str.corr.data$multiple)

#### Correlation strength and csa increases

# Average increases

csa.str.corr.data <- total_strength %>%
  spread(week, str) %>%
  mutate(str.rel.change = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject,  sets, sex, sets,baseline, str.rel.change) %>%

  inner_join(mr.results %>%
                mutate(condition=leg) %>%
                inner_join(participants) %>%
                filter(include=="incl") %>%
                dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
                spread(timepoint, CSA.avg) %>%
                mutate(csa.rel.change = 100 * ((post/pre)-1)) %>%
                dplyr::select(subject, sets, csa.rel.change)) %>%
        ungroup() 





# The code below could possibly be used to calculate the relative contribution of 
# csa increase to muscle strength increases


# m <- lme4::lmer(str.rel.change ~ csa.rel.change + sets + (1|subject), 
#                 data = csa.str.corr.data)
# 
# 
# 
# mx.null <- lme4::lmer(str.rel.change ~ 1 + (1|subject), data = csa.str.corr.data)
# mx.csa <- lme4::lmer(str.rel.change ~ csa.rel.change  + (1|subject), data = csa.str.corr.data)
# mx.sets <- lme4::lmer(str.rel.change ~ sets + (1|subject), data = csa.str.corr.data)
# 
# 
# 
# boot.fun <- function(x) {
#         data.frame(nlme::VarCorr(x))[2, 4]
# }
# 
# 
# null.boot <- lme4::bootMer(mx.null, boot.fun, nsim = 500, parallel = "snow", cl = 12)
# csa.boot <- lme4::bootMer(mx.csa, boot.fun, nsim = 500)
# sets.boot <- lme4::bootMer(mx.sets, boot.fun, nsim = 500)
# 
# 
# 
# qt.null <- quantile(null.boot$t, c(0.025, 0.975))
# qt.csa <- quantile(csa.boot$t, c(0.025, 0.975))
# qt.sets <- quantile(sets.boot$t, c(0.025, 0.975))
# 
# 
# results <- data.frame(variable = c("sets", "csa"),
#                            m = c(1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4],
#                                  1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4]),
#                            lwr = c(1- qt.sets[1] / qt.null[1],
#                                    1- qt.csa[1] / qt.null[1]),
#                            upr = c(1- qt.sets[2] / qt.null[2],
#                                    1- qt.csa[2] / qt.null[2]))
# 
# 
# 
# 
# 
# csa <- 1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# sets <- 1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# exp <- 1 - data.frame(nlme::VarCorr(mx.exp))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# 

# Average csa and strength gains

avg.str.csa <- csa.str.corr.data %>%
        group_by(subject) %>%
        summarise(str = mean(str.rel.change), 
                  csa = mean(csa.rel.change))


csa.str_corr <- cor.test(avg.str.csa$csa, avg.str.csa$str)

## Comparisons with other published studies on per session gains in csa and strength

### Per session ##
per.session <- read_excel("./data/study-1/trainingdata.xlsx", sheet = 1) %>%
  filter(!is.na(date)) %>%
  group_by(subject) %>%
  summarise(sessions = n()) %>%
        
inner_join(csa.str.corr.data) %>%

  mutate(str.change.day = str.rel.change/((sessions/31) * 12*7),
         csa.change.day = csa.rel.change/((sessions/31) * 12*7), 
         str.session = str.rel.change/sessions,
          csa.session = csa.rel.change/sessions ) %>%
   dplyr::select(-baseline) %>%
  ungroup() %>%
        pivot_longer(names_to = "variable", values_to = "gain", cols = str.rel.change:csa.session) 
       

sum.stats <- cbind(per.session %>%
        group_by(sets, variable, sex) %>%
        summarise(mean = mean(gain), 
                  sd = sd(gain)) %>%
        arrange(variable) %>%
        data.frame(), 
        data.frame(reference.mean = c(0.11, 0.11, 0.11, 0.11,
                              NA, NA, NA, NA, 
                              0.0825, 0.1375,0.0825, 0.1375, 
                              NA, NA, NA, NA, 
                               NA, NA, NA, NA,
                              0.6675, 0.475, 0.6675, 0.475), 
           reference.range = c("[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "", "", "", "",
                               "(0.22)b", "(0.14)b",  "(0.22)b", "(0.14)b", 
                               "", "", "", "",
                              "", "", "", "",
                               "(0.35)b", "(0.22)b","(0.35)b", "(0.22)b" ))) %>%
        mutate(m.sd = if_else(mean > 6, paste0(sprintf("%.1f",mean), " (", sprintf("%.1f",sd), ")"), 
                              paste0(sprintf("%.2f", mean), " (", sprintf("%.2f",sd), ")")), 
               reference = if_else(is.na(reference.mean), "",
                                   paste0(sprintf("%.2f",reference.mean), " ", reference.range))) %>%
        filter(!(variable %in% c("str.change.day") )) %>%
                       dplyr::select(variable, sex, sets, m.sd, reference) %>%
        
        mutate(variable = factor(variable, levels = c("csa.rel.change", 
                                                      "csa.change.day", 
                                                      "csa.session", 
                                                      "str.rel.change", 
                                                      "str.session"), 
                                 labels = c("CSA %-change", 
                                            "CSA %-change per day", 
                                            "CSA %-change per session", 
                                            "Average strength %-change", 
                                            "Average strength %-change per session")), 
               sex = factor(sex, levels = c("female", 
                                            "male"), 
                            labels = c("Female", "Male")), 
               sets = factor(sets, levels = c("single", "multiple"), 
                             labels = c("LOW", "MOD"))) 

## Ahtiaianen on average  20 weeks with 2 sessions per week = 40 sessions

## Muscle hypertrophy
## In the younger cohorts, male average 5.5% (5.6) = 5.5/40 ~ 0.1375 per session
# in females 3.3 (8.6)% = 0.825
# 8.6/3.3 * 3.3/40
# 5.6/5.5 * 5.5/40
# Muscle strength in younger cohorts
#  male    19% (8.6)/40 sessions ~  0.5 % per session
#  female  26.7% (13.9)/40 ~ 
# 8.6/19 * (19/40)
# 13.9/26.7 * 26.7/40

# Wernbom increase per day in quadriceps CSA per day was 0.06%

```


In Study I, the average increases in muscle strength and mass regardless of volume condition corresponded to what could be expected based on previous research in young, healthy participants (Table \@ref(tab:csa-str-tab)) 
[@RN346;<!-- Wernbom 2007 -->
@RN1741],<!-- Ahtiainen  -->
indicating the general efficacy of the training program.

For muscle hypertrophy and strength gains, the moderate-volume condition consistently showed greater adaptations when compared to the low-volume condition (Figure \@ref(fig:comb-fig-s1)a).
These patterns were in general agreement with previous meta-analyses indicating that increased training volume leads to more favorable outcomes, both in terms of muscle mass
[@RN1767;<!-- Schenfeld  2016--> 
@RN789]<!-- Krieger 2010 -->
and strength
[@RN2492;<!-- Ralston --> 
@RN793].<!-- Krieger 2009 -->

```{r meta-analyses-descriptive, warning=FALSE, message=FALSE, echo=FALSE}

summary_table_meta <- readRDS( "./data/derivedData/meta/naive_models.RDS")


summary_table_meta <- summary_table_meta %>%
  mutate(stat = paste0(sprintf("%.3f", median), 
                       " [",
                       sprintf("%.3f", cri.lwr), 
                       ", ",
                       sprintf("%.3f", cri.upr), 
                       "]"))


```

Since publication of the most recent meta-analyses [@RN1767;@RN2492],<!-- Schenfeld  2016 / Ralston 2107 --> 
additional studies have been published investigating the effect of resistance training volume on muscle mass and strength. Based on studies included in the meta-analysis performed by Schoenfeld *et al.* [@RN1767], additional results were extracted from studies gathered through a systematic literature search among newly published studies (from January 2015 to June 2020). Compared to Schoenfeld *et al.* [@RN1767], an additional ten studies were included in the present analysis.
Subsequently, effect sizes were extracted from 25 studies, both for muscle mass and strength, and meta-regression models were used to investigate the effect of the weekly number of sets on training-induced changes in these outcomes.
Both muscle mass and strength were shown to be robustly affected by an increased training volume measured as an increase in weekly training sets (after controlling for potential confounding factors^[Models were controlled for age (young, middle age or old), sex (male, female or mixed), length of study (number of weeks), type of measurement (direct or indirect for muscle mass and training-specific or unspecific in strength), training status (trained > six months experience with resistance training), body portion (upper, lower or whole)]).
For every additional weekly set, the effect size measure (Hedges' *g*, [95% credible intervals] (CrI)), increased by
`r summary_table_meta[1, 5]` and `r summary_table_meta[2, 5]`
for muscle mass)
and strength (Figure \@ref(fig:comb-fig-s1)b),
respectively.
Together these data underline the training volume dose-dependence of both muscle hypertrophy and strength gains.

(ref:wernbom-citation) @RN346

(ref:ahtiainen-citation) @RN1741

```{r csa-str-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

sum.stats %>%
        arrange(variable, sex, sets) %>%
  dplyr::select(-variable) %>%
        kable(format = "latex", booktabs = TRUE, linesep = "",
              position = "!b",
              col.names = c("Sex", "Condition", "Mean (SD)", "Reference"), 
              caption = "Training induced changes in muscle CSA and average strength in Study I")  %>%
        collapse_rows(c(1, 4))  %>%
        kable_styling(font_size = 7) %>%
          
          pack_rows("CSA %-change", 1, 4) %>%
        pack_rows("CSA %-change per day", 5, 8) %>%
          pack_rows("CSA %-change per session", 9, 12) %>%
          pack_rows("Average strength %-change", 13, 16) %>%
          pack_rows("Average strength %-change per session", 17, 20) %>%
  
        add_footnote(label = c(paste0("Estimates from Wernbom et al. ", "(ref:wernbom-citation)"),
                               paste0("Estimates from Ahtiainen et al. ", "(ref:ahtiainen-citation)")), 
                     notation = "alphabet")

```


```{r comb-fig-s1, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences in training-induced relative changes in muscle-mass and strength in Study I with estimates derived from ANCOVA models controling for baseline values (a). Estimates from meta-regression models examining the effect of weekly number of sets per muscle group on muscle hypertrophy and strength gains (b). Estimates in \\textit{b} are effect size measures (Hedges g) with 95\\% credible intervals (CrI) for the estimated effect of weekly number of sets in each study and the overall effect (Summary). Filled areas represent the cumulative distribution of each effect.", fig.scap = "Differences in training induced changes to muscle mass and strength measures between volume conditions in Study I and weekly training volume meta-regression.", fig.height=6.5, fig.width=4.72, fig.align='center'}

library(ggplot2); library(ggtext)

# The figure and estimates are from ./R/study-1/muscle-strength-size.R
muscle_strength_fig <- readRDS("./data/derivedData/study1_muscle-strength-size/muscle-strength-fig.RDS")

muscle_strength_fig

```

In an attempt to explain differences in training outcomes between volume conditions, selected molecular markers known to influence adaptations to resistance training were investigated for volume-dependency.
First, acute activation of signaling along the mechanosensitive mTORC1-pathway was measured before and after the fifth training session in Study I.
A commonly used readout of mTORC1-signaling is the phosphorylation of S6K1 at Thr\textsuperscript{389}/Thr\textsuperscript{412} which in turn precedes phosphorylation of rpS6 (see Figure \@ref(fig:mtor-fig)a).
In the present study, exercise induced S6K1\textsuperscript{Thr\textsuperscript{389}/Thr\textsuperscript{412}} phosphorylation was indeed shown to be volume dependent along with phosphorylation of rpS6\textsuperscript{Ser\textsuperscript{235/236}} and mTOR\textsuperscript{Ser\textsuperscript{2448}} (Figure \@ref(fig:mtor-fig) b).
Together these observations indicate larger perturbations along the mTORC1 signaling axis which confirms previous studies showing an exercise-volume dependency of mTORC1-related signaling in the acute phase (< 5 hours) after exercise when the low-volume condition consists of at least three sets
[@RN2716;@RN1837;@RN784;@RN791;@RN834].<!-- Kumar 2012 --><!-- Ahtiainen 5 vs 10 sets p70 and s6 different --><!-- Terzis volume affect activation -->
It is recognized that phosphorylation of mTOR itself at Ser\textsuperscript{2448} primarily should be regarded as indicative for negative feedback as this site is phosphorylated due to S6K1 activity
[@RN2309] (Figure \@ref(fig:mtor-fig) a).<!-- Figueriedo mTOR and ser2448 signalling review -->
It is also recognized that the phosphorylation status of rpS6 at Ser\textsuperscript{235/236} is not solely due to mTORC1 signaling as both mTORC1 and extracellular signal-regulated kinases (ERK)-signaling converges here
[@RN2311].<!-- Roux S6 phopshorylation at ser 235 236 promoted by ras/erk -->
Interestingly, Terzis *et al.*
[@RN785]<!--Terzis et al., 2008-->
and Ahtiainen *et al.*
[@RN1837]
did not report any clear volume dependency in exercise-induced activation of ERK 1/2.
While this may suggest that any volume-dependent regulation of this pathway was outside the timeframe of these studies, it may also suggest that volume selectively modulates specific pathways in the acute phase after resistance exercise.
It should be noted that the present and previous studies are limited in their temporal resolution, and different patterns over time in relation to exercise-volume cannot be ruled out.

```{r p70-csa-correlation-data, warning=FALSE, message=FALSE, echo=FALSE}

### Check association with 

p70_csa <- readRDS("./data/derivedData/study1-western-blot/csa_p70.RDS")

### Calculate raw correlations
cor_est_p70 <- p70_csa %>%
  group_by(sets) %>%
   summarise(p70.fc.cor = cor.test(fc, change, method = "spearman")$estimate, 
            p85.fc.cor = cor.test(fc.p85, change, method = "spearman")$estimate, 
            p70.fc.cor.p = cor.test(fc, change, method = "spearman")$p.value, 
            p85.fc.cor.p = cor.test(fc.p85, change, method = "spearman")$p.value)  %>%
  data.frame()

p70.m <- lme(change ~ w2pre + scale(fc) + sets, 
             random = list(subject = ~ 1), 
             weights = varExp(),
             data = p70_csa)

p85.m <- lme(change ~  w2pre.p85 + scale(fc.p85)  + sets , 
             random = list(subject = ~ 1),
             weights = varExp(),
             data = p70_csa)

# plot(p70.m)
# plot(p85.m)


p70.mod <- paste0(sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 3]), 
                  "]")

p85.mod <- paste0(sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 3]), 
                  "]")

corr_est_p70 <- paste0(sprintf("%.3f",min(c(cor_est_p70[,2],cor_est_p70[,3]))), 
       " - ", 
       sprintf("%.3f",max(c(cor_est_p70[,2],cor_est_p70[,3])))) 

```

The importance of mTORC1 signaling for protein synthesis in the acute phase after resistance exercise is well established.
In humans, administration of rapamycin, a selective inhibitor of mTORC1, before resistance exercise leads to delayed or blunted activation of mTORC1 effectors such as
S6K1 and rpS6, as well as unchanged levels of phosphorylation of 4E-BP1 and eEF2 resulting in abolished exercise-induced increase in protein synthesis
[@RN780].<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
Volume-dependent increase in mTORC1 signaling also coincide with larger protein synthesis rates
[@RN791;<!-- Burd in single session single and multiple sets phosphorylation p70   -->
@RN2716]<!-- Kumar volume affects protein synthesis -->
suggesting that higher exercise volume can be regarded as leading to an increased potential for protein synthesis in the acute-phase after exercise.
Previous studies have indirectly linked mTORC1-related signaling to muscle growth as acute perturbation along this pathway has been correlated with muscle growth
[@RN785;<!--Terzis et al., 2008-->
@RN788].<!-- Mitchell et al., 2013-->
However, this is not a consistent finding in the literature 
[@RN834;<!-- Mitchell et al., 2012-->
@RN2171],<!-- Phillips et al., 2017-->
nor is it consistent with the results in the present study as acute activation of S6K1 in response to the fifth session was not associated with training-induced muscle mass changes.^[Raw Spearman's correlations between exercise-induced change in S6K1 phosphorylation and change in muscle mass (measured with MRI) indicated weak (insignificant) associations ($\rho=$ `r corr_est_p70`) depending on the isoform measured (p70 or p85) and the number of sets. More elaborate modeling confirmed weak relationships between the degree of phosphorylation and muscle mass change. When controlling for baseline values in degree of phosphorylation (as this influence the calculation of fold-change) and number of sets, standardized estimates for the effect of phosphorylation were (increase in %-muscle mass change for every 1SD change in fold-change phosphorylation, with [95% CI])  $\beta=$ `r p70.mod` for p70 S6K1-p70\textsuperscript{Thr389} and $\beta=$ `r p85.mod` for S6K1-p85\textsuperscript{Thr412}.]

```{r mtor-fig, warning=FALSE, message=FALSE, echo=FALSE, fig.cap="Measured phosphorylation sites in context (a) and differences between volume conditions in phosphorylation status of S6K1 at Thr \\textsuperscript{389} (p70) Thr \\textsuperscript{412} (p85), rpS6 at Ser \\textsuperscript{235 236} and mTOR at Ser \\textsuperscript{2448} induced by acute exercise and expressed as fold-changes (b). Estimates in \\textit{b} are derived from ANCOVA models controlling for baseline values and presented as point estimates with 95\\% confidence intervals. Dots in \\textit{a} represents up-stream regulators.", fig.scap = "Differences between volume conditions in exercise induced phosphorylation of proteins related to mTORC1 signaling", fig.height=3.5, fig.width=4.72, fig.align='center'}

western_fig <- readRDS("./figures/results/study1-western-blot.RDS")
western_fig

```

Although mTORC1 signaling most certainly contributes to protein synthesis in the acute recovery phase after resistance exercise [@RN780],<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
several aspects complicate its applicability as a predictor of training-induced muscle hypertrophy.
First, measuring its activity is not straight forward for biological reasons, including pathway cross-talk and negative feedback as well as technical aspects
[@RN1949;@RN2311;@RN1942].<!-- Figueriedo, mTOR feedback --><!-- Cross talk with erk --><!--  Bass technical considerations in western blotting -->
Secondly, mTORC1 signaling is not a stable phenomenon either between or within individuals as short- and long-term resistance training leads to diminished activity along its pathway
[@RN1809;@RN1871].<!-- mTOR signal lost after three weeks -->
Thirdly, mTORC1 readily integrate multiple signals but also convey these to multiple downstream processes
[@RN2320].
As such, mTORC1 activity represents an early, transient response to resistance exercise, with mTORC1 signaling contributing to accumulative responses.
One such downstream target of mTORC1 is ribosomal biogenesis
[@RN1632;@RN1810;@RN1828;@RN2321],
<!-- Nader mTOR regulates ribo biogenthrough cell cycle regulators--><!-- von Walden 2012 mTOR regulates myotube... --><!-- von Walden 2016 --><!-- Chauvin S6K controls RiBi transcriptional program -->
ultimately leading to accumulation of ribosomal RNA, a response typically seen in connection with resistance training
[@RN1755;@RN1644;@RN1809;@RN1897]<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD--><!-- Figuieredo 2015 16 sessions 49% increase young --><!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young--<!-- 11% 36 sessions young -->.

```{r ribo-markers}

descriptive_ribo_markers <- readRDS("./data/derivedData/study1-ribo-biogenesis/descriptive_ribo_markers.RDS")

rrna <- descriptive_ribo_markers %>%
  filter(target != "totalrna") %>%
  summarise(min.w2pre = round(min(w2pre),0), 
            max.w2pre = round(max(w2pre),0), 
            min.w12 = round(min(w12),0), 
            max.w12 = round(max(w12),0)) 
```

As ribosomal RNA is the most abundant constituent of muscle RNA, it provides an estimate of muscle ribosomal abundance when expressed per unit tissue weight
[@RN2054;@RN2145].<!-- Zak 1967 in heart muscle 85% of RNA is ribosomal RNA --><!-- Young -->
From baseline to prior to the fifth training session (Week 2), total RNA per mg tissue increased by $\sim$ `r round(descriptive_ribo_markers[9,2], 0)` and `r round(descriptive_ribo_markers[10,2], 0)`% in the low- and moderate-volume condition, respectively. Total RNA levels were still elevated above baseline after the intervention (Week 12, low-volume $\sim$ `r round(descriptive_ribo_markers[9,3], 0)`; moderate-volume $\sim$ `r round(descriptive_ribo_markers[10,3], 0)`%). Similar patterns were seen in target analysis of ribosomal RNA species using qPCR (low-volume increase from baseline to Week 2 $\sim$ `r rrna[1,2]`-`r rrna[1,3]`% and Week 12 $\sim$ `r rrna[1,4]`-`r rrna[1,5]`%; moderate-volume $\sim$ `r rrna[2,2]`-`r rrna[2,3]`% and Week 12 $\sim$ `r rrna[2,4]`-`r rrna[2,5]`%). Comparing volume conditions revealed higher levels of total RNA and mature ribosomal RNA species in the moderate-volume condition at Week 2 (Figure \@ref(fig:rrna-fig)a). At Week 12, between-volume-condition differences in total RNA were less clear and ribosomal RNA 28S even showed higher levels in the low-volume leg (Figure \@ref(fig:rrna-fig)a).
Analysis of c-Myc mRNA abundance in response to the fifth training session also showed volume-dependent regulation with exercise-induced increases being $\sim$ 1.5-fold higher in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:rrna-fig)b). c-Myc represents a rapamycin-insensitive signaling pathway, parallel to mTORC1, known to also stimulate ribosomal biogenesis 
[@RN1754;@RN1834;@RN1832].<!-- West --><!-- cMyc driver of ribosomal biogenesis -->

```{r rrna-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences between volume conditions in total RNA and ribosomal RNA-species (pre-rRNA 45S, rRNA 18S, rRNA 5.8S and rRNA 28S) measured at rest at Weeks 0, 2 and 12 in Study I (a). Acute changes in abundance of c-Myc mRNA and pre-rRNA 45S in response to acute exercise in Week 2 and differences between volume conditions (b). Errorbars represents 95\\% confidence intervals, transparent points and error bars signifies that the confidence interval contain 1 (the null hypothesis).", fig.scap = "Differences between volume conditions in total RNA and ribosomal RNA", fig.height=2.5, fig.width=4.72, fig.align='center'}

ribo_markers_fig <- readRDS("./figures/results/study1-ribo-markers.RDS")

ribo_markers_fig

```


Ribosomal biogenesis is an early adaptation to resistance training with initial increases in total RNA evident before any hypertrophy can be detected
[@RN1656;@RN1520;@RN1596;@RN2113].<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old --><!-- Bickel 2005 Two boutselectric stimulation, young ~ 30% incr--><!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training --><!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions-->
The increase seen in total RNA serves as a marker of ribosomal accumulation and thus increased translational capacity. This improved translational capacity contributes to protein accretion resulting in muscle hypertrophy, reflected in associations between measures of ribosome abundance (typically total RNA) and muscle hypertrophy
[@RN1644;@RN1755;@RN2055;@RN1897].<!-- Figueriedo 2015 correlation between delta csa and fold change of total RNA --><!-- Stec 2016 only high responders have increased total RNA and rRNA abundance after short training --><!-- Mobley weak correlation r = 0.28 p = 0.026  and no cluster * time interaction effect but increases in total RNA 34% in high responders vs 8% in low --><!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->
Due to the sequential nature of these events (increased translational capacity before muscle hypertrophy), early increases in total RNA would possibly better reflect training-induced changes in muscle hypertrophy.
Any increase in muscle mass would also contribute to a dilution of the RNA pool
[@RN2142]<!-- figueriedo description of dilution effect -->
possibly masking their causal relationship.
Indeed, comparing the relationships between RNA abundance and muscle hypertrophy at different time-points supported this view as the amount of RNA at Week 2 showed the most robust relationship with the resulting muscle growth (Figure \@ref(fig:rrna-csa-fig), \@ref(tab:rna-csa-tab)). This indicates that translational capacity, affected by exercise, early in the training program determines training-induced changes in muscle mass. 

```{r rna-csa-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

rna_csa_change_models <- readRDS("./data/derivedData/study1-rna-csa-cor/regression.RDS")

rna_csa_change_models %>%
  mutate(Estimate = round(Estimate, 2), 
         SE = round(SE, 2), 
         df = df, 
         t.value = round(t.value, 2), 
         ci = paste0("[", sprintf('%#.2f', lower), ", ", sprintf('%#.2f', upper), "]"), 
         beta = paste0(sprintf('%#.2f',scaled.est), 
                       " [", 
                       sprintf('%#.2f', scaled.lower), 
                       ", ", 
                       round(scaled.upper, 2), "]"),
         beta = if_else(term %in% c("(Intercept)", "setsmultiple"), "", beta)) %>%
  dplyr::select(term, Estimate, SE, df, t.value, ci, beta) %>%
  mutate(term = factor(term, 
                       levels = c("(Intercept)",
                                  "setsmultiple", 
                                        "w0", 
                                        "w2pre", 
                                        "w12"), 
                       labels = c("(Intercept)", 
                                  "Moderate- vs. low-volume",
                                  "Week 0", 
                                  "Week 2", 
                                  "Week 12"))) %>%
  kable(format = "latex", 
        booktabs = TRUE, 
        escape = FALSE,
        caption = "Influence of RNA abundance on training-induced muscle growth measured with MRI", 
        col.names = c("", "Estimate", "SE", "df", "\\textit{t}-value", "95\\% CI", "Estimate [95\\% CI]")) %>%
  add_header_above(c(" " = 6, "Standardized coef." = 1), 
                   escape = FALSE) %>%
  pack_rows("RNA abundance (ng mg\\textsuperscript{-1})", 3, 5, escape = FALSE) %>%
  kable_styling(font_size = 7) %>%
  add_footnote("Standardized coefficients scaled by its SD")


```

```{r rrna-csa-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Relationship between total RNA at Weeks 0, 2 and 12 and training induced percentage change in thigh cross sectional-area (CSA change) as measured by MRI. Dashed line represent naive relationship, solid line represent adjusted reliationship as seen in Table 5.2.", fig.scap = "Relationship between total RNA and training induced muscle growth", fig.height=4.72/3, fig.width=4.72, fig.align='center'}

rna_csa_fig <- readRDS("./figures/results/study1-rna-csa.RDS")


rna_csa_fig

```

## Volume-dependent remodeling of muscle fiber-type composition

```{r myhc-stats, warning=FALSE,message=FALSE, echo=FALSE}

mx.summary <- readRDS("./data/derivedData/study1-myhc-composition/myhc2x_stats.RDS")

or.week2 <- paste0(sprintf('%#.2f', exp(mx.summary[5, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[5, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[5, 6])), 
                   "]")

or.week12 <- paste0(sprintf('%#.2f', exp(mx.summary[6, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[6, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[6, 6])), 
                   "]")


```

Resistance training readily leads to remodeling of the muscle, including qualitative changes in fiber type composition through changes in the myosin heavy-chain protein composition
[@RN2220;@RN2799;@RN819;@RN1489].<!-- Adams Down regulation of MyHc 2x content in training group both myhc isoform sds page and atpase staining --><!-- Williamsson less IIx dominant hybrid fibers with training conversion towards IIa --><!-- Schiaffino review, other protein families also different between fiber types--><!-- Ellefsen 2014, training induced switch from 2A to 2X -->
Resistance training-induced changes in fiber-types identified by myosin heavy-chain expression occurs primarily as a IIX $\rightarrow$ IIA transition, which results in a more fatigue-resistant phenotype
[@RN819;@RN2056;@RN2801]<!-- Schiaffino review, other protein families also different between fiber types--><!-- Andersen, primarily transitions fiber in 2x to 2a direction minimal 2 to 1 trans --><!-- Karatzaferi 2001 Drop in ATP most pronounced in type 2x fibers in response to acute exercise -->
and thus represents an important aspect of the adaptive response to resistance training by reversing effects of inactivity and disuse
[@RN2057;@RN2108].<!-- Andersen 2x overshoot after de training --><!-- Bed rest increases type IIx -->
In Study I, muscle fiber-type composition was primarily determined based on myosin heavy-chain protein's immunostaining in muscle cross-sections. The staining protocol allowed for the identification of Type I (red), Type IIA (brown), Type IIX (unstained), and Type IIA/IIX hybrids (light brown, Figure \@ref(fig:myhc-fig)a). Throughout the study, a general reduction in Type IIX fibers was seen as pure Type IIX was replaced with Type IIA/IIX hybrids at Week 2, and these, in turn, were measured at lower proportions at Week 12 (Figure \@ref(fig:myhc-fig)b).

After twelve weeks of training, the proportion of Type IIX fibers was lower in the moderate- compared to the low-volume condition (indicative as an odd-ratio (OR) < 1, OR [95% CI]: `r or.week12`, combined measure of IIX/IIA hybrids and pure IIX fibers, Figure \@ref(fig:myhc-fig)d). Reductions seen in Type IIX myosin heavy-chain expression were reflected on the gene level as moderate-volume training led to more pronounced down-regulation of the *MYH1* gene, coding for Type IIX myosin heavy-chain protein, at all time-points during the intervention (Figure \@ref(fig:myhc-fig)c).
This observation underlines that training volume is an important factor for fiber-type composition remodeling. Although not surprising, similar observations are lacking in the literature as previous studies have not compared this transition directly between volume protocols.
However, when comparing non-exhaustive high-load resistance training to load-matched training to failure, Pareja-Blanco *et al.*
[@RN2217]
observed a blunted IIX $\rightarrow$ IIA transitions in response to the non-exhaustive protocol.
Together these data indicate that increased metabolic stress and/or dosage of neuromuscular activity are plausible candidates for regulating IIX $\rightarrow$ IIA reprogramming in humans in response to resistance training.
This as opposed to tensile stimuli.
Based on observations in animal models, neural input might, however, be considered the primary candidate.
For example, in a rat model of resistance training, the force did not affect fast-to-slow fiber-type transitions when performed with the same electrical stimulation pattern, indicating that neural activity governs fiber-type transitions
[@RN2104].<!-- Eftestol -->

At Week 2, a more pronounced reduction in *MYH1* gene abundance in the moderate-volume condition did not translate to the protein level. Instead, a greater number of fibers were classified as IIX in the moderate-volume condition compared to the low-volume condition (OR: `r or.week2`).
The moderate-volume condition thus showed a greater mismatch between gene and protein levels compared to the low-volume condition.
Such mismatch is seen after resistance-exercise and is indicative for the early transition of muscle fibers reflecting the sequential order of events, whereby changes at the RNA level do not immediately translate into changes at the protein level
[@RN2056].<!-- Andersen mismatch between  -->
This indicated that remodeling occurred faster in response to the low-volume condition despite a greater "transition signal" in the moderate-volume leg.
A plausible explanation for this observation could be that the likely more pronounced cellular repair after damage caused by a greater training stimulus leads to a delayed fiber-type transition. 
This hypothesis is supported by observations indicating that a proportion of the protein synthesis seen early in a resistance-training period can be attributed to tissue repair rather than myofibrillar synthesis
[@RN1521;<!--Kim 2005 -->
@RN2144].<!-- Damas 2016 -->
Regardless of causality, the more rapid early adaptation seen in response to a lower training volume suggests that optimized resistance training should include a progressive volume component.



\pagebreak

```{r myhc-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Representative muscle cross-sections stained for myosin-heavy chain isoforms are shown in (a). Using antibodies for Type I (MyHC Slow) and all but Type IIX (BF-35) enabeled separating Type I fibers from other fibers (red stain, a lower panel). Type IIX fibers were indentified as unstained, and weak brown staining was analyzed as Type IIX/IIA hybrids (a, upper panel). Resistance training led to a reduction in Type IIX and IIX/IIA hybrids, (b) shows participant-average Type IIX and IIX/IIA percentages. Training led to reduction in \\textit{MYH1} gene expression, and more so in the moderate-volume condition, evident from pairwise comparisons in (fold-differences with 95\\% CI, c). Differences in protein expression of Type IIX (counted as pure IIX + 0.5 $\\times$ IIX/IIA hybrids) reversed throughout the study as the moderate-volume condition showed greater proportions of IIX fibers at Week 2 but lesser expression at Week 12 (d). Points in \\textit{d} (right panel) represents average per volume condition.", fig.scap = "Fiber-type composition in Study I", fig.height=6, fig.width=4.72, fig.align='center'}

# load fiber typing figure
img <- ggdraw() + draw_image("./data/study-1/immuno/immuno_control_image2.png")

# load figures
myhc_fig_comps <- readRDS( "./data/derivedData/study1-myhc-composition/myhc_fig.RDS")


## Combine figure directly in manuscript as saving the png takes some space 

myhc_fig <- plot_grid(plot_grid(NULL, img, NULL, ncol = 3, rel_widths = c(0.2, 1, 0.2)), 
                      plot_grid(myhc_fig_comps$overall_type2x, 
                                NULL,
                                plot_grid(myhc_fig_comps$myhc2x_mrna_means, 
                                          myhc_fig_comps$diff_2x_mRNA, 
                                                    ncol = 1, 
                                                    align = "v",
                                                    rel_heights = c(0.5, 0.5)),
                                          ncol = 3, rel_widths = c(0.5,0.05, 0.5)),
                  
                       myhc_fig_comps$fiber_type_insets, 
          ncol = 1, 
          rel_heights = c(0.6, 0.6, 0.6)) + 
  draw_plot_label(x = c(0.25,
                   0, 0.45, 
                   0), 
             y = c(0.98, 
                   0.67, 0.67, 
                   0.36), 
             label = c("a", "b", "c", "d"))

myhc_fig

```

\pagebreak

## Volume-dependent effects on transcriptome characteristics

As already [discussed](#muscle-mass-growth), resistance training in Study I led to an increase in total RNA per unit tissue weight. The magnitude of this increase was different between volume conditions (see Figure \@ref(fig:rrna-fig)).
Studies of gene expression using qPCR and RNA sequencing techniques assume that equal amounts of biological material are used when evaluating the abundance of single transcripts.
It is therefore common to use a set amount of total RNA during sample preparation to satisfy such an assumption
[@RN2298;@RN2400;@RN2401].<!-- Damas 2018 --><!-- Robinson 2017--><!-- Laker 2017.-->
However, if the ratio of RNA to tissue mass for some reason changes throughout the course of a study, this would mean that analyses are performed on different amounts of tissue. This may be the case after a resistance training intervention, and may even be differentially affected by different resistance training modalities, such as in Study I.
Indeed, in Study I, the resistance training protocol resulted in a situation where samples were prepared from a lesser amount of tissue as a fixed amount of RNA was used during sample preparation.
This effect was more pronounced in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:lib-size-fig)a).

Despite the fixed amount of total RNA being used during sample preparation, the total abundance of mRNA increased, evident as increases in total library sizes, quantified from RNA sequencing as the total number of quantified gene counts. The average increase in total library size was more apparent in the low-volume condition
(Figure \@ref(fig:lib-size-fig)b).
When library sizes were normalized to the amount of tissue used in sample preparation, differences between conditions were diminished
(Figure \@ref(fig:lib-size-fig)c).
This suggests that resistance-training lead to increased global mRNA expression per-unit-muscle weight (43-53\% in the present study). This increase was not different between volume-conditions, suggesting that the volume-dependent regulation of RNA transcription primarily relates to transcription of RNA-fractions such as rRNA rather than mRNA (see Figure \@ref(fig:rrna-fig)). 

```{r lib-size-fig, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Less muscle tissue was used for cDNA synthesis in the moderate- compared to low-volume condition as a result of total RNA increases per tissue weight (a). Library sizes tended to be greater in the low-volume condition (b), after normalizing library sizes to muscle weight differences were diminished (c). Lower panels show differences between volume conditions.", fig.scap = "Muscle weight in RNA-sequencing library preparation", fig.height=3, fig.width=4.72, fig.align='center'}

libsize.fig <- readRDS("./data/derivedData/study1-transcriptome/muscle-weight-lib-size.RDS")


libsize.fig +
  draw_plot_label(x = c(0, 0.33, 0.67), 
             y = c(1,1,1), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b", "c"))


tissue_offset_ngenes <- readRDS("./data/derivedData/study1-transcriptome/n_genes_regulated_time.RDS") %>%
  data.frame() %>%
  filter(model == "tissue")


w2pre_unchanged <- sprintf("%.1f", tissue_offset_ngenes[4,5] * 100)
w12_unchanged <- sprintf("%.1f", tissue_offset_ngenes[1,5] * 100)

w2pre_up<- sprintf("%.1f", tissue_offset_ngenes[5,5] * 100)
w12_up<- sprintf("%.1f", tissue_offset_ngenes[2,5] * 100)

w2pre_down<- sprintf("%.2f", tissue_offset_ngenes[6,5] * 100)
w12_down<- sprintf("%.2f", tissue_offset_ngenes[3,5] * 100)


```

<!-- The general increase in mRNA counts seen in both volume-conditions is intriguing as during steady state conditions, total mRNA abundance generally scales with cellular volume
[@RN2876].<!-- Padovan-Merhar 2015 mRNA scale with cell size, increasing cell size increases global transcription to compensate -->
<!-- However, during mechanically induced muscle growth, Kirby *et al.* found that nascent transcription, including mRNA transcription is elevated primarily in small muscle fibers
[@RN2616].<!-- Kirby nascent transcript in smaller fibers during hypertrophy (SA), global promoter activation -->
<!-- This may correlate with small fibers being prime drivers of muscle hypertrophy as these fibers are more likely to increase in size
[@RN2616].<!-- Kirby nascent transcript in smaller fibers during hypertrophy (SA), global promoter activation -->
<!-- Training-induced increases in mRNA may thus represent a mechanism important for protein accretion and hypertrophy.
<!--
Interestingly, rRNA levels are found to be higher in slow vs. fast muscle fibers with selected mRNA found to scale with total RNA indicating that also mRNA levels are greater in more glytolytic fibers
[@RN1707].<!-- Habets 1999 in rat higher 28S RNA and total rna in slow compared to fast fibers, unchanged myhc mRNA/RNA between fibers   -->
<!--This could thus indicate that elevated mRNA levels precedes muscle remodeling to a more-->
Although the observed increase in library sizes suggests global increases in mRNA transcription, it did not resemble global gene amplification seen in models with experimentally manipulated c-Myc protein expression
[@RN2430;@RN2359]<!-- Lin 2012 increased cmyc leads to increased expression of all active genes--><!-- Loven global amplification in the light of gene expression studies  -->
as a considerable fraction of genes were unchanged by training.
When expressing gene abundance per the amount of tissue used in sample preparation, `r w2pre_unchanged`\% and `r w12_unchanged`\% of identified genes were classified^[Up- and down-regulated genes were classified based on log\textsubscript{2}-fold change > 0.5 and < -0.5 together with FDR-corrected *P*-values < 0.05] as unchanged from baseline to Week 2 and 12, respectively. A small fraction of genes was identified as being less abundant at Week 2 (`r w2pre_down`\%) and Week 12 (`r w12_down`\%). The majority of genes were classified as up-regulated from baseline to Week 2 (`r w2pre_up`\%) and Week 12 (`r w12_up`\%).

Gene-set enrichment analyses revealed that training-induced changes in gene expression profiles were related to increased abundance of genes associated with extracellular matrix remodeling, collagen synthesis and inflammatory processes. Although these observations were made without a proper control to investigate training-induced effects *per se*, they resemble previous (controlled and uncontrolled) studies characterizing skeletal muscle tissue in response to heavy mechanical loading in the acute phase
[@RN2451; @RN2453;@RN2454;@RN2457]<!-- Wessner acute exercise gives different regulation of metalloproteinase --><!-- Mackey  repeated bout effect 30 days --><!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load --><!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->,
and after prolonged training [@RN774;@RN2298].<!-- Raue, upregulation of ECM after long term exercise --><!-- Damas long term training ECM up gene-->

```{r week2-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Genes identified as differentially expressed (log2 fold-change > |0.5|, FDR < 0.05) are highlighted in \\textit{a}.  Overlap in differentially expressed genes between models are shown in (b)", fig.scap = "General patterns of differentially expressed genes at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

week2_grid <- readRDS("./data/derivedData/study1-transcriptome/week2_volcano_upset.RDS")

week2_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

To be able to account for differences in the amount of tissue used in sample preparation due to volume-dependent differences in the ratio of total RNA to tissue weight, three different normalization strategies were used to determine volume-dependence in transcriptome characteristics.
By implementing each normalization strategy as variations of generalized linear mixed models (GLMM), we could compare outcomes related to conceptually different normalization techniques using the same statistical framework.
The use of GLMM also had the benefit of accounting for the study design, which included correlated data (within participants)
[@RN2366].<!-- Cui et al what if we ignore random effects -->
In the first model, an offset term expressing gene counts per tissue weight was included. This term was combined with the effective library size, included as a fixed effect in the model together with study conditions, time, and exercise volume (tissue offset-model).
In a second model, the offset term was removed. Thus, only the effective library size was used to normalize the gene counts, as suggested by Cui *et al.*
[@RN2366] (effective library-size model).<!-- Cui et al what if we ignore random effects -->
For comparison, a naive model containing no normalization-parameters was also included in analyses.

At Week 2, tissue-offset normalization (modeling mRNA counts per-mg-muscle weight) revealed a general shift of mRNA counts towards the moderate-volume condition (Figure \@ref(fig:week2-upset-volcano)a).
This contrasted pattern found using "conventional" library-size normalization that in turn more resembled the non-normalized naive model, seen as an overlap of genes identified as differentially expressed between the models (Figure \@ref(fig:week2-upset-volcano)b).
Gene-set enrichment analyzes revealed that genes related to extracellular matrix remodeling and collagen synthesis were driving the difference in transcriptome profiles between volume-conditions at Week 2
(Figure \@ref(fig:week2-go-analysis)a).
This was apparent using both the tissue offset model and the library-size normalized model. However, the volume-dependent effects were more pronounced in the tissue-offset model
(Figure \@ref(fig:week2-go-analysis)b).
Differences in normalization models led to global shifts of estimates of differences between volume conditions. Figure \@ref(fig:gene-set-rug) shows this global shift as density curves representing all fold-changes between volume conditions. Accounting for the amount of muscle mass in sample preparations also shifted specific gene-sets towards greater differences between volume conditions. This is exemplified in Figure \@ref(fig:gene-set-rug) by the gene set "Collagen containing extracellular matrix".

```{r week2-go-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Gene-set enrichment analysis based on gene-ontology gene sets and estimates from the tissue-offset model (a). Adjusted \\textit{P}-values and Normalized enrichment scores are retrieved from analysis of directional rank-based tests (GSEA), labeling of points indicate if gene-sets were also identified from differentially expressed genes (Log2 fold-change >|0.5|, FDR < 0.05). In \\textit{b} comparison between the tissue-offset and library size normalization models are shown in rank-based (non-directional, cerno-test) analysis of gene ontology gene sets.", fig.scap = "Gene-set enrichment analysis at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

go_analysis_week2 <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/go_week2.RDS")


go_analysis_week2 +
  draw_plot_label(x = c(0.02, 0.55), 
             y = c(1, 0.85), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))



```



```{r gene-set-rug, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Differences between volume conditions at Week 2 as seen using different normalization models. Shifts in the full distribution of Log2-fold change values between volume conditions were seen between on normalization models (density curves). As an example, genes associated with the \\textit{Collagen containing extracellular matrix} gene set are highlighted (black bars), differentially expressed genes are identified with gene symbols in each method.", fig.scap = "Global shifts in volume-dependent fold change as an effect of normalization methods at Week 2", fig.height=5, fig.width=3.72, fig.align='center'}

gene_set_rug <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/gene-set-rug.RDS")

gene_set_rug

```

At week 12, differences seen at Week 2 between volume conditions derived from the tissue-offset model were abolished (Figure \@ref(fig:week12-upset-volcano)a). In contrast, volume dependent differences were seen in the library-size normalized model with genes identified as having higher expression in the low-volume condition. However, these overlapped with the naive model, indicating that imbalance in muscle tissue between conditions may have contributed to global shifts in favor of the low-volume condition. This indicates that towards the end of the training intervention, the muscle tissue may have reached a new transcriptional equilibrium.

```{r week12-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Overlap in differentially expressed genes between models are shown in \\textit{b}", fig.scap = "General patterns of differentially expressed genes at Week 12", fig.height=5, fig.width=4.72, fig.align='center'}

week12_grid<- readRDS("./data/derivedData/study1-transcriptome/week12_volcano_upset.RDS")

week12_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

From a general perspective, this analysis underlines the importance of a well-considered choice of normalization strategy in studies of gene expression in non-homeostatic conditions
[@RN2182;@RN2878;@RN2359].<!-- Normalization per cell radically changes the view of increased or decreased global gene expression --><!-- Evans discusses mRNA per cell or per transcriptome  --><!-- Loven 2012 in the case of gene amplification, gene expression changes might be over/underestimated between conditions -->
Specifically, in gene expression studies of training-induced muscle hypertrophy, an ill-advised normalization strategy will mask or overestimate differences between experimental conditions as different amount of tissue is being used in sample preparation. This, as a consequence of a normalized RNA input between different conditions. Differences in the amount of tissue used in sample preparation between volume-conditions in Study I effectively led to underestimating genes identified as differentially expressed at Week 2 and led to nonsensible identification of differentially expressed genes at Week 12.
The choice of normalization strategy ultimately determines the definition of differential expression. By using the tissue offset model, we were interested in gene counts per tissue, as opposed to using the library size as the denominator in analyses, leading to gene counts expressed per transcriptome. Although both strategies may have merit in certain situations, selected genes may be better studied as counts per amount of tissue. It could be argued that one such group of genes are genes related to extracellular matrix remodeling.
First, extracellular matrix-related genes have been shown to display close association between mRNAs abundances and their respective proteins, including collagen-organization proteins [@RN2448].<!-- Makhnovskii cycling exercise, mRNA and protein scales for ECM -->
This indicates that remodeling on the protein level is primarily determined by transcriptional regulation.
Second, selected extracellular-matrix related genes are primarily expressed in other cells than muscle fibers (e.g. fibroblasts)
[@RN2883;<!-- Zou muscle fibroblast but not myogeneic cells express Col6a1  -->
@RN2421].<!-- Kjaer genral review on ECM with collagen synthesis in fibroblast highlighted -->
Thus, assuming that an increase in total RNA due to increased ribosomal biogenesis would scale to other cell types in muscle tissue might introduce bias to such gene expression data.
Consequently, studies of transcriptional regulation of extracellular matrix remodeling concerning muscle hypertrophy should account for the amount of tissue used in analyses.

Extracellular matrix remodeling has not been found to be affected by different exercise variables. Instead, when comparing low- and high-load, Holm *et al.* found similar responses measured as collagen synthesis [@RN2454].<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load -->
Similarly, a comparison of concenctric and eccentric resistance training did not reveal differential effects on muscle collagen synthesis
[@RN2457].<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->
Although it should be noted that the measures of collagen synthesis and measures of gene expression related to extracellular matrix target different facets of extracellular matrix remodeling, volume-dependent regulation of these target genes implicates this remodeling as paralleled with muscle growth.

```{r, echo = FALSE, warning=FALSE, message=FALSE}

acute_time_ngenes <- readRDS("./data/derivedData/study1-transcriptome/n_genes_regulated_time_acute.RDS")


w2post_unchanged<- sprintf("%.1f", acute_time_ngenes[1,4] * 100)

w2post_up<- sprintf("%.1f", acute_time_ngenes[2,4] * 100)

w2post_down<- sprintf("%.1f", acute_time_ngenes[3,4] * 100)




```

### Transcriptome responses to acute exercise

In Week 2, in response to acute exercise, `r w2post_up` and `r w2post_down`\% of identified genes were categorized as up and down-regulated 1 hour after exercise.
These changes were measured as per transcriptome as no change in the total RNA per tissue weight was expected
[@RN2142].<!-- Figueriedo No change in total rna increase in short time frames -->
Instead, fluid shifts [@RN2270]<!-- Ploutz-Snyder Fluid shifts 1995 --> 
could be expected to affect analyses in the short time frame through a biased estimation of wet muscle weight.
The differentially regulated genes were generally associated with stress responses, including inflammation and immune responses and extracellular matrix-related processes. In contrast to the observed changes in response to prolonged resistance training, acute exercise led to down-regulation of extracellular matrix-related genes, similar to what has been found by others [@RN774]<!-- Raue, upregulation of ECM after long term exercise -->.

Regarding volume-dependent regulation in the acute phase, only a single gene was found to be differentially expressed between volume-conditions. This gene, *RFT1*, was found to be down-regulated and more so in response to moderate-volume exercise. The gene is associated with lipid transport, carbohydrate transport, and endoplasmic reticulum membrane and has previously been found to decrease in muscle after exercise [@RN2360]<!-- Pillon -->.
Further analyses of volume-dependent differences not relying solely on arbitrarily defined differential expression confirmed that no robust differences were evident in transcriptome characteristics in this short time-span.
The timing of the post-exercise biopsy likely missed more relevant volume-dependent regulation in the acute phase.

## Determinants of moderate- over low-volume training benefit {#determinants}

To explore determining factors for the additional benefit of moderate- over low-volume training, individual differences in muscle-hypertrophy (CSA) and average muscle strength between the moderate- and low-volume conditions were calculated. In primary analyses, difference scores were dichotomized to *benefit of moderate-volume training* or *no benefit of moderate-volume training* based on the smallest worthwhile change (SWC) in the direction of moderate volume. Participants were thus identified as having *benefit of moderate-volume training* if the difference in raw change scores were greater than the between-participants baseline SD $\times$ 0.2 in favor of moderate-volume training.^[Sex-differences were accounted for by mean-centering variables prior to SD estimation. A weighted SWC was calculated for the strength variable to avoid underestimating SD.]
The choice of using a dichotomized response variable (benefit vs. no benefit) was made to maintain an individual perspective, i.e., regardless of the magnitude of the individual difference, what determines the benefit of moderate- over low-volume training?

```{r csa-str-sets-corr, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Relationship between individual responses to moderate- and low-volume training. Dashed lines are identity lines ($y = x$) and the distance from identity lines to solid lines represent the smallest worthwhile change (SWC) in each variable.", fig.scap = "Relationship between individual responses to moderate- and low-volume training", fig.height=2.4, fig.width=4.72, fig.align='center'}

# Read correlations
corr <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor_df.RDS")
csa_cor <- paste0("*r* = ", sprintf("%.3f", corr[2, 2]), 
                  ", [", 
                  sprintf("%.3f", corr[2, 3]),
                  ", ",
                  sprintf("%.3f", corr[2, 4]),
                  "]")
str_cor <- paste0("*r* = ", sprintf("%.3f", corr[1, 2]), 
                  ", [", 
                  sprintf("%.3f", corr[1, 3]),
                  ", ",
                  sprintf("%.3f", corr[1, 4]),
                  "]")                



csa_str_cor <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor.RDS")

csa_str_cor

```

For each outcome variable, grouping of participants into belonging to the benefit or no benefit group, respectively, is visualized in Figure \@ref(fig:csa-str-sets-corr). Responses to either training condition, allocated in a contralateral fashion were highly correlated for both muscle CSA (`r csa_cor`) and average strength (`r str_cor`). Participants identified as having benefit of moderate-volume training were identified across the whole range of responses (Figure \@ref(fig:csa-str-sets-corr)).

To develop parsimonious models that could explain additional benefits of moderate-volume training for each outcome (muscle CSA and average strength), a set of potential determinants was selected before any association analysis. These included blood variables, baseline strength, muscle mass, volume-dependent molecular responses to training, and baseline fiber-type composition. Additionally, pre-study training habits and training characteristics during the study and dietary data were used in an initial screening process of potential variables. However, as neither pre-study training habits (strength or other types of physical training), within-study training characteristics (number of sessions completed and supervised), nor dietary data displayed any differences between benefit-classifications, they were excluded from further consideration. So was also a set of continuous variables that did not meet the criteria for inclusion, based on preliminary univariate analysis [@RN1998]. 
Continuous variables used in the initial screening process are shown in Figure \@ref(fig:univariate-benefit-analysis). Variables were kept for further consideration when the mean difference 80\% CI (thick error bars) did not contain the null-hypothesis of no difference between benefit-classification groups. Based on the variables selected during this first step, 32 participants were included in the subsequent analysis as two participants had missing values for one or more of the selected predictors.

```{r univariate-benefit-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Differences between groups classified as either having benefit or no benefit of moderate-volume training. For the sake of comparison, all variables have been scaled by their SD. Thick error bars indicate a 80\\% CI and thin error bars indicate the 95\\% CI. Transparent points and error bars indicate that the variables were removed from further modeling. Sex was included as a covariate in all comparisons.", fig.scap = "Univariate analysis of potential determinants of benefit to moderat- over low-volume training", fig.height=5, fig.width=4, fig.align='center'}

univariate_regression <- readRDS("./data/derivedData/study-1-benefit-analysis/univariate_regression_results_fig.Rds")

univariate_regression

```

The choice of a dichotomized response variable (benefit vs. no benefit) entailed the use of logistic regression. An assumption of this type of regression modeling is the linearity of the logit
[@RN1998].
As all variables did not meet this assumption when fitted in logistic regression, they were categorized into biologically meaningful categories (Vitamin D insufficient/sufficient), dichotomized based on measurement detection limits (testosterone in females), or sex-specific median values (lean body mass and testosterone in males).
A stepwise approach was used to remove variables that did not display association with the response variables. The process of stepwise removal of predictors are visualized in Figure \@ref(fig:model-reduction-plot).
Additional benefit of moderate- over low-volume training for changes in muscle cross sectional-area was predicted by individual differences between volume conditions in Total RNA at Week 2 and baseline lean body mass dichotomized to above or below the sex-specific median.
Only individual differences between volume conditions in Total RNA at Week 2 remained in the model after variable selection for muscle strength.

```{r model-reduction-plot, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Sequential removal of predictors of moderate-volume benefit. Point estimates for each predictor are displayed with 95\\% confidence intervals.", fig.scap = "Step-wise variable selection of determinants of moderate- over low-volume training benefit.", fig.height=3.72, fig.width=4.72, fig.align='center'}

model_reduction_plot <- readRDS("./data/derivedData/study-1-determinants/model_reduction_plot.RDS")                    
model_reduction_plot

```

The rationale for using dichotomized outcomes was, as mentioned above, to specifically explore determinants of *individual* benefit of additional training volume, regardless of the magnitude of the difference between conditions.
This choice of analytic approach arose from discussions regarding clinical decision-making, where each individual's potential benefit should guide the choice of training methods.
Although linked to the specific study question, a dichotomized outcome variable could potentially mask other important associations in the data due to loss of information associated with the transformation
[@RN2881].<!-- Harrell 2015 Regression modeling strategies, critique of stepwise procedures-->
A primary hypothesis in the study was that the baseline muscle phenotype, specifically a "glycolytic" vs. a "aerobic" phenotype (measured as high vs. low type IIX content) would not show relative benefit of moderate-volume training.^[[As outlined in the Study's pre-registration](https://clinicaltrials.gov/ct2/show/NCT02179307?term=lillehammer&draw=2&rank=9)]
If such a marker would hold any prognostic value, it could potentially guide decisions regarding the training process.
In analysis, the phenotype marker measured as Type IIX counts was included together with other potential predictors of individual benefit of higher training volume.
Both pre-training characteristics that could aid in clinical decision making and markers arising from the training itself were included in the analysis.
In order to select important determinants from this set of potential determinants, a stepwise procedure for variable selection, known as purposeful selection was used
[@RN1998].<!-- Hosmer -->
Generally, stepwise variable selection has been criticized for a number of reasons, including arbitrarily searching for a single best model, over-estimation of regression parameters, and sensitivity to small changes in the investigated data set
[@RN2882;@RN2881].<!-- Whittingham 2006 information theory shows multiple models that works equally well compete  --><!-- Harrell 2015 Regression modeling strategies, critique of stepwise procedures-->

The question was slightly rephrased in a secondary analysis to address the limitations mentioned above, aiming to explore the association between potential determinants and individual differences between moderate- and low-volume training.
As shown in Figure \@ref(fig:continuous-determinants)a, only Total RNA content at Week 2 in the moderate-volume leg, expressed as a percentage of the low-volume leg RNA content, positively associated with differences between volume conditions in muscle hypertrophy (CSA) and strength gains. These relationships are plotted in Figure \@ref(fig:continuous-determinants)b.

Together with previous studies
[@RN1755;@RN1644;@RN2055],<!-- Stec higher sitgnificant increase in total RNA as wells as rRNA in high responders --><!--Figueiredo correlation between fold change total RNA and delta CSA 8 weeks of training--><!--Mobley larger RNA increases in responders compared to low resp.-->
as well as the direct correlation between RNA content and muscle hypertrophy presented earlier in this text
(Figure \@ref(fig:rrna-csa-fig)),
the identification of volume-dependent total RNA regulation as a predictor of moderate-volume benefit in muscle mass and strength suggests an important role for early-phase ribosomal accumulation as a determining factor for muscle hypertrophy and strength. 
Participants showing greater increases in ribosomal content in response to higher training volume at Week 2 also displayed greater relative benefit of the higher volume.
This likely acts through an increased translational capacity and confirms observations regarding volume-dependence in regulation of total RNA, mature rRNA species (rRNA 18S, 28S and 5.8S) and subsequent muscle hypertrophy (and strength gains).

```{r continuous-determinants, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Determinanats of moderate- over low-volume training benefit", fig.cap = "Predictors of differences in outcomes between moderate- and low-volume training. Values in \\textit{a} are standardized regression coefficients with 95\\% CI from univariate robust regression. Positive estimates indicate a positive association. Variables related to blood parameters, body composition and muscle strength were mean centered per sex. Association between differences in Total RNA between volume-conditions and differences in muscle mass and strength gains between volume-conditions are plotted in \\textit{b}. The linear trends are displayed with 95\\% CI, derived from robust regression analysis.", fig.height=6.5, fig.width=4.72, fig.align='center'}

# 

continous_determinants <- readRDS("./data/derivedData/study-1-determinants/continous_determinants.RDS")


plot_grid(continous_determinants$determinants_overview, 
          plot_grid(continous_determinants$csa_scatter, 
                    continous_determinants$str_scatter, 
                    ncol = 2, 
                    rel_widths = c(0.5, 0.5)), 
          ncol = 1, rel_heights = c(1, 0.5)) +

  draw_plot_label(x = c(0, 0), 
             y = c(1, 0.35), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))



lean_mass_reg_table <- readRDS("./data/derivedData/study-1-determinants/lean_mass_reg_table.RDS")

lean_mass_diff <- paste0(sprintf("%.1f", lean_mass_reg_table[2, 1]), 
                         "%-point [",
                         sprintf("%.1f", lean_mass_reg_table[2, 5]),
                         ", ",
                         sprintf("%.1f", lean_mass_reg_table[2, 6]),
                         "]")

```

In primary analyses, the percentage of lean body-mass was identified as a predictor of additional benefit of moderate-volume training for muscle hypertrophy. 
Albeit associated with a large degree of uncertainty, this indication was in line with current prescription guidelines recommending greater training volume for experienced individuals
[@RN1].
When analyzed between benefit categories, participants who displayed benefit of moderate- over low-volume training had a `r lean_mass_diff` higher lean body mass than participants in the no benefit-group.
Lean body mass did not, however, display a linear relationship with individual differences between volume-conditions.
This, together with the fact that we did not find any robust relationship between pre-training Type IIX fiber proportions and differences in training outcomes between volume conditions, underlines that such markers have low prognostic value, even though they are readily adapted to the training regime as such.
This likely relates to the fact that traits such as fiber type, or body composition are genetically determined
[@RN2884;@RN2798]<!-- Arden lean mass is heritable -->
in addition to influenced by any history of physical activity.

In strength analyses, resting average cortisol levels sampled during the first two weeks showed a negative association with benefit of moderate- over low-volume training. This association was not found in muscle hypertrophy, indicating that factors other than muscle mass drove this relationship. Although speculative, higher levels of cortisol may be a response to the novelty of the training regime
[@RN166].<!-- Kraemer, cortisol drops after short term resistance training -->
Whether this translates in motor learning or other components of the neuromuscular response to different training volumes remains to be determined.


## Predictors of additional benefit of added training volume: Meta-analysis

```{r meta-hypoteses, echo = FALSE, message=FALSE, warning=FALSE}

meta_hypotheses <- readRDS("./data/meta-volume/models/meta_hypotheses.RDS")

upper_body_strength <- paste0(round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,2], 3), 
                    " [", 
                    round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,5], 3), 
                    "]")

upper_body_mass <- paste0(round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,2], 3), 
                    " [", 
                    round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,5], 3), 
                    "]")

unspecific_strength <- paste0("95% CrI: [", 
                    round(meta_hypotheses$strength_unspecific$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$strength_unspecific$hypothesis[1,5], 3), "]")

```


A meta-analysis was conducted to further examine potential factors interacting with benefit of added training volume (previously reported without interactions in Figure \@ref(fig:comb-fig-s1)). Study parameters age, training status, sex, body portion, measurement technique and length of study were sequentially assessed for their interaction with training volume. The weekly number of sets did not interact with age, training status, or sex for either muscle mass or strength (interaction estimates are shown in Table \@ref(tab:meta-interaction-table)), indicating that the literature currently gives little support for such demographic variables to guide prescription of training volume.  
For measures of muscle mass, in contrast to what has been previously reported
[@RN1767],<!-- Schoenfeld meta -->
the present meta-analysis did not indicate any robust differences between unspecific (e.g. DXA) or specific (e.g. MRI or muscle thickness measured with ultrasound) muscle hypertrophy measures, indicated by an interaction effect containing the null-hypothesis (Table \@ref(tab:meta-interaction-table)).
However, specific assessment of muscle strength, e.g., measures of one repetition maximum in exercises used in the training program, consistently gave greater benefit of additional volume (Table \@ref(tab:meta-interaction-table)).
In fact, muscle strength assessed in exercises that were not used in training interventions (unspecific assessment), were shown not to reflect additional training volume (effect size change per one weekly additional set `r unspecific_strength` in unspecific exercises). This may indicate that specific motor learning is a factor influenced by training volume.

```{r meta-interaction-table, message=FALSE, warning=FALSE, echo=FALSE}

library(kableExtra)

## Load data from meta_analysis.R
summary_table_interactions <- readRDS("./data/meta-volume/models/summary_table_interactions.RDS")



summary_table_interactions %>%
  cbind(data.frame(Variable = c("30-50 vs. 18-20 years", 
                                "50- vs. 18-20 years", 
                                "Mixed ages vs. 18-20 years", 
                                "Trained (> 1 year RT) vs. untrained", 
                                "Female vs. male", 
                                "Mixed sex vs. male", 
                                "Upper-body vs. lower-body", 
                                "Whole-body vs. lower-body", 
                                "Unspecific vs. Specific assessment", 
                                "Weeks")), .) %>%
  mutate(m.est = as.character(m.est),
         s.est = as.character(s.est),
         m.ci = paste0("[", 
                       m.lwr, 
                       ", ", 
                       m.upr, 
                       "]"), 
         s.ci = paste0("[", 
                       s.lwr, 
                       ", ", 
                       s.upr, 
                       "]") , 
         s.est = if_else(Variable == "Whole-body vs. lower-body", 
                                 "", s.est), 
         s.ci = if_else(Variable == "Whole-body vs. lower-body", 
                                 "", s.ci)) %>%
  
  
  dplyr::select(Variable, m.est, m.ci, s.est, s.ci) %>%

  kable(col.names = c("Interacting variable", 
                      "Estimate", 
                      "95% CrI", 
                      "Estimate", 
                      "95% CrI"),
        escape = TRUE,
        booktabs = TRUE, 
        caption = "Interaction between study parameter and weekly number of sets from meta-regression models")  %>%
  kable_styling(font_size = 7) %>%
  add_header_above(c(" " = 1, 
                     "Muscle mass" = 2, 
                     "Muscle strength" = 2)) %>%
  pack_rows("Age", 1, 3) %>%
  pack_rows("Training status", 4, 4) %>%
  pack_rows("Sex", 5, 6)  %>%
  pack_rows("Body portion", 7, 8) %>%
  pack_rows("Measurement technique", 9, 9) %>%
  pack_rows("Length of study", 10, 10)
  


```

A consistent finding between muscle mass and strength measures was the negative effect seen in upper- compared to the lower-body measures.
These results suggest that adding a weekly set to upper-body programs will not robustly lead to increased strength measures (upper-body effect size change for every additional weekly set `r upper_body_strength`).
The effect of an additional weekly set in the upper body for measures of muscle mass was still positive (`r  upper_body_mass`) albeit lower when compared to the effect seen in the lower body (Table \@ref(tab:meta-interaction-table)).
Together this indicates that there are differences between muscle groups. Muscles more active in every day activities, i.e. muscles in the lower extremities, will benefit to a greater extent from additional training volume, compared to upper-body muscles.
Studies examining volume-dependence in molecular determinants to resistance-training adaptations between different muscle groups are limited.
However, Hanssen *et al.* found different satellite cell activation between volume-conditions in *m. vastus lateralis* but not in *m. trapezius*, coinciding with volume-dependence in outcomes in lower- but not upper-body muscles
[@RN796;@RN776].
Whether satellite cells *per se* were responsible for this effect or if it represents a general inability of upper-body muscles to differentiate between two relatively strong stimuli remains to be explored.

The duration of interventions interacted differently in measures of muscle mass and strength. More extended training periods seemed to reduce the positive effect of added training volume on muscle mass, albeit likely to a negligible degree. However, strength outcomes were positively influenced by the length of the training period (Table \@ref(tab:meta-interaction-table)). This may be an effect of different time courses in muscle mass and strength measures in response to resistance training.



## Characteristics of early-phase training-induced ribosome biogenesis

Study II aimed to further characterize early-phase total RNA accumulation in muscle tissue in response to resistance training, given its likely importance for subsequent hypertrophy and muscle strength adaptations.
By utilizing a mixed design, Study II assessed the effect of resistance training *per se* by comparing muscular responses in a training group with responses in a non-training negative control group. In the training group, variations in training volume were examined in participants with different volume-protocols allocated to either leg (see figure \@ref(fig:study2-overview) for an overview). Overall, the training protocol resulted in increased muscle mass, regardless of volume condition, measured as *m. vastus lateralis* muscle thickness, compared to the non-training control group (Figure \@ref(fig:study2-muscle-strength-mass)a). This coincided with increased muscle strength, most pronounced when measured as isokinetic torque, but also evident in isometric torque (Figure \@ref(fig:study2-muscle-strength-mass)b).
After the de-training period, muscle thickness and isokinetic strength changes in the training group were still greater than the control group but isometric strength was not robustly greater than the control condition.
The different training protocols (constant vs. variable volume) led to similar changes across the whole study.
These observations indicated the training program's efficacy in inducing changes to the neuro-muscular apparatus, specifically concerning muscle mass.

```{r study2-muscle-strength-mass, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Muscle mass and strength changes in Study II", fig.cap="Training induced changes in muscle mass (a) and strength (b) compared to non-training controls in Study II. Values in the training group are averaged over volume conditions.", fig.height=3, fig.width=4.72, fig.align='center'}

ultra_sound_strength <- readRDS("./data/derivedData/study2-muscle-mass-strength/ultra_sound_strength.RDS")



ultra_sound_strength +
  
  draw_plot_label(x = c(0, 0.5), 
             y = c(1, 1), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))




```


```{r total-rna-stats, echo=FALSE, warning=FALSE, message=FALSE}

# Segmented model analysis in text 
# Read the segmented model 
tc.m2 <- readRDS("./data/study-2/total-rna-analysis/tc.m2.RDS")




time1.hyp <- hypothesis(tc.m2, "time1 = 0")
time2.hyp <- hypothesis(tc.m2, c("time2 + time1 = 0")) # The slope (on log scale) between session 4 and 8
time3.hyp <- hypothesis(tc.m2, c("time2 + time1 +time3 = 0")) # The slope on log scale between session 8 and 12

slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      
slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      

slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,5]) - 1)),
                   "]")

# Estimate changes to end of periods

sessions <- hypothesis(tc.m2, c("time1 * 4 = 0", 
                                     "(time1 * 8) + (time2 * 4)= 0", 
                                     "(time1 * 12) + (time2 * 8) + (time3 * 4)= 0"))



                  

## Effect of detraining 

detrain.hyp <- hypothesis(tc.m2, "detraindetrain = 0")

detrain_rna <- paste0(sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,2]) - 1)),
                           "%, [",
                      sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      
fourth.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,5]) - 1)),
                   "]")


eight.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,5]) - 1)),
                   "]")

twelth.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,5]) - 1)),
                   "]")

```

In the training group, twelve resistance training sessions led to increases in total RNA per-unit-muscle weight. Compared to the control group, this was evident after twelve sessions (post-training, Figure \@ref(fig:study2-rrna-vs-ctrl)a).
Increases were partially reversed after the de-training period (`r detrain_rna`), although still on average elevated compared to baseline, it was no longer robustly different to the control group (de-training in Figure \@ref(fig:study2-rrna-vs-ctrl)a).
In line with the observed change in total RNA, the initial session induced rRNA transcription as 45/47S pre-rRNA increased compared to non-training controls (Session 1, Figure \@ref(fig:study2-rrna-vs-ctrl)b). This increase was still apparent after twelve sessions and then accompanied by greater mature rRNA levels (28S and 18S, Figure \@ref(fig:study2-rrna-vs-ctrl)b). After the period of de-training, pre-rRNA levels returned towards levels seen in the control group, but mature transcripts were still elevated in the training group compared to controls, suggesting a decreased *de novo* rRNA transcription (Post-training + de-training, Figure \@ref(fig:study2-rrna-vs-ctrl)b).

Together with Study I and several others,
<!--@RN2266<  Hortobagyi 2000 NO change in RNA per wet weight after immobilization or 12 weeks retraining low N-->
[@RN2225;@RN2180;@RN1656;@RN1520;@RN1755;@RN1644;@RN1809;@RN1897]<!-- Haun 2019 used a progressive volume troughout a six week study study RNA increased during six weeks (12% week 3; 27% week 6 --><!-- Haddad 2005 Unloading through limb suspension leads to lowered total RNA levels, resistance training increased --><!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old --><!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr--><!-- @RN2358;  Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young--><!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD--><!-- Figuieredo 2015 16 sessions 49% increase young --><!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young--><!-- 11% 36 sessions young -->,
Study II confirms that resistance training leads to RNA accumulation, likely driven by *de novo* transcription of rRNA. After the eight-day de-training period, the apparent normalization of pre-rRNA levels indicate that ribosomal biogenesis is coupled with up-stream stresses, such as resistance training. Such coupling is possibly due to the relative cellular expense associated with ribosomal biogenesis [@RN1920].
To further test the hypothesis that ribosome biogenesis would be reflected by the degree of training-induced stress, training volume was manipulated in fluctuating manner in one leg and held constant in the other leg in participants in the training group.
After the initial four sessions, which were conducted with the same volume (six sets), the variable-volume leg conducted four sessions with reduced volume (three sets), followed by four sessions with increased volume (nine sets). For 45S pre-rRNA this led to small differences between volume conditions at 48 hours after session twelve, indicating that rRNA transcription was affected by the increased volume in the last block.
However, for total RNA and rRNA species, this did not lead to differences between volume conditions as they increased similarly in both conditions (Figure \@ref(fig:study2-rrna-volume-conditions)a and b). Together this indicates that although ribosomal biogenesis to some degree is controlled in an on-demand manner, fluctuations in training volume over a relatively short period ($\sim$ 1-2 weeks) following an initial training period with similar volume does not influence ribosomal accumulation to any meaningful extent.
However, it may also indicate that untrained individuals maximize rRNA transcription and RNA accumulation during initial sessions during a training program.
Indeed, the training-associated increase in total RNA abundance occurred primarily during the first part of the intervention. In sessions one to four, when averaged over volume-conditions, total RNA increased by `r slope0_4` per session, followed by attenuated accumulation from sessions four to eight (`r slope4_8` increase per session) and eight to twelve (`r slope8_12` increase per session). This increase corresponded to relative increases from baseline to 48 hours after session four, eight, and twelve, respectively, of `r fourth.session`, `r eight.session` and `r twelth.session`.

```{r study2-rrna-vs-ctrl, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Total RNA and rRNA changes in response to resistance training in Study II.", fig.cap="Resistance training led to increased total RNA (a) and rRNA (b) abundance per tissue weight measured after Sessions 1 and 12 (Post-training) and after eight days of de-training. Comparisons with the control group are presented in right panels. Values in the training group are averaged over volume conditions. Point estimates are displayed with 95\\% CrI.", fig.height=7, fig.width=4.72, fig.align='center'}


rrna_totrna_vs_control <- readRDS("./data/derivedData/study2-rrna-analysis/rrna_totrna_vs_control.RDS")

rrna_totrna_vs_control +
  
  draw_plot_label(x = c(0.02, 0.02), 
             y = c(0.99, 0.7), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

Although ribosomal biogenesis is a well-known characteristic of resistance training, the time-course of rRNA accumulation after onset of such training has not been properly studied.
Study II indicates that relatively few training sessions are needed to maximize total RNA per unit tissue weight.
Together with data from Study I and other studies that have performed repeated biopsy sampling during a relatively short timeframe [@RN1809], this suggests that young males and females will reach peak training-induced values within four to nine sessions, followed by a plateau.
Since the plateau phase occurred without a concomitant reduction in pre-rRNA levels, it may represent a dilution effect rather than decreased rRNA transcription, whereby ribosomal content is essentially stabilized by concomitant cellular growth and protein accretion
[@RN2142]<!-- Figueiredo review details the concept of rna -->
Possibly, a progressive increase in training volume could abolish such a plateau effect.
Indeed, Haun *et al.*
[@RN2225]
observed progressive increases in total RNA content in muscle throughout six weeks of resistance training in well-trained participants, presumably due to a continuous increase in training volume (from 2-4 to 8-12 sets per exercise). This observation suggests that a progressive increase in volume may increase ribosomal abundance to a higher degree than constant-volume protocols.
However, the lack of a relevant control condition warrants more research.

```{r study2-rrna-volume-conditions, echo=FALSE, warning=FALSE, message=FALSE, fig.cap = "Total RNA (a) and rRNA (b) abundance in response to resistance training in variable- and constant-volume training conditions. Asterisk indicate robust difference between volume conditions.", fig.scap="Total RNA and rRNA abundance in response to training in Study II.", fig.height=6.5, fig.width=4, fig.align='center'}

rna_abundance_exp_group <- readRDS("./data/derivedData/study2-rrna-analysis/rna_abundance_exp_group.RDS")

rna_abundance_exp_group +
  
  draw_plot_label(x = c(0.025, 0.53), 
             y = c(0.73, 0.99), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))
```

```{r ubf-to-rna-stat, echo=FALSE, message=FALSE, warning=FALSE}

# Save model 
comb.m1 <- summary(readRDS("./data/study-2/ubf-tot-rna-model/comb_m1.RDS"))$fixed




ubf_cor_tot_rna <- paste0(sprintf("%.1f", 100 * (exp(comb.m1[2,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,4]) - 1)),
                   "]")


```

```{r ubf-rps6-regression-table}
# Save model 

# Model for rpS6 predicting total RNA
comb.s6.m1 <- readRDS("./data/study-2/ubf-tot-rna-model/comb_m1_s6.RDS")


comb.m1 <- readRDS("./data/study-2/ubf-tot-rna-model/comb_m1.RDS")



comb.m1 <- summary(comb.m1)
comb.s6.m1 <- summary(comb.s6.m1)



reg_results <- 
  cbind(rbind(comb.m1$fixed,
      comb.m1$random$participant, 
      comb.m1$random$`participant:leg`, 
      comb.m1$spec_pars), 
      data.frame(coef = c("Intercept", 
                          "UBF protein levels (SD-units)", 
                          "Session 1-4",
                          "Session 4-8", 
                          "Session 8-12", 
                          "De-training",
                          "Between Participant variation",
                          "Between Participant:Leg variation", 
                          "Residual SD"))) %>%
  remove_rownames() %>%
  dplyr::select(coef, Estimate, Est.Error, lwr = `l-95% CI`, upr = `u-95% CI`)


reg_results_s6 <- 
  cbind(rbind(comb.s6.m1$fixed,
      comb.s6.m1$random$participant, 
      comb.s6.m1$random$`participant:leg`, 
      comb.s6.m1$spec_pars), 
      data.frame(coef = c("Intercept", 
                          "rpS6 protein levels (SD-units)", 
                          "Session 1-4",
                          "Session 4-8", 
                          "Session 8-12", 
                          "De-training",
                          "Between Participant variation",
                          "Between Participant:Leg variation", 
                          "Residual SD"))) %>%
  remove_rownames() %>%
  dplyr::select(coef, Estimate, Est.Error, lwr = `l-95% CI`, upr = `u-95% CI`)



  rbind(reg_results, reg_results_s6) %>%
  
  mutate(Estimate = round(Estimate, 2), 
         Est.Error = round(Est.Error, 2), 
         lwr = round(lwr, 2), 
         upr = round(upr, 2)) %>%
   kable(col.names = c("Coefficient", "Estimate", "SD", "Lower", "Upper"), 
          caption = "Effect of UBF and rpS6 levels, sessions and de-training on RNA-levels.", 
         format = "latex",
         escape = FALSE,
         booktabs = TRUE) %>%
    kable_styling(font_size = 7) %>%
    add_header_above(c(" " = 3, "95\\\\% CrI" = 2), escape = FALSE) %>%
    pack_rows(start_row = 1, 
              end_row = 9, 
              group_label = "UBF") %>%
        pack_rows(start_row = 10, 
              end_row = 18, 
              group_label = "rpS6") %>%
    footnote(general_title = "", 
             footnote_as_chunk = TRUE,
             threeparttable = TRUE,
             general = "The dependent variable is total RNA levels (log). Session 1-4 represents the slope in response to session 1-4 with Session 4-8 and 8-12 representing changes in slopes.")
    
    
  
```

```{r study2-ubf-rps6-rna-fig, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Total RNA levels predicted by UBF and rpS6 protein in Study II.", fig.cap="Upstream binding factor (UBF) (a) and ribosomal protein S6 (rpS6) (b) protein levels predicting Total RNA abundance. Estimates are derived from models presented in Table 5.4 with 95\\% CrI. Points represent underlying observations.", fig.height=2.5, fig.width=4.72, fig.align='center'}


ubf_rps6_fig <- readRDS("./data/derivedData/study2-ubf-rps6-rna/ubf-rps6-rna-fig.RDS")


ubf_rps6_fig

```

The general patterns seen for total RNA and mature rRNA responses to resistance training in the present study coincided with pattern seen for rpS6 and upstream binding factor (UBF) protein levels. This was evident as increases of rpS6 and UBF after twelve training sessions compared to non-training controls, followed by partial reversal of rpS6 but maintained UBF levels after the de-training period
(Figure \@ref(fig:study2-ubf-rps6-fig)a).
Together, this indicates general coordination between training induced synthesis of rpS6 and rRNA transcription and a potential role of increased UBF levels in maintaining an increased rRNA transcription.
The associations between total RNA, rpS6, and UBF protein levels were analyzed to explore such coordination further. When controlling for time, UBF levels positively predicted RNA levels in the training group. For every unit increase in UBF (standard deviations), total RNA increased by `r ubf_cor_tot_rna`. No such relationship was found between RNA levels and rpS6 (Table \@ref(tab:ubf-rps6-regression-table), Figure \@ref(fig:study2-ubf-rps6-rna-fig)).


```{r study2-ubf-rps6-fig, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "UBF and rpS6 protein in response to resistance training in Study II.", fig.cap="Ribosomal protein S6 (rpS6) upstream binding factor (UBF) protein in response to resistance training in compared to non-training controls (a). Values are estimated means with 95\\% CrI. A representative blot is shown in \\textit{b}.", fig.height=4.8, fig.width=4.72, fig.align='center'}

protein_analyses_raw <- readRDS("./data/derivedData/study2-protein-analyses/protein_analyses.RDS")

reps_west <-  ggdraw() + draw_image("./data/study-2/wetlab/western/representative_img/round2_gel4_annotated.png") 




protein_analyses <- plot_grid(plot_grid(protein_analyses_raw$prot_fold_change, 
                                        protein_analyses_raw$prot_interaction, 
                                        ncol = 2, 
                                        rel_widths = c(0.7, 0.5)), 
          reps_west, ncol = 1, rel_heights = c(1, 0.7))



protein_analyses +
  
  draw_plot_label(x = c(0.025, 0.15), 
             y = c(0.99, 0.4), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```


Although resistance training led to a general increase in both rRNA and rpS6, the apparent disconnect in regression analyses suggests that regulation of rpS6 expression and ribosomal RNA transcription displays different temporal characteristics. This may relate to differing half-life among ribosomal components or ribosomal proteins having extra-ribosomal functions which in turn regulates their abundance
[@RN2588].<!-- Warner 2009 General description of   -->
The relationship between UBF levels and total RNA indicate that UBF abundance drives rRNA transcription and thus regulates ribosome biogenesis.
Mechanistically, UBF is needed for rRNA transcription, acting by recruiting secondary factors to the rDNA promoter and thereby enabling RNA Pol-I controlled transcription
[@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->

In line with data from the present study, mechanical loading has previously been shown to lead to increased levels of UBF protein, an effect believed to occur in a mTORC1-independent manner 
[@RN1746;@RN1644;@RN1072],<!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase --><!-- Goodman 2011 UBF increase due to synergist ablation is rapamycin insensitive https://doi.org/10.1113/jphysiol.2011.218255 --> 
but instead mediated by increased c-Myc-induced UBF mRNA transcription [@RN2565].<!-- Pootinga 2004 MAD1 and -->
Further corroborating data from the present study, the availability of UBF *per se* has been shown to regulate rRNA transcription in cell models by stimulating rDNA transcription
[@RN2566;@RN2556].<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes --><!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->
Together, this suggests that training-induced increases in UBF may contribute to rRNA accumulation *in vivo*.
It should however be noted that UBF availability and activation is controlled partly through mTORC1, as its inhibition blocks rRNA transcription
[@RN1632;@RN2564].<!-- Nader rapamycin block UBF availability  --><!-- Hannan 2003 -->
<!-- Hannan 2003 10.1128/MCB.23.23.8862-8877.2003 mTOR is believed to play a role in phosphorylating UBF leading to recruitment of SL-1. The effect of blockage of mTOR through rapamycin is not rescued by hypophosphorylated UBF -->
Indeed, after acute exercise, UBF activates through phosphorylation
[@RN1746;@RN1644],<!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase --><!-- Figueriedo ubf increase total and active form (ser388) -->
indicating that acute exercise also modulates its activity.


### Resistance-training induced increase in total RNA predicts muscle growth

As indicated in Study I, and previous studies
[@RN1644;@RN1897;@RN1755;@RN2055],<!-- Figueriedo  --><!-- Mobely Three groups, no clear group * time in RNA incrinteraction but 32 vs 8 % increase in high vs low responders, although correlation RNA incr with MT incr -->
<!-- Reidy correlation between total RNA and muscle hypertrophy-->
ribosome biogenesis is a determining process for resistance training-induced muscle growth.
To further explore this relationship in the data set of Study II, changes in total RNA abundance over the training period were estimated from each participant's leg (Figure \@ref(fig:study2-rna-muscle-thickness)c) and used for modeling muscle growth measured as increases in *m. vastus lateralis* thickness.
There was a positive association between the magnitude of total RNA increase per session and muscle growth  (Figure \@ref(fig:study2-rna-muscle-thickness)a).
This association was apparent after controlling for the estimated RNA content in muscle tissue, expressed as the RNA level at Session 6 (estimated as the intercept-term in individual models). Conversely, when controlling for the increase in total RNA, total RNA abundance at Session 6 was negatively associated with muscle growth  (Figure \@ref(fig:study2-rna-muscle-thickness)b).
The association between total RNA increases and muscle growth was robust as no single measurement within-participant nor any single participant changed the conclusion from the analysis (Figure \@ref(fig:study2-rna-muscle-thickness)d). Removing single observations or participants from the analysis did, however, lead to overlap between the 95\% CrI and the null-hypothesis (Figure \@ref(fig:study2-rna-muscle-thickness)d).

This analysis supports the view that ribosomal biogenesis is a determining factor behind resistance training-induced muscle growth
[@RN1644;@RN1897;@RN2055;@RN1755].<!-- Figueriedo  --><!-- Mobely Three groups, no clear group * time in RNA incrinteraction but 32 vs 8 % increase in high vs low responders, although correlation RNA incr with MT incr -->
<!-- Reidy correlation between total RNA and muscle hypertrophy--><!-- Stec 2016  -->
However, the analysis suggests that the increase *per se* and not a fixed amount of total RNA, determines muscle growth.
A higher level of total RNA mid-training was instead associated with lowered muscle growth when controlling for the increase in RNA.


```{r study2-rna-muscle-thickness, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Relationship between total RNA and muscle hypertrophy in Study II", fig.cap="Muscle thickness increase in response to twelve sessions predicted by total RNA increase per session and average total RNA abundance at Session 6 (a). Individual plots of estimates total RNA increases per session are shown in \\textit{b} together with results from leave-one-out analyses (c). Leave-one-out analyses shows the effect of removing a single participant (black point and error-bars) and individual values from the total RNA per session estimates where light-blue points represents bounds of the 95\\% CI and blue points represent mean estimates.", fig.height=7, fig.width=4.72, fig.align='center'}

tot_rna_muscle_thickness <- readRDS("./data/derivedData/study2-rna-muscle-mass-fig/tot_rna_muscle_thickness.RDS")

tot_rna_muscle_thickness

```

## Study limitations

In Study I, a main objective was to determine differences in training outcomes and molecular markers between volume conditions. Therefore, the study is limited with respect to what conclusions can be drawn from time effects alone, as no non-training control group was included in the study. Conclusions regarding changes over time in, e.g., total RNA, gene expression, and strictly speaking, muscle growth and strength are limited in their strength of evidence. 
In Study II, the use of a control group strengthens this aspect of the analyses. However, the number of biopsies was limited in the control group. The control group in Study II served the purpose of system validation but did not serve as a true control condition for training *per se*.

In Study I, step-wise regression was used to select important variables relating to moderate-volume benefit. In addition to what has been discussed in previous sections, this analysis was limited by the mere selection of variables. Irrespective of the technique used for statistical analyses, the conclusion regarding the importance of total RNA is drawn when competing with the other available variables. Thus, it cannot be excluded that other unmeasured variables would prove even more important.
From a cellular perspective, a plethora of potential determinants exists, including, but not limited to,
satellite cells [@RN2080],<!-- Blaauw -->
Yes-associated protein [@RN1931] and <!-- Goodman  -->
Myostatin- and transforming growth factor-$\beta$ [@RN1619]-related signaling. The limited availability of techniques in our lab for investigating these and other potential determinants limited analyses.
Additionally, the study did not include detailed documentation of out-of-study lifestyle habits such as habitual physical activity and diet, both of which could have influenced the between-participant analysis.
Lastly, by weighing variables against each other, one implicitly assumes that variables are measured with similar technical variability, which is likely a very optimistic assumption. 

For ethical and practical reasons, the number of biopsy sampling events were minimized. In both studies, the choice of biopsy sampling time-points dictates the resolution of analyses. In Study I, biopsies sampled one hour after the fifth session in Study I likely did not provide reliable information regarding acute changes in gene expression in response to low- and moderate-volume exercise. Similarly, in Study II, biopsies obtained at rest limited the investigation in terms of acute signaling events such as related to mTORC1 or UBF activation. 

The use of total RNA as a proxy marker of ribosomal abundance is widely accepted
[@RN2142;@RN1757;@RN1654;@RN2150].
Indeed, the present data reveal agreement between total RNA and targeted analysis of ribosomal RNA using qPCR, at least during the initial phase of the interventions. However, the apparent disconnect between mRNA and total-RNA pools and the tendency of discrepancies between volume conditions in total RNA and rRNA abundance after twelve weeks of training in Study I warrants further examination, as it suggests that sub-populations of RNA may vary independently from each other. Although the assumption that total RNA represents ribosomal abundance likely holds in the initial phase of the programs examined, it should have been confirmed to strengthen conclusions from both studies.

## Future directions

The importance of early-phase RNA accumulation for training-induced muscle hypertrophy and strength gains was highlighted in both studies presented in this thesis. Future studies that aim to investigate *in vivo* ribosomal biogenesis in response to resistance training should take advantage of the fact that a limited number of training sessions are needed to reach maximal accumulation of total RNA (and thus presumably ribosome abundance). Further use of within-participant designs is also advisable, as it allows investigation of the etiology of inter-individual differences in ribosomal biogenesis. Constant-volume protocols may, in this respect, represent a fruitful model to investigate such differences. Given that training volume likely is the most important exercise variable, governing resistance training dose-response, within-participant models with differentiated volume allocated to either limb could be used to study if, e.g., age-related anabolic resistance is related to impaired sensitivity to the training stimuli.

Although constant volume protocol may prove useful in controlled mechanistic studies, the use of progressive-volume protocols should be evaluated for their use in clinical settings. Ever since the days of DeLorme (see Background), progressive resistance training has been progressive with regard to absolute intensity. However, the use of a fixed training volume could be less effective, as too much training volume during the initial phase of the training period may interfere with adaptations. Additionally, since there is little support for any diagnostic criteria for selecting an individualized training volume, evident from both Study I and the meta-analysis, progression of volume in short-term (1-3 months) training interventions could circumvent this issue.









