# Results and Discussion

```{r settings, include=FALSE}

library(nlme); library(kableExtra)


```


## Within-session training volume affects training-induced changes in muscle mass and strength and molecular determinants of muscle hypertrophy {#muscle-mass-growth}

```{r comparing-avg-strength-csa, echo=FALSE, warning=FALSE, message=FALSE}


source("./R/study-1/mr-and-dxa-data.R")


participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


### Overall % increase in CSA (not corrected for baseline values)
over.all.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(subject) %>%
  summarise(mean.csa = mean(change, na.rm = TRUE)) %>%
  ungroup() %>%
  summarise(m.csa = mean(mean.csa),
            s.csa = sd(mean.csa))

sets.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(sets) %>%
  summarise(m.csa = mean(change),
            s.csa = sd(change))



### Baseline corrected % increase
csa.corrected.data <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>% # Calculates % change
  group_by(sex) %>%
  mutate(pre = pre-mean(pre)) %>% # Mean centered baseline 
  # (no need for sex-diff in model)
  group_by(subject, sex) %>% # Keeps sex for test in model
  summarise(mean.csa = mean(change, na.rm = TRUE),
            mean.baseline = mean(pre)) # Average over both conditions 



## Anocova with baseline values as co-variate
csa.corr.m1 <- lm(mean.csa ~  mean.baseline,
                  data = csa.corrected.data)

csa.m1 <- lm(mean.csa ~  1,
                  data = csa.corrected.data)



csa.corr.confint  <- round(confint(csa.corr.m1)[1,],1)
csa.corr.estimate <- round(summary(csa.corr.m1)$coef[1,1],1)




csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  group_by(timepoint, sets, sex) %>%
  summarise(m=sprintf("%.1f", round(mean(CSA.avg),1)),
            s=sprintf("%.1f", round(sd(CSA.avg),1)))


csa.r <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject, sex, sets, csa.change) %>%
  spread(sets, csa.change)

csa.r.model <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject,leg, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject,leg, sex, sets, pre, csa.change) %>%
  group_by(sex) %>%
  mutate(pre.m = pre-mean(pre, na.rm = TRUE), 
         sets = factor(sets, levels = c("single", "multiple"))) 

#### Change to ancova ###


csa.m0 <-nlme::lme(csa.change ~ sets +  pre.m,
                random = list(subject = ~1),
                na.action = na.omit, 
                data = csa.r.model, method = "ML", 
                control=list(msMaxIter=120,
                             opt = "nloptwrap", msVerbose = FALSE))




csa.estimate <- sprintf("%.2f",round(coef(summary(csa.m0))[2,1], 2))
csa.confint <- intervals(csa.m0)$fixed[2, c(1,3)]

### t-test Does not change the conclusion 
# csa.ttest <- with(csa.r, t.test(multiple,single, paired=T))
# csa.estimate <- sprintf("%.2f",round(csa.ttest$estimate, 2))
# csa.confint <- csa.ttest$conf.int


csa.corr <- cor.test(csa.r$multiple, csa.r$single)


female.t <- with(csa.r[csa.r$sex=="female",], t.test(multiple,single, paired=T))
female.estimate <- sprintf("%.2f",round(female.t$estimate, 2))
female.confint <- female.t$conf.int


male.t <- with(csa.r[csa.r$sex=="male",], t.test(multiple,single, paired=T))
male.estimate <- sprintf("%.2f",round(male.t$estimate, 2))
male.confint <- male.t$conf.int

#### Strength data for overall stats



rm_str <- read_excel("./data/study-1/strength/strengthTests.xlsx") %>%
        filter(!(exercise %in% c("benchpress", "legcurl"))) %>%
        inner_join(participants) %>%
        filter(include == "incl") %>%

  dplyr::select(subject, sex, sets, exercise, timepoint, load)%>%
  mutate(load = as.numeric(load),
         sets = factor(sets, levels = c("single", "multiple"))) %>%
  spread(timepoint, load) %>%
  mutate(baseline = pmax(pre, session1)) %>%
  dplyr::select(subject, sex, sets, exercise, baseline, week2, week5, week9, post) %>%
  gather(week, load, baseline:post) %>%
  mutate(week = factor(week, levels = c("baseline", "week2", "week5", "week9", "week12", "post"))) %>%
  group_by(exercise) %>%
  mutate(str = load / max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) 

# load cybex data 
load("./data/study-1/strength/cybex.rda")


cybex <- data.frame(cybex) %>%
        mutate(leg = if_else(leg == "Left", "L", "R")) %>%
        inner_join(participants) %>%
        filter(include == "incl", 
               timepoint %in% c("pre","session1", "post")) %>%

        filter(!(subject=="FP6" & timepoint == "post" & speed == 240))  # This removes one record, exclusion due to error in measurement, see protocol FP6


# Clean cybex data set and combine with 1RM 
total_strength <- cybex %>%
  ungroup() %>%
  dplyr::select(subject, timepoint, sets,speed, sex, torque.max) %>%
  spread(timepoint, torque.max) %>%
  rowwise()%>%
  mutate(baseline = max(pre, session1, na.rm=T)) %>%
  ungroup() %>%
  dplyr::select(subject, sex, sets, exercise = speed, baseline,  post) %>%
  gather(week, load, baseline:post) %>%
  group_by(exercise) %>%
  mutate(str = load/max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) %>%
  rbind(rm_str) %>% # This includes all strength tests
  mutate(ex_fam = if_else(exercise %in% c("legext", "legpress"), 
                          "rm", if_else(exercise == 0, "isom", "isok"))) %>%
  group_by(subject, sex, sets, week, ex_fam) %>%
  summarise(str = mean(str, na.rm = TRUE)) %>%
  group_by(subject, sex, sets, week) %>%
  summarise(str = mean(str, na.rm = TRUE)) 


strength.stats <- total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  group_by(sex) %>%
  mutate(BASELINE = baseline - mean(baseline)) %>%
    group_by(sex, subject) %>%
    summarise(increase = mean(increase, na.rm = TRUE),
              BASELINE = mean(BASELINE, na.rm = TRUE)) 




## ANCOVA for the strength 1RM
strength.m1 <- lm(increase ~ sex + BASELINE,
                  data = strength.stats)

strength.corr.confint  <- round(confint(strength.m1)[1,],0)
strength.corr.estimate <- round(summary(strength.m1)$coef[1,1],0)



### Correlation in strength gains ###

str.corr.data <-  total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject, sex, sets, increase) %>%
  spread(sets, increase) 

str.corr <- cor.test(str.corr.data$single,
                          str.corr.data$multiple)

#### Correlation strength and csa increases

# Average increases

csa.str.corr.data <- total_strength %>%
  spread(week, str) %>%
  mutate(str.rel.change = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject,  sets, sex, sets,baseline, str.rel.change) %>%

  inner_join(mr.results %>%
                mutate(condition=leg) %>%
                inner_join(participants) %>%
                filter(include=="incl") %>%
                dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
                spread(timepoint, CSA.avg) %>%
                mutate(csa.rel.change = 100 * ((post/pre)-1)) %>%
                dplyr::select(subject, sets, csa.rel.change)) %>%
        ungroup() 





# The code below could possibly be used to calculate the relative contribution of 
# csa increase to muscle strength increases


# m <- lme4::lmer(str.rel.change ~ csa.rel.change + sets + (1|subject), 
#                 data = csa.str.corr.data)
# 
# 
# 
# mx.null <- lme4::lmer(str.rel.change ~ 1 + (1|subject), data = csa.str.corr.data)
# mx.csa <- lme4::lmer(str.rel.change ~ csa.rel.change  + (1|subject), data = csa.str.corr.data)
# mx.sets <- lme4::lmer(str.rel.change ~ sets + (1|subject), data = csa.str.corr.data)
# 
# 
# 
# boot.fun <- function(x) {
#         data.frame(nlme::VarCorr(x))[2, 4]
# }
# 
# 
# null.boot <- lme4::bootMer(mx.null, boot.fun, nsim = 500, parallel = "snow", cl = 12)
# csa.boot <- lme4::bootMer(mx.csa, boot.fun, nsim = 500)
# sets.boot <- lme4::bootMer(mx.sets, boot.fun, nsim = 500)
# 
# 
# 
# qt.null <- quantile(null.boot$t, c(0.025, 0.975))
# qt.csa <- quantile(csa.boot$t, c(0.025, 0.975))
# qt.sets <- quantile(sets.boot$t, c(0.025, 0.975))
# 
# 
# results <- data.frame(variable = c("sets", "csa"),
#                            m = c(1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4],
#                                  1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4]),
#                            lwr = c(1- qt.sets[1] / qt.null[1],
#                                    1- qt.csa[1] / qt.null[1]),
#                            upr = c(1- qt.sets[2] / qt.null[2],
#                                    1- qt.csa[2] / qt.null[2]))
# 
# 
# 
# 
# 
# csa <- 1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# sets <- 1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# exp <- 1 - data.frame(nlme::VarCorr(mx.exp))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# 

# Average csa and strength gains

avg.str.csa <- csa.str.corr.data %>%
        group_by(subject) %>%
        summarise(str = mean(str.rel.change), 
                  csa = mean(csa.rel.change))


csa.str_corr <- cor.test(avg.str.csa$csa, avg.str.csa$str)

## Comparisons with other published studies on per session gains in csa and strength

### Per session ##
per.session <- read_excel("./data/study-1/trainingdata.xlsx", sheet = 1) %>%
  filter(!is.na(date)) %>%
  group_by(subject) %>%
  summarise(sessions = n()) %>%
        
inner_join(csa.str.corr.data) %>%

  mutate(str.change.day = str.rel.change/((sessions/31) * 12*7),
         csa.change.day = csa.rel.change/((sessions/31) * 12*7), 
         str.session = str.rel.change/sessions,
          csa.session = csa.rel.change/sessions ) %>%
   dplyr::select(-baseline) %>%
  ungroup() %>%
        pivot_longer(names_to = "variable", values_to = "gain", cols = str.rel.change:csa.session) 
       

sum.stats <- cbind(per.session %>%
        group_by(sets, variable, sex) %>%
        summarise(mean = mean(gain), 
                  sd = sd(gain)) %>%
        arrange(variable) %>%
        data.frame(), 
        data.frame(reference.mean = c(0.11, 0.11, 0.11, 0.11,
                              NA, NA, NA, NA, 
                              0.0825, 0.1375,0.0825, 0.1375, 
                              NA, NA, NA, NA, 
                               NA, NA, NA, NA,
                              0.6675, 0.475, 0.6675, 0.475), 
           reference.range = c("[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "", "", "", "",
                               "(0.22)b", "(0.14)b",  "(0.22)b", "(0.14)b", 
                               "", "", "", "",
                              "", "", "", "",
                               "(0.35)b", "(0.22)b","(0.35)b", "(0.22)b" ))) %>%
        mutate(m.sd = if_else(mean > 6, paste0(sprintf("%.1f",mean), " (", sprintf("%.1f",sd), ")"), 
                              paste0(sprintf("%.2f", mean), " (", sprintf("%.2f",sd), ")")), 
               reference = if_else(is.na(reference.mean), "",
                                   paste0(sprintf("%.2f",reference.mean), " ", reference.range))) %>%
        filter(!(variable %in% c("str.change.day") )) %>%
                       dplyr::select(variable, sex, sets, m.sd, reference) %>%
        
        mutate(variable = factor(variable, levels = c("csa.rel.change", 
                                                      "csa.change.day", 
                                                      "csa.session", 
                                                      "str.rel.change", 
                                                      "str.session"), 
                                 labels = c("CSA %-change", 
                                            "CSA %-change per day", 
                                            "CSA %-change per session", 
                                            "Average strength %-change", 
                                            "Average strength %-change per session")), 
               sex = factor(sex, levels = c("female", 
                                            "male"), 
                            labels = c("Female", "Male")), 
               sets = factor(sets, levels = c("single", "multiple"), 
                             labels = c("LOW", "MOD"))) 

## Ahtiaianen on average  20 weeks with 2 sessions per week = 40 sessions

## Muscle hypertrophy
## In the younger cohorts, male average 5.5% (5.6) = 5.5/40 ~ 0.1375 per session
# in females 3.3 (8.6)% = 0.825
# 8.6/3.3 * 3.3/40
# 5.6/5.5 * 5.5/40
# Muscle strength in younger cohorts
#  male    19% (8.6)/40 sessions ~  0.5 % per session
#  female  26.7% (13.9)/40 ~ 
# 8.6/19 * (19/40)
# 13.9/26.7 * 26.7/40

# Wernbom increase per day in quadriceps CSA per day was 0.06%







```

In Study I, the average increases in muscle strength and mass in each volume condition corresponded to what could be expected based on previous research in young, healthy participants (Table \@ref(tab:csa-str-tab)) 
[@RN2007;<!-- Wernbom 2007 -->
@RN1741],<!-- Ahtiainen  -->
indicating the general efficacy of the training program.


(ref:wernbom-citation) @RN2007

(ref:ahtiainen-citation) @RN1741

```{r csa-str-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

sum.stats %>%
        arrange(variable, sex, sets) %>%
  dplyr::select(-variable) %>%
        kable(format = "latex", booktabs = TRUE, linesep = "",
              col.names = c("Sex", "Condition", "Mean (SD)", "Reference"), 
              caption = "Training induced changes in muscle CSA and average strength in Study I")  %>%
        collapse_rows(c(1, 4))  %>%
        kable_styling(font_size = 7) %>%
          
          pack_rows("CSA %-change", 1, 4) %>%
        pack_rows("CSA %-change per day", 5, 8) %>%
          pack_rows("CSA %-change per session", 9, 12) %>%
          pack_rows("Average strength %-change", 13, 16) %>%
          pack_rows("Average strength %-change per session", 17, 20) %>%
  
        add_footnote(label = c(paste0("Estimates from Wernbom et al. ", "(ref:wernbom-citation)"),
                               paste0("Estimates from Ahtiainen et al. ", "(ref:ahtiainen-citation)")), 
                     notation = "alphabet")

```

The moderate-volume condition consistently showed favorable adaptations when compared to the low-volume condition in measures of muscle hypertrophy and strength gains (Figure \@ref(fig:comb-fig-s1)a).
This was in general agreement with previous meta-analyses indicating that an increased training volume will lead to more favorable outcomes, both in terms of muscle mass
[@RN1767;<!-- Schenfeld  2016--> 
@RN789]<!-- Krieger 2010 -->
and strength
[@RN2492;<!-- Ralston --> 
@RN793].<!-- Krieger 2009 -->

```{r meta-analyses-descriptive, warning=FALSE, message=FALSE, echo=FALSE}

summary_table_meta <- readRDS( "./data/derivedData/meta/naive_models.RDS")


summary_table_meta <- summary_table_meta %>%
  mutate(stat = paste0(sprintf("%.3f", median), 
                       " [",
                       sprintf("%.3f", cri.lwr), 
                       ", ",
                       sprintf("%.3f", cri.upr), 
                       "]"))


```

Since publication of the most recent meta-analyses [@RN1767;@RN2492],<!-- Schenfeld  2016 / Ralston 2107 --> 
additional studies have been published investigating the effect of resistance training volume on muscle mass and strength. Based on studies included in the meta-analysis performed by Schoenfeld *et al.* [@RN1767], additional results were coded based on a systematic literature search among newly published studies (from January 2015 to June 2020). Compared to Schoenfeld *et al.* [@RN1767], an additional ten studies were included in the present analysis.
Subsequently, effect sizes were coded from 25 studies, both for muscle mass and strength and meta-regression models were used to investigate the effect of weekly number of sets on training-induced changes in these outcomes.
Both muscle mass and strength were shown to be robustly affected by an increased training volume measured as weekly number of sets (after controlling for potential confounding factors^[Models were controlled for age (young, middle age or old), sex (male, female or mixed), length of study (number of weeks), type of measurement (direct or indirect for muscle mass and training-specific or unspecific in strength), training status (trained > 1 year experience with resistance training), body portion (upper, lower or whole)]).
For every additional weekly set, the effect size measure (Hedges' *g*, [95% credible intervals] (CrI)), increased by
`r summary_table_meta[1, 5]` and `r summary_table_meta[2, 5]`
for muscle mass)
and strength (Figure \@ref(fig:comb-fig-s1)b),
respectively.
Together these data underlines the training volume dose-dependence of both muscle hypertrophy and strength gains.

\pagebreak

```{r comb-fig-s1, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences in training-induced relative changes in muscle-mass and strength (a). Estimates are derived from ANCOVA models controling for baseline values. Estimates from meta-regression models examining the effect of weekly number of sets per muscle group on muscle hypertrophy (b) and strength gains (c). Estimates are effect size measures (Hedges g) with 95\\% credible intervals (CrI).", fig.scap = "Differences in training induced changes to muscle mass and strength measures between volume conditions in Study I and weekly training volume meta-regression.", fig.height=6, fig.width=4.72, fig.align='center'}

library(ggplot2); library(ggtext)

# The figure and estimates are from ./R/study-1/muscle-strength-size.R
muscle_strength_fig <- readRDS("./data/derivedData/study1_muscle-strength-size/muscle-strength-fig.RDS")

muscle_strength_fig

```

\pagebreak

In an attempt to explain differences in training outcomes between volume-conditions, selected molecular markers, known to influence adaptations to resistance training were investigated for volume-dependency.
First, acute activation of signaling along the mechanosensitive mTORC1-pathway was measured in before and after the fifth training session in Study I.
A commonly used readout of mTORC1-signaling is the phosphorylation of S6-kinase (S6K1) at Thr\textsuperscript{389}/Thr\textsuperscript{412} which in turn precedes phosphorylation of ribosomal protein S6 (rpS6, see Figure \@ref(fig:mtor-fig)a).
In the present study, exercise-induced S6K1\textsuperscript{Thr\textsuperscript{389}/Thr\textsuperscript{412}} phosphorylation was indeed shown to be volume dependent along with phosphorylation of rpS6\textsuperscript{Ser\textsuperscript{235/236}} and mTOR\textsuperscript{Ser\textsuperscript{2448}} (Figure \@ref(fig:mtor-fig) b).
Together these observations indicates larger perturbations along the mTORC1 signaling axis which confirms previous studies showing exercise-volume dependency of mTORC1-related signaling in the acute phase (< 5 hours) after exercise, with the low-volume condition doing at least three sets activating the muscle under study
[@RN2716;<!-- Kumar 2012 -->
@RN1837;<!-- Ahtiainen 5 vs 10 sets p70 and s6 different -->
@RN784;<!-- Terzis volume affect activation -->
@RN791;
@RN834].
It is recognized that phosphorylation of mTOR itself at Ser\textsuperscript{2448} primarily should be regarded as indicative for negative feedback as this site is phosphorylated due to S6K1 activity
[@RN2309] (Figure \@ref(fig:mtor-fig) a).<!-- Figueriedo mTOR and ser2448 signalling review -->
It is also recognized that the phosphorylation status of rpS6 at Ser\textsuperscript{235/236} is not solely due to mTORC1 signalling as both mTORC1 and extracellular signal-regulated kinases (ERK)-signalling converges here
[@RN2311].<!-- Roux S6 phopshorylation at ser 235 236 promoted by ras/erk -->
Interestingly, Terzis *et al.* and Ahtiainen *et al.* did not report any clear volume dependency in exercise-induced activation of ERK 1/2
[@RN785;<!--Terzis et al., 2008-->
@RN1837], 
suggesting that any volume-dependent regulation of this pathway was outside the time-frame of these studies. However, it might also suggest that volume selectively modulates specific pathways in the acute phase after resistance exercise.
It should however be noted that the present, and previous studies are limited in their temporal resolution and different patterns over time in related to exercise-volume cannot be ruled out.

```{r p70-csa-correlation-data, warning=FALSE, message=FALSE, echo=FALSE}

### Check association with 

p70_csa <- readRDS("./data/derivedData/study1-western-blot/csa_p70.RDS")

### Calculate raw correlations
cor_est_p70 <- p70_csa %>%
  group_by(sets) %>%
   summarise(p70.fc.cor = cor.test(fc, change, method = "spearman")$estimate, 
            p85.fc.cor = cor.test(fc.p85, change, method = "spearman")$estimate, 
            p70.fc.cor.p = cor.test(fc, change, method = "spearman")$p.value, 
            p85.fc.cor.p = cor.test(fc.p85, change, method = "spearman")$p.value)  %>%
  data.frame()

p70.m <- lme(change ~ w2pre + scale(fc) + sets, 
             random = list(subject = ~ 1), 
             weights = varExp(),
             data = p70_csa)

p85.m <- lme(change ~  w2pre.p85 + scale(fc.p85)  + sets , 
             random = list(subject = ~ 1),
             weights = varExp(),
             data = p70_csa)

# plot(p70.m)
# plot(p85.m)


p70.mod <- paste0(sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 3]), 
                  "]")

p85.mod <- paste0(sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 3]), 
                  "]")

corr_est_p70 <- paste0(sprintf("%.3f",min(c(cor_est_p70[,2],cor_est_p70[,3]))), 
       " - ", 
       sprintf("%.3f",max(c(cor_est_p70[,2],cor_est_p70[,3])))) 

```

The importance of mTORC1 signalling for protein synthesis in the acute phase after resistance exercise is well established.
In humans, administration of rapamycin, a selective inhibitor of mTORC1, prior to resistance exercise leads to delayed or blunted activation of mTORC1 effectors such as
S6K1 and rpS6 as well as unchanged levels of phopshorylation of 4E-BP1 and eEF2 concomitantly with abolished exercise-induced increase in protein synthesis
[@RN780].<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
Volume dependent increase in mTORC1 signalling also coincide with larger protein synthesis rates
[@RN791;<!-- Burd in single session single and multiple sets phosphorylation p70   -->
@RN2716]<!-- Kumar volume affects protein synthesis -->
As such, volume dependence of mTORC1-related signalling suggests that higher within-session volume can be regarded as leading to an increased potential for protein synthesis in the acute-phase after exercise.
Previous studies has indirectly linked mTORC1-related signaling to muscle growth as acute perturbation along this pathway has been correlated with muscle growth
[@RN785;<!--Terzis et al., 2008-->
@RN788].<!-- Mitchell et al., 2013-->
However, this is not a consistent findings in the literature 
[@RN834;<!-- Mitchell et al., 2012-->
@RN2171],<!-- Phillips et al., 2017-->
nor is it consistent with results in the present study as acute activation of S6K1 in response to the fifth session was not associated with training-induced changes in muscle mass.^[Raw Spearman's correlations between exercise induced change in S6K1 phosphorylation and change in muscle mass (measured with MRI) indicated weak (insignificant) associations ($\rho=$ `r corr_est_p70`) depedning on the isoform measured (p70 or p85) and number of sets. More elaborate modeling confirmed weak relationships between degree of phsophorylation and muscle mass change. When controlling for baseline values in degree of phsophorylation (as this influence the calculation of fold-change) and number of sets, standardized estimates for the effect of phosphorylation were (increase in %-muscle mass change for every 1SD change in fold-change phsophorylation, with [95% CI])  $\beta=$ `r p70.mod` for p70 S6K1-p70\textsuperscript{Thr389} and $\beta=$ `r p85.mod` for S6K1-p85\textsuperscript{Thr412}.].

```{r mtor-fig, warning=FALSE, message=FALSE, echo=FALSE, fig.cap="Measured phosphorylation sites in context (a) and differences between volume conditions in phosphorylation status of S6K1 at Thr \\textsuperscript{389} (p70) Thr \\textsuperscript{412} (p85), rpS6 at Ser \\textsuperscript{235 236} and mTOR at Ser \\textsuperscript{2448} induced by acute exercise and expressed as fold-changes (b). Estimates are derived from ANCOVA models controling for baseline values. Values in b are point estimates with 95\\% confidence intervals.", fig.scap = "Differences between volume conditions in exercise induced phosphorylation of proteins related to mTORC1 signaling", fig.height=3.5, fig.width=4.72, fig.align='center'}

western_fig <- readRDS("./figures/results/study1-western-blot.RDS")
western_fig

```

Although mTORC1 signaling most certainly contributes to protein synthesis in the acute recovery phase after resistance exercise [@RN780],<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
its use as a predictor of training induced muscle hypertrophy is complicated by a number of reasons.
First, measuring its activity is not straight forward for biological reasons including pathway cross-talk and negative feedback as well as technical aspects
[@RN1949;<!-- Figueriedo, mTOR feedback -->
@RN2311;<!-- Cross talk with erk -->
@RN1942].<!--  Bass technical considerations in western blotting -->
Secondly, mTORC1 signaling is not a stable phenomenon either between or within individuals as short- and long-term resistance training leads to diminished activity along its pathway
[@RN1809;<!-- mTOR signal lost after three weeks -->
@RN1871].
Thirdly, mTORC1 readily integrate multiple signals but also convey these to multiple downstream processes
[@RN2320].
As such, mTORC1 activity represent an early, transient response to resistance exercise with its signaling contributing to accumulative responses.
One such downstream target of mTORC1 is ribosomal biogenesis
[@RN1632;<!-- Nader mTOR regulates ribo biogenthrough cell cycle regulators -->
@RN1810;<!-- von Walden 2012 mTOR regulates myotube... -->  
@RN1828;<!-- von Walden 2016 -->
@RN2321],<!-- Chauvin S6K controls RiBi transcriptional program -->
ultimately leading to accumulation of ribosomal RNA, a response typically seen in connection with to resistance training
[@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].

```{r ribo-markers}

descriptive_ribo_markers <- readRDS("./data/derivedData/study1-ribo-biogenesis/descriptive_ribo_markers.RDS")

rrna <- descriptive_ribo_markers %>%
  filter(target != "totalrna") %>%
  summarise(min.w2pre = round(min(w2pre),0), 
            max.w2pre = round(max(w2pre),0), 
            min.w12 = round(min(w12),0), 
            max.w12 = round(max(w12),0)) 
```

As ribosomal RNA is the most abundant constituent of muscle RNA it provides an estimate of muscle ribosomal abundance when expressed per unit tissue weight
[@RN2054;<!-- Zak 1967 in heart muscle 85% of RNA is ribosomal RNA -->
@RN2145].<!-- Young -->
From baseline to prior to the fifth training session (Week 2), total RNA per mg tissue increased by $\sim$ `r round(descriptive_ribo_markers[9,2], 0)` and `r round(descriptive_ribo_markers[10,2], 0)`% in the single-set and multiple-sets condition, respectively. Total RNA levels were still elevated above baseline after the intervention (Week 12, single-set $\sim$ `r round(descriptive_ribo_markers[9,3], 0)`; multiple-sets $\sim$ `r round(descriptive_ribo_markers[10,3], 0)`%). Similar patterns were seen in target analysis of ribosomal RNA using qPCR (single-set increase from baseline to Week 2 $\sim$ `r rrna[1,2]`-`r rrna[1,3]`% and Week 12 $\sim$ `r rrna[1,4]`-`r rrna[1,5]`%; multiple-sets $\sim$ `r rrna[2,2]`-`r rrna[2,3]`% and Week 12 $\sim$ `r rrna[2,4]`-`r rrna[2,5]`%). Comparing volume conditions revealed higher levels of total RNA and mature ribosomal RNA species in the multiple-sets condition at Week 2 (Figure \@ref(fig:rrna-fig)a). At Week 12 differences between volume conditions were in total RNA less clear and ribosomal RNA 28S showing higher levels in the single-set leg (Figure \@ref(fig:rrna-fig)a).

```{r rrna-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences between volume conditions in total RNA and ribosomal RNA-species (pre-rRNA 45S, rRNA 18S, rRNA 5.8S and rRNA 28S) measured at rest during the course of Study I (a). Acute changes in abundance of c-Myc mRNA and pre-rRNA 45S in response to acute exercise in Week 2 and differences between volume conditions (b). Errorbars represents 95\\% confidence intervals, transparent points and errorbars signifies that the confidence interval contain 1.", fig.scap = "Differences between volume conditions total RNA and ribosomal RNA", fig.height=2.5, fig.width=4.72, fig.align='center'}

ribo_markers_fig <- readRDS("./figures/results/study1-ribo-markers.RDS")

ribo_markers_fig

```

Analysis of c-Myc mRNA abundance in response to the fifth training session also showed volume-dependent regulation with exercise-induced increases being $\sim$ 1.5-fold higher in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:rrna-fig)b). c-Myc represents a rapamycin-insesitive signaling pathway, paralell to mTORC1, known to also stimulating ribosomal biogenesis 
[@RN1754;<!-- West -->
@RN1834;
@RN1832].<!-- cMyc driver of ribosomal biogenesis -->

Ribosomal biogenesis is an early adaptation to resistance training with initial increases in total RNA evident before any hypertrophy can be detected
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113].<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions-->
The increase in ribosomal RNA represents an accumulation of ribosomes and an increased translational capacity. This improved translational capacity contributes to protein accreetion resulting in muscle hypertrophy, reflected in associations between measures of ribosome abundance (typically total RNA) and muscle hypertrophy
[@RN1644;<!-- Figueriedo 2015 correlation between delta csa and fold change of total RNA -->
@RN1755;<!-- Stec 2016 only high responders have increased total RNA and rRNA abundance after short training -->
@RN2055;<!-- Mobley weak correlation r = 0.28 p = 0.026  and no cluster * time interaction effect but increases in total RNA 34% in high responders vs 8% in low -->
@RN1897].<!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->
Due to the sequential nature of these events (increased translational capacity before muscle hypertrophy), early increases in total RNA would possible better reflect training-induced changes in muscle hypertrophy.
Any increase in muscle mass would also contribute to a dilution of the RNA pool
[@RN2142]<!-- figueriedo description of dilution effect -->
possibly masking their causal relationship.
Comparing the relationships between RNA abundance and muscle hypertrophy at different time-points supported this view as the amount of RNA at Week 2 showed the strongest relationship with the resulting muscle growth (Figure \@ref(fig:rrna-csa-fig), \@table(rna-csa-tab)). This indicates that translational capacity, affected by exercise, early in the training program is determining training-induced changes in muscle mass.  

```{r rna-csa-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

rna_csa_change_models <- readRDS("./data/derivedData/study1-rna-csa-cor/regression.RDS")

rna_csa_change_models %>%
  mutate(Estimate = round(Estimate, 2), 
         SE = round(SE, 2), 
         df = df, 
         t.value = round(t.value, 2), 
         ci = paste0("[", sprintf('%#.2f', lower), ", ", sprintf('%#.2f', upper), "]"), 
         beta = paste0(sprintf('%#.2f',scaled.est), 
                       " [", 
                       sprintf('%#.2f', scaled.lower), 
                       ", ", 
                       round(scaled.upper, 2), "]"),
         beta = if_else(term %in% c("(Intercept)", "setsmultiple"), "", beta)) %>%
  dplyr::select(term, Estimate, SE, df, t.value, ci, beta) %>%
  mutate(term = factor(term, 
                       levels = c("(Intercept)",
                                  "setsmultiple", 
                                        "w0", 
                                        "w2pre", 
                                        "w12"), 
                       labels = c("(Intercept)", 
                                  "Moderate- vs. low-volume",
                                  "Week 0", 
                                  "Week 2", 
                                  "Week 12"))) %>%
  kable(format = "latex", 
        booktabs = TRUE, 
        escape = FALSE,
        caption = "Influence of RNA abundance on training-induced muscle growth measured with MRI", 
        col.names = c("", "Estimate", "SE", "df", "\\textit{t}-value", "95\\% CI", "Estimate [95\\% CI]")) %>%
  add_header_above(c(" " = 6, "Standardized coefficients" = 1), 
                   escape = FALSE) %>%
  pack_rows("RNA abundance (ng mg\\textsuperscript{-1})", 3, 5, escape = FALSE) %>%
  kable_styling(font_size = 8) %>%
  add_footnote("Standardized coefficients scaled by its SD")


```



```{r rrna-csa-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Relationship between total RNA and training induced change in thigh cross sectional area (CSA) as measured by MRI. Dashed line represent naive relationship, solid line represent adjusted reliationship as seen in \\@ref(table:rna-csa-tab).", fig.scap = "Relationship between total RNA and training induced muscle growth", fig.height=4.72/3, fig.width=4.72, fig.align='center'}

rna_csa_fig <- readRDS("./figures/results/study1-rna-csa.RDS")


rna_csa_fig

```

## Volume-dependent remodeling of muscle fiber-type composition

```{r myhc-stats, warning=FALSE,message=FALSE, echo=FALSE}

mx.summary <- readRDS("./data/derivedData/study1-myhc-composition/myhc2x_stats.RDS")

or.week2 <- paste0(sprintf('%#.2f', exp(mx.summary[5, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[5, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[5, 6])), 
                   "]")

or.week12 <- paste0(sprintf('%#.2f', exp(mx.summary[6, 1])), 
                           " [", 
                   sprintf('%#.2f', exp(mx.summary[6, 5])), 
                   ", ", 
                   sprintf('%#.2f', exp(mx.summary[6, 6])), 
                   "]")


```

Resistance training readily leads to remodeling of the muscle, this includes qualitative changes. These include fiber type transitions through changes in the myosin heavy-chain protein composition
[@RN2220;<!-- Adams Down regulation of MyHc 2x content in training group both myhc isoform sds page and atpase staining -->
@RN2799;<!-- Williamsson less IIx dominant hybrid fibers with training conversion towards IIa -->
@RN819;<!-- Schiaffino review, other protein families also different between fiber types-->
@RN1489].<!-- Ellefsen 2014, training induced switch from 2A to 2X -->
Resistance-training induced changes in fiber-types identified by myosin heavy-chain expression occurs primarily as a IIX $\rightarrow$ IIA transition which results in a more fatigue-resistant phenotype
[@RN819;<!-- Schiaffino review, other protein families also different between fiber types-->
@RN2056;<!-- Andersen, primarily transitions fiber in 2x to 2a direction minimal 2 to 1 trans -->
@RN2801]<!-- Karatzaferi 2001 Drop in ATP most pronounced in type 2x fibers in response to acute exercise -->
and thus represent an important aspect of the adaptive response to resistance training by reversing effects of inactivity and disuse
[@RN2057;<!-- Andersen 2x overshoot after de training -->
@RN2108].<!-- Bed rest increases type IIx -->
In Study I, muscle fiber-type compositions were determined based on immunostaining of myosin heavy-chain protein in muscle cross-sections. The staining protocol allowed for the identification of Type I (red), Type IIA (brown), Type IIX (unstained), and Type IIA/IIX hybrids (light brown, Figure \@ref(fig:myhc-fig)a). Throughout the study, a general reduction in Type IIX fibers was seen as pure Type IIX was replaced with Type IIA/IIX hybrids at Week 2, and these, in turn, were measured at lower proportions at Week 12 (Figure \@ref(fig:myhc-fig)b).

When comparing volume conditions, moderate training volume led to more pronounced down regulation of the *MYH1* gene, coding for Type IIX myosin heavy-chain protein, at all time-points during the intervention (Figure \@ref(fig:myhc-fig)c). This presumably led to the accumalative effect seen after twelve weeks of training where the proportion of Type IIX fibers was lower in the moderate- compared to the low-volume condition (indicative as an odd-ratio (OR) < 1, OR [95% CI]: `r or.week12`, combined measure of IIX/IIA hybrids and pure IIX fibers, Figure \@ref(fig:myhc-fig)d).  
This underlines that training volume is an important factor for fiber-type composition remodeling. Although not surprising, similar observations are lacking in the litterateur as previous studies have not compared this transition directly between volume protocols.
However, when comparing non-exhaustive high-load resistance training to load-matched training to volatile failure, Pareja-Blanco *et al.* observed a blunted IIX $\rightarrow$ IIA transitions in response to the non-exhaustive protocol 
[@RN2217].
Together these data indicates that increased metabolic stress and/or dosage of neuromuscular activity are plausible candidates for regulation of IIX $\rightarrow$ IIA reprogramming in humans in response to resistance training.
This as opposed to a tensile stimuli.
Based on observations in animal models, neural input might however be considered the primary candidate.
For example, in a rat model of resistance training, force did not affect fast-to-slow fiber-type transitions when performed with the same electrical stimulation pattern, indicating that neural activity governs fiber-type transitions
[@RN2104].<!-- Eftestol -->

At Week 2, a more pronounced reduction in *MYH1* gene abundance in the moderat-volume condition did not translate to the protein level. Instead a greater number of fibers were classified as IIX in the moderate-volume condition compared to the low-volume condition (OR: `r or.week2`).
The moderate-volume condition thus showed an even greater mismatch between gene and protein levels.
Such mismatch is seen after resistance-exercise and is indicative for early transition of muscle fibers
[@RN2056].<!-- Andersen mismatch between  -->
Interestingly, the difference between volume conditions was driven by less Type IIX expression in the low-volume condition.
This indicates that remodeling occurred faster in response to the low-volume condition despite a greater "transition signal" in the moderate-volume leg.
A plausible explanation to this observation could be that the greater damage caused by a greater training stimuli lead to a delayed fiber-type transition. 
This hypothesis is supported by observations indicating that a proportion of the protein synthesis seen early in a resistance-training period can be attributed to tissue repair rather than myofibrilar synthesis
[@RN1521;<!--Kim 2005 -->
@RN2144].<!-- Damas 2016 -->
Regardless of causality, the more rapid early adaptation seen in response to a lower training volume suggests that optimized resistance training should include a progressive volume component.



\pagebreak

```{r myhc-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Representative muscle cross-sections stained for myosin-heavy chain isoforms are shown in (a). Using antibodies for Type I (MyHC Slow) and all but Type IIX (BF-35) enabeled separating Type I fibres from other fibres (red stain, a lower panel). Type IIX fibers were indentified as unstained, and weak brown staining was analysed as Type IIX/IIA hybrids (a, upper panel). Resistance training led to a reduction in Type IIX and IIX/IIA hybrids, (b) shows participant-average Type IIX and IIX/IIA percentages. Training led to reduction in \\textit{MYH1} gene expression, and more so in the moderate-volume condition, evident from pairwise comparisons in (fold-differences with 95\\% CI, c). Differeces in protein expression of Type IIX fiber (counted as pure IIX + 0.5 $\\times$ IIX/IIA hybrids) reversed throught the study as the moderate-volume condition showed greater proprtions of IIX fibers at Week 2 but lesser expression at Week 12 (d).", fig.scap = "Fiber-type composition in Study I", fig.height=6, fig.width=4.72, fig.align='center'}

# load fiber typing figure
img <- ggdraw() + draw_image("./data/study-1/immuno/immuno_control_image2.png")

# load figures
myhc_fig_comps <- readRDS( "./data/derivedData/study1-myhc-composition/myhc_fig.RDS")


## Combine figure directly in manuscript as saving the png takes some space 

myhc_fig <- plot_grid(plot_grid(NULL, img, NULL, ncol = 3, rel_widths = c(0.2, 1, 0.2)), 
                      plot_grid(myhc_fig_comps$overall_type2x, 
                                NULL,
                                plot_grid(myhc_fig_comps$myhc2x_mrna_means, 
                                          myhc_fig_comps$diff_2x_mRNA, 
                                                    ncol = 1, 
                                                    align = "v",
                                                    rel_heights = c(0.5, 0.5)),
                                          ncol = 3, rel_widths = c(0.4,0.05, 0.5)),
                  
                       myhc_fig_comps$fiber_type_insets, 
          ncol = 1, 
          rel_heights = c(0.6, 0.6, 0.6)) + 
  draw_plot_label(x = c(0.25,
                   0.005, 0.45, 
                   0.005), 
             y = c(0.98, 
                   0.67, 0.67, 
                   0.36), 
             label = c("a", "b", "c", "d"))

myhc_fig

```

\pagebreak

## Volume-dependent effects on transcriptome characteristics

As already [discussed](#muscle-mass-growth), resistance training in Study I led to an increase in total RNA per unit tissue weight. Furthermore, the magnitude of this increase was different between volume conditions (see Figure \@ref(fig:rrna-fig)).
Studies of gene expression, using techniques such as qPCR or RNA sequencing, comes with the assumption that an equal amount of biological material is used when evaluating abundance of a single transcript.
It is therefore common that a normalized amount of total RNA is used in sample preparation to satisfy such an assumption
[@RN2298;<!-- Damas 2018 -->
@RN2400; <!-- Robinson 2017-->
@RN2401].<!-- Laker 2017.-->
However, if the ratio of RNA to tissue mass has changed throughout a study, analyses are performed on a different amount of tissue. This may be the case after a resistance-training intervention, or between study conditions, as in Study I.
Indeed, by calculating the amount of tissue used when preparing samples for RNA sequencing it is obvious that the increased amount of total RNA per-unit-muscle weight results in a situation where samples are prepared from a lesser amount of tissue.
This effect was more pronounced in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:lib-size-fig)a).
Total library sizes were quantified from RNA sequencing as the total number of quantified gene counts, this revealed a general increase in library sizes in response to resistance training, however, the increase was more apparent in the low-volume condition
(Figure \@ref(fig:lib-size-fig)b).
When library sizes were normalized to the amount of tissue used in sample preparation, differences between conditions were diminished
(Figure \@ref(fig:lib-size-fig)c).
This suggests that resistance-training leads to an increased global mRNA expression per-unit-muscle weight (43-53\% in the present study). However, this increase was not different between volume-conditions, suggesting that a volume-dependent regulation of global RNA transcription primarily relates to transcription of RNA-fractions other than mRNA (likely rRNA, see Figure \@ref(fig:rrna-fig)). 

```{r, lib-size-fig, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Less muscle tissue was used for cDNA synthesis in the moderat- compared to low-volume condition as a result of total RNA increases per tissue weight (a). Library sizes tended to be greater in the low-volume condition (b), after normalizing library sizes to muscle weight differences were diminished (c). Lower panels shows differences between volume conditions", fig.scap = "Muscle weight and RNA-seq library size", fig.height=3, fig.width=4.72, fig.align='center'}

libsize.fig <- readRDS("./data/derivedData/study1-transcriptome/muscle-weight-lib-size.RDS")


libsize.fig +
  draw_plot_label(x = c(0, 0.33, 0.67), 
             y = c(1,1,1), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b", "c"))


tissue_offset_ngenes <- readRDS("./data/derivedData/study1-transcriptome/n_genes_regulated_time.RDS") %>%
  data.frame()


w2pre_unchanged<- sprintf("%.1f", tissue_offset_ngenes[4,4] * 100)
w12_unchanged<- sprintf("%.1f", tissue_offset_ngenes[1,4] * 100)

w2pre_up<- sprintf("%.1f", tissue_offset_ngenes[5,4] * 100)
w12_up<- sprintf("%.1f", tissue_offset_ngenes[2,4] * 100)

w2pre_down<- sprintf("%.2f", tissue_offset_ngenes[6,4] * 100)
w12_down<- sprintf("%.2f", tissue_offset_ngenes[3,4] * 100)


```

<!-- The general increase in mRNA counts seen in both volume-conditions is intriguing as during steady state conditions, total mRNA abundance generally scales with cellular volume
[@RN2876].<!-- Padovan-Merhar 2015 mRNA scale with cell size, increasing cell size increases global transcription to compensate -->
<!-- However, during mechanically induced muscle growth, Kirby *et al.* found that nascent transcription, including mRNA transcription is elevated primarily in small muscle fibers
[@RN2616].<!-- Kirby nascent transcript in smaller fibers during hypertrophy (SA), global promoter activation -->
<!-- This may correlate with small fibers being prime drivers of muscle hypertrophy as these fibers are more likely to increase in size
[@RN2616].<!-- Kirby nascent transcript in smaller fibers during hypertrophy (SA), global promoter activation -->
<!-- Training-induced increases in mRNA may thus represent a mechanism important for protein accretion and hypertrophy.
<!--
Interestingly, rRNA levels are found to be higher in slow vs. fast muscle fibers with selected mRNA found to scale with total RNA indicating that also mRNA levels are greater in more glytolytic fibers
[@RN1707].<!-- Habets 1999 in rat higher 28S RNA and total rna in slow compared to fast fibers, unchanged myhc mRNA/RNA between fibers   -->
<!--This could thus indicate that elevated mRNA levels precedes muscle remodeling to a more-->
Although the observed increase in library sizes suggested a global increase in mRNA transcription, it did not resemble global gene amplification seen in models with experimentally manipulated c-Myc protein expression
[@RN2430;<!-- Lin 2012 increased cmyc leads to increased expression of all active genes -->
@RN2359].<!-- Loven global amplification in the light of gene expression studies  -->
Instead, the majority of identified genes were categorized as having unchanged levels per amount of tissue from baseline to Week 2 (`r w2pre_unchanged`\%) and Week 12 (`r w12_unchanged`\%). A small fraction of genes were identified as being less abundant (Week 2, `r w2pre_down`\%; Week 12, `r w12_down`\%) and the remaining genes were identified as more abundant at Week 2 (`r w2pre_up`\%) and 12 (`r w12_up`\%), respectively.
Gene-set enrichment analyses revealed that training-induced changes in gene expression profiles were related to increased abundance of genes related to extracellular matrix remodeling, collagen synthesis and inflammatory processes. Although these observations were made without a proper control to investigate training-induced effects *per se*, they resembles previous (controlled and uncontrolled) studies characterizing skeletal muscle tissue in response to heavy mechanical loading, in the acute phase
[@RN2451; <!-- Wessner acute exercise gives different regulation of metalloproteinase -->
@RN2453;<!-- Mackey  repeated bout effect 30 days -->
@RN2454;<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load --> 
@RN2457]<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->,
and after prolonged training [@RN774; <!-- Raue, upregulation of ECM after long term exercise --> 
@RN2298].<!-- Damas long term training ECM up gene-->

```{r, week2-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Genes identified as differentially expressed (log2 fold-change > |0.5|, FDR < 0.05) are highlighted in a.  Overlap in differentially expressed genes between models are shown in (b)", fig.scap = "General patterns of differentially expressed genes at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

week2_grid <- readRDS("./data/derivedData/study1-transcriptome/week2_volcano_upset.RDS")

week2_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

As we observed volume-dependent differences in the ratio of total RNA to tissue weight, leading to differences in biological input in sample preparation, we used three different normalization strategies to characterize volume-dependence in transcriptome characteristics.
By implementing each normalization strategy as variations of generalized linear mixed models (GLMM), we could compare outcomes related to conceptually different normalization techniques using the same statistical framework.
The use of GLMM also had the benefit of accounting for the study design which included correlated data (within participants)
[@RN2366].<!-- Cui et al what if we ignore random effects -->
The model with most parameters included an offset term expressing gene counts per tissue weight. This was combined with library size included as a fixed effect in the model together with study conditions, time and volume-conditions (tissue offset-model).
In a second model, the offset term was removed, thus only the library size was used to normalize the gene counts, as suggested by Cui *et al.*
[@RN2366] (effective library-size model).<!-- Cui et al what if we ignore random effects -->
For comparison, a naive model containing no normalization-parameters was also included in analyses.
At Week 2, tissue-offset normalization (modeling mRNA counts per-mg-muscle weight) revealed a general shift of mRNA counts towards the moderate-volume condition (Figure \@ref(fig:week2-upset-volcano)a).
This was in contrast to patterns found when using library-size and naive models resulting in a pattern where the "conventional" normalization strategy, accounting only for library sizes more resembled the non-normalized naive model as there was an overlap of genes identified as differentially expressed between the models (Figure \@ref(fig:week2-upset-volcano)b).
Gene-set enrichment analyzes revealed that genes related to extracellular matrix remodeling and collagen synthesis was driving the difference in transcriptome profiles between volume-conditions at Week 2
(Figure \@ref(fig:week2-go-analysis)a).
This was apparent using both the tissue offset model and the library-size normalized model, however volume-dependent effects were more pronounced in the tissue-offset model
(Figure \@ref(fig:week2-go-analysis)b).

```{r week2-go-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Gene-set enrichment analysis based on gene-ontology gene sets and estimates from the tissue-offset model (a). Adjusted \\textit{P}-values and Normalized enrichment scores are retrieved from analysis of directional rank-based tests (GSEA), labeling of points indicate if gene-sets were also identified from differentially expressed genes (Log2 fold-change >|0.5|, FDR < 0.05). In (b) comparison between the tissue-offset and library size normalization models are shown in rank-based (non-directional, cerno-test) analysis of gene ontology gene sets.", fig.scap = "Gene-set enrichment analysis at Week 2", fig.height=5, fig.width=4.72, fig.align='center'}

go_analysis_week2 <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/go_week2.RDS")


go_analysis_week2 +
  draw_plot_label(x = c(0.02, 0.55), 
             y = c(1, 0.85), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))



```

Differences in normalization models led to global shifts of estimates of differences between volume conditions. Figure \@ref(fig:gene-set-rug) shows this global shift as density curves representing all fold-changes between volume conditions. Accounting for the amount of muscle mass in sample preparations shifted also specific gene-sets towards greater differences between volume conditions, this is exemplified in Figure \@ref(fig:gene-set-rug) by the gene set "Collagen containing extracellular matrix".

```{r gene-set-rug, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Differences between volume conditions at Week 2 as seen using different normalization models. Shifts in the full distribution of Log2-fold change values between volume conditions were seen between on normalization models (density curves). As an example, genes associated with the \\textit{Collagen containing extracellular matrix} gene set are highlighted (black bars), differentially expressed genes are identified with gene symbols in each method.", fig.scap = "Global shifts in volume-dependent fold change as an effect of normalization methods at Week 2", fig.height=5, fig.width=3.72, fig.align='center'}

gene_set_rug <- readRDS("./data/derivedData/study1-gene-set-enrichment-week2/gene-set-rug.RDS")

gene_set_rug

```

At week 12, differences seen at Week 2 between volume conditions derived from the tissue-offset model were abolished (Figure \@ref(week12-upset-volcano)a). Volume dependent differences were seen in the library-size normalized model, however, these overlapped with the naive model indicating that imbalance in muscle tissue between conditions may have contributed to global shifts in favor of the low-volume condition. This indicates that after initial perturbations related to different training volumes, the muscle may have reached a new transcriptional equilibrium.   

```{r week12-upset-volcano, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Volume dependent differences in gene expression in different models (a) either accounting for the amount of tissue used in cDNA synthesis (Tissue offset), library sizes or un-normalized (Naive). Overlap in differentially expressed genes between models are shown in (b)", fig.scap = "General patterns of differentially expressed genes at Week 12", fig.height=5, fig.width=4.72, fig.align='center'}

week12_grid<- readRDS("./data/derivedData/study1-transcriptome/week12_volcano_upset.RDS")

week12_grid +
  draw_plot_label(x = c(0, 0.25), 
             y = c(1, 0.57), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))


```

From a general perspective, this analysis underlines the importance of a well considered choice of normalization strategy in studies of gene expression in non-homeostatic conditions
[@RN2182;<!-- Normalization per cell radically changes the view of increased or decreased global gene expression -->@RN2878;<!-- Evans discusses mRNA per cell or per transcriptome  -->
@RN2359].<!-- Loven 2012 in the case of gene amplification, gene expression changes might be over/underestimated between conditions -->
Specifically, in gene expression studies of training-induced muscle hypertrophy, an ill-advised normalization strategy could mask or overestimate differences between experimental conditions as different amount of tissue is being used in sample preparation. This as a consequence of a normalized RNA input between different conditions. Differences in tissue used in sample preparation between volume-conditions in Study I effectively led to an underestimation of genes identified as differentially expressed at Week 2 and led to nonsensible identification of differentially expressed genes at Week 12.
The choice of normalization strategy ultimately determines the definition of differential expression. By using the tissue offset model we were interested in gene counts per tissue, as opposed to using the the library size as the denominator in analyses, leading to gene counts expressed per transcriptome. Although both strategies may have merit in certain situations, selected genes may be better studied as counts per amount of tissue. It could be argued that one such group of genes are genes related to extracellular matrix remodeling.
First, extracellular matrix-related genes have been shown to display close association between mRNAs abundances and their respective proteins, including collagen-organization proteins [@RN2448].<!-- Makhnovskii cycling exercise, mRNA and protein scales for ECM -->.
This indicates that remodeling on the protein level to a large degree is determined transcriptionally.
Secondly, selected extracellular-matrix related genes are not expressed in muscle fibers but instead in neighboring cells (e.g. fibroblasts)
[@RN2883;<!-- Zou muscle fibroblast but not myogeneic cells express Col6a1  -->
@RN2421].<!-- Kjaer genral review on ECM with collagen synthesis in fibroblast highlighted -->
Thus assuming that an increase in total RNA due to increased ribosomal biogenesis would scale to other cell types in muscle tissue might affect the interpretation of such gene expression data.
Consequently, studies of transcriptional regulation of extracellular matrix remodeling in relation to muscle hypertrophy should account for the amount of tissue used in analyses.

Previous studies have generally not found extracellular matrix remodeling to be affected by different exercise variables. Instead, when comparing low- and high-load, Holm *et al.* found similar responses measured as collagen synthesis [@RN2454].<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load -->.
Similarly, eccentric versus concentric resistance exercise did not affect muscle collagen synthesis differently
[@RN2457].<!-- Moore 2005 collagen synthesis rates not affected by ecc vs. conc -->
Although it should be noted that the measurement of collagen synthesis and expression of genes related to extracellular matrix are targeting different aspects of extracellular matrix remodeling. Volume-dependent regulation of these target genes, implicates this remodeling as paralleled with muscle growth.

```{r, echo = FALSE, warning=FALSE, message=FALSE}

acute_time_ngenes <- readRDS("./data/derivedData/study1-transcriptome/n_genes_regulated_time_acute.RDS")


w2post_unchanged<- sprintf("%.1f", acute_time_ngenes[1,4] * 100)

w2post_up<- sprintf("%.1f", acute_time_ngenes[2,4] * 100)

w2post_down<- sprintf("%.1f", acute_time_ngenes[3,4] * 100)




```


In response to acute exercise, `r w2post_up` and `r w2post_down`\% of identified genes were categorized as up and down-regulated 1 hour after exercise.
These changes were measured as per transcriptome as no changes in the total RNA per tissue weight
[@RN2142],<!-- Figueriedo No change in total rna increase in short time frames -->
but instead fluid shifts [@RN2270]<!-- Ploutz-Snyder Fluid shifts 1995 --> 
could be expected in the short time frame affecting the estimate of muscle weight from wet tissue.
Differentially regulated genes were generally associated with stress responses including inflammation and immune responses as well as extracellular matrix related processes. In contrasts to long term changes, acute exercise led to down-regulation of extracellular matrix-related genes, similarly to what have been found by others [@RN774]<!-- Raue, upregulation of ECM after long term exercise -->.
Regarding volume-dependent regulation in the acute phase, only a single gene was found to be differentially expressed between volume-conditions. This gene, *RFT1*, which was found to be down-regulated and more so in response to moderate-volume exercise is associated lipid transport, carbohydrate transport, and endoplasmic reticulum membrane and has previously been found to be decreased in muscle after exercise [@RN2360]<!-- Pillon -->.
Further analyses of volume-dependent differences not relying solely on arbitrarily defined differential expression confirmed that no strong differences were evident in transcriptome characteristics in this short time-span.
It is likely that the timing of the post-exercise biopsy missed more relevant volume-dependent regulation in the acute phase.


## Determinants of moderate- over low-volume training benefit

To explore determining factors for additional benefit of moderate-volume training, individual differences in muscle-hypertrophy (CSA) and average muscle strength between the moderate- and low-volume condition were calculated. In primary analyses, difference-scores were dichotomized to *additional benefit of moderate-volume training* or *no additional benefit of moderate-volume training* based on the smallest worthwhile change (SWC) in the direction of moderate-volume. Participants were thus identified as having *additional benefit of moderate-volume training* if the difference in raw change scores were greater than the between-participants baseline SD $\times$ 0.2 in favor of moderate-volume training.^[Sex-differences were accounted for by mean-centering variables prior to SD estimation. A weighted SWC was calculated for the strength variable to avoid underestimating SD.]
The choice of using a dichotomized response variable (benefit vs. no benefit) was made to maintain an individual perspective; irrespective of the magnitude of the individual difference, what determines benefit of moderate- over low-volume training?

```{r csa-str-sets-corr, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Relationship between individual responses to moderate- and low-volume training. Dashed lines are identity lines ($y = x$) and the distance from identity lines to solid lines represent the smallest worthwhile change (SWC) in each variable.", fig.scap = "", fig.height=2.4, fig.width=4.72, fig.align='center'}

# Read correlations
corr <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor_df.RDS")
csa_cor <- paste0("*r* = ", sprintf("%.3f", corr[2, 2]), 
                  ", [", 
                  sprintf("%.3f", corr[2, 3]),
                  ", ",
                  sprintf("%.3f", corr[2, 4]),
                  "]")
str_cor <- paste0("*r* = ", sprintf("%.3f", corr[1, 2]), 
                  ", [", 
                  sprintf("%.3f", corr[1, 3]),
                  ", ",
                  sprintf("%.3f", corr[1, 4]),
                  "]")                



csa_str_cor <- readRDS("./data/derivedData/study-1-determinants/csa_str_cor.RDS")

csa_str_cor

```

Identification of participants as belonging to the benefit- or no benefit-group, respectively, for each outcome variable is visualized in Figure \@ref(fig:csa-str-sets-corr). Responses to either training condition were highly correlated for both muscle CSA (`r csa_cor`) and average strength (`r str_cor`), but participants identified as having benefit of moderate-volume training were identified across the whole range of responses (Figure \@ref(fig:csa-str-sets-corr)).

With the aim to develop parsimonious models that explained benefits of moderate-volume training in each outcome (muscle CSA and average strength), a set of potential determinants was selected prior to any association analysis. These included blood variables, baseline strength and muscle mass, volume-dependent molecular responses to training and baseline fiber-type composition. Additionally, pre-study training habits and training characteristics during the study as well as dietary data were used in an initial screening process of potential variables. As neither pre-study training habits (strength or other type physical training), within-study training characteristics (number of sessions completed and supervised) or dietary data displayed any differences between benefit-classifications, they were excluded from further consideration. So were also a set of continuous variables that did not meet the criteria for inclusion, based on preliminary univariate analysis   
[@RN1998].
Figure \@ref(fig:univariate-benefit-analysis) shows continuous variables used in the initial screening process. Variables were kept for further consideration when the mean difference 80\% CI (thick error bars) did not containing the null-hypothesis of no difference between benefit-classification groups. Based on the variables selected in this first step, 32 participants were included in the subsequent analysis as two participants had missing values in one or more of the selected predictors.

```{r univariate-benefit-analysis, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Differences between groups classified as either having benefit or no benefit of moderate-volume training. For the sake of comparison, all variables have been scaled by their SD. Thick errorbars indicate a 80\\% CI and thin errorbars indicate the 95\\% CI. Transparent points and errorbars indicate that the variables was removed from further modeling. Sex was included as a covariate in all comparisons.", fig.scap = "Univariate analysis of potential determinants of benefit to moderat- over low-volume training", fig.height=5, fig.width=4, fig.align='center'}

univariate_regression <- readRDS("./data/derivedData/study-1-benefit-analysis/univariate_regression_results_fig.Rds")

univariate_regression

```

The choice of a dichotomized response variable (benefit vs. no benefit) entailed the use of logistic regression. An assumption of this type of regression modeling is linearity of the logit
[@RN1998].
As all variables did not meet this assumption when fitted in logistic regression they were categorized into biologically meaningful categories (Vitamin D insufficient/sufficient), dichotomized based on measurement detection limits (testosterone in females) or sex-specific median values (lean body mass and testosterone in males).
A stepwise approach was used to remove variables that did not display association with the response variables. Figure \@ref(fig:model-reduction-plot) shows the process of stepwise removal of predictors.
Additional benefit of moderate- over low-volume training on muscle CSA was predicted by individual differences between volume conditions in Total RNA at Week 2 and baseline lean body mass dichotomized to above or below the sex-specific median.
For muscle strength, only individual differences between volume conditions in Total RNA at Week 2 remained in the model after variable selection.

```{r model-reduction-plot, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Sequential removal of predictors of moderate-volume benefit. Point estimates for each predictor are displayed with 95\\% confidence intervals.", fig.scap = "Step-wise variable selection of determinants of moderate- over low-volume training benefit.", fig.height=3.72, fig.width=4.72, fig.align='center'}

model_reduction_plot <- readRDS("./data/derivedData/study-1-determinants/model_reduction_plot.RDS")                    
model_reduction_plot

```

The rationale for using dichotomized outcomes was, as mentioned above, to specifically explore determinants of *individual* benefit of additional training volume, regardless of the magnitude of the difference between conditions.
This choice of analytic approach arose from discussions regarding clinical decision-making, where the potential benefit in each individual should guide the choice of training methods.
Although linked to the specific study question, a dichotomized outcome variable could potentially mask other important associations in the data due to loss of information associated with the transformation
[@RN2881].<!-- Harrell 2015 Regression modeling strategies, critique of stepwise procedures-->
A primary hypothesis in the study was that the baseline muscle phenotype, specifically a "glycolytic" vs. a "aerobic" phenotype (measured as high vs. low type IIX content) would not show relative benefit of moderate-volume training.^[[As outlined in the Study's pre-registration](https://clinicaltrials.gov/ct2/show/NCT02179307?term=lillehammer&draw=2&rank=9)]
If such a marker would hold any prognostic value, it could potentially guide decisions regarding the training process.
In analysis, the phenotype marker measured as Type IIX counts was included together with other potential predictors of individual benefit of higher training-volume.
Both pre-training characteristics that could aid in clinical decision making and markers arising from the training itself were included in the analysis.
In order to select important determinants from this set of potential determinants, a stepwise procedure for variable selection, known as purposeful selection was used
[@RN1998].<!-- Hosmer -->
Generally, stepwise variable selection has been criticized for a number of reasons, including arbitrarily searching for a single best model, over-estimation of regression parameters and sensitivity to small changes in the investigated data set
[@RN2882;<!-- Whittingham 2006 information theory shows multiple models that works equally well compete  -->
@RN2881].<!-- Harrell 2015 Regression modeling strategies, critique of stepwise procedures-->
In order to address the above mentioned limitations, the question was slightly rephrased in a secondary analysis presented here to explore the association between potential determinants and individual differences between moderate- and low-volume training.
As shown in Figure \@ref(fig:continuous-determinants)a, only Total RNA content at Week 2 in the moderate-volume leg, expressed as a percentage of the low-volume leg RNA content, positively associated with differences between volume conditions in muscle hypertrophy (CSA) and strength gains. These relationships are plotted in Figure \@ref(fig:continuous-determinants)b.

Together with previous studies
[@RN1755; <!-- Stec higher sitgnificant increase in total RNA as wells as rRNA in high responders -->
@RN1644;<!--Figueiredo correlation between fold change total RNA and delta CSA 8 weeks of training--> 
@RN2055],<!--Mobley larger RNA increases in responders compared to low resp.-->
as well as the direct correlation between RNA content and muscle hypertrophy presented earlier in this text
(Figure \@ref(fig:rna-csa-fig)),
this underlines the importance of early-phase ribosomal accumulation as a determining factor for training-induced muscle hypertrophy. 
Participants showing greater increases in ribosomal content in response to higher training volume also displayed greater relative benefit of the higher volume.
This likely acts through an increased translational capacity and confirms observations regarding volume-dependence in regulation of total RNA, mature rRNA species (rRNA 18S, 28S and 5.8S) and subsequent muscle hypertrophy (and strength gains).

```{r continuous-determinants, echo=FALSE, warning=FALSE, message=FALSE, fig.scap = "Determinanats of ", fig.cap="Analysis of association between potential predictors and individual differences between moderate- and low-volume training in muscle cross-sectional area and muscle strength as the dependent variable (a).", fig.scap = "Predictors of differences in outcomes between moderate- and low-volume training. Values in (a) are standardized regression coefficients with 95\\% CI from univariate robust regression. Positive estimates indicate a positive association. Variables related to blood parameters, body composition and muscle strength was mean centered per sex.", fig.height=6, fig.width=4.72, fig.align='center'}

# 

continous_determinants <- readRDS("./data/derivedData/study-1-determinants/continous_determinants.RDS")


plot_grid(continous_determinants$determinants_overview, 
          plot_grid(continous_determinants$csa_scatter, 
                    continous_determinants$str_scatter, 
                    ncol = 2, 
                    rel_widths = c(0.5, 0.5)), 
          ncol = 1, rel_heights = c(1, 0.5)) +

  draw_plot_label(x = c(0, 0), 
             y = c(1, 0.35), 
             vjust = 1,
             hjust = 0,
             label = c("a", "b"))



lean_mass_reg_table <- readRDS("./data/derivedData/study-1-determinants/lean_mass_reg_table.RDS")

lean_mass_diff <- paste0(sprintf("%.1f", lean_mass_reg_table[2, 1]), 
                         "%-point [",
                         sprintf("%.1f", lean_mass_reg_table[2, 5]),
                         ", ",
                         sprintf("%.1f", lean_mass_reg_table[2, 6]),
                         "]")

```

In primary analyses, percentage of lean body-mass was identified as a predictor of additional benefit of moderate-volume training on on muscle hypertrophy. 
Albeit associated with a large degree of uncertainty, this indication was in line with current prescription guidelines recommending greater training volume for experienced individuals
[@RN1].
When analyzed between benefit categories, participants that displayed benefit of moderate- over low-volume training had a `r lean_mass_diff` higher lean body mass compared to participants in the no benefit-group.
Lean body mass did not, however, display a linear relationship with individual differences between volume-conditions.
This together with the fact that we did not find any robust relationship between pre-training Type IIX fiber proportions and differences in training outcomes between volume conditions underlines that such markers have low prognostic value, even though they are readily adapted to the training regime as such.
This likely relates to the fact that traits such as fiber type, or body composition are genetically determined
[@RN2884;<!-- Arden lean mass is heritable -->
@RN2798]
in addition to influenced by any history of physical activity.

### Predictors of additional benefit of added training volume: Meta-analysis

```{r meta-hypoteses, echo = FALSE, message=FALSE, warning=FALSE}

meta_hypotheses <- readRDS("./data/meta-volume/models/meta_hypotheses.RDS")

upper_body_strength <- paste0(round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,2], 3), 
                    " [", 
                    round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$strength_upper_body_weeklysets$hypothesis[1,5], 3), 
                    "]")

upper_body_mass <- paste0(round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,2], 3), 
                    " [", 
                    round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$muscle_upper_body_weeklysets$hypothesis[1,5], 3), 
                    "]")

unspecific_strength <- paste0("95% CrI: [", 
                    round(meta_hypotheses$strength_unspecific$hypothesis[1,4], 3), 
                    ", ", 
                    round(meta_hypotheses$strength_unspecific$hypothesis[1,5], 3), "]")

```


To further explore potential factors associated with benefit of added training volume, interactions between study parameters and weekly number of sets were estimated from meta-regression models (previously reported without interactions in Figure \@ref(fig:comb-fig-s1)). 

Weekly number of sets did not interact with age, training status or sex for either muscle mass or strength (interaction estimates are shown in Table \@ref(tab:meta-interaction-table)), indicating that the literature currently gives little support for such demographic variables to guide prescription regarding training volume.  

In contrast to what has been previously reported
[@RN1767],<!-- Schoenfeld meta -->
the present analysis did not indicate any robust differences between unspecific (e.g. DXA) or specific (e.g. MRI or muscle thickness measured with ultrasound) measures of muscle mass.
However, specific assessment of muscle strength, e.g. measures of one repetition maximum in exercises used in the training program consistently showed larger positive effects of additional volume (Table \@ref(tab:meta-interaction-table)).
Unspecific assessments of muscle strength were shown not to reflect additional training volume (effect size change per one weekly additional set `r unspecific_strength`). This may indicate that specific motor learning is a factor influenced by training volume.

A consistent finding between measures of muscle mass and strength was the negative effect seen in upper- compared to the lower-body measures.
These results suggests that adding a weekly set to upper-body programs will not robustly lead to an increased effect in strength measures (upper-body effect size change for every additional weekly set `r upper_body_strength`).
The effect of an additional weekly set in the upper-body for measures of muscle mass was still positive (`r  upper_body_mass`) albeit lower when compared to the effect seen in the lower body (Table \@ref(tab:meta-interaction-table)).
Together this indicate that muscles more active in every day activities will benefit to a greater extent by additional training volume, compared to upper-body muscles.
Studies examining volume-dependence in molecular determinants to resistance-training adaptations between different muscle groups are limited.
However, Hanssen *et al.* found different satellite cell activation between volume-conditions in *m. vastus lateralis* but not in *m. trapezius*, coinciding with volume-dependence in outcomes in lower- but not upper-body muscles
[@RN796;
@RN776].
Whether satellite cells *per se* were responsible for this effect or if it represents a general inability of upper-body muscles to differentiate between two relatively strong stimuli remains to be explored.

The length of study interacted differently in muscle mass and strength models were longer training periods seemed to reduce the positive effect of added training volume on muscle mass, albeit likely to negligible degree. However strength outcomes were positively influenced by the length of the training period (Table \@ref(tab:meta-interaction-table)). This may be an effect of different time courses in muscle mass and strength measures in response to resistance training.





```{r meta-interaction-table, message=FALSE, warning=FALSE, echo=FALSE}

library(kableExtra)

## Load data from meta_analysis.R
summary_table_interactions <- readRDS("./data/meta-volume/models/summary_table_interactions.RDS")



summary_table_interactions %>%
  cbind(data.frame(Variable = c("30-50 vs. 18-20 years", 
                                "50- vs. 18-20 years", 
                                "Mixed ages vs. 18-20 years", 
                                "Trained (> 1 year RT) vs. untrained", 
                                "Female vs. male", 
                                "Mixed sex vs. male", 
                                "Upper-body vs. lower-body", 
                                "Whole-body vs. lower-body", 
                                "Unspecific vs. Specific estimation", 
                                "Weeks")), .) %>%
  mutate(m.est = as.character(m.est),
         s.est = as.character(s.est),
         m.ci = paste0("[", 
                       m.lwr, 
                       ", ", 
                       m.upr, 
                       "]"), 
         s.ci = paste0("[", 
                       s.lwr, 
                       ", ", 
                       s.upr, 
                       "]") , 
         s.est = if_else(Variable == "Whole-body vs. lower-body", 
                                 "", s.est), 
         s.ci = if_else(Variable == "Whole-body vs. lower-body", 
                                 "", s.ci)) %>%
  
  
  dplyr::select(Variable, m.est, m.ci, s.est, s.ci) %>%

  kable(col.names = c("Interacting variable", 
                      "Estimate", 
                      "95% CrI", 
                      "Estimate", 
                      "95% CrI"),
        escape = TRUE,
        booktabs = TRUE, 
        caption = "Interaction between study parameter and weekly number of sets from meta-analyses on muscle mass and strength")  %>%
  kable_styling(font_size = 7) %>%
  add_header_above(c(" " = 1, 
                     "Muscle mass" = 2, 
                     "Muscle strength" = 2)) %>%
  pack_rows("Age", 1, 3) %>%
  pack_rows("Training status", 4, 4) %>%
  pack_rows("Sex", 5, 6)  %>%
  pack_rows("Body portion", 7, 8) %>%
  pack_rows("Measurement technique", 9, 9) %>%
  pack_rows("Length of study", 10, 10)
  


```




## Characteristics of early-phase training-induced ribosome biogenesis



Total RNA seems to be a valid proxy marker of ribosomal density, as most of the RNA is assumed to be ribosomal RNA
[@RN2223],<!-- Young  -->
which in turn is a valid marker of translational capacity
[@RN2145].
Several studies have shown that total RNA content is altered by RT
<!--@RN2266<  Hortobagyi 2000 NO change in RNA per wet weight after immobilization or 12 weeks retraining low N-->
[@RN2225;<!-- Haun 2019 used a progressive volume troughout a six week study study RNA increased during six weeks (12% week 3; 27% week 6 -->
@RN2180;<!-- Haddad 2005 Unloading through limb suspension leads to lowered total RNA levels, resistance training increased --> 
@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN2358; <!-- Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->],
as was also the case in the present data set.
However, the time course of total RNA/rRNA changes in response to RT has so far remained speculative, with no study investigating responses to prolonged interventions with multiple sampling time points.
In the present data, RT led to a clear session-to-session increase in total RNA per unit tissue weight in response to the first four session, whereupon the changes gradually leveled out before peaking after the 8^th^ session, with the peak increase from baseline being $\sim$ 50%, defining an accumulation phase.
This corroborates well with previous suggestions of peak values being reached within four to nine sessions in young males and females
[@RN1809;
@RN2358], and may be essential for preparing muscle fibers for subsequent growth
[@RN1809;<!-- Correlation between increase and muscle growth -->
@RN1644; <!--Figueiredo correlation between incr and muscle hypertrophy -->
@RN2358].<!-- Early accumulation of RNA predicts volume depdent growth -->

After the 8^th^ session, no meaningful increase or decrease were observed for total RNA/rRNA content within the training period, suggesting a plateau phase with attenuated net synthesis of novel ribosomes. Within this last part of the intervention, synthesis of novel rRNA still seemed to be elevated per weight unit muscle tissue compared to baseline, as suggested by sustained elevation of pre-rRNA transcripts, coinciding with peak values of UBF protein levels. This may indicate that during the plateau phase, the ribosomal concentration is balanced by muscle growth [@RN2142].<!-- Figueiredo review details the concept of rna concentration -->

The observed rates of RNA accumulation over the entirety of the intervention were found to be a determinant of changes in muscle thickness (after controlling for average total RNA levels).
Individuals with higher rates of accumulation showed larger accretion of muscle mass.
This supports the notion that ribosomal biogenesis is an important determinant of RT-induced muscle hypertrophy, with previous studies showing that increases in total RNA are positively correlated with increases in muscle mass 
[@RN1644;<!-- Figueriedo  -->
@RN1897;
@RN2055],<!-- Mobely Three groups, no clear group * time in RNA incrinteraction but 32 vs 8 % increase in high vs low responders, although correlation RNA incr with MT incr -->
<!-- Reidy correlation between total RNA and muscle hypertrophy-->
differs between individuals displaying low vs. high levels of muscle hypertrophy in response to RT
[@RN1755]<!-- Stec 2016  -->
an contribute to explain RT volume-dependent changes in muscle mass and strength
[@RN2358].
In addition, supression of ribosomal biogenesis in *in vitro* models leads to halted muscle cellular growth in some 
[@RN1632;<!-- Nader 2005 blockage of mTOR halts ribo bio and growth -->
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1755]<!-- Stec specific pol I inhibition -->
but not all studies
[@RN1929].<!-- Crossland 2017 inhibition of ribo bio did not limoit igf-1 stimulated growth  -->
Conversely, individual variation in fixed amounts of total RNA was not found to determine muscle mass accretion, and higher levels of total RNA was instead associated with a tendency towards lowered muscle growth.
Overall, the rate of increases in ribosomal density thus seems to be a better predictor of individual RT-induced changes in muscle mass than absolute ribosomal density, suggesting that net increases in ribosomal biogenesis may be a core determinant of RT responsiveness.
Interestingly, the interaction between rRNA synthesis rate and muscle mass accretion (but not between ribosomal content and muscle mass accretion) may shed light on observed differences in muscular responses to RT between young and old individuals.
Whereas aged muscle display higher levels of total RNA at rest 
[@RN1656]<!-- Stec 2015 higher levels of RNA/mg in old -->
they show reduced changes in total RNA levels in response to RT
[@RN1809],<!-- Brook 2016 increase in RNA per mg not seen in old, data at baseline collected at different timepoints? -->
potentially explaining their alleged poorer overall hypertrophic responses [@RN1809].
Whether these cellular characteristics are related to e.g. differences in fiber type distributions [@RN1707] remains to be determined.
Together, these results and perspectives emphasizes on the potentially crucial role of RT-induced ribosomal synthesis for adaptations to training, making ribosomal responses to RT an interesting biomarker in relation to manipulation of training loads for specific populations.

<!-- rpS6 dissconect from total RNA levels. -->
In the present study, training induced increases in rRNA and total RNA coincided with increases in rpS6. Changes in total RNA levels and rpS6 in response to de-training did however not correspond as rpS6 protein levels remained elevated after the de-training period. Training induced increases in rpS6 seen in the present study are in agreement to what has previously been reported in young men
[@RN1642],<!-- Brook 2015 rpS6 increase in resposne to RT over control -->
but not in elderly men and women where a decrease was observed in response to training despite increases in total RNA and rRNA
[@RN1755].<!-- Stec 2016 rpS6 decreased 22% after four weeks of training in older  -->
Although increases were seen in both rpS6 and and total RNA, rpS6 did not explain variations in total RNA when number of sessions were controlled for. Together with a disconnect after the de-training period, this suggest that regulations of rpS6 expression and ribosomal RNA transcription displays different temporal characteristics in response to RT. Additionally, ribosomal proteins may have extra-ribosomal functions 
affecting their expression
[@RN2588].<!-- Warner 2009 General description of   -->  

UBF levels robustly explained total RNA levels over the entire course of the intervention. These analyses were done with the number of sessions accounted for, allowing unbiased estimates. Unrealistically strong relationships could have been otherwise expected as both the dependent variable (total RNA) and the covariate (UBF levels) varies with the number of sessions.
From a mechanistic perspective, UBF is an important transcription factor for rDNA transcription as it, in its active state recruits a secondary transcription factor (SL1) to the rDNA promoter and enables transcription by RNA Pol I
[@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
Activation of UBF is controlled by the mechanosensitive mTOR pathway, and rapamycin, a specific mTOR inhibitor, blocks UBF from recruiting SL1 and subsequent rRNA transcription
[@RN1632; <!-- Nader rapamycin block UBF availability  -->
@RN2564].<!-- Hannan 2003 -->
<!-- Hannan 2003 10.1128/MCB.23.23.8862-8877.2003 mTOR is believed to play a role in phosphorylating UBF leading to recruitment of SL-1. The effect of blockage of mTOR through rapamycin is not rescued by hypophosphorylated UBF -->
Evidence from human exercise studies confirms training-induced activation of UBF through phosphorylation
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644].<!-- Figueriedo ubf increase total and active form (ser388) --> 
In addition to exercise-induced activation of UBF, mechanical loading also leads to increased levels of total UBF
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644]. Increases in UBF was determined to be rapamycin insensitive after synergist ablation in mice
[@RN1072]<!-- Goodman 2011 UBF increase due to synergist ablation is rapamycin insensitive https://doi.org/10.1113/jphysiol.2011.218255 --> 
pointing to an effect observed in cell models where c-Myc induces UBF mRNA transcription [@RN2565].<!-- Pootinga 2004 MAD1 and c-Myc controls UBF transcription UBF is required for c-Myc induced rDNA transcription -->
Interestingly the availability of UBF *per se* has been shown to regulate rRNA transcription
[@RN2566]<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->  through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->
Together with our observations, this underlines the importance of UBF as a regulator of RT-induced ribosomal biogenesis.

After eight days of de-training, total RNA and rRNA levels per weight unit muscle tissue returned toward baseline levels, though without concomitant reversal of muscle thickness, which remained at elevate levels.
This was likely caused by attenuated rRNA transcription, a notion that was supported by reversal of pre-rRNA abundances and possibly by lowered UBF protein levels, though this was not confirmed as statistically robust.
This supports the idea that ribosomal biogenesis is a cellular activity on demand, possibly relating to its relative expense [@RN1920] also in muscle tissue.
Based on this notion, and the fact that RT volume is known to be a potent modulator of molecular mechanisms determining protein synthesis and ribosomal biogenesis including induction of c-Myc expression, mTOR activation
[@RN2358; <!-- Hammarstrm volume affects ribo biogenesis and mTOR and c-myc -->
@RN1837; <!-- Ahtianinen 2015 volume affects mTOR differently -->
@RN791]<!-- Burd ex volume protein synthesis and anabolic [rpS6] phospho signalling-->,
subsequent total RNA increases [@RN2358] and post exercise protein synthesis [@RN791]
and subsequent training outcomes [@RN1767; @RN2358], we hypothesized that fluctuations in training volume would be reflected in markers of ribosomal biogenesis.
When comparing VAR to CONST in the present study we found only one part of the pre-rRNA, 45S ETS, to be differentially expressed and only so after Session 12 in favor of VAR together with a tendency towards rescued UBF levels after de-training in response to increased volume in the VAR but not CONST protocol.
These observations do not give support to a clear effect of fluctuations in training volume on total RNA levels or rRNA expression within a relatively short and training-intensive intervention, though it should be noted that the time point with increased 45S ETS expression was preceded by a period of increased training volume, suggesting a potential interaction between time and volume. 
Indeed, both training protocols utilized in the present study increased muscle strength and induced muscle hypertrophy to a similar degree.
From a general perspective, albeit volume is an important determinant of increases in muscle strength and mass
[@RN1767;<!-- schoenfeld 2016 dose response  -->
@RN2492],<!-- Ralston 2017 Weekly sets volume on muscle strength  -->
differences in organization of training loads is likely of minor importance when training volumes are equated over time
[@RN2572].<!-- Grgic 2017 volume equated periodization schemes led to similar muscle hyeprtrophy -->
It is important to note that RT in the current study was performed with the same volume in the first four sessions, something that could have been more than enough to maximize rRNA transcription in previously untrained individuals.
This is supported by the observation that pre-rRNA increased rapidly initially in both protocols with minimal changes in response to subsequent sessions, regardless of exercise volume.
The CONST protocol in the present study corresponded to volumes used in the moderate volume condition in a previous study from our lab (three sets in two exercises activating knee extensor muscles) [@RN2358].
There, higher levels of total RNA were observed after four sessions in the moderate compared to a low volume protocol [@RN2358].
Interestingly, using a progressive volume protocol in well-trained participants, increases in total RNA have been reported throughout six weeks of training
[@RN2225].<!-- Haun total RNA increase througout a 6 week intervention -->
Altough this observation was done in well-trained participants performing a high volume protocol without a control condition with constant volume, compared to constant volume protocols
[@RN1809;<!-- Brook 2016 -->
@RN2358; and the present study],<!-- Hammarstrm 2020 -->
progressive volume may thus increase ribosomal abundance to a higher degree and provide a measure to avoid the plateau phase seen in the present study.

In conclusion, RT-induced ribosome accumulation reached peak values in the initial phase of RT (8 sessions) and was interconnected with increases in UBF protein levels. The rate of total RNA accumulation predicted RT-induced muscle hypertrophy. Fluctuations in training volume did not transfer to fluctuations in ribosomal biogenesis, but training cessation led to decreased ribosomal content.  
