---
output: html_document
editor_options: 
  chunk_output_type: console
---
# Methodological considerations {#methodological-considerations}


## Reliability of micro-biopsy sampling

The micro biopsy technique used in works presented in this thesis generally produces smaller samples compared to other
biopsy techniques
[@RN2549]<!--Ekblom describes the Bergström and conchotome -->, and thus
requires several passes to produce sufficient material for multiple
downstream experiments. However, reports confirms that the micro-biopsy
technique is comparable to the traditionally used Bergström technique in
several measures of muscle characteristics at the same time as being
well tolerated [@RN824; @RN2553].
Any reported differences in fiber type distributions between sampling techniques have been suggested to be related
to differences in sampling depth [@RN2553; @RN2552].<!-- Hayot -->
Thus, reproducibility of any biopsy technique may relate to its ability to obtain sufficient material at similar depths between samples.

```{r immuno-data-prep, echo=FALSE, message=FALSE, warning=FALSE}

source("./R/libraries.R")
source("./R/themes.R")


# Sample set up (included samples)
legs <- read_csv2("./data/study-1/body-composition/oneThreeSetLeg.csv") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


# Total number of fibers per sample
low_fiber_counts <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total = type1 + type2a + type2ax + type2x) %>%
        mutate(low_count = if_else(total < 200, "low", "acceptable")) %>%
        group_by(low_count) %>%
        summarise(n = n()) %>%
        mutate(percentage = round((n / sum(n)) * 100, 1)) 

 
average_n_fibers <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total = type1 + type2a + type2ax + type2x) %>%
        summarise(n_avg = mean(total, na.rm = TRUE)) %>%
        pull(n_avg)




between_legs <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
    inner_join(legs) %>%
    filter(include == "incl") %>%
    mutate(total  = type1 + type2a + type2ax + type2x, 
           
           typeI = (type1 / total)*100,
           typeII = ((type2a + type2ax +type2x)/total) *100,
           
           type1  =(type1 / total)*100, 
           type2a =(type2a / total)*100, 
           type2ax=(type2ax / total)*100, 
           type2x =(type2x / total)*100 ) %>% 
    
    filter(total > 300) %>%
    
    filter(timepoint %in% c(1)) %>%
    dplyr::select(subject, timepoint, leg,total, type1:type2x, typeI:typeII) %>%
    pivot_longer(names_to = "type", values_to = "percentage", cols = c(type1:type2x, typeI:typeII)) %>%
    mutate(timepoint = paste0("t", timepoint)) %>%
    pivot_wider(names_from = leg, values_from = c(percentage, total)) %>%
    
    mutate(type = factor(type, levels = c("type1", 
                                          "type2a", 
                                          "type2ax", 
                                          "type2x", 
                                          "typeI", 
                                          "typeII"), 
                         labels = c("Type I", 
                                    "Type IIA", 
                                    "Type IIAX", 
                                    "Type IIX", 
                                    "Type I_tot", 
                                    "Type II_tot"))) %>%
     filter(complete.cases(.)) %>%
        rowwise() %>%
        mutate(mean.perc = mean(c(percentage_L, percentage_R)), 
               diff.perc = percentage_L - percentage_R, 
               diff.sq = (percentage_L - percentage_R)^2,
               diff.mean.sq = (diff.perc/mean.perc)^2,
               min.count = min(c(total_L, total_R))) %>%
        filter(mean.perc > 0) %>%
        
        group_by(type) %>%
        
        summarise(m = mean(mean.perc), 
                  S2 = sum(diff.sq)/(2*n()), 
                  S = sqrt(S2), 
                  RMS = 100 * sqrt(sum(diff.mean.sq) / (2*n())), 
                  ws = 100 * (sqrt(sum(diff.perc^2) / (2*n()) ))/m, 
                  n = n()) %>%
        mutate(cv  = 100 * (S/m)) 
     


 
between_duplicates <-  read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
         inner_join(legs) %>%
         filter(include == "incl") %>%
         mutate(total  = type1 + type2a + type2ax + type2x, 
                type1  =(type1 / total)*100, 
                type2a =(type2a / total)*100, 
                type2ax=(type2ax / total)*100, 
                type2x =(type2x / total)*100) %>%  
     
    filter(total > 300) %>%
         filter(timepoint %in% c(2, 3)) %>%
         dplyr::select(subject, timepoint, leg,total, type1:type2x) %>%
         pivot_longer(names_to = "type", values_to = "percentage", cols = type1:type2x) %>%
         mutate(timepoint = paste0("t", timepoint)) %>%
         pivot_wider(names_from = timepoint, values_from = c(percentage, total)) %>%
         
 
         mutate(type = factor(type, levels = c("type1", 
                                               "type2a", 
                                               "type2ax", 
                                               "type2x"), 
                              labels = c("Type I", 
                                         "Type IIA", 
                                         "Type IIAX", 
                                         "Type IIX"))) %>%
     filter(complete.cases(.)) %>%
         
         rowwise() %>%
         mutate(mean.perc = mean(c(percentage_t2, percentage_t3)), 
                diff.perc = percentage_t2 - percentage_t3, 
                diff.sq = (percentage_t2 - percentage_t3)^2,
                diff.mean.sq = (diff.perc/mean.perc)^2,
                min.count = min(c(total_t2, total_t3))) %>%
         filter(mean.perc > 0) %>%  
     group_by(type) %>%
     
     summarise(m = mean(mean.perc), 
               S2 = sum(diff.sq)/(2*n()), 
               S = sqrt(S2), 
               RMS = 100 * sqrt(sum(diff.mean.sq) / (2*n())), 
               ws = 100 * (sqrt(sum(diff.perc^2) / (2*n()) ))/m, 
               n = n()) %>%
     mutate(cv  = 100 * (S/m)) 






```

In Study I, one or several pieces of muscle (total weight
$\sim$\SI{15}{mg}) were chosen per sampling for analysis of fiber type
composition. The average total number of counted fibers per samples was `r round(average_n_fibers, 0)`.
Only a small percentage (`r low_fiber_counts[2,3]`\%) of samples did not contain enough fibers for representative analysis of fiber type composition (> 200 fibers)
[@RN874]<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
and as many as 1400 fibers were counted in some specimens (\@ref(fig:fiber-methods-fig)a).
The smaller amount of tissue obtained using the micro-biopsy technique did thus not limit analyses of fiber type composition.

In order to asses the reproducibility of the technique, samples obtained during the same day (separated by $\sim$ 2 hours) were analyzed as duplicate samples. 
Similar to Blomstrand *et al.* [@RN874], standard deviations from duplicate measures were calculated as 

$$SD = \sqrt{\frac{\sum{d^2}}{2n}}$$

where $d$ is the difference between paired observations and $n$ is the number of pairs.
The variation (SD) between duplicate samples were `r round(between_duplicates[1,4],2)` and `r round(between_duplicates[2,4],2)` for Type I and IIA fibers, respectively (\@ref(fig:fiber-methods-fig)b).
This did only marginally differ from between leg variations (Type I, `r round(between_legs[1,4],2)`; Type IIA, `r round(between_legs[2,4],2)`; Figure \@ref(fig:fiber-methods-fig)c).
These estimates are slightly lower than those reported by Blomstrand *et al.*, using the Bergström needle
[@RN874].<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
This suggests that the micro-biopsy technique is reproducible in terms fiber type composition.



```{r fiber-methods-fig, fig.cap="Number of fibres in immunohistochemistry analyses (a), correlation between duplicate samples from the same leg (separated by 2 hours) (b) and correlations between samples from left and right leg pre-intervention.", fig.scap="Characteristics of biopsy samples used in immunohistochemistry analyses.", warning=FALSE, echo = FALSE, warning=FALSE, fig.width = 3.5, fig.height = 6, fig.align='center'}



 ############## Figures myhc fiber typing  ######################      
total_histogram <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total = type1 + type2a + type2ax + type2x) %>%        
        
        ggplot(aes(total)) + geom_histogram(binwidth = 30, 
                                            fill = group.study.color[5], alpha = 0.8) + 
        dissertation_theme() + 
        scale_x_continuous(limits = c(0, 1600),
                           expand = c(0,0),
                           breaks = c(0, 200, 400, 600, 800, 1000, 1200, 1400, 1600), 
                           labels = c(0, "", 400, "", 800, "", 1200, "", 1600)) + 
        scale_y_continuous(limits = c(0, 20), 
                           breaks = c(0, 5, 10, 15, 20), 
                           expand = c(0,0)) +
        
        labs(x = "Total number of fibers per sample", 
             y = "Counts") + 
        theme(axis.text.x = element_text(hjust = c(0, rep(0.5, 7), 1)))
        
### Comparison between "duplicate samples" at week 2
corr_plot_fiber_types <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total  = type1 + type2a + type2ax + type2x, 
               type1  =(type1 / total)*100, 
               type2a =(type2a / total)*100, 
               type2ax=(type2ax / total)*100, 
               type2x =(type2x / total)*100) %>%  
        filter(timepoint %in% c(2, 3)) %>%

        dplyr::select(subject, timepoint, leg,total, type1:type2a) %>%
        pivot_longer(names_to = "type", values_to = "percentage", cols = type1:type2a) %>%
        mutate(timepoint = paste0("t", timepoint)) %>%
        pivot_wider(names_from = timepoint, values_from = c(percentage, total)) %>%
        ungroup() %>%
        mutate(type = factor(type, levels = c("type1", 
                                              "type2a"), 
                             labels = c("Type I", 
                                        "Type IIA"))) %>%
     
        ggplot(aes(percentage_t2, percentage_t3)) + 
        geom_point(shape = 21,
                   size = 1.8,
                   alpha = 0.7,
                   fill = group.study.color[4]) + 
        
        facet_wrap(~type, ncol = 2, strip.position = "top")  +
        
        scale_x_continuous(limits = c(0, 100), 
                           expand = c(0,0), 
                           breaks = c(25, 50 ,75)) + 
        scale_y_continuous(limits = c(0, 100), 
                           expand = c(0,0), 
                           breaks = c(25, 50 ,75)) + 
        
        labs(x = "Duplicate 1", 
             y = "Duplicate 2") +
        
        geom_abline(intercept = 0, slope = 1, lty = 2, color = "gray50") + 
        dissertation_theme() + 
        theme(strip.background = element_blank(),
              strip.text = element_text(size = 8, hjust = 0),
              panel.border = element_rect(color = "black", fill = NA))
        
### Left right at baseline 

baseline_between_legs <- read_excel("./data/study-1/fiber-type/fibertype.xlsx") %>%
        inner_join(legs) %>%
        filter(include == "incl") %>%
        mutate(total  = type1 + type2a + type2ax + type2x, 
               type1  =(type1 / total)*100, 
               type2a =(type2a / total)*100, 
               type2ax=(type2ax / total)*100, 
               type2x =(type2x / total)*100) %>%  
        filter(timepoint %in% c(1)) %>%

        dplyr::select(subject, timepoint, leg,total, type1:type2a) %>%
        pivot_longer(names_to = "type", values_to = "percentage", cols = type1:type2a) %>%
        pivot_wider(names_from = leg, values_from = c(percentage, total)) %>%
        ungroup() %>%
        mutate(type = factor(type, levels = c("type1", 
                                              "type2a"), 
                             labels = c("Type I", 
                                        "Type IIA"))) %>%
     
        ggplot(aes(percentage_L, percentage_R)) + 
        geom_point(shape = 21,
                   size = 1.8,
                   alpha = 0.7,
                   fill = group.study.color[2]) + 
        
        facet_wrap(~type, ncol = 2, strip.position = "top")  +
        
        scale_x_continuous(limits = c(0, 100), 
                           expand = c(0,0), 
                           breaks = c(25, 50 ,75)) + 
        scale_y_continuous(limits = c(0, 100), 
                           expand = c(0,0), 
                           breaks = c(25, 50 ,75)) + 
        
        labs(x = "Left leg", 
             y = "Right leg") +
        
        geom_abline(intercept = 0, slope = 1, lty = 2, color = "gray50") + 
        dissertation_theme() + 
        theme(strip.background = element_blank(),
              strip.text = element_text(size = 8, hjust = 0),
              panel.border = element_rect(color = "black", fill = NA))


fiber_type_rel_plot <- plot_grid( plot_grid(NULL, total_histogram, NULL, ncol = 3, rel_widths = c(0.2, 1, 0.2)),
          corr_plot_fiber_types, 
          baseline_between_legs,
          ncol = 1, rel_heights = c(1.5, 2, 2)) + 
        
        draw_plot_label(label=c("a", "b", "c"),
                        x = c(0.02, 0.02, 0.02), 
                        y = c(0.98, 0.75, 0.35),
                        hjust=.5, vjust=.5, 
                        size = label.size)

fiber_type_rel_plot
```




## Feasibility of model-based normalization of RT-qPCR-based gene-expression data in a case of human muscle under hypertrophic stress

Quantitative reverse-transcription real-time polymerase chain-reaction (RT-qPCR) is the method of choice for conducting targeted gene-expression studies.
For accurate interpretation of such data the selection of normalization strategy needs to account for biological as well as non-biological variation.
Whereas analysis using endogenous control genes is the most commonly adopted approach, model-based strategies using estimates of technical variation from multiple genes could offer increased sensitivity and accuracy.  
Here, we compare the feasibility gene-specific and model-based normalization strategies for analyzing the effect of two resistance-training programs on abundance of IGF-1 and myostatin mRNA in humans. As for gene-specific normalization, neither of the eleven commonly utilized endogenous control genes investigated exhibited stability suggesting that the use of endogenous internal controls for normalization may affect interpretation. In contrast, using mixed-effects models utilized to estimate technical variation between samples without relying on endogenous controls proved to be feasible. 
Simulations further showed that model-based strategies were computationally robust and had higher power than traditional analysis.


## Increased relevance of RNA-seq data through data-driven selection of 

Analysis of high-dimensional data such as those retrieved from RNA sequencing experiments requires the use of a multitude bioinformatic and statistical tools. Such tools are continuously being developed and optimized for different tasks. This continuing development requires that such tools are validated for specific study conditions [@RN2426]<!-- Conesa 2016 pipelines need to be adopted to study specific conditions-->
To improve interpretability of the RNA sequancing data obtained from Study I we sought to compare mapping tools for their validity. As a first step, we first utilized myosin heavy-chain compositions determined by immunohistochemistry to relate mRNA levels to protein. Myosin heavy-chain compositions are well suited for this purpose as their mRNA and protein levels are known to correlate
[@RN2444; <!-- Serrano Correlation between mRNA and prot abundance MyHC -->
@RN1489; <!-- Ellefsen Reliable determination of myosin heavy chain -->
@RN2445].<!-- Marx 2002 correlations were observed between the isoform expression of different MyHCs at the protein and mRNA-->
To avoid concerns with normalization assumptions, both mRNA and protein abundances were expressed relative to the sum of counts for the whole gene- and protein-family 
[@RN825; <!--Ellefsen Gene family profiling  -->
@RN1489]. This analysis indicated that transcript-based mapping tools
(RSEM [@RN2387]<!-- Li Rsem-->,
kallisto [@RN2389]<!-- Bray kallisto --> and Salmon [@RN2390])
resulted in stronger correlations between mRNA and protein profiles than genome-based mapping tools
(STAR [@RN2386]<!-- Dobin STAR -->
and  HISAT2 [@RN2385]; Figure \@ref(fig:rna-seq-myhc-validation)).
A second analysis utilized the within-participant design of the study. With the assumption that paired observations between legs, at rest, would result in similar expression profiles if read mapping were more accurate, we determined the pairwise Log\textsubscript{2}-differences of a set of genes with known robust expression across tissues [@RN1759].<!-- Eisenberg housekeeping -->
An initial observation indicated that genom-based mapping tools 


They were also associated with the less technical variation, evident as lower Log2-fold differences in expression for a selection of highly expressed genes between the two legs at baseline [@RN2403]<!-- Tarnopolsky 2007 -->, assuming minimal biological variation between such paired samples. In these analyses, RSEM displayed slightly lower average variation between paired samples, thus outperforming kallisto and Salmon (Figure 1J).


To this end, we first sought to select a suitable read-trimming method. This is necessary to provide high-quality downstream alignment and k-mer search in reads [@RN2441].<!-- Del Fabbro commonly used tools include trimmomatic and cutadapt(trimgalore) -->
Two commonly used algorithms were compared [@RN2382; <!-- Bolger Trimmomatic --->
@RN2441]. <!-- Del Fabbro commonly used tools include trimmomatic and cutadapt(trimgalore) --> Trimmomatic provided data with higher quality than Trimgalore (Figure 1G), which did not improve quality scores compared to non-filtered data. Second, we compared five mapping tools for performing transcript quantification of trimmed reads: two genome-based mapping tools
(STAR [@RN2386]<!-- Dobin STAR -->
and  HISAT2 [@RN2385])<!-- Kim HIsat -->
and three transcript based mapping tools (RSEM [@RN2387]<!-- Li Rsem-->,
kallisto [@RN2389]<!-- Bray kallisto --> and Salmon [@RN2390])<!-- Patro Salmon -->.
Transcript-based mapping tools resulted in stronger correlations between mRNA and protein profiles, measured as relationships between myosin heavy chain mRNA profiles and protein abundances in rested state biopsies [@RN2358] (Figure 2A),<!-- Hammarström et al. -->
which is known to correlate in resting human skeletal muscle 
This comparison was performed using gene-/protein-family normalization
[@RN825; <!--Ellefsen Gene family profiling  -->
@RN1489],
allowing the deduction of mRNA-to-protein relationships without the need for other normalization assumptions. Notably, a marked skew was observed in the relationship between *MYH1* proportions and its corresponding Type IIX fiber following the initial part of the training intervention. This coincided with robust changes in *MYH1* expression, as is typically seen in response to mechanical loading in such short time frames [@RN2056]<!-- Andersen Rapid switch off MyHCIIX-->.
Furthermore, the transcript-based mapping tools (RSEM, kallisto, and Salmon) resulted in transcriptome profiles with an expected bimodal distribution of counts and a larger subset of detected genes compared to genome-based tools (Figure 1H). 


```{r rna-seq-myhc-validation, fig.cap="Validation of gene quantification using mRNA to protein correlations of myosin heavy-chain genes", fig.scap="Correlations between gene-family normalized protein and gene data from different mRNA quantification methods.", warning=FALSE, echo = FALSE, warning=FALSE, fig.width = 4.72, fig.height=7, fig.align='center'}

myhc_rnaseq_fig <- readRDS("./data/derivedData/study1-myhc-rnaseq-validation/myhc_rnaseq_fig.RDS")

myhc_rnaseq_fig

```




```{r rna-seq-pairwise-validation, fig.cap="Variation pairwise", fig.scap="Within-participant variation between quantification methods.", warning=FALSE, echo = FALSE, warning=FALSE, fig.width = 4.72, fig.height=2.5, fig.align='center'}

rna_seq_validation_pairwise <- readRDS("./data/derivedData/study1-pairwise-variation-rnaseq/pairwise_fig.RDS")


rna_seq_validation_pairwise


```










