---
author: 'Daniel Hammarström'
date: 'May 20xx'
year: '2019'
isbn: 'Provided by the library'
place: 'Stockholm'
printedby: 'Printer service, Stockholm, 2019'
coverinfo: ' '
sernr: '999' # Add number in "Avhandlingsserie" 
title: 'Determinants of intra-individual variation in adaptability to resistance training of different volumes with special reference to skeletal muscle phenotypes'
subtitle: 'Subtitle of my thesis' # currently not in use ##
knit: "bookdown::render_book"
site: bookdown::bookdown_site
output: 
  thesisdowngih::thesis_pdf: default
  latex_engine: xelatex
#  thesisdown::thesis_gitbook: default
#  thesisdown::thesis_word: default # The word and e-pub versions are not tested.
#  thesisdown::thesis_epub: default
# If you are creating a PDF you'll need to write your preliminary content here or
# use code similar to line 20 for the files.  If you are producing in a different
# format than PDF, you can delete or ignore lines 20-31 in this YAML header.
abstract: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-abstract.Rmd"), collapse = '\n  ')`
# If you'd rather include the preliminary content in files instead of inline
# like below, use a command like that for the abstract above.  Note that a tab is 
# needed on the line after the |.
listofpapers: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-listofpapers.Rmd"), collapse = '\n  ')`
dedication: |
  You can have a dedication here if you wish. 
# preface: | # If you wish to add a preface, remove comments on these lines
#   This is an example of a thesis setup to use the reed thesis document class
#   (for LaTeX) and the R bookdown package, in general.
defence: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-defence.Rmd"), collapse = '\n  ')`
bibliography: 
        - bib/thesis.bib
        - bib/complete-bib.bib
# Download your specific bibliography database file and refer to it in the line above.
csl: csl/vancouver.csl
# Download your specific csl file and refer to it in the line above.
lot: true
lof: true
#space_between_paragraphs: true
# Delete the # at the beginning of the previous line if you'd like
# to have a blank new line between each paragraph
header-includes:
#- \usepackage[utf8]{inputenc}
#- \usepackage{erewhon}
#- \usepackage{lipsum}
#  \usepackage[utf8]{inputenc}
#  \usepackage[T1]{fontenc}
#  \usepackage{libertine}
#  \usepackage[svgnames]
#- \usepackage{GoudyIn}
#- \usepackage{xcolor}
  \usepackage{lettrine}
  \usepackage{booktabs}
  \usepackage{longtable}
  \usepackage{array}
  \usepackage{multirow}
  \usepackage{wrapfig}
  \usepackage{float}
  \usepackage{colortbl}
  \usepackage{pdflscape}
  \usepackage{tabu}
  \usepackage{threeparttable}
  \usepackage{threeparttablex}
  \usepackage[normalem]{ulem}
  \usepackage{makecell}
  \usepackage{siunitx}

---


<!--
Above is the YAML (YAML Ain't Markup Language) header that includes a lot of metadata used to produce the document.  Be careful with spacing in this header!

If you'd prefer to not include a Dedication, for example, simply delete lines 27 and 28 above or add a # before them to comment them out.  If you have other LaTeX packages you would like to include, delete the # before header-includes and list the packages after hyphens on new lines.

If you'd like to include a comment that won't be produced in your resulting file enclose it in a block like this.
-->


```{r include_packages, include = FALSE}
# This chunk ensures that the thesisdown package is
# installed and loaded. This thesisdown package includes
# the template files for the thesis.
library(devtools)
library(thesisdowngih)
```


```{r create_package_bib, include = FALSE}
knitr::write_bib(c("base", "qpcR"), "./bib/packages.bib") ### This creates a bibfile for R packages
```

\setcounter{DefaultLines}{3}


# Introduction
\lettrine{S}keletal muscle health is essential for physical independence. In a lifespan perspective, measures of muscle mass and/or strength are inversely associated with mortality
[@RN2512;<!-- Ran 2018 Low muscle strength was independently associated with all cause mortality -->
@RN2513; <!-- Fukasawa 2016 81 hemodialysis patients, low thigh muscle mass associated with all cause and cv mortality-->
@RN2514; <!-- Miyake 2019 Low muscle mass associated with ACM in type 2 diabetes, SMI = appendicular skeletal muscle mass kg / height m^2 -->
@RN2376;<!-- Ruiz n ~ 8000, strength inversely related to all cause and cancer  mortality in men -->
@RN2515; <!-- Szulc 2010 715 men Loss of appendicular muscle mass predicts all cause mortality in elderly -->
@RN2516]<!-- Abramowitz 2018 Higher ASMI was independently associated with lower mortality -->
and disability
[@RN2517].<!-- Janssen 2002 Low muscle mass associated with disabilities -->
Besides adverse associations between of low muscle mass and strength and clinical conditions, muscle weakness also accounts for increased health care costs in patient populations
[@RN2532;<!-- Sousa 2016 --> 
@RN2184].<!-- Pinedo-Villanuev Excess health care costs associated with muscle weakness was estimated to 1.3 billion punds in UK in population above 70 yrs   -->
The intercept between muscle mass, muscle function and health status is interrelated with variables such as age and primary illness or injury
[@RN763].<!-- Wolfe 2006 amino acid availability in acute illness and injury, link between metabolic functioning of mm.-->
This highlights that interventions designed to increase muscle mass and strength are likely to prevent adverse health outcomes across the lifespan. A higher level of muscle mass and functional capacity would counteract the effects of muscle loss due to illness, age or inactivity.

Although a large degree of the observed variations in lean mass and strength are attributed to genetic components 
[@RN2526; <!-- Arden genetic factors explain leg extensor strength (60%), lean body mass (52%) and grip strength (36%) -->
@RN2527],<!-- Roth review, > 50% genetic component in muscle mass -->
environmental factors also contribute, leaving a window of opportunity to increase muscle mass and functional capacity. Among factors affecting muscle mass and functioning are nutrition and pharmacological agents. However, physical activity and specifically systematic resistance training of sufficient volume, intensity and frequency provides a stimulus that promote morphological and functional changes to the human neuromuscular system without adverse side-effects. Irrespective of age, resistance training generally leads to increased muscle mass and strength
[@RN1741;<!-- Ahtiainen 2016 pooled analysis including controls   -->
@RN2534]<!-- Grgic 2020 > 75 of age increase strength and muscle mass relative to control and training is safe -->
and is considered safe when performed in a well organized manner
[@RN2534;<!-- Grgic safety in elderly -->
@RN2536].<!-- Faigenbaum 2010 resistance training in young safety -->

Resistance training can be modulated indefinitely through combined variations of training variables such as frequency, intensity and volume 
[@RN1;<!-- ACSM guidelines -->
@RN798].<!-- Bird 2005 acute programming variables -->
Well designed training prescriptions should incorporate information about the current state and goals of the trainee to maximize the potential outcome of the training program
[@RN1;<!-- ACSM guidelines -->
@RN2538;<!-- Feigenbaum consider personal status in programming-->
@RN798].<!-- Bird 2005 acute programming variables -->
Training volume has received particular attention in the scientific community for many reasons. Evidence suggests that exercise volume affects selected molecular determinants of muscle hypertrophy in a dose-dependent manner
[@RN791; <!-- Burd  -->
@RN784; <!-- Terzis 2010 training volume p70 and s6 volume dependent -->
@RN1837].<!-- Ahtiainen 2015 10 X 10 > 5 X 10 in p70, S6 phospho --> 
Such effects are believed to facilitate long-term training effects as training programs with higher volume generally result in higher gains in muscle mass and strength with little evidence of differences between age groups or participants with different training backgrounds
[@RN793; <!-- Krieger 2009 meta on strength, 2-3 set more efficient than 1 set but not 4-6 sets -->
@RN789; <!-- Krieger 2010 meta analysis on training volume for hypertrophy, multiple sets more effective than signle set -->
@RN1767]. <!-- Shoenfeld 2016 weekly number of sets related to gains in muscle mass  -->  
A consequence of a more extensive training program is the increased time required to complete such a program. As time constraints has been reported as a limiting factor for engaging in physical activity
[@RN2063]<!-- Time constraints limits participation in physical activity -->
some merit can be given to arguments against guidlines suggesting higher volume in resistance training prescription 
[@RN794;<!-- Carpinelli argues that single set programs are as effective as multiple set programs -->
@RN2538].<!-- Feigenbaum, argues that more volume does not outweigh the time cost -->
From an individual perspective, training prescription that balances time-requirement with efficacy presumably increases the likelihood of participation in physical activity [@RN2063]<!-- Choi time-constrains and expectations of outcomes are related to participation -->.
From a more general perspective, increased knowledge about mechanisms governing responses to physical training could improve training prescription also for individuals and populations that experience attenuated benefit of resistance training
[@RN2547]. <!-- Pickering 2019, non-responders might not exists, changing training prescription could increase efficacy -->
<!-- Add "Individualized prescrioption is needed -->
The overreaching goal of the present thesis is to contribute to understanding individualized training loads. To this end, training volume was used to study the effects of variable training<!-- Change this? --> stimulus in within-participant models of exercise-training.   





<!--chapter:end:index.Rmd-->


# Background
```{r Backgroundsettings, echo = FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R"); source("./R/themes.R")
```

## Resistance-exercise prescription, influences and challenges
Recommendations of systematic physical exercise for the purpose of improving health or physical performance has long been part of human culture, evident from records dating back to ancient Chinese, Indian and Greek civilizations
[@RN2640].<!-- Tipton Ancient China India and greek/roman written records on exercise precription -->
Today's exercise-training prescription still bear traces of ideas from these eras, further developed during the renaissance and formalized in systems like German Turnen and Ling gymnastics during the nineteenth century 
[@RN2663].<!-- Pfister 2003 describes differences and common ancestry of gymnastics and sports during 1700-1800 -->
German Turnen as a system of physical activities was established in a time when Germany developed from aristocracy to a unified nation.
Turnen not only served as a system of preparing men to fight for the developing nation but also as a way of establishing a national identity.
Ling gymnastics shared common origins with German Turnen and also served as a system of military preparation.
However, Ling also established systems for medical, pedagogical and aesthetic gymnastics.
Especially Lings medical gymnastics was important for the development of modern exercise prescription as it was scientifically oriented, based on the physiological and medical understanding of that time [@RN2663].
The medical gymnastics of the nineteenth century is referenced in twentieth century texts on therapeutic exercise prescription
[@RN2634].<!-- Nicoll 1943 Dosage of exercise therapy "never exercised against more than 25% of their all out efficiency" with increased exercise periods and decreased rest periods as progression -->

With the introduction of "heavy resistance exercises" for the development of muscle strength and mass after injury, DeLorme outlined a system on which modern resistance-exercise prescription is based [@RN2633].<!-- DeLorme 1945 case reports of systematic resistance training for restoration of muscle power and mass. First mention of RM -->
DeLorme published his system short after the Second World War [@RN2633]
during which he, as a newly graduated physician, had been working with war injury rehabilitation
[@RN2639]<!-- Todd 2012 DeLorme biography -->.
Inspired by practitioners of weight training [@RN2639],<!-- Todd 2012 DeLorme biography --> 
DeLorme specifically emphasized high-resistance, low-repetition exercise where progression was achieved with increased resistance [@RN2633] as opposed to previous recommendations of endurance-like exercise where progression was achieved through increased number of repetitions
[@RN2634].<!-- Nicoll 1943 -->
DeLorme originally used the term "heavy resistance exercises" in contrasts to low-resistance exercises [@RN2633], but as this could be perceived as exercises performed only with heavy weights, the system was renamed *progressive resistance exercise* to better reflect the method
[@RN2641].<!-- DeLorme and Watkins 1948 -->
Indeed, central to the system was the concept of repetition maximum [@RN2633].
Repetition maximum refers to the external resistance that can be overcome with a given number of repetitions.
By adjusting external resistance to each individuals progression over the course of a training program, exercises are both individualized and progress is monitored [@RN2633].
DeLorme originally prescribed sessions of up to 100 repetitions performed in sets of 10 repetitions [@RN2633] but later revised this recommendation to three sets of 10 repetitions performed with increasing intensities
[@RN2641]. <!-- DeLorme 1948 Technics of progressive resistance exercise -->

Scientific inquiries into prescription of resistance training from the first part of the twentieth century concerned its therapeutic use
[e.g. @RN2633;<!-- DeLorme case report -->
@RN2646]<!-- DeLorme controlled trial of RT vs. traditional conservative (low-resistance) treatment -->
but was also evaluated in the context of improving strength and physical performance in healthy populations
[e.g. @RN2632;<!-- 1946 Houtz 16 untrained females increased strength over 4 weeks follwing DeLormes system -->
@RN2644; <!-- Chui 1950 Effect of strength training vs. control on physical performance in college stud -->
@RN2642]. <!-- Capen 1950 Effect of strength training in college stud.   -->
Scientific contributions soon moved from questions regarding the effectiveness of resistance training *per se* to comparing outcomes from different modes of resistance training 
[@RN2645;<!-- Hettinger 1953 comparing different loading patterns for isometric RT--> 
@RN1477;<!-- Capen 1956 Four different variations on the DeLorme system -->
@RN1472;<!-- Berger 1962 effect of varied weight training program on strength -->
@RN2659;<!-- Berger 1965 Increased %of 1RM increases strength gains -->
@RN2658;<!-- Berger 1967 Supra-maximal efforts (max of each rep) more efficient than 10RM loads   -->
@RN2656].<!-- O'Shea 1966 3 sets of 10, 5 or 3 reps on muscle strength and girth squat -->
A vocabulary for progressive resistance exercise-training developed through these investigations and parallel practice,
the introduction of repetition maximum by DeLorme being one example. These concepts established as modern definitions of exercise variables enabling precise prescription of training loads for a variety of populations and training goals
[@RN2537<!-- Fleck & Kraemer 2014 Textbook ch 5 : based on a needs analysis acute variables are selected   -->
@RN798].<!-- Bird 2005: Acute training variables leads to different training outcomes -->

Although this development started after the Second World War, resistance training was not part of general exercise guidelines until much later.
The American College of Sports Medicine (ACSM) position statement on exercise for healthy individuals from 1978 primarily concerned physical fitness in terms of cardio-respiratory fitness
[@RN2655]<!-- ACSM 1978 guidelines -->.
Since the updated 1990 ACSM statement, resistance training is recommended to be included as part of a sensible, general training program
[@RN2654].<!-- ACSM 1990 guidelines --> 
The introduction of resistance-training as part of the ACSM recommendation also coincides with specific recommendations on resistance training being part of other consensus statements
[@RN2666, Ch. 2].<!-- Physical Activity and Health: A Report of the Surgeon General provides an overview of recommendation guidelines evolution  -->
Consequently, informed by epidemiological data, the most recent general guidelines for physical activity do include resistance training [@RN2667].<!-- WHO guidelines -->

It could be argued that the above account reflects the fact that common understandings of *why* and *how to* exercise are influenced by societal norms and historic events such as the search of national identity in the nineteenth century or war injuries in the twentieth century
[@RN2663;
@RN2639].
In attempting to outline contemporary influences on exercise prescription, one could argue that the development of techniques to collect large amount of biological data is one such influence. The continuously decreasing cost of a sequenced human genome
[@RN2696] can serve as an example of this development.<!-- Wetterstrand webpage genome sequence costs -->
Such molecular techniques has enabled efforts to describe mechanisms by which exercise training induce favorable adaptations. The newly established Molecular Transducers of Physical Activity Consortium is an example of a large scale effort, explicitly initiated to develop personalized exercise recommendations and identify molecular targets through which effects of exercised may be mimicked
[@RN2678].<!-- MToPAC explicitly stating diagnostics is a goal --> 
Advances in bio-medical technologies are enablers of this enterprise and the quest to *individualize* exercise based on molecular diagnostics can be seen as a motivation for modern exercise science
[@RN2678;<!-- MToPAC explicitly stating diagnostics is a goal -->
@RN2677].<!-- Sparks 2017, exercise response heterogenity, goal to test appropriate exercise dosa based on e.g. DNA -->
Scientific research into contemporary exercise prescription can thus be understood as a part of the era of *personalized medicine*. 

A challenge facing this program is to accurately describe etiologies of response heterogeneity associated with physical training. A wide variation of individual responses are commonly observed after standardized resistance-training programs where measures of muscle strength-changes varies from -32 to +250% and muscle size-changes varies from negative to (-11%) to impressively large (+59%)
[@RN764;<!-- Hubal wide range of responses to standardized exercise  -->
@RN1741].<!-- Ahtiainen 2016 7% (strength) and 29% (muscle size) categorized as non responders (accounting for controls)  -->
By relating such variations to individuals genome (DNA)
[@RN1263],<!-- Pescatello review of the famuss data set -->
and messenger RNA (mRNA) profiles
[@RN826;<!-- Thalacker-Mercer, Cluster analysis, non-responders pro-inflammatory mRNA profile at baseline -->
@RN2684]<!-- Stokes co-relation between mRNA profiles and LMM changes  -->
we are beginning to gain knowledge about the genetic influence on training responses.
In such studies, a common strategy has been to dichotomize responses into "responders" and "non-responders" to exercise training. From a public health perspective this is probably fruitful when *non-response* is defined as the absence of meaningful health-related adaptations, or even adverse effects in response to a given training regime
[@RN2677;<!-- Sparks 2017 response heterogeneity -->
@RN2698].<!-- Roth viewpoint 3 scenarios for genomic screening -->
The existence of non-responders would have large implications regarding exercise prescription on the population scale
[@RN758],<!-- Timmons 2011  -->
and for any given individual as it, if properly diagnosed, would guide clinical decision-making.

<!-- Although "molecular exercise prescription" as a mean of eliminating adverse responses to a given training regime shows potential for future applications
[@RN2698; <!-- Roth viewpoint 3 scenarios for genomic screening 
@RN2547],
the rise of data intensive techniques has also highlighted its complexity.

In simplistic terms, genetic information is transferred from DNA to RNA through transcription and from RNA to protein through translation.
This flow of information, indicates a deterministic role of DNA in controlling the characteristics of an organism. It is however apparent that the transcription of specific genes are under tight control of several mechanisms including a group of non-genetic modifications to the DNA organization of each cell, known as epigenetic modifications
[@RN2690;<!-- Seaborn sharples review 2020 exercise metabolism as a determinant for epigenetic changes 
@RN2691].<!-- McGee Hargreaves 2019 review, epigentics/metabolism/exercise and mention intergenerational trandmission thorugh epigentic modification 
Such control determines gene expression in response to progressive resistance training with modifications persisting after a period of de-training
[@RN2683;<!-- Transcriptome is associated with epigentic changes in response to RE 
@RN2012].<!-- Seaborn 2018 epigenetic changes persist after de-training, epigenetic memory of training 
Regulation of gene expression is thus related to modifiable variables such as exercise and diet
[@RN2690;<!-- Seaborn sharples review 2020 exercise metabolism as a determinant for epigenetic changes 
@RN2691;<!-- McGee Hargreaves 2019 review, epigentics/metabolism/exercise and mention intergenerational trandmission thorugh epigentic modification -->
@RN2694;<!-- Yoshie 2020 13 mo old mice have epigenetic changes in skm due to childhood diet 
@RN2695]<!-- Godfrey 2016 review childhood environmental conditions affects epigenetics 
linking environmental factors to the complexity of gene expression.
-->
A key aspect of successful exercise diagnostics would be to take advantage of the relationship between exercise variables (i.e. modality, intensity, volume etc.) and exercise response for a given individual.
By adapting an individuals training program based on some prior knowledge about the individual, it is possible that the response could be positively affected. 
Observations supporting such notion exists as an individual classified as non-responsive to a specific exercise modality (e.g. endurance training) may be classified as a responder to another (e.g. resistance training)
[@RN2681].<!-- Changing from end to res training removed non responsiveness --> 
Even changing training variables within a specific modality have been shown to convert non-responders to responders as endurance training volume was increased
[@RN2699].<!-- Lundby increasing training volume improved non-responders -->

Although apparent reversal of non-response to exercise training has been observed by manipulating training variables, decisive indications for such manipulations are still lacking.



## Adaptations to resistance training

### Muscle hypertrophy and strength
Systematic resistance training typically increases muscle mass and strength, adaptations through which many beneficial effects on health (increased amino acid storage physical independence) and athletic performance are conveyed. 
Muscle growth is a well characterized response to resistance training.
On the whole muscle level, healthy untrained individuals can be expected to increases their muscle mass by $\sim$ 5-20% when training is conducted over two weeks to 6 months
[@RN1741; <!-- Ahtianen -->
@RN346;<!-- Wernbom  -->
@RN764].<!-- Hubal 2005 elbow flexor 18.9% increase -->
Over this time span, muscle growth is approximately linear with time
[@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113;<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions -->
@RN2736]<!-- Narici 1989 Increase in CSA (MRI, linearly ~ 0.14%/day total 8.5%) and MVC (total 20.8%) no change in untrained control in CSA, numerical increases occured in untrained in EMG/MVC   -->
<!-- Although some of the early increase muscle mass can be attributed muscle damage-induced edema
[@RN2746],Damas 2016 increased echo intensity indicating muscle edema at 3 weeks -->
and has been detected as early as 3-4 weeks after training initiation, without apparent muscle edema 
[@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113].<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions -->

Relative muscle growth can be expected to be more pronounced in upper- compared to lower-body muscles when loading patterns are similar
[@RN346;<!-- Wernbom  -->
@RN2739].<!-- Welle 1996 larger incr in csa in upper body -->
This possibly relates to greater every day activity levels of lower-body muscles requiring a larger stimuli for adaptation
[@RN767].<!-- Folland review -->
Small, but detectable differences in muscle growth can be expected between sexes for training induced muscle growth in the upper-body
[@RN764]<!-- Hubal CSA men: 20.4% vs 17.9% -->
but not for lower-body muscles
[@RN2740].<!-- Roberts 2020, meta analysis when all muscles combined no diff between sexes in within sex effects sizes -->
Hypertrophic responses can be expected to be reduced with increasing age 
[@RN2739; <!-- Welle 1996 larger incr in csa in upper body --> 
@RN752]<!-- Peterson 2011 Meta analysis, age (negatively) and training volume (positively) affects rt outcomes  -->
but increased with sufficient addition of dietary protein
[@RN2199]<!-- Morton meta Dietary protein supplementation significantly enhanced changes in muscle strength and size during prolonged RET in healthy adults. Increasing age reduces and training experience increases the efficacy of protein supplementation during RET. -->
Additionally, training variables such as intensity, volume, frequency (reviewed below) together with other training aids, e.g. manipulation of blood flow through pressure cuffs
[@RN2745]<!-- Lixandrao BFR meta analysis hypertrophy similar between low load BFR and high load -->
can effectively modulate resistance training induced hypertrophy.
Together, this underlines that both non-modifiable (e.g. sexual dimorphism and age) and modifiable factors (e.g. training variables and protein supplementation) affects training induced muscle hypertrophy.

An increased muscle mass transfers to muscle strength trough an increase of contractile elements in parallel.
The muscle cell is to a large degree (about 80% of the cell volume 
[@RN2731]<!-- Lüthi 1986 Myofibrils occupie more space in the mucsle fiber after resistance training mitochondria occupie less. No sign increase in fiber CSA with strength training no control group -->)
occupied by these contractile elements, called myofibrils, where the interaction between actin and myosin determines force production
[@RN2756].
With resistance training, the number of myofibrils and thus the muscles force generating capacity increases
[@RN2731].<!-- Lüthi 1986 Myofibrils occupie more space in the mucsle fiber after resistance training mitochondria occupie less. No sign increase in fiber CSA with strength training no control group -->
<!-- Although the number of fibers per muscle changes as a consequence of age
[@RN2193] (decrease with age) Lexell 1988 number of fibers reduced with age 
and activity
[@RN2742] (increase with activity), Sjöström 1991 Increase in left/right tibialis anterior -->
Whole muscles growth in response to short term resistance training occurs primarily through addition of myofibrils in a set number of muscle fibers.
This can be assumed as training-induced fiber splitting or myogenesis is likely a slow process and any increase in the number of fibers would only represent a small addition to the whole muscle  
[@RN2741;<!-- Murach 2019 review on hyperplasia  -->
@RN2742; <!-- Sjöström 1991 Increase in left/right tibialis anterior -->
@RN2754;<!-- MacDougall no diff between body builders and untrained in number of fibers, equal relative amounts of collagen -->
@RN767].<!-- Folland 2007 review on mechanism for hypertrophy and strength gains -->


[@RN2669].<!-- Straight 2020 muscle fiber size increases with RT, in midlle age old, age inversely related to training effects -->



Muscle fiber hypertrophy is difficult to study in humans as such studies are restricted to small biopsy samples.
Muscle fiber characteristics varies across the muscle biopsy specimens
[@RN2742] <!-- Sjöström 1991 Increase in left/right tibialis anterior -->



[@RN2629]<!-- Ikai 1970 Arm flexor training within-subject protocol, trained arm 23% increase in CSA and 90 % strength 100 days of 6 days/week training. Control arm 30% str incr, 5% CSA incr -->
[@RN2737]<!-- Young 1983 diproportional higher strength gains than CSA, increase in CSA with ultrasound in response to RT -->

[@RN2736]<!-- Narici 1989 Increase in CSA (MRI, linearly ~ 0.14%/day total 8.5%) and MVC (total 20.8%) no change in untrained control in CSA, numerical increases occured in untrained in EMG/MVC   -->
[@RN2735]<!-- Narici 1996 Local changes in whole muscle CSA after 6 months of RT, no stat sig increase in fiber CSA at a site with low MR based increase in CSA -->  


[@RN787]<!-- MacDougall 1980 increases in fiber are in response to training and immobilization fiber areas affected pos and neg in resp condition (small n one-tailed tests) -->

[@RN2734]<!-- Alway 1988 larger fiber area but not stat sig myofibrill larger in strength trained (n=6) than endurance trained (n=6), active (n=6) and sedentary controls (n=6). -->









[@RN1710]<!-- Straight 2015 resistance training increases muscle power in middle age and older adults, training volume influences the magnitude of change -->


### Muscle fiber-type transitions
Skeletal muscle tissue is heterogeneous in terms 


Muscle fibre composition is another potential determinant of muscular responses to resistance training. Type II fibres have greater growth potential compared to type I fibres
[@RN1755;<!-- Stec 2016 8 (type I) vs 21% (type II) and 44 vs 83% CSA change in mo and xtr responders --> 
@RN2208]<!-- Jespersen Type II fibers larger fluctuations in CSA higher hypertrophy and atrophy rates-->, and readily switch from IIX to IIA phenotypes in response to mechanical loading
[@RN2056;<!-- Andersen 2018 type IIx to 2a most prominent switch-->
@RN2169;<!-- Widrick 2002 2a to 2x switch; quantitative changes related to contractile proteins-->
@RN1489]<!-- Ellefsen 2014, training induced switch from 2A to 2X -->,
suggesting that these fibres display greater plasticity in response to resistance training.



### Mitochondrial function

Increased mitochondrial respiration after 12 weeks of RT in young men [@RN2608]

Fiber type distributions do not predict muscle mitochondrial density in endurance trained individuals [@RN2615]

PMID:158694 Reduced mitochondrial density per fiber area in reponse to RT

## Effects of exercise prescription on muscle mass and strength
Precise exercise-training^[Exercise is herein defined as an acute bout of physical activity designed to affect physical characteristics such as strength, speed or endurance. Training is defined as the systematic process of combining multiple exercise-sessions performed in sequence over time. Resistance-exercise is defined as an acute strength-promoting program requiring the neuromuscular system to exert force against resistance. Resistance training is defined as a long-term process of multiple resistance exercise-sessions performed over a defined period of time.]
prescription gives information on exercises, their sequential order, intensity and volume, rest periods between efforts or sessions and the frequency at which exercise sessions are to be performed
[@RN789].<!-- Bird designing resistance training programs -->
By manipulating these variables, resistance training programs can be tailored to better fit goals and starting points of any individual. 
The relative importance of exercise-training variables for training outcomes has been examined in numerous studies including (but not limited to) the overall organization of exercise sessions,
[@RN2575;<!-- Evans mini review on periodization --> 
@RN2572]<!-- Grgic 2017 No effect of linear and daily unduluting periodization on muscle hypertrophy -->
training frequency 
[@RN2571]<!-- Schoenfeld 2016 training frequency twice per week better in volume equated studies for hypertrophy -->,
<!-- Higher frequencies gives higher gains in strength but not when volume equated -->
and intensity
[@RN2569].<!-- Schoenfeld, no difference between high vs low load strength training in hypertrophy but 1rm favors high loads -->
It could be argued that training volume is of particular importance for muscle growth as when this variable is held constant, manipulation of other variables has little or no effect hypertrophy
[@RN1612;<!-- Schoenfeld 2014 7*3Rm vs 3*10RM equal muscle hypertrophy but dissimilar strength -->
@RN2569].<!-- Meta analysis high vs low resistance has little effect on muscle mass gains but strength is affected positively with higher intensities. --> 
For development of strength, factors such as intensity and within session organization of exercises is of importance 
[@RN2570;<!-- Frequency has little effect on strength when volume is equated -->
@RN2591],<!-- Exercise order influences strength but not hypertrophy -->
however, when other factors are held constant, increased training volume generally leads to increased strength
[@RN2570;
@RN2492;<!-- Ralston 2017 meta analysis medium to high volume more effective strength gains -->
@RN793],
similarly to effects of training volume on muscle growth
[@RN1767;
@RN789].

### Effects of resistance exercise volume on muscle strength and mass
Exercise volume can be prescribed as the within session number of sets performed per muscle group. This unit is practical as it comparable between individuals and muscle groups [@RN2130].<!-- Baz-Valle 2018 sets per exercise better than mechanical load times rep or total work -->
Berger conducted an early study concerning effects of resistance exercise volume with the goal to determine what method most efficiently produced strength gains (in healthy young males) [@RN1476]. Berger compared one, two and three sets performed with two, six or ten repetition maximum (RM) in the bench press, three times per week, over twelve weeks. As the combined effect of three sets per session was superior regardless of the number of repetitions performed Berger concluded in favor of three sets. This conclusion was later challenged on the basis of data interpretation
[@RN794;<!-- Carpenelli and Otto questions the analysis-->
@RN2538].<!-- Feigenbaum -->
Reveiwing the study by Berger and others, Carpinelli and Otto arrived to the conclusion that there was "insufficient evidence to support the prevalent belief that a greater volume of exercise (through multiple sets) will elicit superior muscular strength or hypertrophy" [@RN794]. This stand has since been repeatedly put forward as a criticism of higher volume training programs 
[@RN2568; 
@RN2201] and sparked considerable scientific activity. The main argument against the recommendation of additional volume in strength training programs has been the lack of statistically significant results in single studies [@RN2568; @RN2538]. 
Indeed, individual studies do not generally agree on dose-dependent effects of training volume on muscle mass and strength gains
[@RN2465; <!-- Ribiero 2015 Strength but not body composition change with dose dep older women -->
@RN2464;<!-- Correa 2015 older women, increase muscle thickness and strength with volume  -->
@RN2463;<!-- Bottaro 2011 untrained young male similar results in muscle thickness and strength -->
@RN1570; <!-- Radaelli 2014 1 vs 3vs 5 sets, 5 sets most effective in muscle thickness and some strength exercise -->
@RN1518;<!-- Radaelli 2014 older women no diff between volume groups in muscle thickness or strength gains -->
@RN1474;<!-- McBride 1474 Men and women DXA no diff but strength increase more in multiple sets -->
@RN1472; <!-- Galvao 2005 Older adults, no diff in body comp DXA, but increase in strength  -->
@RN1456;<!-- Starkey 1996 young male and female no diff in strength or muscle thickenss gains -->
@RN1454;<!-- Ostrowski 1997 trained young med, low volume decrease from previous levels no difference between groups -->
@RN1384; <!-- Rhea 2002 young men bod pod no diff in body comp changes but diff in strength gains in favour of multi set -->
@RN1382; <!-- Cannon 2010, Older women and younger women, muscle volume and strength increased similarly irrespective of age or training volume and without interaction -->
@RN776],<!-- Rønnestad, three set more effectiv in increaseing strength and muscle mass in lower body compared to upper body -->
including studies performed within participants, where different training volumes are allocated to either extremity 
[@RN834;<!-- Mitchell 2012 young men no sig difference in strength or mass gains between 1 and 3 sets knee extension -->
@RN1607].<!-- Sooneste 2013 young men p=0.076 on strength gains in favor of three sets, CSA increased more in 3 sets leg exercise -->
For example, differences in strength are between volume conditions are found in older individuals 
[@RN2465; <!-- Ribiero 2015 Strength but not body composition change with dose dep older women -->
@RN2464;<!-- Correa 2015 older women, increase muscle thickness and strength with volume  -->
@RN1472] <!-- Galvao 2005 Older adults, no diff in body comp DXA, but increase in strength  -->
but not confirmed in another study
@RN1518].<!-- Radaelli 2014 older women no diff between volume groups in muscle thickness or strength gains -->
Studies shows that more volume does not lead to increased muscle gains in young individuals
[@RN1384; <!-- Rhea 2002 young men bod pod no diff in body comp changes but diff in strength gains in favour of multi set -->
@RN1456;<!-- Starkey 1996 young male and female no diff in strength or muscle thickenss gains -->
@RN2463]<!-- Bottaro 2011 untrained young male similar results in muscle thickness and strength -->
a conclusion challenged by others 
[@RN776;<!-- Rønnestad, three set more effectiv in increaseing strength and muscle mass in lower body compared to upper body -->
@RN1570]. <!-- Radaelli 2014 1 vs 3vs 5 sets, 5 sets most effective in muscle thickness and some strength exercise -->

As previously noted, combining the above results and additional studies, meta-analyses concluded that training volume dose-dependency exists for the development of muscle mass and strength
[@RN2570;
@RN2492;<!-- Ralston 2017 meta analysis medium to high volume more effective strength gains -->
@RN793;
[@RN1767;
@RN789].
As a second argument against additional volume in resistance training recommendation has been the cost/benefit relationship of adding training volume without meaningful or substantial additional gains
[@RN2538;<!-- Feigenbaum 1999 questions the analysis and the practical utility of volume -->
@RN2568],<!-- Carpinelli Berger in retrospect questions the practical significance of 1.8 kg diff in maximal strength -->
a subsequent question is, whom would benefit from greater volumes and whom would not?
Schoenfeld *et al.* combined data from published studies to explore if participant characteristics of the above mentioned studies interacted with training volume in explaining study outcomes. Neither sex, muscle groups nor age interacted with volume prescription indicating that no such factor would be able refine training prescription guidelines
[@RN1767].<!-- Schoenfeld 2016 meta-->
As the number of studies used to synthesis the meta-analysis was relatively low (*n* = 15) and the studies were heterogeneous in terms of e.g. outcome measurements, it may have lacked in power to detect any meaningful interactions. Additionally, included studies may not have been reporting relevant characteristics for such analysis.

Collectively, the available evidence suggest that there is overlap between training outcomes in studies were different volume has been utilized.
The overlap cannot, with available data, be explained by general population characteristics such as age or sex.
Studying the effect of different training volumes within participants could potentially help to define determinants of training outcomes in response to different volume conditions.
Two within-participant studies have investigated the effects of training volume on strength and hypertrophy outcomes.
Sooneste *et al.* compared strength outcomes in response to three- and one-set elbow flexor training for 12 weeks in young males using a whitin-participant protocol (arms allocated to either volume condition).
The results showed general benefit of three- over one-set training for muscle hypertrophy and tended to do so also for strength gains [@RN1607].
No attempts were made to relate baseline characteristics to the magnitude of differences between volume conditions, presumably due to the small sample size (*n* = 8).
Mitchell *et al.* compared muscle hypertrophy and strength gains in response to three- and one-set of knee-extension exercise performed three times per week for ten weeks.
The study contained an additional training condition (low intensity, 30% of 1RM performed with three sets) with participants legs assigned to either of the three conditions in a random fashion.
No significant differences were reported between volume conditions for muscle mass or strength gains [@RN834].
However, the analyses were performed without taking the correlation between individuals into account due to the mixed design [@RN834].
No attempts were made to relate any measured characteristic to differences in responses.

## Molecular determinants of training-induced muscle hypertrophy
Muscle mass fluctuates as a consequence of the balance between muscle protein synthesis and breakdown. When a net-positive balance is achieved, muscle protein accumulates and muscle mass increases.
This is a scenario observed in response increased mechanical stress.
Following a single bout of resistance exercise, muscle protein synthesis increases
over resting levels up to 48-h post-exercise 
[@RN2710;<!-- Dreyer 2006, blunting of protein synthesis during ex, increase post ex -->
@RN2711;<!-- McDougall 1995 Highest values at 24h after ex, at 36 not sig higher than ctrl -->
@RN786;<!-- Phillips 1997 fasted state, FSR and FBR both increased but FBR to a lesser extent giving increase in net balance. Fasted state -->
@RN2712;<!-- Biolo 1995, fasted state ex increase both breakdown and synthesis FSR/FBR at 3h net balance increase but not positive -->
@RN2717;<!-- Increase in FSR over control at 4 and 24 h post ex in biceps, no change in RNA -->
@RN1521]<!-- Kim, increase after exercise in mixed and myofibrillar prot synth in trained and untrained state, lower mixed after training, higher basal protein synt rate after training period 8 weeks 2-4set 8RM 8 weeks biopsy 12 h after ex Same absolute intensity -->
after being blunted during exercise
[@RN2710].<!-- Dreyer 2006, blunting of protein synthesis during ex, increase post ex -->
Mixed muscle protein synthesis and breakdown are highly correlated
[@RN2713]<!-- Phillips 1999, training alters both breakdown and synthesis --> 
[@RN786]<!-- Phillips 1997, correlation between fbr and fsr-->
indicating that these processes are mechanistically coupled.
And similarly to muscle protein synthesis, acute resistance exercise also stimulates protein breakdown but to a lesser extent than protein synthesis leading to an increase in the net protein balance from baseline
[@RN2713]<!-- Phillips 1999, training alters both breakdown and synthesis --> 
[@RN786].<!-- Phillips 1997, correlation between fbr and fsr-->
When resistance exercise is performed in the fasted state, post exercise net protein balance can be expected to be negative
[@RN2713]<!-- Phillips 1999, training alters both breakdown and synthesis --> 
[@RN786],<!-- Phillips 1997, correlation between fbr and fsr-->
in contrast when performed in the fed state where net balances are positive
[@RN2714]<!-- Børsheim 2002 Post ex prot synthesis increased with ingestion of essential amiono acids -->
[@RN2715].<!-- Tipton 1999, post-exercise net balance is increased when amino acids are administrated -->
This indicates an additive effect of 

Measurement of protein synthesis (and breakdown) rates in response to a single resistance exercise session has been given a lot of attention


In recent years, this view has been supplemented<!-- supported? --> by evidence suggesting that chronic RT leads to increased resting synthetic rate of muscle protein
[@RN1521;<!-- Kim 2005 Fasted state skeletal muscle PS altered with training  --> 
@RN1866;<!-- Wilkingson 2008 -->
@RN1897],<!-- Reidy 2017 Increased rested state protein synthesis -->
which has been postulated to be associated with increased translational capacity, i.e. accumulation of ribosomes
[@RN1897;<!-- Reidy 2017 -->
@RN1912]<!-- Figueiredo 2017 Ribosome biogenesis review -->.
This notion is supported by exercise-induced increases in total RNA, a proxy marker of ribosome abundance, which is closely connected to protein synthesis
[@RN1754;<!-- West 2016 correlation between total RNA and protein synthesis in C2C12-->
@RN2145]<!-- Millward 1973 relationship between RNA and protein synthesis in rat in rats in vivo constant infusion -->
and muscle hypertrophy
[@RN1644;<!-- Figueiredo 2015 Correlation between total RNA fold change and CSA hyp -->
@RN1755; <!-- Stec Ribosome bio higher in high responders -->
@RN2358].<!-- Volume dependent RNA accum relates to volume depend hyper -->
Conversely, inhibition of ribosomal RNA (rRNA) transcription and inhibition of its up-stream transcription factors act to diminish muscle cell growth
[@RN1755<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->;
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1810]<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->.  

Biosynthesis of novel ribosomes is a complex, highly coordinated and energy demanding process that involves synthesis of both ribosomal proteins and the four mature rRNA transcripts
[@RN1920;<!-- Warner review yeast -->
@RN1820;<!-- Moss review -->
@RN1810]<!-- Inhibition of Pol II affects trasncription at Pol I -->.
Ribosomal accumulation is believed to be determined by the rates of pre-rRNA transcription by RNA polymerase I (Pol I), which in turn is regulated by coordinated assembly of a complex of transcription factors at the rDNA promoter
[@RN1820]<!-- Moss review -->.
Specifically, activation of the of the upstream binding factor (UBF) through phosphorylation is needed to initiate transcription
[@RN2582; <!-- Tuan UBF phosphorylation important for SL1-UBF interaction -->
@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
Such activation is at least partly controlled by the mechanosensitive mTOR pathway, with its inhibition being associated with blocked UBF phosphorylation and subsequent rRNA transcription
[@RN1632; <!-- Nader rapamycin block UBF availability  -->
@RN2564].<!-- Hannan mTOR  -->
Interestingly the availability of UBF *per se* has been shown to be a determinant of rRNA transcription
[@RN2566]<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->
through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->




[@RN2724]<!-- Phillips 2002 fed state same absolute intensity, training attenuated ex induced FSR and FBR, FSR/FBR incr at rest after training   -->
[@RN2723]<!-- Tang 2008 -->

[@RN2144]<!-- Damas 2016 Correlation between integrated MyoPS and CSA increase after muscle damage accounted for. Deuterium used for MyoPS as opposed to infused tracer -->

### mTOR

The discovery of an immunosuppressant organic compound called rapamycin in the 1960’s led to the characterization of a rapamycin sensitive protein involved in cell growth. The protein was later named mechanistic target for rapamycin (mTOR)
[@RN1049]. This protein has since been shown play a key role in skeletal muscle hypertrophy in relation to mechanical loading. Bodine *et al.* made a comprehensive characterization of mTOR-mediated skeletal muscle hypertrophy using rodent models in 2001, showing that mTOR activation was essential for load-induced hypertrophy. Additionally, using transfection techniques, they showed that constitutively activated Akt signaling led to hypertrophy in an mTOR-dependent manner, confirmed with concurrent administration of rapamycin [@RN782]<!-- Bodine 2001 -->.


[@RN860]<!-- Kubica 2004, rapamycin prevents increases in ex induced protein synthesis trough eukaryotic initiation factor 2B mRNA translation (rats) -->



In this context, molecular signatures conveyed by the mechanistic target of rapamycin complex 1 (mTORC1) has been in particular focus. Inhibition of mTORC1 impairs protein synthesis in humans [@RN780] and activation of its associated downstream target S6 kinase 1 (S6K1) correlates with increases in muscle protein synthesis and subsequent muscle growth</span>
[@RN791;<!-- Burd Correlation in single session between p70 and  -->
@RN785].<!-- Terzis correlation between p70S6K and hypertrophy-->
In line with this, surplus <span custom-style="CHANGE">exercise</span> volume leads to greater phosphorylation of S6K1 
[@RN791;<!-- Burd in single session single and multiple sets phosphorylation p70   -->
@RN784;<!-- Terzis volume affect activation -->
@RN1837]<!-- Ahtiainen 5 vs 10 sets p70 and s6 different --> and <span custom-style="CHANGE">is accompanied by increases in </span>myofibrillar protein synthesis [@RN791]<!-- Burd -->,
fitting the notion that increased training volume provides more pronounced adaptations<span custom-style="CHANGE"> through repeated episodes of increased protein synthesis</span>. 
<!-- However, also from a cell biological perspective, present findings on effects of different training volumes are heterogeneous. For example, Mitchell *et al.* [-@RN834] failed to show differences in S6K1 phosphorylation between volume protocols, corroborating with similar effects of different volumes on muscle strength and mass. -->


Individual response patterns to resistance training, including muscle strength and mass, correlate closely with muscle cell characteristics, measured in both rested-state and acute training-phase conditions
[@RN826; <!-- Thalacker-Mercer 2013 differences between response cluster in gene expression -->
@RN774; <!-- Raue 2012 Gene expression signatures correlate with strength and CSA gains -->
@RN1755; <!-- Stec 2016 Differential ribosome biogenesis between responders and non responders -->
@RN785]<!-- Terzis correlation between P70 and hypertrophy -->. <span custom-style="CHANGE">



### Ribosomal biogenesis 

Conversely, inhibition of ribosomal RNA (rRNA) transcription and inhibition of its up-stream transcription factors act to diminish muscle cell growth
[@RN1755<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->;
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1810]<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->.  

Biosynthesis of novel ribosomes is a complex, highly coordinated and energy demanding process that involves synthesis of both ribosomal proteins and the four mature rRNA transcripts
[@RN1920;<!-- Warner review yeast -->
@RN1820;<!-- Moss review -->
@RN1810]<!-- Inhibition of Pol II affects trasncription at Pol I -->.
Ribosomal accumulation is believed to be determined by the rates of pre-rRNA transcription by RNA polymerase I (Pol I), which in turn is regulated by coordinated assembly of a complex of transcription factors at the rDNA promoter
[@RN1820]<!-- Moss review -->.
Specifically, activation of the of the upstream binding factor (UBF) through phosphorylation is needed to initiate transcription
[@RN2582; <!-- Tuan UBF phosphorylation important for SL1-UBF interaction -->
@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->





[@RN2563] <!-- Lin 2007 Phosphorylation at Ser389 is needed for interaction with SL1, but ser584 do not, non of these phosphorylation sites leads to location to nucleolar   -->
[@RN2602]<!-- UBF is a target for cyclin d1 and cyclin E NIH 3T3 fibroblasts,Mutation of Ser484 impairs rDNA transcription  -->

[@RN2604]<!-- Stefanovsky 2001 ERK inhibition through MEK specifik inhibitor blocks EGF induced rRNA transcription in vitro. Direct Raf activation -> ERK activity stimulated rRNA transcription. UBF is a target of ERK signalling  -->

[@RN2717]<!-- Chesley 1992 increase in FSR in response to re-ex i biceps, no change in RNA content, increase in "RNA activity" prot synth/RNA -->

[@RN2155]<!-- Welle 1999 No increase in mRNA or total RNA after 7 days in older subjects but 30% increase in myofibrillar protein synthesis -->

### Transcriptional regulation of muscle mass

[@RN2618]<!-- Newlands 1998 Transcriptional activity of myonuclei are not maximised, increase in transcription factors increase the chance of a loci being transcribed... -->

[@RN2616]<!-- Kirby 2016 nascent RNA is detected in myonuclei after SA (mice) 47S is detected in EU labeled RNA pool -->


#### Ribosomal biogensis as a determinant of RT-induced hypertrophy

Resistance exercise is a potent stimuli for rRNA transcription as a single session leads to increases in pre-rRNA
[@RN1656<!-- A single session upregulates 45 S 24 h post ex -->;
@RN1037<!-- Nader 2014 45S up in response to 4 h after acute exercise -->]
and repeated bouts lead to accumulation of mature rRNA thus also total RNA and presumably functional ribosomes
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN2358; <!-- Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].
However, the true time course of ribosomal transcription and accumulation in response to RT remains largely unstudied, with a mere few studies having investigated exercise-induced changes in rRNA over multiple time-points, all of which are either limited to a selected few time-points or a limited time frame.
For example, two consecutive bouts of electrically evoked muscle contractions were associated with increased levels of total RNA, with peak values being observed 72 hrs after the second bout
[@RN1520]<!-- Total RNA increase 72 h after a second bout -->. 
Using voluntary contractions, peak values were reported after nine sessions, followed by a slight decrease to after 18 sessions [@RN1809],
resembling data from our lab where five sessions of RT led to marked increase in total RNA levels (per-unit muscle tissue), whereby a numerical lowering occurred to after the last training session of the 12 wk interventions (31 sessions).
[@RN2358].
Interestingly, during the initial phase of RT, total RNA accumulation seems to be volume-dependent, as three sets per exercise in leg exercises led to augmented total RNA and rRNA levels compared to one set per exercise, coinciding with the differences in muscle hypertrophy seen after 12 weeks of RT.
[@RN2358].
These data suggest that ribosome accumulation reaches a plateau in the early phase of RT and that increases are sensitive to training volume in constant volume protocols. 


Recent observations in humans are challenging this view by indicating that translational capacity is a limiting factor for training-induced muscle hypertrophy. First, increased abundances of <span custom-style="CHANGE">ribosomal RNA (rRNA)</span> in response to resistance training, measured as total RNA per-weight-unit muscle tissue, correlate with muscle hypertrophy [@RN1644]<!-- Figueiredo et al., 2015)-->.
In accordance with this, training-induced increases in rRNA are larger in muscle hypertrophy high-responders than in low-responders
[@RN1755;<!--Stec 2016 4 weeks rRNA and total RNA abundance larger in high responders -->
@RN2055].<!-- Mobely 2018 Post-hoc comparisons at week 12 indicate weaker association -->
Secondly, elderly participants typically show blunted ribosome biogenesis, coinciding with attenuated hypertrophic responses
[@RN1656;<!-- Stec 2015 acute bout, blunted 45S induction --> 
@RN1809]<!-- Brook 2016 blunted responses in elderly together with blunted rna induction-->.
Collectively, these observations suggest that muscle growth depends at least in part on increased translational capacity, making it a prime candidate for explaining the diverse response patterns seen in resistance training with different volume in different individuals. To date, no study has investigated the association between training volume, ribosome biogenesis and regulation, and gross training adaptations.


<!-- OUTLINE 






<!-- 
Also in humans, administration of rapamycin, hindering the activity of mTOR leads to an abolished exercise-induced increase in protein synthesis
[@RN780]<!-- Drummond rapamycin blocks protein synthesis in humans -->. 

<!-- 
Furthermore, observational evidence linking mTOR to load-induced hypertrophy comes from human and rodent studies correlating acute phosphorylation the downstream target of mTOR, ribosomal protein S6 kinase (S6K) in response to acute mechanical loading and hypertrophic responses following a subsequent training period
[@RN785;<!-- Terzis 2008 mTOR activity correlate with subsequent adaptation 
@RN866]
--> 

<!-- 
### Protein synthesis


Positive net protein balance in responce to exercise <!-- Tipton 10.1152/ajpendo.00234.2002.-->

<!-- Coordinated mTOR 10.1038/nature13492. -->

<!-- Inhibition of RNA synthesis restrict protein synthesis Goldspink 1977 -->

<!--Indicated in Goldspink 1977 and 1976 RNA reflects ribosomal availability  Goldspink 1977-->

<!--Protein synthesis is proportional to RNA content  Millward 1973 -->


<!-- Increase loading leads to increased RNA Goldberg 1975 Mechanism of work-induced hypertrophy of skeletal muscle 
```{r Millward1973, echo=FALSE, message=FALSE, warning=FALSE, fig.height = 2.5, fig.width = 2.5, fig.cap="Data from Millward et al. 1973. Group A were fed a diet containing protein, group B were starved or fed a diet not containing protein."}

millward <- read_excel("./data/litrev.xlsx", sheet = "millward1973") 

millward %>%
        mutate(group = if_else(group == "A", "Protein fed", "Protein starved")) %>%
        ggplot(aes(RNA, proteinSynthesis, fill = group, color = group)) +
        geom_smooth(method = "lm", se = FALSE) + 
        geom_point(color = "black", shape = 21) +
        scale_x_continuous(name = TeX("$RNA\\;(\\mu g \\; mg^{-1} \\, protein)$"), 
                           breaks = c(0, 4, 8, 12, 16),
                           minor_breaks = c(2, 6, 10, 14),
                           limits = c(0, 16), 
                           expand = c(0, 0)) +
        scale_y_continuous(name = TeX("$Protein\\,synthesis\\,rate\\;(days^{-1})"), 
                           breaks = c(0, 0.05, 0.1, 0.15, 0.20, 0.25),
                           minor_breaks = c(0.02, 0.06, 0.10, 0.14, 0.18, 0.22),
                           limits = c(0, 0.25), 
                           expand = c(0, 0)) + 
        

        dissertation_theme() +
                theme(legend.position = c(0.2, 0.8), 
                      legend.title = element_blank()) 



```

### The mammalian target of rapamycin (mTOR) and translational efficiency
The mammalian target of rapamycin (mTOR) is a large serine-threonine protein kinase wich in complex with oyher regulatory proteins forms a signaling hub responsible for responses to environmental cues such as nutrients and mechanical stress.

mTOR has several phosphorylation sites 

Phosphorylation of Ser2448 is mediated by S6K1 to reduce mTOR activity in a negative feedback loop <!--[@RN1902]-->. 

<!--Ser2448 is phosphorylated by S6K1, changes in nutrient avaliability modifies S6K1 and Ser2448, Ser2448 phosphorylation is abolished when S6K1 is depleted [@RN1902]-->

<!--
When the C-terminal is deleted, mTOR gets constitutively active




## Ribsome biogenesisand muscle growth

<!-- Stec 2016 growth factor induced hypertrophy is abolished when ribo biog is inhibited with the specific CX-5461  -->
<!-- Crossland 2017 in contrasts, IGF-1 induced growth was not impaired when ribo biog was inhibited by CX-5461 -->
<!--Crossland et al. noted that using IGF-1 induces specific growth related effects in muscle cells. These may be different than the effects induced by sreum as in Stec et al. 

 Studies showing correlation between growth and ribo biogenesis -->


## Effects of exercise volume on molecular determinants of muscle growth


[@RN2716]<!-- Kumar 2012  -->

[@RN2720]<!-- Kumar 2009 increasing exercise intensities maximises post ex prot synthesis at 75% in young (plateau at 60-90% of 1RM)  -->




### From Training response

In humans, the biological adaptation to resistance training varies with exercise-training variables such as volume, intensity, rest between repetitions and sets, selection and order of exercises, repetition velocity and frequency of training sessions [@RN1].
In addition, genetic and epigenetic disposition and environmental factors play a role in variations in adaptations
[@RN758;<!-- Timmons 2011 reports gene polymorphism studies -->
@RN2012;<!-- Seaborn 2018 methylation leads to increased gene expression relates to lean mass increase with exercise-->
@RN2199]<!-- Morton 2018 Systematic review of protein supplementation i.e. protein availability as environmental factors -->.
As time constraints often hinder participation in exercise training-programs [@RN2063], numerous studies have searched for the minimal required exercise dose to promote beneficial adaptations. Within-session volume has received particular attention, and although a handful of studies have shown that low-volume training provides similar gains in strength and muscular mass as moderate-volume training
[@RN1382;<!-- Cannon et al 2010 No difference between volume conditions in strength or hypert in women  -->
@RN1454;<!-- Ostrowski no diff between volumes in strength and hypert. in men -->
@RN834]<!-- Mitchell no stat different between volume conditions -->, meta-analyses conclude in favor of moderate volume protocols [@RN793; @RN789; @RN1486; @RN1767].
This apparent discrepancy of specific studies to demonstrate benefits of increased training volume is likely due to a combination of small sample sizes and substantial variation in training responses between individuals and experimental groups. In theory, within-participant designs should alleviate these limitations.


## From RNA-seq

Skeletal muscle is a highly adaptable tissue that responds to environmental stress by altering growth rates and differentiation processes. During resistance training, signaling cascades that stimulate muscle plasticity are triggered. Upon repeated exposures, this facilitates growth and a phenotypic shift in a metabolically active direction [@RN2364],<!-- Egan et al -->
with the opposite happening during inactivity [@RN2365]<!-- Dirks 2016, one week of bed rest muscle loss reduxed ox-capacity and whole body insuline resistance -->.
Despite this generalized view, muscle responsiveness and plasticity vary, both in response to different resistance-training protocols [@RN2358]<!-- Hammarstrom 2020 --> and, perhaps more importantly, between individuals [@RN764; ><!-- Hubal 2005 n=585, large variability reported -->
@RN1741]<!-- Ahtiainen 2016 Heterogenity --->. 
Selected individuals show a near-complete absence of muscle growth after prolonged resistance training, which markedly reduces the beneficial outcomes of such interventions for muscle function and overall health [@RN1741; <!-- Ahtiainen 2016 Heterogenity --->
@RN764]<!-- Hubal 2005 n=585, large variability reported -->.
Currently, little is known about the etiology of this variation. However, it is usually associated with phenotypic traits of skeletal muscle [@RN765;<!-- Bamann cluster analysis showes different expression profiles of e.g. MGF between resp groups -->
@RN1755;  <!-- Stec et al, Ribosome biogenesis different between responders and nonresponders -->
@RN755 <!-- Davidsen 2011 miRNA in response to RT different between responder groups --> 
], which implies interactions with environmental factors, genetics, epigenetics, and composites of the intra physiological milieu [@RN2199;<!-- Morton systematic review showing extra benefit of protein on muscle mass--> 
@RN1966].<!-- Brook 2016 rev. Anabolic resistance in age, nutritional and training variations to potimize outcomes -->
This multifaceted origin makes the training-response-spectrum difficult to study directly, with each of the underlying factors offering limited explanatory value alone [@RN758].<!-- Timmons 2011 Review critique of "single gene master regulator" -->
Instead, a more indirect approach is necessary, whereby the combined effects of the factors are targeted by studying global patterns of mRNA, protein expression, and skeletal muscle biology.

Previous studies have investigated transcriptome responses to acute resistance exercise [@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN2298;<!-- Damas reghulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> @RN2408]<!-- Hyldahl 2011 acute activation in reponse to 100 eccentric contractions-->
and chronic resistance training [@RN2400;<!-- Robinson 2017 differences on common regulation between exercise types in you and old--> 
@RN2397;<!-- Melow 2007 Age was associated with DE of 596 genes, training in aged led to RNA profiles more like young. --> 
@RN2298;<!-- Damas reghulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> @RN2299;<!-- Murton Exercise induced genes with no response in control group-->
@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN774]<!-- Raue 2012. Differential expression  -->,
as well as described associations between transcriptome characteristics and degrees of muscle growth [@RN774;<!--Raue 2012. 661 genes regulated by exercise and correlated with muscle size and strength--> 
@RN753],<!-- Phillips 2013 association with  -->
and function [@RN2402;<!-- Hangelbroek 2018 gene cluster predicts 1RM, training induces expression of ECM/collagen -->
@RN2398].<!-- Stepto 2009 Difference between trained (strength and endurance) and untrained controls. A gene clsuter correlated with strength -->
Whereas these studies have merited interesting findings, they lack clear coherences in terms of differential expression events, even for classical exercise-inducible genes such as PGC1$\alpha$ [@RN2360].<!-- Pillon et al -->
This lack of clear coherence is potentially due to a combination of issues such as differences in study design and methods for synthesis and analysis of transcriptome data. First, biologically founded variability can be attributed to differences in exercise protocols (e.g., differences in exercise-volume or intensity). This makes it difficult to discern a general transcriptome exercise response, as training variables are not standardized between studies. Biological heterogeneity is also caused by differences between research participants, affecting signal-to-noise ratios and making it difficult to discern the effects of single independent factors such as training variables. Design stage decisions such as the use of within-participant designs [@RN2358;<!-- Hammarstrom 2020 -->
@RN2403]<!-- Tarnopolsky no diff (only 0.6% differentially expressed) between legs in rested state (small n=8) --> are likely to reduce this variation and to provide transcriptome data with increased biological meaningfulness. Second, technical variability can be attributed to decisions made during the bioinformatical treatment of data. As described by Concea *et al.*, there is no optimal pipeline for sequencing technology as new tools keep evolving and emerging, different tools should be explored to an optimum pipeline for the specific type of data [@RN2426].<!-- Conessa 2016 best practices fom RNA seq-->
To exploit the potential of any study design, there is a need for identifying an appropriate pipeline for transcriptome analyses to ensure a biologically valid interpretation of data. This entails identifying potential violations of common assumptions caused by the experimental model at hand, relating to, for example, data normalization [@RN2414;<!-- Robinson 2010 TMM method normalization assumes most genes not affected by treatment-->
@RN2363]<!-- Dillies et al -->. 

For transcriptome data to provide adequate biological information about a given experimental set-up, numerous bioinformatic steps need to be adopted in a customized manner [@RN2426;<!-- Conessa 2016 best practices fom RNA seq-->
@RN2757].<!-- Arora 2020 differences between pipelines in gene abundance estimates --> 
Of these steps, data normalization is particularly decisive [@RN2363]<!-- Dillies et al -->,
as it aims to transform naïve transcript counts into biologically meaningful results. This essentially means expressing them as *per*-cell abundances [@RN2359]<!-- Loven 2012 et al  -->.
For most experimental models, this is equivalent to providing transcript-to-total RNA ratios, given the fulfillment of the assumption that total RNA levels remain stable between conditions on a per-unit-cell or per-unit-tissue basis [@RN2359]<!-- Loven 2012 et al  -->.
In cell models that exhibit high degrees of plasticity, gene expression events result in increased amounts of total RNA and mRNA transcripts per cell [@RN2430]<!-- Lin 2012 c-Myc induced gene amplification through -->, specifically violating the assumption that most genes are not differentially expressed [@RN2359;<!-- Loven 2012 et al  -->
@RN2414].<!-- Robinson 2010 TMM method normalization assumes most genes not affected by treatment-->
We are not aware of any study that has addressed the need to account for such perspectives during transcriptome analyses of skeletal muscle subjected to mechanical stress, such as resistance training. Indeed, this assumption can be expected to be violated, as total RNA content increases markedly on a per-unit-weight basis [@RN2358]<!-- Hammarstrom 2020 -->, with potential global changes also occurring for the mRNA pool, though this remains unknown. The extent to which total RNA, and therefore ribosomal RNA, increases, coincides with the increase in muscle mass [@RN2358<!-- Hammarstrom 2020 -->;
@RN1755]<!-- Stec -->, underlining its importance for cellular growth but also its inevitable presence as a potential confounding factor in RNA sequencing experiments.

## From tr10

Skeletal muscle is a critical target for interventions that promotes health across the lifespan [@RN763]<!-- Wolfe 2006 -->,
with resistance training (RT) being the advocated remedy.  
Prolonged RT leads to changes in the balance between muscle protein breakdown and synthesis, with one bout of resistance exercise acutely increasing protein synthesis for up to 48 hrs after exercise [@RN786],<!-- Phillips 1997 -->
with subsequent repeated bouts leading to accumulation of muscle protein over time
[@RN1272<!-- Phillips review 2014 -->;
@RN1642]<!-- Brook 2015 experiment -->. 





<!--chapter:end:01-Background.Rmd-->

# Aims
<!-- Aim = what you hope to achieve; Objectives the action you will take to achieve the aims --> 
The primary aim of this thesis was to relate the adaptive response to resistance training with low- and moderate-volume to skeletal-muscle characteristics in previously untrained individuals. The key question was whether manipulation of exercise-volume will have diverse effects in different individuals related to muscular intrinsic characteristics. A further aim was to characterize exercise-volume dependence in muscle molecular characteristics and determine a time course profile of markers of ribosomal biogenesis in response to resistance training. <!-- A final aim of the present thesis was to develop an implementation of unbiased reference gene selection to aid gene expression analysis in repeated measures designs   --> Based on these aims, the objectives of the present thesis were;

- to relate skeletal muscle and systemic characteristics to benefit of moderate- compared to low-volume resistance training;
- To determine volume-dependence in molecular networks related to muscle growth and remodeling in response to resistance training
- To determine a time course of markers related to ribosome biogenesis in the early phase of resistance training.

<!-- To create a software implementation of reference gene selection algorithms -->


<!--chapter:end:02-Aims.Rmd-->

---
output: html_document
bibliography: bib/complete-bib.bib
editor_options: 
  chunk_output_type: console
  markdown: 
    wrap: 72
---

# Methods

```{r libs, echo=FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R")
source("./R/themes.R")
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

## Study participants, protocols and training interventions

Study I was designed to examine effects of low- and moderate-volume on
responses to acute exercise and long-term training within participants.
Forty-one healthy individuals were recruited and 34 of these completed
at least 85% of the prescribed sessions and were thus included in
subsequent data analyses. Reasons for not completing the trial included
injury not related to the study (*n =* 1), pain or discomfort during
exercises (*n =* 5) and non-adherence to the study protocol. There were
no differences in characteristics between participants included in or
excluded from data analysis in Study I.

Study II was designed to study the effects of resistance training *per
se*, and effects of variable volume on selected markers related to
ribosome biogenesis. Participants were therefore recruited to a training
group (*n =* 11) and a non-training control group (*n =* 8). Eligible
for participation in both studies were young (Study I age 18-40; Study
II 18-35), non-smoking men and women. Exclusion criteria included a
training history of more than one weekly session during the last 12
(Study I) or six (Study II) months leading up to the study. Participants
were also screened for intolerance to local anesthetic, current or
previous injuries affecting their ability to perform resistance
training, self-reported symptoms or history of disease, intake of
medication or supplements with known effects on adaptations to training.
Participant characteristics for both studies are shown in Table
\@ref(tab:characteristics-table).

```{r characteristics-table, results = "asis", message=FALSE, warning=FALSE, echo=FALSE}

### Read data on included/excluded, body composition and cybex data
characteristics <- read_excel("./data/study-1/body-composition/bodycomp_DXA.xlsx") 
participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";")
load("./data/study-1/strength/cybex.rda")

library(knitr); library(kableExtra)

# Study 1 characteristics
cybex.temp <- data.frame(cybex) %>%
  filter(timepoint %in% c("pre", "session1"), type=="Isom")%>%
  group_by(subject)%>%
  summarise(trq=max(torque.max))

studyI <- participants %>%
  inner_join(characteristics) %>%
  filter(subject!="FP10") %>% # removes participant excluded prior to comecing the study
  filter(timepoint=="pre") %>%
  inner_join(cybex.temp) %>%
  dplyr::select(subject, sex, include, age, height, weight, fat.whole, lean.whole,BMC.whole, trq)%>%
  mutate(fat=((fat.whole/1000)/weight)*100,
         trq.lean=trq/(lean.whole/1000)) %>%
  inner_join(participants) %>%
        mutate(group = if_else(include == "incl", "Included", "Excluded"), 
               sex = factor(sex, levels = c("female", "male"), 
                            labels = c("Female", "Male")), 
               study = "Study I", 
               bmi = weight/(height/100)^2) %>%
        dplyr::select(participant = subject, 
                      study,
                      sex,group, age, height, weight,bmi,
                      fat.whole, 
                      lean.whole,
                      BMC.whole) 

# 
# age.m1 <- lme(age ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# weight <- lme(weight ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# height <- lme(height ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# fat    <- lme(fat ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# mvc    <- lme(trq.lean ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# 
# summary(age.m1)

# summary(weight)

# summary(height)

# summary(fat)

# summary(mvc)
  
### No differences between included not included.

# Study 2 characteristics 



dxa_whole <- read_excel("./data/study-2/body-composition/tr010_dxa_standarddefinitions.xlsx") %>%
  dplyr::select(Etternavn, Kjønn,Målingsdato,Alder, Vekt:`HeleVev%fett`) %>%
  pivot_longer(names_to = "variable", values_to = "value", cols = Vekt:`HeleVev%fett`) %>%
  mutate(value = gsub("kg", "", value), 
         value = gsub("[[:space:]]", "", value, fixed = FALSE), 
         value = as.numeric(value)) %>%

  separate(Etternavn, into = c("Study", "participant")) %>%
  dplyr::select(participant, Date = Målingsdato, variable, value) %>%
  filter(!(participant %in% c("P08", "P20", "P17"))) %>%
  group_by(participant) %>%
  mutate(time = if_else(Date == min(Date), "S0", 
                        if_else(Date == max(Date), "post1w", 
                                "S12")), 
         leg = if_else(variable == "BeinhøyreMagermasse", "R", "L")) %>%
  filter(variable %in% c("HeleBeinmasse", 
                         "HeleFettmasse", 
                         "HeleVevmasse", 
                         "HeleMagermasse" ,
                         "Vekt", 
                         "HeleTotalmasse")) %>%
  pivot_wider(names_from = variable, values_from = value) %>%
  mutate(fat_rel = 100*(HeleFettmasse/HeleTotalmasse)) %>%
          filter(time == "S0") 




studyII <- read_excel("./data/study-2/body-composition/tr010_age_participant.xlsx") %>%
  inner_join(dxa_whole) %>%
  mutate(age = as.numeric(difftime(recruited, dob)/365.2425), 
         bmi = weight/(height/100)^2, 
         study = "Study II") %>%
          dplyr::select(participant, 
                        study,
                        sex,
                        group,
                        age,
                        height, 
                weight = Vekt,
                bmi,

                fat.whole = HeleFettmasse,
                lean.whole = HeleMagermasse,
                BMC.whole = HeleBeinmasse) %>%
        mutate(sex = factor(sex, levels = c("F", "M"), 
                            labels = c("Female", "Male"))) 
        

# Combine both studies and make table
rbind(studyI, studyII) %>%

        mutate(rel_lean = ((lean.whole/1000) / weight) * 100, 
               rel_fat =  ((fat.whole/1000) / weight) * 100) %>%
        dplyr::select(participant:bmi, rel_lean:rel_fat) %>%
        pivot_longer(names_to = "variable", values_to = "values", cols = age:rel_fat ) %>%

        
        group_by(study, group, sex, variable) %>%
        summarise(m = mean(values), 
                  sd = sd(values)) %>% 
        ungroup() %>%
        mutate(m.sd = if_else(variable == "age", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                              if_else(variable == "bmi", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                      if_else(variable == "height", paste0(sprintf("%.0f", m), " (", sprintf("%.0f", sd), ")"), 
                                              if_else(variable == "rel_fat", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                      if_else(variable == "rel_lean", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                              paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")")))))),
               group = factor(group, levels = c("Included", "Excluded", "experiment", "control"), 
                              labels = c("Included", "Excluded", "Training", "Control"))) %>%
        dplyr::select(study:variable, m.sd) %>%
        pivot_wider(names_from = variable, values_from = m.sd) %>%
        arrange(study, group) %>%
        dplyr::select(study, group, sex, age, height, weight, rel_fat, rel_lean) %>%

  kable(format = "latex", 
        booktabs = TRUE, 
        caption = "Participant characteristics", 
        escape = FALSE, 
        col.names = c(" ", " ", "Sex", "Age (years)", "Stature\n(cm)", "Mass (kg)", "Fat mass (\\%)", "Lean mass (\\%)")) %>%
        
        kable_styling(font_size = 7) %>%
        collapse_rows(1:2) %>%

  footnote(general = "Data are means and (SD)",
           general_title = "", 
           footnote_as_chunk = TRUE)


```

## Resistance training interventions

Each training session started with a light standardized warm-up (5 min
ergometer cycling and 10 repetitions each of push-ups, sit-ups,
back-extensions and squats). Before each exercise in the main program,
one set of 10 repetitions were performed in the specific exercise with
approximately 50% of 1RM.

Studies were fully or partially performed as within-participant studies
as each participant had their legs assigned to different training
conditions (not including the control group in Study II). Allocation was
performed after enrollment where each participant had their legs
randomized to either low- or moderate volume (Study I), or variable or
constant volume (Study II).

In Study I, the low-volume protocol consisted of a single set of each
exercise and the moderate-volume consisted of three sets per exercise.
Three unilateral leg exercises were used (leg press, leg curl and knee
extension). The moderate volume-leg commenced all sessions and the low
volume-leg performed a single set of each exercise in the rest between
second and third set of the moderate volume training protocol.

In Study II, only unilateral knee-extension was performed in an effort
to concentrate the stimulus to the quadriceps
muscles.<!-- Check this --> The constant-volume leg performed six sets
of 10RM throughout the study and variable leg performed six sets in
session one to four, three sets in session five to eight and nine sets
in session nine to twelve with same intensity (10RM).

### Ethical considerations

Both studies were approved by the local ethics committee Lillehammer
University College/Inland Norway University of Applied Sciences and the
Norwegian Centre for Research Data. In accordance with the *Declaration
of Helsinki*[@RN2548] the studies were pre-registered in publicly
accessible databases (Study I, ClinicalTrials.gov Identifier:
NCT02179307; Study II, <https://osf.io/wa96y>). Participants were
informed of the study design, potential risks and sources of discomfort
prior to giving their informed consent.


## Muscle strength assessments
<!-- From training response -->

Isokinetic and isometric unilateral knee-extension strength was assessed in a dynamometer (Cybex 6000, Cybex International, Medway USA).
Participants were seated and secured in the dynamometer with the knee joint aligned with the rotation axis of the dynamometer. Maximal isokinetic torque was assessed at three angular speeds (60$^{\circ}$, 120$^{\circ}$ and 240$^{\circ} ~\times$ sec$^{-1}$). Prior to testing, participants were familiarized with the test protocol by performing three submaximal efforts at each angular speed.  Participants were given two attempts at 60$^{\circ} ~\times$ sec$^{-1}$ and three attempts at 120 and 240$^{\circ}~\times$ sec$^{-1}$ performed in immediate succession. The highest value was used for statistical analyses. After isokinetic testing, maximal voluntary contraction torque (MVC) was assessed at a knee angle of 30$^{\circ}$ (full extension $=90^{\circ}$). Participants were instructed to push with maximal force against the lever for 5 sec. Participants were given two attempts, with 30 sec rest in-between. The highest value was used for downstream analyses.

Maximal strength was assessed as one repetition-maximum (1RM) in unilateral leg-press and knee-extension. The test session for each exercise started with a specific warm-up consisting of ten, six and three repetitions at $50$, 75 and $85\%$ of the anticipated maximum. Thereafter, 1RM was found by increasing the resistance progressively until the weight could not be lifted through the full range of motion. For each exercise, the highest load successfully attempted was defined as 1RM. Each participant was given four to six attempts. 

At baseline, 1RM, isokinetic and isometric strength assessments were performed twice, separated by at least four days. The maximum value achieved for each of the tests was used in subsequent analysis. Strength tests were separated by at least 48 hours from preceeding training sessions. A combined measure of muscle strength was calculated as the average of all tests (1RM, isometric and isokinetic), wherein each test modality was given equal weight. A subset of the participants (n=18) performed strength assessment during the course of the study (at week two, five and nine). For the remaining participants, ordinary training sessions were prioritised when participants missed training or testing due to illness or scheduling difficulties. 

<!-- From TR010 -->

Muscle strength was assessed as maximal voluntary isokinetic (90&deg; sec<sup>-1</sup>) and isometric (60&deg; angle, fully extended leg 0&deg;) knee extension torque. After a brief warm-up (5-min cycling, RPE 12-14), participants were seated and secured in the individually adjusted dynamometer. Participants were instructed to gradually increase their effort during three warm-up repetitions (50, 60 and, 70% of subjective maximal effort). After a 30-sec restitution period participants were instructed to perform three repetitions with maximal effort in the concentric phase. Sixty seconds after the isokinetic test the lever automatically moved to a 60&deg; angle and participants were instructed to push against the lever enough to see feedback from the visual feedback system. After an additional 15-sec restitution period, participants were instructed to push against the lever with maximal effort. Within the same assessment session, participants remained seated in the dynamometer for measurement perfomed on both legs. The first measurement was alternated between legs every other session. For statistical treatment of the data, all successful attempts were used. The last strength assessment at baseline was performed at least seven days prior to the first biopsy sampling. At least one of the baseline strength tests was performed on separate day with two sessions allowed to be perform on the same day with a short rest between assessments. Post training assessments were performed 48 hours and eight days after the last session.






## Measures of muscle mass

In Study I muscle mass was measured by magnetic resonance imaging (MRI)
and dual energy X-ray absorptiometry (DXA) prior to and after the
intervention. Both MRI and DXA measurements were completed during the
same visit to the laboratory. Participants were instructed to refrain
from strenuous physical activity during the last 48 h leading up to the
measurements. The post-training measurements were completed at least 48
h after the last strength testing session. Participants were asked to
refrain from food consumption during 2 h leading up to the measurements.

MRI images were obtained from the mid-thigh and analyzed by the same
investigator blinded for time (pre- and post-training) and condition
(low- and moderate-volume). Multiple images were used to estimate the
cross-sectional area of the extensor muscles at the same distance from
the knee-joint.

See figure <!-- @ref(fig:body-comp-measures)-->

<!--Dallin et al. recently estimated the [@RN2541]
     MRI has shown better agreement with cadaver... see ref in Dallin 

    ## Muscle strength assessments

    Muscle strength was with  -->

In Study II, m. vastus lateralis muscle thickness was measured by B-mode ultrasonography (SmartUs EXT-1M, Telemed, Vilnius, Lithuania) using a 39 mm 12 MHz, linear array probe. At least three images were captured for each leg per time-point. Between each image acquisition the probe was relocated to the same position where on the skin and subsequently marked on a soft transparent plastic sheet superimposed on the thigh.  

Muscle thickness of m. vastus lateralis and m. rectus femoris were measured using  Transverse images were obtained $\sim$ 60 % distally from the trochanter major towards the femoral lateral epicondyle. Landmarks such as moles and scars were also marked on the plastic sheet for relocation of the scanned areas during post-training measurements. During analysis, pre and post images from the same participant were analyzed consecutively using the Fiji software,66 and by two independent researchers. The average muscle thickness of the three images captured per muscle was used for further analyses.

<!-- From tr010 -->
For determination of body composition participants were scanned using DXA (Lunar Prodigy densitometer, GE Healthcare, Madison, WI, USA) with the standard scanning mode (13-25 cm). Participants were lying supine within the scanning bed reference lines, with a strap secured around the ankles to ensure a standardized body position in each scan. The scans were conducted with participants in a fasted state between 07.00-10.00 AM, with empty bladder and wearing only under-wear. Prior to each scan, a phantom scan was run to prevent baseline drifting from affecting analyses. The same technician was used at each time point. Analyses was performed using GE enCORE version 17.0 software (GE Healthcare). Region of interest was customized for covering upper thigh, marked with a sqaure from pubic symphysis to lateral part of tuberculum major, and distal to art. genu.

Muscle thickness (MT) was measured using a B-mode ultra sound unit (SmartUS EXT-1M, Telemed, Vilnius, Lithuania). Participants lay supine in a relaxed position for 20 min before assessments, with their feet strapped in a standardized position. A mark was set on the line 60% of the distance between Spinia Iliac Anterior Superior and the lateral femur condyle. MT of *m.vastus lateralis* was measured applying a water-soluble transmission gel (Aquasonic 100 Ultrasound Transmission Gel; Parker Laboratories Inc., Fairfield, NJ, USA), and a 39 mm 12 MHz ultrasound probe was placed perpendicular to the site of interest without pressing the skin. When the quality of the image was satisfactory, evident as distinct upper and lower muscle fascia, three images were captured, where the probe was relocated to the same position between each image. Position of the probe was marked on the skin and subsequently marked on a transparent paper to ensure similar probe placement for both the right and left *m.vastus lateralis* at subsequent assessments. Analyses were done in ImageJ Fiji [@RN2561] with images cropped and coded to ensure blinding of the assessor.



## Muscle tissue sampling and preparations for downstream analyses

Muscle samples were obtained under local anesthesia (Study I, Xylocaine,
\SI{10}{\mg\per\ml} with adrenalin \SI{5}{\micro\gram\per\ml},
AstraZeneca, Oslo, Norway; Study II, Lidocaine Mylan,
\SI{10}{\mg\per\ml}, Mylan Ireland Ltd, Ireland) with a fine needle
(12-14 gauge; Universal-plus, Medax, Italy) operated with a
spring-loaded instrument (Bard Magnum, Bard Norway AS, Norway). Sampling
was performed as previously described
[@RN824],<!-- Hayot 2005 relible micriboiopsy technique --> with
modifications. Anesthesia was injected in the subcutaneous tissue with
care taken not to inject anesthesia into the muscle itself. Following
pilot experiments we decided not to use an insertion cannula as
described in [@RN824] as the biopsy needle itself could be used to
puncture the skin and muscle fascia. This also resulted in less
discomfort. Several passes through the same skin puncture was made to
obtain sufficient material for downstream analyses. A smaller needle (14
vs. 12 gauge) was used to further minimize discomfort in Study II where
more biopsies were sampled over a shorter time span, with exception from
when material was used for immunohistochemistry. The first biopsy was
sampled at one third of the distance between the patella to the
*anterior superior iliac spinae* with subsequent biopsies sampled
$\sim$\SI{2}{cm} proximal to previous samples. In Study II, samples
obtained more than one week apart were sampled with closer proximity and
distally from previous samples but never at previous sampling sites.

The micro biopsy technique produces smaller samples compared to other
biopsy techniques
[@RN2549]<!--Ekblom describes the Bergström and conchotome -->, and thus
requires several passes to produce sufficient material for multiple
downstream experiments. However, reports confirms that the micro biopsy
technique is comparable to the traditionally used Bergström technique in
several measures of muscle characteristics at the same time as being
well tolerated [@RN824; @RN2553]. Any reported differences in fiber type
distributions between sampling techniques have been suggested relating
to differences in sampling depth [@RN2553; @RN2552].<!-- Hayot -->

For determination of fiber type distributions, a threshold of 200-300
fibers has been suggested as a suitable sample size per specimen as more
fibers does not reduce the variation between dupliacte samples
[@RN874].<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
In Study I, one or several pieces of muscle (total weight
$\sim$\SI{15}{mg}) were chosen per sampling for analysis of fiber type
distributions (described in detail below). The total number of fibers
were counted from these specimens (Figure ref fig). Using an average of
fibers from the first sampling time point the between leg coefficient of
variation was determined to 14% for Type I fibers and 11.3 for type II
fibers. The between leg variation in Type I fibers is similar to what
has been previously reported...

```{=html}
<!-- Latest paper on variability between samples from the same leg and due to number of fibers counted
Appl Physiol Nutr Metab
. 2020 Apr;45(4):368-375. doi: 10.1139/apnm-2019-0263. Epub 2020 Mar 24.

The within-leg was similar to between leg comparison in our samples. This might highlight sampling depth variation in microbiopsy technique in immunohistochemistry ?

To calculate variation in proportions

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3875499/



```




#### Total RNA extraction
```{r rna-muscle-weight}

# Descriptive data on total RNA extractions from both studies

study1_sample <- read_excel("./data/study-1/sample_weight.xlsx") %>%
  mutate(rna = (conc1.5 * 5) * elution.volume, 
         sample = paste(subject, leg, timepoint, sep = "_")) %>%
  dplyr::select(sample, weight, rna) %>%
    mutate(study = "Study1") 


complete_sample <- readRDS("./data/derivedData/study-2/tot-rna.RDS") %>%
  filter(outlier != "out") %>%
  mutate(sample = paste(participant, leg, time, sep = "_")) %>%
  group_by(sample) %>%
  summarise(weight = mean(tissue_weight, na.rm = TRUE), 
            rna = mean(rna, na.rm = TRUE)) %>%
  mutate(study = "Study2") %>%
  rbind(study1_sample) 

## Calculate mean weights and rna per weight ##

desc_to_rna <- complete_sample %>%
  group_by(study) %>%
  summarise(m.weight = mean(weight, na.rm = TRUE), 
            s.weight = sd(weight, na.rm = TRUE), 
            m.rna_weight = mean(rna/weight, na.rm = TRUE), 
            s.rna_weight = sd(rna/weight, na.rm = TRUE)) 




## Figure on RNA descriptives

rna_fig_a <- complete_sample %>%
  ggplot(aes(weight, rna, fill = study)) + 
  geom_point(shape = 21, alpha = 0.2) +
    scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
      scale_y_continuous(limits = c(0, 25000), 
                     expand = c(0,0), 
                     breaks = c(0, 5000, 10000, 15000, 20000, 25000)) +
  dissertation_theme() + 
  theme(axis.title.x = element_blank(), 
        axis.text.x = element_blank(), 
        axis.line.x = element_blank(), 
        axis.ticks.x = element_blank(), 
        legend.position = "none") + 
  labs(y = "RNA (ng)")


rna_fig_b <- complete_sample %>%
  ggplot(aes(weight, fill = study)) + geom_histogram() +
  facet_grid(study ~ .) +
  dissertation_theme() + 
  scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
  theme(legend.position = "none", 
        strip.background = element_blank(),
        strip.text = element_blank()) +

  labs(y = "Counts", 
       x = "Weight (mg)")

# plot_grid(rna_fig_a, rna_fig_b,
#           ncol = 1,
#          rel_heights = c(1, 0.75), 
#          align = "v", 
#          axis = "trbl")
# 


### Calculate RNA per tissue at baseline
# 
# complete_sample %>%
#   separate(sample, into = c("subject", "leg", "timepoint")) %>%
#   filter(timepoint %in% c("S0", "w0")) %>%
#   mutate(rna_corrected = if_else(study == "Study1", rna * 1.125, rna)) %>%
#   lmer(rna ~ study + weight + (1|subject),
#       
#       data = .) %>%
#   summary()
#   
#   ggplot(aes(rna_corrected / weight, fill = study)) + geom_histogram()
#   
#   
#   group_by(study) %>%
#   summarise(rna_weight = mean(rna/weight), 
#             rna_corrected = mean(rna_corrected/weight)) %>%
#   print()
# 
# 
# 

```

Total-RNA was extracted from pre-weighed, frozen muscle samples with a protocol modified from
[@RN2672]<!-- Chomczynski and Sacchi -->
using Trizol reagent (Life Technologies).
Muscle tissue was homogenized in \SI{300}{ul} Trizol with mechanical disruption achieved by Zirconium Oxide Beads (0.5 mm, Next Advance, Inc., New York, USA) and a bead mill (Bullet blender, Next Advance). External, non-mammalian RNA (Lambda PolyA External Standard Kit, Takara Bio Europe, Saint-Germain-en-Laye, France) was added with the initial volume of Trizol to enable per-weight normalization in subsequent analyses. After homogenization, additional Trizol was added to a total volume of \SI{1}{ml}. Phase separation was achieved by centrifugation after addition of chloroform (\SI{200}{ul}). The upper phase (\SI{400}{ul} in Study I; \SI{450}{ul} in Study II) was transferred to a fresh tube and RNA was precipitated using isopropanol (\SI{500}{ul}). After incubation (10 min, room-temperature) and centrifugation (12000 g, 10 min at 4$^{\circ}$C) the resulting RNA pellet was washed three times in chilled 75% ethanol.

As previously mentioned, to minimize discomfort with a larger number of biopsies sampled over a short time, a smaller needle was used for most biopsies in Study II. This generally led to a less tissue used for RNA extractions (Figure XX).  


#### Protein extraction immunoblotting 

<!-- TRaining response -->

Aliquots of muscle-tissue (approximately 25 mg wet weight) were homogenised using a plastic pestle in ice-cold lysis buffer (2 mM HEPES pH 7.4, 1 mM EDTA, 5 mM EGTA, 10 mM MgCl~2~, 1$\%$ Triton X-100) spiked with protease and phosphatase inhibitors (Halt, Thermo Fischer Scientific, Life Technologies AS, Oslo Norway), incubated at $4^{\circ}$ for 1 hr and centrifuged for 10 min at 10 000 g and 4$^{\circ}$C, after which the supernatants were collected.
Total protein concentrations were determined on a 1:10 dilution (Pierce Detergent Compatible Bradford Assay Reagent, Thermo Fischer Scientific). The remaining supernatant was diluted to 1.5 $\mu g  \times \mu l^{-1}$ total protein in lysis buffer and 4X Laemmli sample buffer (Bio-Rad Laboratories AB, Oslo Norway) containing 2-Mercaptoethanol.
Samples were heated to 95$^{\circ}$C for 5 min and stored at -20$^{\circ}$C until further processing.
During analyses, protein samples (20 $\mu g$ of total protein) were separated at 300 V for 30 min using 4-20% gels (Criterion TGX, Bio-Rad), followed by wet transfer to PVDF membranes (0.2 $\mu m$ Immun-Blot, Bio-Rad) at 300 mA for 3 h.
Gel electrophoresis and protein transfer were performed at 4$^{\circ}$C.
Membranes were then stained using a reversible total protein stain (Pierce Reversible Protein Stain, ThermoFischer Scientific) to ensure appropriate protein transfer.
Primary antibodies were purchased from Cell Signaling Technology (Leiden, The Netherlands): mTOR (mTOR^Ser2448^: #5536; pan: #4517), S6 kinase 1 (p85 S6K1^Thr412^: #9206; p70 S6K1^Thr389^: #9234; pan: #2708), ribosomal protein S6 (rpS6^Ser235/236^: #4858; pan: #2317).
Membranes were blocked for 2 h in tris-buffered saline (TBS, 20 mM Tris, 150 mM NaCl) containing $3\%$ bovine serum albumin and $0.1\%$ Tween-20, followed by over-night incubation with primary antibodies targeting either the phosphorylated or non-phosphorylated epitope diluted in blocking buffer, followed by 2 h incubation with secondary horseradish peroxidase-conjugated antibodies diluted in TBS containing 0.1% Tween-20 and $5\%$ skimmed milk. Membranes were washed in TBS containing $0.1\%$ Tween-20 for 6 $\times$ 5 min after incubation with primary antibody, and for 8 $\times$ 5 min after incubation with secondary antibodies. For rpS6 and mTOR antibodies, following chemiluminescent detection (SuperSignal™ West Femto Maximum Sensitivity Substrate, ThermoFischer Scientific), membranes were incubated with hydrogen peroxide (15 min, 37$^{\circ}$C) to inactivate the horseradish peroxidase (HRP), as described by Sennepin *et al.* [-@RN1930], followed by over-night incubation with primary or secondary antibodies as described above. If the phosphorylated epitope was targeted during the first incubation, antibodies for the non-phosphorylated epitope were used in the second and vice versa. HRP inactivation did not affect the phosphospecific to non-phosphorylated signal ratios. Importantly, as this technique did not involve removing the first primary antibody, antibodies from different hosts (mouse or rabbit) were used for phosphorylated and non-phosphorylated epitopes respectively. As the antibody targeting p70 S6K1^Thr389^ had the same host as the pan-antibody, total-protein was used to normalise chemiluminescent signals.  All incubation and washing steps were performed at $4^{\circ}$C using an automated membrane processor (BlotCycler, Precision Biosystems, Mansfield, MA, USA), except for p70 S6K1 experiments, which were performed by hand at room temperature with incubations at $4^{\circ}$C. For mTOR and rpS6, total-protein and chemiluminescence quantification was calculated as the mean value of two separate experiments. S6K1 was quantified once for each phospho-specific antibody. Total-protein content was quantified using ImageJ [@RN2259], and was defined as the mean grey value of the whole well with between-well values subtracted as background. Chemiluminescence signals were quantified using Image Studio Lite (LI-COR Biotechnology, Lincoln, Nebraska USA). 

<!-- Tr010 -->

Protein was extracted from Trizol preparations according to manufacturers instructions and [@RN2050]<!-- Kopec trizol protocol --> with modifications. The remaining aqueous phase was removed and DNA was precipitated by the addition of 300 &#956;l of absolute ethanol followed by gentle centrifugation (2000 g, 5 min at room temperature). An aliquot of the phenol-ethanol phase, corresponding to ~1.75 mg of tissue, was transferred to to a fresh tube. After addition of at least two volumes of isopropanol and incubation (10 min at room temperature), samples were centrifuged (7500 g, 10 min 4&deg;C) and a pellet formed. The pellet was washed three times in 95% ethanol with each wash separated by centrifugation (5000g, 5 min at room temperature). After the last wash all liquid was removed and 45 &#956;l of Kopec buffer [@RN2050] was added (5% SDS, 10 mM Tris, 140 mM NaCl and 20 mM EDTA, pH 8; containing protease and phosphatase inhibitors). Pellets were incubated at 50&deg;C for three hours after which the majority of samples were dissolved. Any undissolved material was sedimented by centrifugation (10000 g, 10 min at room temperature). Protein concentrations were measured (Pierce Detergent Compatible Bradford Assay, ThermoFisher Scientific). Sample were normalized in 4X Laemmli buffer (Bio-Rad Norway AS, Oslo, Norway), boiled (95&deg;C, 5 min) and stored at -20&deg;C before later use.

## RNA analysis

<!-- TR010 -->
Complementary DNA (cDNA) was synthesized in technical duplicates from 500 ng of total RNA using random hexemer and anchored Oligo-dT primers (Thermo Fisher Scientific) together with Superscript IV (Thermo Fisher Scientific) according to manufacturer's instruction. qPCR reactions were performed with diluted cDNA (2 &#956;l, 1:25 dilution), a SYBR-green based commercial master mix (PowerUp™ SYBR™ Green Master Mix, Thermo Fisher) and, target-specific primers (500 nM) in 10 &#956;l reaction volumes using a real-time detection system (QuantStudio 5 Real-Time PCR System, Thermo Fisher Scientific). Fast cycling was used (1 sec denaturing, 30 sec annealing) after UNG (2 min, 50&deg;C) and polymerase  (2 min, 95&deg;C) activation. Melt curves were collected from all reactions to confirm single product amplification. Primers were further evaluated by agarose gel electrophoresis which confirmed primer sizes and non-template control experiments confirming no amplification without template. Primer sequences and their respective average performances are shown in Table 2.

<!-- Training response -->

Approximately 25 $mg$ of wet muscle-tissue was homogenised in a total volume of 1 ml of TRIzol reagent (Invitrogen, Life technologies AS, Oslo, Norway) using 0.5 mm RNase-free Zirconium Oxide beads and a bead homogeniser (Bullet Blender, Next Advanced, Averill Park, NY, USA) according to the manufacturer’s instructions.
In order to enable analysis of target gene-expression per-unit tissue weight, an exogenous RNA control ($\lambda$ polyA External Standard Kit, Takara Bio Inc, Shiga, Japan) was added at a fixed amount ($0.04  ~ng  \times ml^{-1}$ of Trizol reagent) per extraction prior to homogenisation, as previously described
[@RN1162;<!-- Ellefsen Improved normalization using an external RNA control -->
@RN1512]<!-- Ellefsen Irisin and FNDC5 -->.
Following phase-separation, 400 $\mu l$ of the upper phase was transferred to a fresh tube and RNA was precipitated using isopropanol.
The resulting RNA pellet was washed three times with 70% EtOH and finally eluted in TE buffer.
RNA quantity and purity was evaluated using a spectrophotometer, all samples had a 260/280 $nm$ ratio > 1.95. RNA was stored at -80$^{\circ}$C until further processing. In the analysis of total RNA content per-unit tissue weight, one sample was excluded prior to analysis due to negative deviation from the expected value based on the relationship between sample weight and RNA content suggesting sample loss in washing steps. 
RNA integrity was assessed by capillary electrophoresis (Experion Automated Electrophoresis Station using RNA StdSens Assay, Bio-Rad) with average integrity scores (RQI) 
Five-hundred nanograms of RNA were reverse transcribed using anchored Oligo-dT, random hexamer primers (Thermo Scientific) and SuperScript IV Reverse Transcriptase (Invitrogen) according to manufacturer's instructions.
All samples were reverse transcribed in duplicates and diluted 1:50 prior to real-time polymerase chain reaction (qPCR).
qPCR reactions were run on a fast-cycling real-time detection system (Applied Biosystems 7500 fast Real-Time PCR Systems, Life technologies AS), with a total volume of 10 $\mu l$, containing 2 $\mu l$ of cDNA, specific primers (0.5 $\mu M$ final concentration) and a commercial master mix (2X SYBR Select Master Mix, Applied Biosystems, Life technologies AS). qPCR reactions consisted of 40 cycles (three seconds 95$^{\circ}$C denaturing and 30 seconds 60$^{\circ}$C annealing).
Melt-curve analyses were performed for all reactions to verify single-product amplification. Gene-specific primers were designed for all targets using Primer-BLAST [@RN1815] and Primer3Plus [@RN1816] and ordered from Thermo Scientific, except for the external RNA control, for which primers were supplied with the kit.
Raw fluorescence data was exported from the platform specific software and amplification curves were modelled with a best-fit sigmoidal model using the qpcR-package [@RN1768] written for R [@R-base]. Threshold cycles (Ct) were estimated from the models by the second-derivate maximum method with technical duplicates modelled independently.
Amplification efficiencies were estimated for every reaction [as described by @RN1934; implemented in @RN1768]. For every primer pair, mean amplification efficiencies ($E$) were utilised to transform data to the linear scale using $E^{-Ct}$. Primer sequences and primer characteristics (i.e. average primer efficiencies and Ct values) are presented in Table 2. Gene expression data was log-transformed prior to statistical analysis. As Ct-values, but not efficiencies are related to RNA integrity [@RN2248], RQI scores were used in the statistical treatment of qPCR data to control for potential degradation effects on a by target basis (see below). 




### Hormonal measurements
Hormone analyses were performed on blood samples collected at five time points: alongside muscle biopsies (Figure 1, four sampling events) and 10 minutes after completion of the fifth training session. Samples were drawn from the antecubital vein into serum-separating tubes and kept at room temperature for 30 min before centrifugation (1500 g, 10 min). Serum was immediately aliquoted and stored at -80$^{\circ}$C until further processing. Serum concentrations of total testosterone, cortisol, growth hormone and insulin-like growth-factor 1 (IGF-1) were measured on an Immulite 1000 analyzer, using kits from the Immulite Immunoassay System menu (Siemens Medical Solutions Diagnostics, NY, USA), performed according to manufacturer’s protocols. Serum Vitamin D (S-25-OH-D) levels were measured in samples collected before and after the intervention using a electrochemiluminescence immunoassay (Roche Cobas Vitamin D total assay, Roche Diagnostics GmbH., Mannheim, Germany) using automated instrumentation (Roche Cobas 6000’s module e601, Roche Diagnostics GmbH., Mannheim, Germany). 




## Statistics and data analysis

TO DO:

-   For methods discussion, compare product length, efficiencies and ct
    values in relation to RQI-values. See Fleige 2006 for reference.


### Normalization

-   An external reference gene was added at a constant amount in Trizol
    preps
-   A normalization factor was used to express relative target gene
    abundance per-weight tissue.
-   In qPCR the linearised expression (effectivety \^cq) was used to
    express the fraction of external reference per total RNA.
-   In RNA-seq the external reference gene was sequenced and counts were
    used to express external RNA as a fraction of total RNA.
-   In both cases the normalization factor was calculated as mw \*
    counts.

A simulation to see that this is equivalent to tissue used in prep when
no measurement errors exists.

```{r}

library(tidyverse)

expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 250, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) 



expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 50, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) %>%
         ggplot(aes(rna.mg, mg.inprep,  color = mg)) + geom_point(size = 2)




```

## Meta-analysis of within-session training volume

### Literature search, inclusion criteria and coding of studies

A first set of studies were coded based previously published
meta-analyses [@RN1767; @RN789]. For more recent studies, PubMed, Google
Scholar and SportDiscuss searches were made with search terms being
"training volume", "resistance training", "strength training", "set",
"muscle strength", "muscle hypertrophy" used in different combinations.
Studies examining the effect of within-session training volume on muscle
strength and mass, with all other training variables kept constant
between study groups were considered for inclusion in the meta-analysis.
Studies were further assessed for inclusion based on criteria being; (i)
participants described as healthy without medications affecting muscle
metabolism, (ii) interventions lasting at least 6 weeks and (iii) RT
performed without additional stimuli (e.g. blood flow restriction) at
intensities above 65% of 1RM or 20RM.

All available outcome measures of muscle mass and strength gains in
response to RT were extracted from each study with exception of outcomes
reported both as summaries and individual measures (e.g. muscle
thickness reported as individual muscles and summarized for the whole
muscle group). In such cases the summary was used as outcome. Weekly
training volume was calculated as product of weekly sessions, number of
sets and exercises for each muscle group assessed for muscle hypertrophy
or strength gains. An intervention average of weekly sessions was used
when the number of sessions per week differed over the course of the
intervention. An exercise was assumed to influence an outcome when it
targeted prime movers also assessed for strength or muscle hypertrophy
measures. Participant characteristics were coded based on sex (male,
female or mixed when a study failed to discern between male and
females), age (young, middle-aged, old or mixed), body-mass index (BMI,
calculated from average body mass and height when BMI values were not
available), training status (trained, \> 1 session per week during the
last 6 months leading up to the intervention; untrained \< 1 session).
Study groups were considered independent also in studies utilizing
within-participant models.

### Calculations of effect sizes and statistical analysis

Group-wise effect sizes were calculated for each outcome measure based
on the within-group change score pre- to post-training divided by the
pre-training standard deviation (SD). Pre-training SD's were calculated
as a pooled SD within outcome and study. Variances of the effect size
were calculated using an average effect size across all outcomes within
muscle strength or mass, and correlations specific to each measurement
type (isokinetic-, isometric- or repetition maximum strength tests;
muscle thickness, magnetic resonance imaging, dual energy X‐ray
absorptiometry) estimated from previous studies.\
A correction factor () was applied to both effect sizes and their
variances.

Mixed-effects meta-regression models were used to model the effect of
weekly number of sets on RT-induced muscle mass and strength gains.
Models were fitted in a Bayesian framework using the brms-package
[@RN2562].

<!--chapter:end:03-Methods.Rmd-->

# Results and Discussion

```{r settings, include=FALSE}

library(nlme); library(kableExtra)


```


## Effects of different training volume on changes in muscle size and function

```{r comparing-avg-strength-csa, echo=FALSE, warning=FALSE, message=FALSE}


source("./R/study-1/mr-and-dxa-data.R")


participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


### Overall % increase in CSA (not corrected for baseline values)
over.all.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(subject) %>%
  summarise(mean.csa = mean(change, na.rm = TRUE)) %>%
  ungroup() %>%
  summarise(m.csa = mean(mean.csa),
            s.csa = sd(mean.csa))

sets.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(sets) %>%
  summarise(m.csa = mean(change),
            s.csa = sd(change))



### Baseline corrected % increase
csa.corrected.data <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>% # Calculates % change
  group_by(sex) %>%
  mutate(pre = pre-mean(pre)) %>% # Mean centered baseline 
  # (no need for sex-diff in model)
  group_by(subject, sex) %>% # Keeps sex for test in model
  summarise(mean.csa = mean(change, na.rm = TRUE),
            mean.baseline = mean(pre)) # Average over both conditions 



## Anocova with baseline values as co-variate
csa.corr.m1 <- lm(mean.csa ~  mean.baseline,
                  data = csa.corrected.data)

csa.m1 <- lm(mean.csa ~  1,
                  data = csa.corrected.data)



csa.corr.confint  <- round(confint(csa.corr.m1)[1,],1)
csa.corr.estimate <- round(summary(csa.corr.m1)$coef[1,1],1)




csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  group_by(timepoint, sets, sex) %>%
  summarise(m=sprintf("%.1f", round(mean(CSA.avg),1)),
            s=sprintf("%.1f", round(sd(CSA.avg),1)))


csa.r <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject, sex, sets, csa.change) %>%
  spread(sets, csa.change)

csa.r.model <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject,leg, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject,leg, sex, sets, pre, csa.change) %>%
  group_by(sex) %>%
  mutate(pre.m = pre-mean(pre, na.rm = TRUE), 
         sets = factor(sets, levels = c("single", "multiple"))) 

#### Change to ancova ###


csa.m0 <-nlme::lme(csa.change ~ sets +  pre.m,
                random = list(subject = ~1),
                na.action = na.omit, 
                data = csa.r.model, method = "ML", 
                control=list(msMaxIter=120,
                             opt = "nloptwrap", msVerbose = FALSE))




csa.estimate <- sprintf("%.2f",round(coef(summary(csa.m0))[2,1], 2))
csa.confint <- intervals(csa.m0)$fixed[2, c(1,3)]

### t-test Does not change the conclusion 
# csa.ttest <- with(csa.r, t.test(multiple,single, paired=T))
# csa.estimate <- sprintf("%.2f",round(csa.ttest$estimate, 2))
# csa.confint <- csa.ttest$conf.int


csa.corr <- cor.test(csa.r$multiple, csa.r$single)


female.t <- with(csa.r[csa.r$sex=="female",], t.test(multiple,single, paired=T))
female.estimate <- sprintf("%.2f",round(female.t$estimate, 2))
female.confint <- female.t$conf.int


male.t <- with(csa.r[csa.r$sex=="male",], t.test(multiple,single, paired=T))
male.estimate <- sprintf("%.2f",round(male.t$estimate, 2))
male.confint <- male.t$conf.int

#### Strength data for overall stats



rm_str <- read_excel("./data/study-1/strength/strengthTests.xlsx") %>%
        filter(!(exercise %in% c("benchpress", "legcurl"))) %>%
        inner_join(participants) %>%
        filter(include == "incl") %>%

  dplyr::select(subject, sex, sets, exercise, timepoint, load)%>%
  mutate(load = as.numeric(load),
         sets = factor(sets, levels = c("single", "multiple"))) %>%
  spread(timepoint, load) %>%
  mutate(baseline = pmax(pre, session1)) %>%
  dplyr::select(subject, sex, sets, exercise, baseline, week2, week5, week9, post) %>%
  gather(week, load, baseline:post) %>%
  mutate(week = factor(week, levels = c("baseline", "week2", "week5", "week9", "week12", "post"))) %>%
  group_by(exercise) %>%
  mutate(str = load / max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) 

# load cybex data 
load("./data/study-1/strength/cybex.rda")


cybex <- data.frame(cybex) %>%
        mutate(leg = if_else(leg == "Left", "L", "R")) %>%
        inner_join(participants) %>%
        filter(include == "incl", 
               timepoint %in% c("pre","session1", "post")) %>%

        filter(!(subject=="FP6" & timepoint == "post" & speed == 240))  # This removes one record, exclusion due to error in measurement, see protocol FP6


# Clean cybex data set and combine with 1RM 
total_strength <- cybex %>%
  ungroup() %>%
  dplyr::select(subject, timepoint, sets,speed, sex, torque.max) %>%
  spread(timepoint, torque.max) %>%
  rowwise()%>%
  mutate(baseline = max(pre, session1, na.rm=T)) %>%
  ungroup() %>%
  dplyr::select(subject, sex, sets, exercise = speed, baseline,  post) %>%
  gather(week, load, baseline:post) %>%
  group_by(exercise) %>%
  mutate(str = load/max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) %>%
  rbind(rm_str) %>% # This includes all strength tests
  mutate(ex_fam = if_else(exercise %in% c("legext", "legpress"), 
                          "rm", if_else(exercise == 0, "isom", "isok"))) %>%
  group_by(subject, sex, sets, week, ex_fam) %>%
  summarise(str = mean(str, na.rm = TRUE)) %>%
  group_by(subject, sex, sets, week) %>%
  summarise(str = mean(str, na.rm = TRUE)) 


strength.stats <- total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  group_by(sex) %>%
  mutate(BASELINE = baseline - mean(baseline)) %>%
    group_by(sex, subject) %>%
    summarise(increase = mean(increase, na.rm = TRUE),
              BASELINE = mean(BASELINE, na.rm = TRUE)) 




## ANCOVA for the strength 1RM
strength.m1 <- lm(increase ~ sex + BASELINE,
                  data = strength.stats)

strength.corr.confint  <- round(confint(strength.m1)[1,],0)
strength.corr.estimate <- round(summary(strength.m1)$coef[1,1],0)



### Correlation in strength gains ###

str.corr.data <-  total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject, sex, sets, increase) %>%
  spread(sets, increase) 

str.corr <- cor.test(str.corr.data$single,
                          str.corr.data$multiple)

#### Correlation strength and csa increases

# Average increases

csa.str.corr.data <- total_strength %>%
  spread(week, str) %>%
  mutate(str.rel.change = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject,  sets, sex, sets,baseline, str.rel.change) %>%

  inner_join(mr.results %>%
                mutate(condition=leg) %>%
                inner_join(participants) %>%
                filter(include=="incl") %>%
                dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
                spread(timepoint, CSA.avg) %>%
                mutate(csa.rel.change = 100 * ((post/pre)-1)) %>%
                dplyr::select(subject, sets, csa.rel.change)) %>%
        ungroup() 





# The code below could possibly be used to calculate the relative contribution of 
# csa increase to muscle strength increases


# m <- lme4::lmer(str.rel.change ~ csa.rel.change + sets + (1|subject), 
#                 data = csa.str.corr.data)
# 
# 
# 
# mx.null <- lme4::lmer(str.rel.change ~ 1 + (1|subject), data = csa.str.corr.data)
# mx.csa <- lme4::lmer(str.rel.change ~ csa.rel.change  + (1|subject), data = csa.str.corr.data)
# mx.sets <- lme4::lmer(str.rel.change ~ sets + (1|subject), data = csa.str.corr.data)
# 
# 
# 
# boot.fun <- function(x) {
#         data.frame(nlme::VarCorr(x))[2, 4]
# }
# 
# 
# null.boot <- lme4::bootMer(mx.null, boot.fun, nsim = 500, parallel = "snow", cl = 12)
# csa.boot <- lme4::bootMer(mx.csa, boot.fun, nsim = 500)
# sets.boot <- lme4::bootMer(mx.sets, boot.fun, nsim = 500)
# 
# 
# 
# qt.null <- quantile(null.boot$t, c(0.025, 0.975))
# qt.csa <- quantile(csa.boot$t, c(0.025, 0.975))
# qt.sets <- quantile(sets.boot$t, c(0.025, 0.975))
# 
# 
# results <- data.frame(variable = c("sets", "csa"),
#                            m = c(1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4],
#                                  1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4]),
#                            lwr = c(1- qt.sets[1] / qt.null[1],
#                                    1- qt.csa[1] / qt.null[1]),
#                            upr = c(1- qt.sets[2] / qt.null[2],
#                                    1- qt.csa[2] / qt.null[2]))
# 
# 
# 
# 
# 
# csa <- 1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# sets <- 1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# exp <- 1 - data.frame(nlme::VarCorr(mx.exp))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# 

# Average csa and strength gains

avg.str.csa <- csa.str.corr.data %>%
        group_by(subject) %>%
        summarise(str = mean(str.rel.change), 
                  csa = mean(csa.rel.change))


csa.str_corr <- cor.test(avg.str.csa$csa, avg.str.csa$str)

## Comparisons with other published studies on per session gains in csa and strength

### Per session ##
per.session <- read_excel("./data/study-1/trainingdata.xlsx", sheet = 1) %>%
  filter(!is.na(date)) %>%
  group_by(subject) %>%
  summarise(sessions = n()) %>%
        
inner_join(csa.str.corr.data) %>%

  mutate(str.change.day = str.rel.change/((sessions/31) * 12*7),
         csa.change.day = csa.rel.change/((sessions/31) * 12*7), 
         str.session = str.rel.change/sessions,
          csa.session = csa.rel.change/sessions ) %>%
   dplyr::select(-baseline) %>%
  ungroup() %>%
        pivot_longer(names_to = "variable", values_to = "gain", cols = str.rel.change:csa.session) 
       

sum.stats <- cbind(per.session %>%
        group_by(sets, variable, sex) %>%
        summarise(mean = mean(gain), 
                  sd = sd(gain)) %>%
        arrange(variable) %>%
        data.frame(), 
        data.frame(reference.mean = c(0.11, 0.11, 0.11, 0.11,
                              NA, NA, NA, NA, 
                              0.0825, 0.1375,0.0825, 0.1375, 
                              NA, NA, NA, NA, 
                               NA, NA, NA, NA,
                              0.6675, 0.475, 0.6675, 0.475), 
           reference.range = c("[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "", "", "", "",
                               "(0.22)b", "(0.14)b",  "(0.22)b", "(0.14)b", 
                               "", "", "", "",
                              "", "", "", "",
                               "(0.35)b", "(0.22)b","(0.35)b", "(0.22)b" ))) %>%
        mutate(m.sd = if_else(mean > 6, paste0(sprintf("%.1f",mean), " (", sprintf("%.1f",sd), ")"), 
                              paste0(sprintf("%.2f", mean), " (", sprintf("%.2f",sd), ")")), 
               reference = if_else(is.na(reference.mean), "",
                                   paste0(sprintf("%.2f",reference.mean), " ", reference.range))) %>%
        filter(!(variable %in% c("str.change.day") )) %>%
                       dplyr::select(variable, sex, sets, m.sd, reference) %>%
        
        mutate(variable = factor(variable, levels = c("csa.rel.change", 
                                                      "csa.change.day", 
                                                      "csa.session", 
                                                      "str.rel.change", 
                                                      "str.session"), 
                                 labels = c("CSA %-change", 
                                            "CSA %-change day", 
                                            "CSA %-change session", 
                                            "Average strength %-change", 
                                            "Average strength %-session")), 
               sex = factor(sex, levels = c("female", 
                                            "male"), 
                            labels = c("Female", "Male")), 
               sets = factor(sets, levels = c("single", "multiple"), 
                             labels = c("LOW", "MOD"))) 

## Ahtiaianen on average  20 weeks with 2 sessions per week = 40 sessions

## Muscle hypertrophy
## In the younger cohorts, male average 5.5% (5.6) = 5.5/40 ~ 0.1375 per session
# in females 3.3 (8.6)% = 0.825
# 8.6/3.3 * 3.3/40
# 5.6/5.5 * 5.5/40
# Muscle strength in younger cohorts
#  male    19% (8.6)/40 sessions ~  0.5 % per session
#  female  26.7% (13.9)/40 ~ 
# 8.6/19 * (19/40)
# 13.9/26.7 * 26.7/40

# Wernbom increase per day in quadriceps CSA per day was 0.06%







```


In Study I, the average increases (Table \@ref(tab:csa-str-tab)) in muscle strength and mass in each volume condition corresponded to what could be expected based on previous studies
[@RN2007;<!-- Wernbom 2007 -->
@RN1741].<!-- Ahtiainen  -->

(ref:wernbom-citation) @RN2007

(ref:ahtiainen-citation) @RN1741

```{r csa-str-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

sum.stats %>%
        arrange(variable, sex, sets) %>%
        kable(format = "latex", booktabs = TRUE, linesep = "", 
              col.names = c("", "Sex", "Volume condition", "Mean (SD)", "Reference"), 
              caption = "Training induced changes in muscle CSA and average strength in Study I")  %>%
        collapse_rows()  %>%
        kable_styling(font_size = 7) %>%
        add_footnote(label = c(paste0("Estimates from Wernbom et al. ", "(ref:wernbom-citation)"),
                               paste0("Estimates from Ahtiainen et al. ", "(ref:ahtiainen-citation")), 
                     notation = "alphabet")

```

Average within participant differences in responses between LOW and MOD were consistent across measures of muscle hypertrophy and strength gains (Figure \@ref(fig:comb-fig-s1)). These differences were in agreement to what could be expected based on published meta-analyses
[@RN2492;<!-- Ralston weekly sets volume, medium/higher volume better than low volume for strength gains  -->
@RN789;<!-- Krieger muscle hypertrophy -->
@RN793;<!-- Kriger muscle strength -->
@RN1767].<!-- Schoenfeld  -->
Taken together, these observations confirmed the efficacy of training programs in general and a dose-response with regard to within-session exercise volume.

In Study II, training efficacy was assessed by comparing outcomes to a non-training control group. The training group displayed increases compared to the control group for both strength muscle thickness measures.

```{r comb-fig-s1, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences in training induced relative changes in muscle mass and strength measures. Estimates are derived from ANCOVA models controling for baseline values and sex.", fig.scap = "Differences in training induced changes to muscle mass and strength measures between volume conditions in Study I", fig.height=3, fig.width=5, fig.align='center'}

library(ggplot2); library(ggtext)

# The figure and estimates are from ./R/study-1/muscle-strength-size.R
muscle_strength_fig <- readRDS("./data/derivedData/study1_muscle-strength-size/muscle-strength-fig.RDS")

muscle_strength_fig
```

## Acute effects of diffrent training volume on determinants of muscle protein synthesis
<!--
Higher volume of acute resistance exercise led to increased phosphorylation of S6K1, ribosomal protein S6 and mTOR. These phosphorylation sites are indicative of activity along the mTORC1 pathway but mechanistic studies show that multiple sites exists for signal redundancy 
Inhibiting mTOR by Torin1 after electrically stimulated muscle contractions still led to S6K1 phsophorylation<!-- doi: 10.1016/j.cellsig.2013.05.005. -->

<!--
Regardless of upstream paralell signaling patterns the present result confirm volume dependency in a pathway controlling protein synthesis and ribosome biogenesis.

A limitation in the present study is that the limited panel of antibodies and time-points used to capture volume 
-->






<!--chapter:end:04-results.Rmd-->

# General Discussion
<!--
Constant volume protocols with higher volume within a tolerable range produces favorable adaptations regarding muscle hypertrophy. 
-->

<!--chapter:end:05-Discussion.Rmd-->

# Conclusion {-}

If we don't want Conclusion to have a chapter number next to it, we can add the `{-}` attribute.

**More info**

And here's some other random info: the first paragraph after a chapter title or section head _shouldn't be_ indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there.


<!--chapter:end:06-conclusion.Rmd-->

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->

\backmatter

<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->

# References {-}
<!--
This manually sets the header for this unnumbered chapter.
-->
\markboth{References}{References}
<!--
To remove the indentation of the first entry.
-->
\noindent

<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->

\setlength{\parindent}{-0.20in}
\setlength{\leftskip}{0.20in}
\setlength{\parskip}{8pt}



<!--chapter:end:99-references.Rmd-->

