---
author: 'Daniel Hammarström'
date: 'May 20xx'
year: '2019'
isbn: 'Provided by the library'
place: 'Stockholm'
printedby: 'Printer service, Stockholm, 2019'
coverinfo: ' '
sernr: '999' # Add number in "Avhandlingsserie" 
title: 'Determinants of intra-individual variation in adaptability to resistance training of different volumes with special reference to skeletal muscle phenotypes'
subtitle: 'Subtitle of my thesis' # currently not in use ##
knit: "bookdown::render_book"
site: bookdown::bookdown_site
output: 
  thesisdowngih::thesis_pdf: default
  latex_engine: xelatex
  citation_package: natbib
#  thesisdown::thesis_gitbook: default
#  thesisdown::thesis_word: default # The word and e-pub versions are not tested.
#  thesisdown::thesis_epub: default
# If you are creating a PDF you'll need to write your preliminary content here or
# use code similar to line 20 for the files.  If you are producing in a different
# format than PDF, you can delete or ignore lines 20-31 in this YAML header.
abstract: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-abstract.Rmd"), collapse = '\n  ')`
# If you'd rather include the preliminary content in files instead of inline
# like below, use a command like that for the abstract above.  Note that a tab is 
# needed on the line after the |.
listofpapers: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-listofpapers.Rmd"), collapse = '\n  ')`
dedication: |
  To Tage. 
# preface: | # If you wish to add a preface, remove comments on these lines
#   This is an example of a thesis setup to use the reed thesis document class
#   (for LaTeX) and the R bookdown package, in general.
defence: |
  `r if(knitr:::is_latex_output()) paste(readLines("00-defence.Rmd"), collapse = '\n  ')`
bibliography: 
        - bib/complete-bib.bib
# Download your specific bibliography database file and refer to it in the line above.
csl: csl/vancouver.csl
# Download your specific csl file and refer to it in the line above.
lot: true
lof: true
#space_between_paragraphs: true
# Delete the # at the beginning of the previous line if you'd like
# to have a blank new line between each paragraph
header-includes:
#- \usepackage[utf8]{inputenc}
#- \usepackage{erewhon}
#- \usepackage{lipsum}
#  \usepackage[utf8]{inputenc}
#  \usepackage[T1]{fontenc}
#  \usepackage{libertine}
#  \usepackage[svgnames]
#- \usepackage{GoudyIn}
#- \usepackage{xcolor}
  \usepackage{lettrine}
  \usepackage{booktabs}
  \usepackage{longtable}
  \usepackage{array}
  \usepackage{multirow}
  \usepackage{wrapfig}
  \usepackage{float}
  \usepackage{colortbl}
  \usepackage{pdflscape}
  \usepackage{tabu}
  \usepackage{threeparttable}
  \usepackage{threeparttablex}
  \usepackage[normalem]{ulem}
  \usepackage{makecell}
  \usepackage{siunitx}

---


<!--
Above is the YAML (YAML Ain't Markup Language) header that includes a lot of metadata used to produce the document.  Be careful with spacing in this header!

If you'd prefer to not include a Dedication, for example, simply delete lines 27 and 28 above or add a # before them to comment them out.  If you have other LaTeX packages you would like to include, delete the # before header-includes and list the packages after hyphens on new lines.

If you'd like to include a comment that won't be produced in your resulting file enclose it in a block like this.
-->


```{r include_packages, include = FALSE}
# This chunk ensures that the thesisdown package is
# installed and loaded. This thesisdown package includes
# the template files for the thesis.
library(devtools)
library(thesisdowngih)
```


```{r create_package_bib, include = FALSE}
knitr::write_bib(c("base", "qpcR"), "./bib/packages.bib") ### This creates a bibfile for R packages
```

\setcounter{DefaultLines}{3}


# Introduction
\lettrine{S}keletal muscle health is essential for physical independence. In a lifespan perspective, measures of muscle mass and/or strength are inversely associated with mortality
[@RN2512;<!-- Ran 2018 Low muscle strength was independently associated with all cause mortality -->
@RN2808; <!-- Muscular strength and mortality meta analysis higher levels of strength lower risk of death -->
@RN2513; <!-- Fukasawa 2016 81 hemodialysis patients, low thigh muscle mass associated with all cause and cv mortality-->
@RN2514; <!-- Miyake 2019 Low muscle mass associated with ACM in type 2 diabetes, SMI = appendicular skeletal muscle mass kg / height m^2 -->
@RN2809;<!-- Ruiz n ~ 8000, strength inversely related to all cause and cancer  mortality in men -->
@RN2515; <!-- Szulc 2010 715 men Loss of appendicular muscle mass predicts all cause mortality in elderly -->
@RN2516]<!-- Abramowitz 2018 Higher ASMI was independently associated with lower mortality -->
and disability
[@RN2517].<!-- Janssen 2002 Low muscle mass associated with disabilities -->
Besides adverse associations between of low muscle mass and strength and clinical conditions, muscle weakness also accounts for increased health care costs in patient populations
[@RN2532;<!-- Sousa 2016 --> 
@RN2184].<!-- Pinedo-Villanuev Excess health care costs associated with muscle weakness was estimated to 1.3 billion punds in UK in population above 70 yrs   -->
The intercept between muscle mass, muscle function and health status is interrelated with variables such as age and primary illness or injury
[@RN763].<!-- Wolfe 2006 amino acid availability in acute illness and injury, link between metabolic functioning of mm.-->
This highlights that interventions designed to increase muscle mass and strength are likely to prevent adverse health outcomes across the lifespan. A higher level of muscle mass and functional capacity would counteract the effects of muscle loss due to illness, age or inactivity.

Although a large degree of the observed variations in lean mass and strength are attributed to genetic components 
[@RN2526; <!-- Arden genetic factors explain leg extensor strength (60%), lean body mass (52%) and grip strength (36%) -->
@RN2527],<!-- Roth review, > 50% genetic component in muscle mass -->
environmental factors also contribute, leaving a window of opportunity to increase muscle mass and functional capacity. Among factors affecting muscle mass and functioning are nutrition and pharmacological agents. However, physical activity and specifically systematic resistance training of sufficient volume, intensity and frequency provides a stimulus that promote morphological and functional changes to the human neuromuscular system without adverse side-effects. Irrespective of age, resistance training generally leads to increased muscle mass and strength
[@RN1741;<!-- Ahtiainen 2016 pooled analysis including controls   -->
@RN2534]<!-- Grgic 2020 > 75 of age increase strength and muscle mass relative to control and training is safe -->
and is considered safe when performed in a well organized manner
[@RN2534;<!-- Grgic safety in elderly -->
@RN2536].<!-- Faigenbaum 2010 resistance training in young safety -->

Resistance training can be modulated indefinitely through combined variations of training variables such as frequency, intensity and volume 
[@RN1;<!-- ACSM guidelines -->
@RN798].<!-- Bird 2005 acute programming variables -->
Well designed training prescriptions should incorporate information about the current state and goals of the trainee to maximize the potential outcome of the training program
[@RN1;<!-- ACSM guidelines -->
@RN2538;<!-- Feigenbaum consider personal status in programming-->
@RN798].<!-- Bird 2005 acute programming variables -->
Training volume has received particular attention in the scientific community for many reasons. Evidence suggests that exercise volume affects selected molecular determinants of muscle hypertrophy in a dose-dependent manner
[@RN791; <!-- Burd  -->
@RN784; <!-- Terzis 2010 training volume p70 and s6 volume dependent -->
@RN1837].<!-- Ahtiainen 2015 10 X 10 > 5 X 10 in p70, S6 phospho --> 
Such effects are believed to facilitate long-term training effects as training programs with higher volume generally result in higher gains in muscle mass and strength with little evidence of differences between age groups or participants with different training backgrounds
[@RN793; <!-- Krieger 2009 meta on strength, 2-3 set more efficient than 1 set but not 4-6 sets -->
@RN789; <!-- Krieger 2010 meta analysis on training volume for hypertrophy, multiple sets more effective than signle set -->
@RN1767]. <!-- Shoenfeld 2016 weekly number of sets related to gains in muscle mass  -->  
A consequence of a more extensive training program is the increased time required to complete such a program. As time constraints has been reported as a limiting factor for engaging in physical activity
[@RN2063]<!-- Time constraints limits participation in physical activity -->
some merit can be given to arguments against guidlines suggesting higher volume in resistance training prescription 
[@RN794;<!-- Carpinelli argues that single set programs are as effective as multiple set programs -->
@RN2538].<!-- Feigenbaum, argues that more volume does not outweigh the time cost -->
From an individual perspective, training prescription that balances time-requirement with efficacy presumably increases the likelihood of participation in physical activity [@RN2063]<!-- Choi time-constrains and expectations of outcomes are related to participation -->.
From a more general perspective, increased knowledge about mechanisms governing responses to physical training could improve training prescription also for individuals and populations that experience attenuated benefit of resistance training
[@RN2547]. <!-- Pickering 2019, non-responders might not exists, changing training prescription could increase efficacy -->
<!-- Add "Individualized prescrioption is needed -->
The overreaching goal of the present thesis is to contribute to understanding individualized training loads. To this end, training volume was used to study the effects of variable training<!-- Change this? --> stimulus in within-participant models of exercise-training.   





<!--chapter:end:index.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
---

# Background
```{r Backgroundsettings, echo = FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R"); source("./R/themes.R")
```

## Resistance-exercise prescription, a historical note and current challenges
Recommendations of systematic physical exercise with the purpose of improving health or physical performance has long been part of human culture, evident from records dating back to ancient Chinese, Indian and Greek civilizations
[@RN2640].<!-- Tipton Ancient China India and greek/roman written records on exercise precription -->
Today's exercise-training prescription still bear traces of ideas from these eras, further developed during the renaissance and formalized in systems like German Turnen and Ling gymnastics during the nineteenth century 
[@RN2663].<!-- Pfister 2003 describes differences and common ancestry of gymnastics and sports during 1700-1800 -->
German Turnen as a system of physical activities was established in a time when Germany developed from aristocracy to a unified nation.
Turnen not only served as a system of preparing men to fight for the developing nation but also as a way of establishing a national identity.
Ling gymnastics shared common origins with German Turnen and also served as a system of military preparation.
However, Ling also established systems for medical, pedagogical and aesthetic gymnastics.
Especially Ling's medical gymnastics was important for the development of modern exercise prescription as it was scientifically oriented, based on the physiological and medical understanding of that time [@RN2663].
The medical gymnastics of the nineteenth century is referenced in twentieth century texts on therapeutic exercise prescription
[@RN2634].<!-- Nicoll 1943 Dosage of exercise therapy "never exercised against more than 25% of their all out efficiency" with increased exercise periods and decreased rest periods as progression -->

With the introduction of "heavy resistance exercises" for development of muscle strength and mass after injury, DeLorme outlined a system on which modern resistance-exercise prescription is based [@RN2633].<!-- DeLorme 1945 case reports of systematic resistance training for restoration of muscle power and mass. First mention of RM -->
DeLorme published his system short after the Second World War [@RN2633]
during which he, as a newly graduated physician, had been working with war injury rehabilitation
[@RN2639]<!-- Todd 2012 DeLorme biography -->.
Inspired by practitioners of weight training [@RN2639],<!-- Todd 2012 DeLorme biography --> 
DeLorme specifically emphasized high-resistance, low-repetition exercises where progression was achieved with increased resistance [@RN2633] as opposed to previous recommendations of endurance-like exercise where progression was achieved through increased number of repetitions
[@RN2634].<!-- Nicoll 1943 -->
DeLorme originally used the term "heavy resistance exercises" in contrasts to low-resistance exercises [@RN2633], but as this could be perceived as exercises performed only with heavy weights, the system was renamed *progressive resistance exercise* to better reflect the method
[@RN2641]
^[In the current text, exercise is defined as an acute bout of physical activity designed to affect physical characteristics such as strength, speed or endurance. Training is defined as the systematic process of combining multiple exercise-sessions performed in sequence over time. DeLorme first used the adjective *heavy* to describe the resistance prescribed to overcome during exercises but later changed this adjective to *progressive*. In modern texts, the adjective is commonly omitted from the description and *resistance exercise/training* is used to describe strength-promoting exercises and training regimes requiring the neuromuscular system to exert force against (heavy) resistance. Omitting the adjective has led to many heated debates among exercise physiologists as "endurance exercises are also performed against a[n] (external) resistance". With no ambition to resolve any conflict in the area, *resistance exercise/training* will be used synonymous with *progressive* or *heavy* *resistance exercise/training*.]
.<!-- DeLorme and Watkins 1948 -->
Indeed, central to the system was the concept of repetition maximum [@RN2633].
Repetition maximum refers to the external resistance that can be overcome with a given number of repetitions.
By adjusting external resistance to each individuals progression over the course of a training program, exercises are both individualized and progress is monitored [@RN2633].
DeLorme originally prescribed sessions of up to 100 repetitions performed in sets of 10 repetitions [@RN2633] but later revised this recommendation to three sets of 10 repetitions performed with increasing intensities
[@RN2641]. <!-- DeLorme 1948 Technics of progressive resistance exercise -->

Scientific inquiries into prescription of resistance training from the first part of the twentieth century concerned its therapeutic use
[e.g. @RN2633;<!-- DeLorme case report -->
@RN2646]<!-- DeLorme controlled trial of RT vs. traditional conservative (low-resistance) treatment -->
but was also evaluated in the context of improving strength and physical performance in healthy populations
[e.g. @RN2632;<!-- 1946 Houtz 16 untrained females increased strength over 4 weeks follwing DeLormes system -->
@RN2644; <!-- Chui 1950 Effect of strength training vs. control on physical performance in college stud -->
@RN2642]. <!-- Capen 1950 Effect of strength training in college stud.   -->
Scientific contributions soon moved from questions regarding the effectiveness of resistance training *per se* to comparing outcomes from different modes of resistance training 
[@RN2645;<!-- Hettinger 1953 comparing different loading patterns for isometric RT--> 
@RN1477;<!-- Capen 1956 Four different variations on the DeLorme system -->
@RN1472;<!-- Berger 1962 effect of varied weight training program on strength -->
@RN2659;<!-- Berger 1965 Increased %of 1RM increases strength gains -->
@RN2658;<!-- Berger 1967 Supra-maximal efforts (max of each rep) more efficient than 10RM loads   -->
@RN2656].<!-- O'Shea 1966 3 sets of 10, 5 or 3 reps on muscle strength and girth squat -->
A vocabulary for progressive resistance exercise-training developed through these investigations and parallel practice,
the introduction of repetition maximum by DeLorme being one example. These concepts established as modern definitions of exercise variables enabling precise prescription of training loads for a variety of populations and training goals
[@RN2537<!-- Fleck & Kraemer 2014 Textbook ch 5 : based on a needs analysis acute variables are selected   -->
@RN798].<!-- Bird 2005: Acute training variables leads to different training outcomes -->

Although this development started after the Second World War, resistance training was not part of general exercise guidelines until much later.
The American College of Sports Medicine (ACSM) position statement on exercise for healthy individuals from 1978 primarily concerned physical fitness in terms of cardio-respiratory fitness
[@RN2655]<!-- ACSM 1978 guidelines -->.
Since the updated 1990 ACSM statement, resistance training is recommended to be included as part of a sensible, general training program
[@RN2654].<!-- ACSM 1990 guidelines --> 
The introduction of resistance-training as part of the ACSM recommendation also coincides with specific recommendations on resistance training being part of other consensus statements
[@RN2666, Ch. 2].<!-- Physical Activity and Health: A Report of the Surgeon General provides an overview of recommendation guidelines evolution  -->
Consequently, informed by epidemiological data, the most recent general guidelines for physical activity do include resistance training [@RN2667].<!-- WHO guidelines -->

The above account reflects the fact that common understandings of *why* and *how to* exercise are influenced by societal norms and historic events such as the search of national identity in the nineteenth century or treatment of war injuries in the twentieth century
[@RN2663;
@RN2639].
In attempting to outline contemporary influences on exercise prescription, one could argue that the development of techniques to collect large amount of biological data is one such influence. The continuously decreasing cost of a sequenced human genome
[@RN2696] can serve as an example of this development.<!-- Wetterstrand webpage genome sequence costs -->
Such molecular techniques has enabled efforts to describe mechanisms by which exercise training induce favorable adaptations. The newly established Molecular Transducers of Physical Activity Consortium is an example of a large scale effort, explicitly initiated to develop personalized exercise recommendations and identify molecular targets through which effects of exercised may be mimicked
[@RN2678].<!-- MToPAC explicitly stating diagnostics is a goal --> 
Advances in bio-medical technologies are enablers of this enterprise, and the quest to *individualize* exercise based on molecular diagnostics can be seen as a motivation for modern exercise science
[@RN2678;<!-- MToPAC explicitly stating diagnostics is a goal -->
@RN2677].<!-- Sparks 2017, exercise response heterogenity, goal to test appropriate exercise dosa based on e.g. DNA -->
Scientific research into contemporary exercise prescription can thus be understood as a part of the era of *personalized medicine*. 

A challenge facing this program is to accurately describe etiologies of response heterogeneity associated with physical training. A wide variation of individual responses are commonly observed after standardized resistance-training programs where measures of muscle strength-changes varies from -32 to +250\% and muscle size-changes varies from negative to (-11\%) to impressively large (+59\%)
[@RN764;<!-- Hubal wide range of responses to standardized exercise  -->
@RN1741].<!-- Ahtiainen 2016 7% (strength) and 29% (muscle size) categorized as non responders (accounting for controls)  -->
By relating such variations to individuals genome (DNA)
[@RN1263],<!-- Pescatello review of the famuss data set -->
and messenger RNA (mRNA) profiles
[@RN826;<!-- Thalacker-Mercer, Cluster analysis, non-responders pro-inflammatory mRNA profile at baseline -->
@RN2684]<!-- Stokes co-relation between mRNA profiles and LMM changes  -->
we are beginning to gain knowledge about the genetic influence on training responses.
In such studies, a common strategy has been to dichotomize responses into "responders" and "non-responders" to exercise training. From a public health perspective this is probably fruitful when *non-response* is defined as the absence of meaningful health-related adaptations, or even adverse effects in response to a given training regime
[@RN2677;<!-- Sparks 2017 response heterogeneity -->
@RN2698].<!-- Roth viewpoint 3 scenarios for genomic screening -->
The existence of non-responders would have large implications regarding exercise prescription on the population scale
[@RN758],<!-- Timmons 2011  -->
and for any given individual as it, if properly diagnosed, would guide clinical decision-making.

<!-- Although "molecular exercise prescription" as a mean of eliminating adverse responses to a given training regime shows potential for future applications
[@RN2698; <!-- Roth viewpoint 3 scenarios for genomic screening 
@RN2547],
the rise of data intensive techniques has also highlighted its complexity.

In simplistic terms, genetic information is transferred from DNA to RNA through transcription and from RNA to protein through translation.
This flow of information, indicates a deterministic role of DNA in controlling the characteristics of an organism. It is however apparent that the transcription of specific genes are under tight control of several mechanisms including a group of non-genetic modifications to the DNA organization of each cell, known as epigenetic modifications
[@RN2690;<!-- Seaborn sharples review 2020 exercise metabolism as a determinant for epigenetic changes 
@RN2691].<!-- McGee Hargreaves 2019 review, epigentics/metabolism/exercise and mention intergenerational trandmission thorugh epigentic modification 
Such control determines gene expression in response to progressive resistance training with modifications persisting after a period of de-training
[@RN2683;<!-- Transcriptome is associated with epigentic changes in response to RE 
@RN2012].<!-- Seaborn 2018 epigenetic changes persist after de-training, epigenetic memory of training 
Regulation of gene expression is thus related to modifiable variables such as exercise and diet
[@RN2690;<!-- Seaborn sharples review 2020 exercise metabolism as a determinant for epigenetic changes 
@RN2691;<!-- McGee Hargreaves 2019 review, epigentics/metabolism/exercise and mention intergenerational trandmission thorugh epigentic modification 
@RN2694;<!-- Yoshie 2020 13 mo old mice have epigenetic changes in skm due to childhood diet 
@RN2695]<!-- Godfrey 2016 review childhood environmental conditions affects epigenetics 
linking environmental factors to the complexity of gene expression.
-->
A key aspect of successful exercise diagnostics would be to take advantage of the relationship between exercise variables (i.e. modality, intensity, volume etc.) and exercise response for a given individual.
It is possible that the response to training could be positively affected by adapting an individuals training program based on some prior knowledge about the individual. 
Observations supporting such notion exists as an individual classified as non-responsive to a specific exercise modality (e.g. endurance training) may be classified as a responder to another (e.g. resistance training)
[@RN2681].<!-- Changing from end to res training removed non responsiveness --> 
Even changing training variables within a specific modality have been shown to convert non-responders to responders as endurance training volume was increased
[@RN2699].<!-- Lundby increasing training volume improved non-responders -->

Although apparent reversal of non-response to exercise training has been observed by manipulating training variables, decisive indications for such manipulations are still lacking.

## Adaptations to resistance training

### Muscle hypertrophy and strength
Systematic resistance training typically increases muscle mass and strength, adaptations through which many beneficial effects on health (e.g. increased amino acid storage physical independence) and athletic performance are conveyed. 
Muscle growth is a well characterized response to resistance training.
Healthy untrained individuals can be expected to increases their muscle mass by $\sim$ 5-20\% when training is conducted over two weeks to 6 months
[@RN1741; <!-- Ahtianen -->
@RN346;<!-- Wernbom  -->
@RN764].<!-- Hubal 2005 elbow flexor 18.9% increase -->
Over this time span, muscle growth is approximately linear with time
[@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113;<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions -->
@RN2736]<!-- Narici 1989 Increase in CSA (MRI, linearly ~ 0.14%/day total 8.5%) and MVC (total 20.8%) no change in untrained control in CSA, numerical increases occured in untrained in EMG/MVC   -->
<!-- Although some of the early increase muscle mass can be attributed muscle damage-induced edema
[@RN2746],Damas 2016 increased echo intensity indicating muscle edema at 3 weeks -->
and has been detected as early as 3-4 weeks after training initiation, without apparent muscle edema 
[@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113].<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions -->

Relative muscle growth can be expected to be more pronounced in upper- compared to lower-body muscles when loading patterns are similar
[@RN346;<!-- Wernbom  -->
@RN2739].<!-- Welle 1996 larger incr in csa in upper body -->
This possibly relates to greater every day activity levels of lower-body muscles requiring a larger stimuli for adaptation
[@RN767].<!-- Folland review -->
Small, but detectable differences in muscle growth is typically seen between sexes for training induced muscle growth in the upper-body
[@RN764]<!-- Hubal CSA men: 20.4% vs 17.9% -->
but not for lower-body muscles
[@RN2740].<!-- Roberts 2020, meta analysis when all muscles combined no diff between sexes in within sex effects sizes -->
Furthermore, hypertrophic responses can be expected to be reduced with increasing age 
[@RN2739; <!-- Welle 1996 larger incr in csa in upper body --> 
@RN752]<!-- Peterson 2011 Meta analysis, age (negatively) and training volume (positively) affects rt outcomes  -->
but increased with sufficient addition of dietary protein
[@RN2199].<!-- Morton meta Dietary protein supplementation significantly enhanced changes in muscle strength and size during prolonged RET in healthy adults. Increasing age reduces and training experience increases the efficacy of protein supplementation during RET. -->
Additionally, training variables such as intensity, volume, frequency (reviewed below) together with other training aids, e.g. manipulation of blood flow through pressure cuffs
[@RN2745]<!-- Lixandrao BFR meta analysis hypertrophy similar between low load BFR and high load -->
can effectively modulate resistance-training induced hypertrophy.
Together, this underlines that both non-modifiable (e.g. sexual dimorphism and age) and modifiable factors (e.g. training variables and protein supplementation) affects resistance training-induced muscle hypertrophy.

Whole muscles growth in response to short term resistance training occurs primarily through growth of individual muscle fibers (muscle cells or myofibers).
This can be assumed as training-induced splitting of existing fibers and/or myogenesis, formation of new muscle cells, are likely slow processes and any increase in the number of fibers by such mechanisms would only represent a small addition to the whole muscle mass 
[@RN2741;<!-- Murach 2019 review on hyperplasia  -->
@RN2742; <!-- Sjöström 1991 Increase in left/right tibialis anterior -->
@RN2754;<!-- MacDougall no diff between body builders and untrained in number of fibers, equal relative amounts of collagen -->
@RN767].<!-- Folland 2007 review on mechanism for hypertrophy and strength gains -->
Growth of muscle fibers transfers to greater muscle strength trough an expansion of the fibers contractile elements.
The muscle cell is to a large degree occupied by myofibrils
(about 80% of the cell volume 
[@RN2731]<!-- Lüthi 1986 Myofibrils occupie more space in the mucsle fiber after resistance training mitochondria occupie less. No sign increase in fiber CSA with strength training no control group -->),
which in turn contain sarcomeres, arranged in series.
Upon neural activation and subsequent Ca\textsuperscript{2+} release, the sarcomere shortens through interaction between actin and myosin resulting in force generation
[@RN2756].<!-- Sweenney review on muscle contraction -->
With resistance training, the number of parallel myofibrils increases with the growth of individual fibers
[@RN2669]<!-- Straight 2020 muscle fiber size increases with RT, in midlle age old, age inversely related to training effects -->
leading to a greater force generating capacity of the whole muscle
[@RN2731].<!-- Lüthi 1986 Myofibrils occupie more space in the mucsle fiber after resistance training mitochondria occupie less. No sign increase in fiber CSA with strength training no control group -->

Measures of whole muscle size corresponds well with strength, particularly when a measure of muscle size reflects the muscles cross-sectional area
[@RN2758;<!-- Bamman 2000 Anatomical and estimates physiological cross section better prediction for strength on whole muscle level in vivo in humans -->
@RN1142].<!-- Erskin 2013 muscle volume and CSA to strength r = 0.812 and 0.806  -->
However, in response to resistance training, increases in strength are typically greater in magnitude than muscle growth
[@RN2629;<!-- Ikai 1970 Arm flexor training within-subject protocol, trained arm 23% increase in CSA and 90 % strength 100 days of 6 days/week training. Control arm 30% str incr, 5% CSA incr -->
@RN2737;<!-- Young 1983 diproportional higher strength gains than CSA, increase in CSA with ultrasound in response to RT -->
@RN2736;<!-- Narici 1989 Increase in CSA (MRI, linearly ~ 0.14%/day total 8.5%) and MVC (total 20.8%) no change in untrained control in CSA, numerical increases occured in untrained in EMG/MVC   -->
@RN2735].<!-- Narici 1996 Local changes in whole muscle CSA after 6 months of RT, no stat sig increase in fiber CSA at a site with low MR based increase in CSA -->
When relationships between resistance-training induced change in muscle size and strength in previously untrained individuals are analyzed, only a portion of the variation in muscle strength can be accounted for by changes in muscle size ($\sim$ 2.5-28\%)
[@RN1142;<!-- Erskine 2013 23% and 28% muscle volume and mvf/1rm  -->
@RN1741;<!-- Ahtiainen r = 0.157, r2 = 0.025 -->
@RN2158]<!-- Vigotsky 2018 within subject model 12-18% accounted for, smaller when between subjects -->
depending on type of measurements and statistical model used
[@RN2158].<!-- Vigotsky 2018 within subject model 12-18% accounted for, smaller when between subjects -->
This underlines that muscle hypertrophy is one contributing factor to muscle strength gains, but not the only one.

Different experimental models have shown that strength can increase without concomitant muscle hypertrophy.
Getting acquainted to the actual strength test through repeated training of maximal performance produces similar gains in strength without apparent hypertrophy 
[@RN2760].<!-- Mattocks 2017 Practicing 1RM leads to similar strength increases as RT  -->
If resistance training is performed unilaterally and the contralateral limb act as a control, strength gains are typically also seen here
[@RN2219;<!-- Carroll 2006 contralateral effects review -->
@RN2766].<!-- Green 2018 training has contralateral effects both for strength and skill -->
Additionally, systematic imagery training, without muscle activation produces greater strength gains than control and low-intensity training conditions 
[@RN2767].<!-- Plantar flexion imaginary more effective than control in strength gains -->
Together these observations indirectly points to the central nervous system and motor learning as important factors for strength gains.
In addition to effects that mainly can be attributed to motor learning, resistance training leads to changed behavior of motor units, estimated from surface electromyograms 
[@RN2763].<!-- Del Vechchio 2019 rt changes fiering rates in motor units   -->
Such changes could be attributed to morphological and functional changes of motoroneurons
[@RN2764].<!-- Gardiner aplha motorneuron changes with RT -->


### Changes in muscle fiber contractile and metabolic characteristics with resistance training 
In adult human skeletal muscles, muscle fiber types can be identified based on their myosin heavy-chain isoform composition. Pure fibers express a single mysoin heavy chain isoform whereas hybrid fibers co-express isoforms.
Primarily three myosin heavy-chain protein isoforms are expressed in adult human skeletal muscle, determined transcriptionally through expression of the genes *MYH7*, *MYH2* and *MYH1* corresponding to myosin heavy-chain I, IIA and IIX
[@RN819]<!-- -->

Single fibers of different fiber types have specific contractile properties regardless of muscular origin
[@RN846],<!-- Harridge 1996 specific contraction characteristics in different fiber types determined by myosin heavy chain-->
with type II fibers displaying greater force-generating capacity and shortening velocities than type I fibers when normalized to fiber cross sectional area
[@RN2169;<!-- Widrick IIx > IIa > I in force/power unloaded shortening, training alters these charecteristics in proportion to hypertrophy -->
@RN846]<!-- Harridge 1996 specific contraction characteristics in different fiber types determined by myosin heavy chain-->
These differences directly relates to the myosin heavy-chain proteins displaying different characteristics when interacting with actin
[@RN849].<!-- Larsson diffrenet motility speeds of different myosin isoforms -->
Importantly, *in vitro* assays performed at physiological temperatures shows that myosin heavy-chain isoforms extracted from type II fibers are two-fold faster compared to type I fibers with no difference between type IIX and IIA
[@RN849].<!-- Larsson diffrenet motility speeds of different myosin isoforms -->

In addition to contractile characteristics, fiber types identified on the basis of their myosin heavy chain content also differs in metabolic profiles.
Type I fibers are characterized as having lower glycolytic but higher oxidative potential compared to type II fibers
[@RN1885].<!-- Essen 1975 Type II fibers lower oxidative potential compared to type I assumed from activity of oxidative enzymes -->
Differences in metabolic profiles relates to fatigue resistance were type I fibers can maintain power output as well as ATP levels during intense exercise but type II and especially type IIX fibers fail to do so 
[@RN819;<!-- -->
@RN2801].<!-- Karatzaferi 2001 Drop in ATP most pronounced in type 2x fibers in response to acute exercise -->

Fiber type characteristics effectively modulate the muscle's ability to perform specific activities. Differences in fiber type composition within individuals, between different muscles, reflects this as anti-gravity muscle of the lower body typically express more type I fibers compared to upper-body muscles
[@RN819;<!-- -->
@RN846].<!-- Harridge 1996 specific contraction characteristics in different fiber types determined by myosin heavy chain-->
Differences in fiber type composition between individuals and sexes are to some degree genetically determined
[@RN2798;<!-- Simoneau 1995 40% variation in type I fibers can be attributed to genetic factors, 15% to sampling error and 45 to environment -->
@RN2795;<!-- Differences between men and women more type i in women -->
@RN285],<!-- Fiber types in yopung men and women, high percentage of 2X (b) fibers ~ 20%-->
but resistance training is among non-genetic factors influencing fiber-type composition. Short term resistance training, designed for muscle hypertrophy and strength gains specifically converts type IIX fibers to more fatigue resistant type IIA fibers with unaltered type I fiber proportions
[@RN2220;<!-- Adams Down regulation of MyHc 2x content in training group both myhc isoform sds page and atpase staining -->
@RN2799;<!-- Williamsson less IIx dominant hybrid fibers with training conversion towards IIa -->
@RN2056;<!-- Andersen 2018 type IIx to 2a most prominent switch-->
@RN1489].<!-- Ellefsen 2014, training induced switch from 2A to 2X -->
Such conversion is apparent both when measured on the protein and mRNA level
[@RN1489;<!-- Ellefsen  -->
@RN2056].<!-- 2x gene swithoff Anderssen --> 
In contrast, reduced activity or inactivity readily increases the proportion of type IIX expressing fibers 
[@RN2057;<!-- Andersen 2x overshoot after de training -->
@RN2108].<!-- Bed rest increases type IIx -->

Concomitant with resistance training-induced muscle hypertrophy, and fiber type switch, the relative mitochondrial density of the muscle decrease as myofibrilar protein fractions increases, evident from electron microscopy examination
[@RN2731].<!-- Lüthi  -->
However, a single session of resistance training, albeit with low resistance (30\% of 1RM), was shown to increase mitochondrial, as well as myofibrilar and sarcoplasmic protein synthesis rates. When exercise was performed with slower movement speeds (longer time under tension), the increase in mitochondrial protein synthesis was shown to be greater
[@RN1505].<!-- Burd 2012 A single session of resistance exercise (30%1RM) increases mitochondrial protein synthesis, more so with higher metabolic stress (6s repetitions compared to 1s reps). -->
This indicates that the magnitude of metabolic stress induced by resistance training affects the subsequent mitochondrial remodeling.
Such remodeling could explain improved mitochondrial function, measured as mitochondrial respiration in response to 12-weeks of resistance training with less pronounced changes seen in mitochondrial proteins
[@RN2608].<!-- Higher mitochondrial respiration after 12 wk RT --> 
An improved mitochondrial efficiency could also be linked to fiber type transitions.
Mitochondria have the ability to form dynamic networks within cells by fusion (and fission) of individual mitochondria, a characteristic important for normal function
[@RN2817].
Such behavior have been shown to be fiber type specific in muscle, with oxidative fibers (type I and IIA) compared to glycolytic fibers (type IIX and IIB in mice) displaying greater, elongated mitochondrial networks
[@RN2816].<!-- Mishra 2015 elongated   -->
In response to endurance training, fiber type switch from glycolytic to oxidative, coincided with switch to less fragmented mitochondria
[@RN2816].
Such coordinated remodeling can be linked to common molecular mechanisms regulating both fiber type and mitochondrial biogenesis
[@RN2818;<!-- Lin 2002 PGC1a expression drives type II to type I characteristics-->
@RN2819].<!-- PGC1a knock out shift from type I to type IIx and reduced endurance capacity -->


### Connective tissue
In addition to adaptive changes to the contracting apparatus of single muscle fibers and neural mechanisms regulating their activity, resistance training modulates bone, tendon and connective tissue.
From a general perspective, tissues enabling e.g. locomotion by conveying forces produced by contracting muscles and stabilizing body segments adapts in an activity-specific manner
[@RN2775;<!-- greendale 2003 activity patterns affect bone mineral density, sporting activities more effective than active living home activities -->
@RN2771].<!-- Magnusson 2008 review tendon CSA increase with endurance training -->
Specifically, short term resistance training leads to changes in mechanical properties of bone without increases in bone mineral content or density suggesting qualitative changes
[@RN2774].<!-- Miller 2009 rt increases tibial bending stiffness without changes in bobne mineral content 20 weeks, 3 sessions/week con and ecc, females 18-26 years-->
Similarly, tendons respond to short term resistance training by increasing stiffness at high mechanical stress-levels without increased cross sectional area
[@RN2772].<!-- Massey 2017 short term (12 weeks) rt increase patellar tendon stiffness at high stress to a similar degree as long (4 years) compared to controls. Long term training led to increase stiffness at low stress . No increase in tendon CSA in any training group -->
Interestingly, these adaptations seem to reach a plateau as no additional change in this characteristic is seen in individuals who have exercised over four years as opposed to twelve weeks
[@RN2772].
<!--However increased stiffness at lower mechanical stress levels has been documented
@RN2771]. Magnusson 2008 review tendon CSA increase with endurance training -->
Changes in tendon properties in response to resistance training may thus primarily be associated with qualitative changes
[@RN2771]<!-- Magnusson 2008 review tendon CSA increase with endurance training -->
as a consequence of exercise and training-induced increased turnover of collagen in tendons, indicating remodeling
[@RN2771;<!-- Magnusson review -->
@RN2770].<!-- Kjaer review, focus on collagen metabolism in  -->

Muscle fibers are embedded in connective tissue, observed as surrounding the whole muscle (epimysium), muscle fasicles (perimysium) and muscle fibers (endomysium)
[@RN2783].<!-- Borg Rat skeletal muscle electron micropscopy  -->
Connective tissue structures constituting the endomysium connects muscle fibers to adjacent fibers, capillaries and nerves which together with higher order structures make up the extracellular matrix enabeling mechanical and biochemical interaction between cell types
[@RN2783;<!-- Rat skeletal muscle  -->
@RN2421].<!-- Kjaer review -->
Together with the myotendinous junction, intramuscular structures (primarily perimysium) transmits forces originating from contracting muscle fibers to tendon and bone and act as an elastic energy storage during e.g. locomotion
[@RN2421].<!-- Kjaer review -->

The mechanical properties of the extracellular matrix also allows for mechanical stimuli to be converted to biochemical signaling initiating e.g. responses to exercise.
There is a general coordination between connective tissue and muscle-cell remodeling in response to loading, evident from coordinated responses of different cell types in response to exercise
[@RN2421].<!-- Kjaer review -->
The major constituent of the extracellular matrix is collagen, produced in fibroblasts.
In response to acute endurance-type exercise, collagen synthesis and muscle cell specific protein synthesis (myofibrillar and sarcoplasmic fractions) rise in a coordinated fashion
[@RN2788].<!-- Miller coordinated prot synthesis response to exercise (not RT) -->
Also, in response to short- and long-term resistance training with subsequent muscle hypertrophy, relative collagen content of muscle tissue remains stable
[@RN2754].<!-- Macdougall Similal relative collagen content and non contracting tissue in untrained and trained human skm  -->
However, fine tuning of such coordination could exist as contraction mode has shown to differentially affect myofibrillar protein but not collagen synthesis after acute exercise
[@RN2454].<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load -->

Increased remodeling of components of the extracellular matrix is a typical response to resistance training evident from gene expression studies
[@RN2298;<!-- Damas long term training ECM up gene -->
@RN774],<!-- Raue, upregulation of ECM after long term exercise -->
and studies of acute protein synthesis
[@RN2457;<!-- Moore 2005 Collagen synthesis up irrespective of ecc or conc 6 * 10 maximal contractions -->
@RN2454].<!-- Holm 2010 Contraction mode.. No diff between fasted/fed nor heavy/light load -->
Such remodeling may contribute to increases in specific force (force generated per muscle cross section) seen after resistance training through improved lateral force transfer
[@RN1504].<!-- Erskine 2011 Hypothesized increased lateral force transfer increase specific tension in vivo in response to strength training in humans as single fiber specific tension nor mhc composition could explain -->


## Effects of exercise prescription on muscle mass and strength
Precise exercise-training prescription gives information on exercises, their sequential order, intensity and volume, rest periods between efforts or sessions and the frequency at which exercise sessions are to be performed
[@RN789].<!-- Bird designing resistance training programs -->
By manipulating these variables, resistance training programs can be tailored to better fit goals and starting points of any individual. 
The relative importance of exercise-training variables for training outcomes has been examined in numerous studies including (but not limited to) the overall organization of exercise sessions,
[@RN2575;<!-- Evans mini review on periodization --> 
@RN2572]<!-- Grgic 2017 No effect of linear and daily unduluting periodization on muscle hypertrophy -->
training frequency 
[@RN2571]<!-- Schoenfeld 2016 training frequency twice per week better in volume equated studies for hypertrophy -->,
<!-- Higher frequencies gives higher gains in strength but not when volume equated -->
and intensity
[@RN2569].<!-- Schoenfeld, no difference between high vs low load strength training in hypertrophy but 1rm favors high loads -->
It could be argued that training volume is of particular importance for muscle growth as when this variable is held constant, manipulation of other variables has little or no effect hypertrophy
[@RN1612;<!-- Schoenfeld 2014 7*3Rm vs 3*10RM equal muscle hypertrophy but dissimilar strength -->
@RN2569].<!-- Meta analysis high vs low resistance has little effect on muscle mass gains but strength is affected positively with higher intensities. --> 
For development of strength, factors such as intensity and within session organization of exercises is of importance 
[@RN2570;<!-- Frequency has little effect on strength when volume is equated -->
@RN2591],<!-- Exercise order influences strength but not hypertrophy -->
however, when other factors are held constant, increased training volume generally leads to increased strength
[@RN2570;
@RN2492;<!-- Ralston 2017 meta analysis medium to high volume more effective strength gains -->
@RN793],
similarly to effects of training volume on muscle growth
[@RN1767;
@RN789].

### Effects of resistance exercise volume on muscle strength and mass
Resistance exercise volume can be manipulated as the within-session number of sets performed per muscle group. This unit is practical as it is comparable between individuals and muscle groups [@RN2130].<!-- Baz-Valle 2018 sets per exercise better than mechanical load times rep or total work -->
Berger conducted an early study concerning effects of resistance exercise volume with the goal to determine what method would most efficiently produced strength gains (in healthy young males) [@RN1476]. Berger compared one, two and three sets performed with two, six or ten repetition maximum (RM) in the bench press, three times per week, over twelve weeks. As the combined effect of three sets per session was superior regardless of the number of repetitions performed Berger concluded in favor of three sets. This conclusion was later challenged on the basis of data interpretation
[@RN794;<!-- Carpenelli and Otto questions the analysis-->
@RN2538].<!-- Feigenbaum -->
Reveiwing the study by Berger and others, Carpinelli and Otto arrived to the conclusion that there was "insufficient evidence to support the prevalent belief that a greater volume of exercise (through multiple sets) will elicit superior muscular strength or hypertrophy" [@RN794]. This stand has since been repeatedly put forward as a criticism of higher volume training programs 
[@RN2568; 
@RN2201] and sparked considerable scientific activity. The main argument against the recommendation of additional volume in strength training programs has been the lack of statistically significant results in single studies [@RN2568; @RN2538]. 
Indeed, individual studies do not generally agree on dose-dependent effects of training volume on muscle mass and strength gains
[@RN2465; <!-- Ribiero 2015 Strength but not body composition change with dose dep older women -->
@RN2464;<!-- Correa 2015 older women, increase muscle thickness and strength with volume  -->
@RN2463;<!-- Bottaro 2011 untrained young male similar results in muscle thickness and strength -->
@RN1570; <!-- Radaelli 2014 1 vs 3vs 5 sets, 5 sets most effective in muscle thickness and some strength exercise -->
@RN1518;<!-- Radaelli 2014 older women no diff between volume groups in muscle thickness or strength gains -->
@RN1474;<!-- McBride 1474 Men and women DXA no diff but strength increase more in multiple sets -->
@RN1472; <!-- Galvao 2005 Older adults, no diff in body comp DXA, but increase in strength  -->
@RN1456;<!-- Starkey 1996 young male and female no diff in strength or muscle thickenss gains -->
@RN1454;<!-- Ostrowski 1997 trained young med, low volume decrease from previous levels no difference between groups -->
@RN1384; <!-- Rhea 2002 young men bod pod no diff in body comp changes but diff in strength gains in favour of multi set -->
@RN1382; <!-- Cannon 2010, Older women and younger women, muscle volume and strength increased similarly irrespective of age or training volume and without interaction -->
@RN776],<!-- Rønnestad, three set more effectiv in increaseing strength and muscle mass in lower body compared to upper body -->
including studies performed with comparisons between volume conditions within participants, where different training volumes are allocated to either extremity 
[@RN834;<!-- Mitchell 2012 young men no sig difference in strength or mass gains between 1 and 3 sets knee extension -->
@RN1607].<!-- Sooneste 2013 young men p=0.076 on strength gains in favor of three sets, CSA increased more in 3 sets leg exercise -->
For example, differences in strength between volume conditions are found in older individuals 
[@RN2465; <!-- Ribiero 2015 Strength but not body composition change with dose dep older women -->
@RN2464;<!-- Correa 2015 older women, increase muscle thickness and strength with volume  -->
@RN1472] <!-- Galvao 2005 Older adults, no diff in body comp DXA, but increase in strength  -->
but not confirmed in another study
[@RN1518].<!-- Radaelli 2014 older women no diff between volume groups in muscle thickness or strength gains -->
Studies shows that more volume does not lead to increased muscle mass gains in young individuals
[@RN1384; <!-- Rhea 2002 young men bod pod no diff in body comp changes but diff in strength gains in favour of multi set -->
@RN1456;<!-- Starkey 1996 young male and female no diff in strength or muscle thickenss gains -->
@RN2463]<!-- Bottaro 2011 untrained young male similar results in muscle thickness and strength -->
a conclusion challenged by others 
[@RN776;<!-- Rønnestad, three set more effectiv in increaseing strength and muscle mass in lower body compared to upper body -->
@RN1570]. <!-- Radaelli 2014 1 vs 3vs 5 sets, 5 sets most effective in muscle thickness and some strength exercise -->

As previously noted, combining the above results and additional studies, meta-analyses concluded that training volume dose-dependency exists for the development of muscle mass and strength
[@RN2570;
@RN2492;<!-- Ralston 2017 meta analysis medium to high volume more effective strength gains -->
@RN793;
@RN1767;
@RN789].
As a second argument against additional volume in resistance training recommendation has been the cost/benefit relationship of adding training volume without meaningful or substantial additional gains
[@RN2538;<!-- Feigenbaum 1999 questions the analysis and the practical utility of volume -->
@RN2568],<!-- Carpinelli Berger in retrospect questions the practical significance of 1.8 kg diff in maximal strength -->
a subsequent question is, who would benefit from greater volumes and who would not?
Schoenfeld *et al.* combined data from published studies to explore if participant characteristics of the above mentioned studies interacted with training volume in explaining study outcomes. Neither sex, muscle groups nor age interacted with volume prescription indicating that no such factor would be able refine training prescription guidelines
[@RN1767].<!-- Schoenfeld 2016 meta-->
As the number of studies used to synthesis the meta-analysis was relatively low (*n* = 15), and the studies were heterogeneous in terms of e.g. outcome measurements, it may have lacked in power to detect any meaningful interactions. Additionally, included studies may not have been reporting relevant characteristics for such analysis.

Collectively, the available (synthesized) evidence suggest that there is overlap between training outcomes in studies were different volume has been utilized.
The overlap cannot, with available data, be explained by general population characteristics such as age or sex.
Studying the effect of different training volumes within participants could potentially help to define determinants of training outcomes in response to different volume conditions.
Two within-participant studies have investigated the effects of training volume on strength and hypertrophy outcomes.
Sooneste *et al.* compared strength outcomes in response to three- and one-set elbow flexor training for 12 weeks in young males using a whitin-participant protocol (arms allocated to either volume condition).
The results showed general benefit of three- over one-set training for muscle hypertrophy and tended to do so also for strength gains [@RN1607].
No attempts were made to relate baseline characteristics to the magnitude of differences between volume conditions, presumably due to the small sample size (*n* = 8).
Mitchell *et al.* compared muscle hypertrophy and strength gains in response to three- and one-set of knee-extension exercise performed three times per week for ten weeks.
The study contained an additional training condition (low intensity, 30% of 1RM performed with three sets) with participants legs assigned to either of the three conditions in a random fashion.
No significant differences were reported between volume conditions for muscle mass or strength gains [@RN834].
However, the analyses were performed without taking the correlation between individuals into account due to the mixed design [@RN834].
No attempts were made to relate any measured characteristic to differences in responses.

## Molecular determinants of training-induced muscle hypertrophy
Muscle mass fluctuates as a consequence of the balance between muscle protein synthesis and breakdown. When a net-positive balance is achieved, muscle protein accumulates and muscle mass increases.
Following a single bout of resistance exercise, muscle protein synthesis increases
over resting levels up to 48-h post-exercise 
[@RN2710;<!-- Dreyer 2006, blunting of protein synthesis during ex, increase post ex -->
@RN2711;<!-- McDougall 1995 Highest values at 24h after ex, at 36 not sig higher than ctrl -->
@RN786;<!-- Phillips 1997 fasted state, FSR and FBR both increased but FBR to a lesser extent giving increase in net balance. Fasted state -->
@RN2712;<!-- Biolo 1995, fasted state ex increase both breakdown and synthesis FSR/FBR at 3h net balance increase but not positive -->
@RN2717;<!-- Increase in FSR over control at 4 and 24 h post ex in biceps, no change in RNA -->
@RN1521]<!-- Kim, increase after exercise in mixed and myofibrillar prot synth in trained and untrained state, lower mixed after training, higher basal protein synt rate after training period 8 weeks 2-4set 8RM 8 weeks biopsy 12 h after ex Same absolute intensity -->
after being blunted during exercise
[@RN2710].<!-- Dreyer 2006, blunting of protein synthesis during ex, increase post ex -->
Muscle protein synthesis and breakdown rates are highly correlated
[@RN2713;<!-- Phillips 1999, training alters both breakdown and synthesis --> 
@RN786]<!-- Phillips 1997, correlation between fbr and fsr-->
indicating that these processes are mechanistically coupled and fluctuates together.
While acute resistance exercise thus also stimulates to breakdown of muscle protein, it does so to a lesser extent, leading to an increase in the net protein balance from baseline under favorable conditions
[@RN2713;<!-- Phillips 1999, training alters both breakdown and synthesis --> 
@RN786;<!-- Phillips 1997, correlation between fbr and fsr-->
@RN2714;<!-- Børsheim 2002 Post ex prot synthesis increased with ingestion of essential amiono acids -->
@RN2715].
<!--[@RN1908] Millward 1979 Muscle protein mass is regulated primarily through alterations in protein synthesis in all except emergency conditions. --> 
When resistance training is performed under such favorable conditions, in the fed state with availability of dietary protein, a net positive protein balance can be expected after exercise
[@RN2714;<!-- Børsheim 2002 Post ex prot synthesis increased with ingestion of essential amiono acids -->
@RN2715].<!-- Tipton 1999, post-exercise net balance is increased when amino acids are administrated -->

Indispensable for the process of protein synthesis is the ribosome, a cellular machine capable of translating genetic information in the form of messenger RNA (mRNA) to proteins.
A synthesizing ribosome consists of four ribosomal RNA species (rRNA 18S, 5.8S, 28S and 5S) and about 80 proteins constructing two ribosomal subunits. Translation of mRNA occurs at the ribosomal core as ribosomal RNA catalyzes binding of amino acids to form a polypeptide chain corresponding to the ribosome-bound mRNA sequence.
The rate of translation per ribosome (i.e. translational efficiency) can be modified by specific stimulus, of which mechanical stress and nutrients are of importance to this discussion. 
Mechanical stress (such as resistance exercise) or amino acid availability (as increased after ingestion of dietary protein) stimulates to protein synthesis through increased translational efficiency.
This can be observed as the formation of polysomes as functional ribosomes bind to mRNA in response to e.g. mechanical stress
[@RN866;<!-- Baar/esser increased formation of polysomes after resistance exercise in rats  -->
@RN860].<!-- Kubica formation of polysomes in response to resistance exercise are abolished with rapamycin in rats -->
The available data from acute studies on protein synthesis in humans suggests that resistance training leads to muscle hypertrophy through the accumulative effect of repeated bouts of an anabolic stimuli. In recent years, this view has been supplemented by evidence suggesting that chronic resistance training also leads to increased rates protein synthesis at rest
[@RN1521;<!-- Kim 2005 Fasted state skeletal muscle PS altered with training  --> 
@RN1866;<!-- Wilkingson 2008 -->
@RN1897],<!-- Reidy 2017 Increased rested state protein synthesis -->
which has been postulated to be associated with an accumulation of ribosomes, i.e. an increased translational capacity
[@RN1897;<!-- Reidy 2017 -->
@RN1912]<!-- Figueiredo 2017 Ribosome biogenesis review -->.
This notion is supported by training-induced increases in muscle RNA abundance.
As the RNA pool to a large degree consists of ribosomal RNA
[@RN2054;<!-- Zak 1967 85% of RNA is rRNA in heart muscle -->
@RN2223]<!-- Young Ch 40 ~ 85% of RNA is ribosomal in rat skeletal muscle -->,
total RNA can be used as a surrogate measure of ribosomal abundance. 

Given the above discussion, the concepts of translational efficiency and capacity can be illustrated by data from Millward
[@RN2145],
wherein RNA concentrations and its association with protein synthesis rates were measured in rats starved or fed a diet containing protein.
Protein feeding increased the rate of protein synthesis, but the relationship to RNA (ribosomal) abundance was largely maintained (Figure \@ref(fig:Millward1973)). This underlines the fundamental importance of ribosomal abundance and activity in determining protein synthesis. 


(ref:millward1973) [@RN2145]

```{r Millward1973, echo=FALSE, message=FALSE, warning=FALSE, fig.height = 2.5, fig.width = 2.5, fig.cap="Relationship between RNA abundance and protein synthesis in rat skeletal muscle. Rats were either starved or fed a protein rich diet stimulating protein synthesis. Data from (ref:millward1973).", fig.scap="Relationship between RNA content and protein synthesis in rat skeletal muscle, data from (ref:millward1973)", fig.align='center'}

millward <- read_excel("./data/litrev.xlsx", sheet = "millward1973") 

source("./R/themes.R")

millward %>%
        mutate(group = if_else(group == "A", "Protein fed", "Protein starved")) %>%
        ggplot(aes(RNA, proteinSynthesis, fill = group, color = group)) +
        geom_smooth(method = "lm", se = FALSE, show.legend = FALSE) + 
        geom_point(color = "black", shape = 21, size = 2.5, alpha = 0.6) +
        scale_x_continuous(name = TeX("$RNA\\;(\\mu g \\; mg^{-1} \\, protein)$"), 
                           breaks = c(0, 4, 8, 12, 16),
                           minor_breaks = c(2, 6, 10, 14),
                           limits = c(0, 16), 
                           expand = c(0, 0)) +
        scale_y_continuous(name = TeX("$Protein\\,synthesis\\,rate\\;(days^{-1})"), 
                           breaks = c(0, 0.05, 0.1, 0.15, 0.20, 0.25),
                           minor_breaks = c(0.02, 0.06, 0.10, 0.14, 0.18, 0.22),
                           limits = c(0, 0.25), 
                           expand = c(0, 0)) + 
        scale_color_manual(values = c(group.study.color[2], group.study.color[5])) +
        scale_fill_manual(values = c(group.study.color[2], group.study.color[5])) +
        

        dissertation_theme() +
                theme(legend.position = c(0.3, 0.9), 
                      legend.title = element_blank(), 
                      legend.text = element_text(size = 7)) 



```


### mTORC1 a multifaceted coordinator of cell growth

The discovery of an organic compound called rapamycin in the 1960’s led to the characterization of a rapamycin sensitive protein involved in cell growth. The protein was later named mechanistic target for rapamycin (mTOR)
[@RN1049].<!-- Goodman review -->
mTOR is found in two protein complexes (mTOR complex 1, mTORC1; mTOR complex 2, mTORC2) where primarily mTORC1 has been found to be sensitive to rapamycin treatment
[@RN1049].<!-- Goodman review -->
Bodine *et al.* performed a comprehensive characterization of mTORC1-mediated skeletal muscle hypertrophy using rodent models, showing that mTORC1 activation was essential for load-induced hypertrophy. Additionally, using transfection techniques, they showed that constitutively activated signaling upstream of mTORC1 (Akt) led to hypertrophy in an mTORC1-dependent manner, confirmed with concurrent administration of rapamycin
[@RN782].<!-- Bodine 2001 -->
Further utilizing rapamycin in genetically modified mice where mTOR was made rapamycin-resistant, specifically in skeletal muscle cells, confirmed that muscle-fiber specific rapamycin-sensitive mTORC1 signaling was needed to induce muscle hypertrophy in response to mechanical loading
[@RN1072].<!-- Goodman transgenic muscle-specific rapamycin insensitive mice confirm that skeletal muscle mTORC1 is responsible for the effect of rapamycin   -->
These mechanistic studies supports previous observational evidence connecting mTORC1 signaling to muscle growth in rats
[@RN866],<!-- Baar Esser -->
and more recently, in humans
[@RN791;<!-- Burd Correlation in single session between p70 and  -->
@RN785].<!-- Terzis correlation between p70S6K and hypertrophy-->

Administration of rapamycin in humans has also confirmed that mTORC1 signaling is important for protein synthesis in the acute phase after resistance training (2 hours) and in response to protein ingestion
[@RN780;<!-- Drummond administration of rapamycin blocks protein synthesis induced by res ex -->
@RN781].
However, extending the time-frame (up to 24 hours), differences in responses to resistance exercise between rapamycin treatment and control conditions were less pronounced
[@RN2826].
This could in part be explained by the lower dosage of rapamycin administrated in human compared to animal trials but also indicate rapamycin-insensitive mechanisms controlling translational efficiency
[@RN2839;<!-- Choo 2008 mTOR/raptor is needed but 4E-BP1 phosphorylation is recovered in spite of rapamycin treatment -->
@RN2836].<!-- Ogasawara 2018 rapamycin is mTORC1/S6K1 specific in inhibition, additional inhibiton of mTOR kinase (both mTORC1 and 2) also leads to complete downregulation of 4E-BP1 -->

mTORC1 functions as a signaling hub by integrating multiple environmental cues to regulate cellular growth. Among such cues is mechanical stimulation which leads to accumulation of phosphatidic acid in muscle cells
[@RN2844].<!-- O Neil 2009 Increase in phosphatidic acid and mTORC1 activity, not affected by PI3K/Akt inhibiton -->
Such accumulation was shown to be independent of regulators upstream of mTORC1 (phosphoinositide 3-kinase (PI3K)/AKT and ERK signaling)
[@RN2844;<!-- O Neil 2009 Increase in phosphatidic acid and mTORC1 activity, not affected by PI3K/Akt inhibiton -->
@RN2119]<!-- ERK inhibition not inhibiting PA induced mTORC1 activation -->
but still readily led to mTORC1 activation
[@RN2844;<!-- O Neil 2009 Increase in phosphatidic acid and mTORC1 activity, not affected by PI3K/Akt inhibiton -->
@RN2119].<!-- ERK inhibition not inhibiting PA induced mTORC1 activation -->
In cellular models, phosphatidic acid has been shown to interact with mTORC1 on the same site targeted by rapamycin 
[@RN2126]<!-- Fang PA leads to activation of mTOR, PA binds to FRB domain (same as rapamycin) -->
indicating a more direct link between mechanical stimulation and mTORC1 activity.
The enzyme primarily responsible for mechanically induced increases of phosphatidic acid has been shown to be diacylglycerol kinase $\zeta$
[@RN1728].<!-- You 2014,   -->

In the context of muscle growth in response to resistance training, adequate supplementation of dietary protein augments responses
[@RN2199].<!-- Morton 2018 meta, ~ 1.6g/kg/day maximizes fat free mass gains in response to RT -->
Mechanistically, dietary protein intake increase the availability of amino acids in muscle cells and these in turn stimulate protein synthesis through mTORC1 by multiple mechanisms
[@RN2848].<!-- Goberdhan cell metabolism perspective, amino acids are sensed through multiple mechanisms by mTORC1 -->
mTORC1 capabilities to fine tune its response based on cellular status can be exemplified from studies examining responses to different amino acid compositions. 
Providing a mixture of essential amino acids potentiated mTORC1 signaling in response to resistance exercise more than provision of essential amino acids without leucin or leucin alone
[@RN1641].<!-- Apro 2015 leucin + EAA vs placebo,  leucin and EAA-leucin. Full mixture leads to the most potent mTORC1 activity (S6K1) -->

In addition to mechanical stimuli and amino acids, mTORC1 integrate several environmental cues related to growth factors, energy and oxygen status with downstream signaling differing depending on upstream signaling and cellular characteristics
[@RN2139].<!-- Laplante mTOR signaling different between tissues and input -->

Two downstream targets of mTORC1 rely much of the information related to translational control one of which is the well characterized eIF-4E (eukaryotic translation initiation factor 4E)-bining protein 1 (4E-BP1). Upon activation, 4E-BP1 releases eIF-4E
[@RN2837]<!-- Akt dependent phosphorylation of 4e-bp1, upon activation eIF-4E no longer associated with 4E bp1 -->
which enables formation of a preinitiation complex and subsequent recruitment of the small ribosomal subunit to mRNA
[@RN2838].<!-- Gingras review 43S recruitment to mRNA is the rate limiting step in translation initiation mediated by eIF4E and thus 4E-BP1 -->
eEIF-4E-dependent initiation of translation is believed to be rate limiting and thus a control point for protein synthesis.
Interestingly, formation of the preinitiation complex, induced by mTORC1-4E-BP1-mediated release of eIF-4E results in enhanced translation of special classes of mRNAs, containing a 5' structures that does not permit efficient translation
[@RN2838;
@RN2840].
Among the resulting gene products from such mRNA are growth factors, cell cycle regulators such as cyclin D1 and c-Myc as well as ribosomal proteins
[@RN2838;
@RN1049].<!-- Goodman review --->

Parallell to 4E-BP1, is another well described downstream target of mTORC1, S6 kinase 1 (S6K1).
S6K1 was named after its ability to phosphorylate ribosomal protein S6, but has since been shown to have multiple roles related to both translational efficiency and indirectly to translational capacity
[@RN1049].<!-- Goodman review --->
The importance of S6K1 in control of muscle mass is apparent from S6K1 depletion in mice that results in reduces muscle growth and experiments showing that constitutevly active S6K1 resulting in increased myotube growth in cell cultures
[@RN2828;<!-- Ohanna  -->
@RN783].<!-- Rommel 2001 Cell culture, constituvely active s6k1 leads to increased  -->
S6K1 deficient mice did however not show a reduced translation of 5'TOP mRNA, an effect still sensitive to rapamycin
[@RN2827].<!-- Pende 2004, S6K1/S6K2 null mice,  -->
Instead, S6K1 deficient mice have been shown to be unable to induce transcription of genes related to ribosomal biogenesis
[@RN2321].<!-- Chauvin S6K1/S6K2 null mice does not respond with ribosomal biogenesis transcription after feeding -->
Upon activation of Akt, such mice fail to respond with increased ribosomal biogenesis, estimated through accumulation of total RNA and confirmed by targeted rRNA analysis
[@RN2849].<!-- Marabita 2016 S6K1 null mice compensatory 4EBP1 activity leads to hypertrophy but abolished ribosomal biogenesis and less strength -->
S6K1 activity also leads to phosphorylation of downstream targets that enables translation initiation and elongation in addition to its most well known substrate ribosomal protein S6 (rpS6)
[@RN2320].<!-- Goodman review 2019 -->
Although a target of mTORC1, rpS6 have been shown to have conterintuitive role in protein synthesis.
Despite its location within the ribosome close to its core, mice genetically modified to be unable to phosphorylate rpS6 upon stimulation still form polysomes indicating that rpS6 phsophorylation is not needed for translational initiation
[@RN2824].<!-- Ruvinsky 2006 review of rpS6 -->
Protein synthesis rates in the same mice are also higher compared to wild type mice suggesting an inhibitory role of rpS6 phosphorylation in protein synthesis
[@RN2824].<!-- Ruvinsky 2006 review of rpS6 -->
Interestingly, mice depleted of S6K1 showed reduced specific force compared to wild-type mice, coinciding with forming of protein aggregates
[@RN2849].<!-- Marabita 2016 S6K1 null mice compensatory 4EBP1 activity leads to hypertrophy but abolished ribosomal biogenesis and less strength -->
Together these observations points to fine-tuning mechanisms in the S6K1-rpS6-axis, balancing protein synthesis, protein quality and energy wastage
[@RN2849;<!-- Marabita 2016 S6K1 null mice compensatory 4EBP1 activity leads to hypertrophy but abolished ribosomal biogenesis and less strength -->
@RN2824].<!-- Ruvinsky 2006 review of rpS6 -->
Fine-tuning may also exist within the mTORC1-S6K1 axis as S6K1 stimulates to mTOR phsophorylation at Ser\textsuperscript{2448}
[@RN1902],
a commonly used read-out for mTORC1 activity
[@RN1949].
Additinally, mTORC1 signaling is sensitive to training status evident from changes in acute signaling in response to resistance training depending on the acute training status
[@RN2320;<!-- Goodman unknown mechanism in training-induced changes in exercise induced mTORC activity changes-->
@RN1809].<!-- Brook abolished mTORC1 induction after 3 and 6 weeks of training -->
After three and six weeks of training, the acute exercise-induced response (60-90 minutes post-exercise) in mTORC1-related signaling is practically abolished in young males
[@RN1809].
Similarly, in well-trained participants accustom to resistance training, acute resistance-exercise does not lead to perturbations along the Akt-mTORC1-axis in comparison to endurance-trained participants
[@RN1871].
These studies are limited in their temporal resolution as only signaling events in the early recovery phase were investigated.
However they indicate that exercise induced mTORC1-signaling is sensitive to aspects relating to training status.

### Ribosomal biogenesis and resistance training induced muscle hypertrophy

The overall role of ribosomal abundance in determining protein synthesis and subsequent cellular and tissue size was briefly mentioned above.
In addition to correlations between RNA abundance and protein synthesis in mice 
[@RN2145]<!-- Millward -->
and cell culture
[@RN1754], <!-- West correlation in cell culture -->
inhibition of ribosomal RNA (rRNA) transcription or inhibition of up-stream transcription factors act to diminish muscle cell growth upon stimulation
[@RN1755<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->;
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1810].<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->
In the context of resistance training-induced muscle growth, observational evidence from human studies further supports a determining role of ribosomal biogenesis to achieve increased translational capacity and enable hypertrophy.
Figueiredo *et al.* observed a correlation between the changes in RNA abundance and magnitude of muscle growth over eight weeks of resistance training
[@RN1644].<!-- Figueriedo 2015 correlation between delta csa and fold change of total RNA -->
Stec and collegues observed increased ribosomal RNA and total RNA abundance only in participants that were classified as modest or extreme responders in terms muscle growth but not low responders after four weeks of resistance training
[@RN1755].<!-- Stec 2016 only high responders have increased total RNA and rRNA abundance after short training -->
Similarly, Mobley *et al.* found larger increases in total RNA in participants classified as high- vs. low-responders to 12 week resistance training (34% vs. 8% increase in total RNA) together with a correlation between total RNA increases and muscle growth over the same time period
[@RN2055].<!-- Mobley weak correlation r = 0.28 p = 0.026  and no cluster * time interaction effect but increases in total RNA 34% in high responders vs 8% in low -->
Finally, Reidy *et al.* reported a correlation between changes in total RNA content and muscle growth
[@RN1897]<!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->
Together these studies underlines the importance of ribosomal biogenesis and translational capacity in resistance training-induced muscle hypertrophy.
It should be noted however, that protein synthesis and cellular growth may occur in the absence of ribosomal biogenesis.
In cultured myotubes stimulated with IGF-1, inhibition of ribosomal RNA transcription led to reduced RNA content but not myotube size compared to non-inhibited controls
[@RN1929].<!-- Crossland inhibition of Pol 1 upon stimulation of IGF-1 does not inhibit growth -->
In aged male participants, three sessions of resistance training did not lead to increased levels of RNA but a 30% increase in protein synthesis rates
[@RN2155].<!-- Welle three sessions over a seven day period no change in RNA but prot synth -->
Furthermore, in response to a greater number of training sessions, absence or reduced ribosomal biogenesis are observed in selected individuals but muscle growth may still be detected
[@RN1809].<!-- Brook -->
However, when e.g. comparing aged and young skeletal muscle, aged muscle typically respond with reduced hypertrophy after mechanical stimuli, coinciding with reduced ribosomal biogenesis
[@RN1809;
@RN1631],
indicating that potent ribosomal biogenesis is needed to support greater hypertrophy.
Similarly, when comparing cell culture experiments, a "broader" stimuli induced by serum compared to a single growth factor could induce greater cellular growth that subsequently require support of an increased translational capacity
[@RN1929;<!-- Crossland -->
@RN1810;<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->
@RN1755].<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->

Synthesis of ribosomes is indeed a hallmark of the early response to resistance training as a single resistance-exercise session leads to increases in precursor rRNA (pre-rRNA 45S)
[@RN1656<!-- A single session upregulates 45 S 24 h post ex -->;
@RN1037<!-- Nader 2014 45S up in response to 4 h after acute exercise -->]
and repeated bouts lead to accumulation of rRNA/total RNA and thus presumably functional ribosomes
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->].
However, a time course of ribosomal transcription and accumulation in response to resistance training in humans remains largely unstudied.
Only a few studies have investigated exercise- or training-induced changes in markers of ribosomal abundance over multiple time-points.
For example, two consecutive bouts of electrically evoked muscle contractions were associated with increased levels of total RNA, with peak values being observed 72 hrs after the second bout
[@RN1520]<!-- Total RNA increase 72 h after a second bout -->. 
Using voluntary contractions, peak values in total RNA were reported after nine sessions, followed by a slight decrease to after 18 sessions [@RN1809].
These data suggest that ribosomes accumulates with some delay from initiation of training, reaches a plateau in the early phase of resistance training (three weeks) and slightly decreases as muscle mass further increases
[@RN1809;
@RN1520].

Synthesis of new ribosomes is a complex and energy demanding process. It involves the synthesis of both ribosomal proteins and four mature rRNA species as well as their assembly into functional ribosomal subunits
[@RN1920;<!-- Warner review yeast -->
@RN1940;
@RN1820].<!-- Moss review -->
All mature rRNA (except rRNA 5S) are derieved from a single pre-rRNA transcript (45S pre-rRNA). After being transcribed from ribosomal DNA (rDNA), the 45S pre-rRNA transcript goes through several "splitting" events ultimately leading to the formation of three rRNA species, 18S, 5.8S and 28S. Coinciding with step-wise splitting of the pre-cursor transcript, modifications to the rRNA structure and assembly with ribosomal proteins into precursor ribosomal subunits occur in the nucleolar compartment
[@RN1940].<!-- Ribosome biogenesis at a glance -->
After export to the cytoplasm, additional maturation steps are required in both subunits before they can form functional ribosomes  
[@RN1940].<!-- Ribosome biogenesis at a glance -->

Ribosomal biogenesis is believed to be determined by the rates of pre-rRNA transcription by RNA polymerase I (Pol I), which in turn is regulated by coordinated assembly of a complex of transcription factors at the rDNA promoter
[@RN1820].<!-- Moss review -->
This process is co-regulated with translation as initiation of rDNA transcription is controlled by mTORC1 through multiple mechanisms.
Activation of the of the upstream binding factor (UBF) through phosphorylation is needed to initiate transcription
[@RN2582; <!-- Tuan UBF phosphorylation important for SL1-UBF interaction -->
@RN2563].<!-- Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
This activation is partly controlled by mTORC1 activity, with its inhibition being associated with blocked UBF phosphorylation and reduced UBF availability trough de-phosphorylation of retinablastoma protein.
This in turn leads to reduced availability and ability of UBF to recruit secondary factors to the rDNA promotor and stimulate rRNA transcription
[@RN1632; <!-- Nader rapamycin block UBF availability  -->
@RN2564].<!-- Hannan mTOR block in postmitotic cardimyocytes -->
mTORC1 also controls one of these secondary factors, TIF-1A, as rapamycin leads to specific phosphorylation and translocation away from the nuclei
[@RN2851],<!-- Mayer rapamycin translocates TIF1-A away from rDNA transcription in HeLa cells -->
in addition to inducing chromatin modulations at the rDNA promotor
[@RN1828].<!--  von Walden 2012 histone acetylation at the promotor inhibited by rapamycin -->

Parallell to mTORC1 control, specific inhibition of MEK showed that MEK/ERK signaling is important for UBF binding to rDNA
[@RN2604].<!-- Stefanovsky 2001 MEK specific inhibition under mitogen activation inhibits rRNA transcriptionand UBF phosphorylation at 117 and 201 -->
Furthermore, the transcription factor c-Myc has also been implicated in ribosomal biogenesis as its inhibition coincides with less UBF activity and rRNA transcription irrespective of mTORC1 signaling
[@RN1754].<!-- West -->
c-Myc is also found at the rDNA promotor and is required for rDNA transcription
[@RN1834;<!--  -->
@RN1828],<!-- von Walden c-myc is enriched at the cdna promotor  -->
in addition to its role in transcription of genes imoportant for rRNA transcription, e.g. UBF
[@RN2565].<!-- c-myc activate transcription of the ubf promoter -->
Interestingly, the availability of UBF *per se* has also been shown to be a determinant of rRNA transcription
[@RN2566;<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->
@RN2564]<!-- mTOR controls UBF availability, rapamycin block serum induced increase in UBF -->
through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->

In summary, ribosomal biogensis as well as the regulation of translation is under coordinated control of several pathways integrating multiple stressors and environmental cues to regulate cellular protein synthesis. 

### Transcriptional regulation of training-induced muscle tissue remodeling (with analytical challanges) 

Skeletal muscle fibers are polynuclear cells. Each fiber contains multiple nuclei which supports its transcriptional requirements. Muscle fiber-specific nuclei (myonuclei) lack the ability to proliferate and are therefore relying on dormant cell populations located in muscle tissue to activate and supply the fiber with transcriptional capacity during growth or in response to injury
[@RN2080].<!-- Blaauw 2014 review satellite cells and hypertrophy -->
Among these cells, satellite cells are the primary source of myonuclear accretion in adult skeletal muscle, however other cell types may also contribute
[@RN2080;<!-- Blaauw 2014 review satellite cells and hypertrophy -->
@RN2864].<!-- Mccarthy ablation of satellite cells still led to some new myonuclei, suggests other cell types contribute --> 
Satellite cells can activate, proliferate or differentiate and fuse with the muscle fiber upon specific stimuli such as hormonal activation
[@RN912]<!-- Egner 2013 steroid treatment leads to increased number of myonuclei -->
or mechanical stress
[@RN2104],<!-- Eftestøl 2016 activation of satellite cells is dependent on mechanical strain isometric force or 50-60% of maximal force -->
or return to their quiescent state
[@RN2080]<!-- Blaauw 2014 review satellite cells and hypertrophy -->
Satellite cells readily activates in humans in response resistance exercise by exiting their quiescent state to proliferate
[@RN2872;<!-- McKay in young G2/M phase increase ~ 70%, G0/G1 ~ 81%, s phase 100% total number of pax7+ cells ~ 83% incr -->
@RN1048;<!-- Snijders 2012 using DLK -->
@RN2081],<!-- Mackey using Pax7, Ki67 and CD56 -->
leading to increased number of satellite cells after resistance training
[@RN2617].

A possible growth limiting role for satellite cells in the context of muscle hypertrophy is to maintain a fixed number of myonuclei per muscle fiber volume through myonuclear accretion, in order to maintain transcriptional capacity (a maintained myonuclear domain)
[@RN2080].<!-- Blaauw 2014 review satellite cells and hypertrophy -->
However, human studies are contradicting that such a role would be a determining factor for hypertrophy.
Kadi *et al.* reported that although the number of satellite cells increased in response to training, the number of myonuclei did not, 
suggesting that existing myonuclei were sufficient to provide transcriptional capacity in hypertrophied muscle
[@RN2004].<!-- Kadi increase in satellite cells, not in myonuclei, more csa per nuclei -->
Similarly, other studies have reported increases in muscle fiber cross-sectional area, without or with minimal concomitant enlargement of the myonuclei pool
[@RN2112;<!-- Damas, no increase in myonuclei in response to resistance training, although 16% increase in type II csa, correlation with mps at tenth week ok training... -->
@RN2874].
In contrast, Petrella and colleagues reported that extreme responders to 16 weeks of resistance training also increased their myonuclear number more than modest and non-responders
[@RN2868].<!-- Petrella 2008 larger increase in myonuclei in high responders, confunded with the fact that high responders were to a large extent young males and low responders older women and men, see @RN765 -->
However, this analysis may have been confounded by the fact that the clustering did not account for age or sex. For example, the extreme-responder group contained 9 young (20-35 years) but only 2 elderly (60-75 years) male participants whereas the non-responder group contained 1 young man and 6 elderly
[@RN765].<!-- description of the age/sex characteristics of clusters in RN2868 -->
This is of importance as satellite cell responses to exercise are different between young and elderly
[@RN2872].<!-- McKay significant age effects in satellite cell response to exercise and at baseline -->
Using a similar clustering-approach as performed in [@RN2868] did not result in differences in myonuclear addition between different response-clusters in a homogeneous sample in terms of age (and sex)
[@RN2055].<!-- Mobley increase in numbers of satellite cell content and myonuclei but no differences between responder clusters -->
A further argument against the idea of a determining role of myonuclear addition in resistance-training induced hypertrophy comes from studies on mice where initial muscle growth induced by synergist ablation is unaffected by specific depletion of satellite cells 
[@RN2864].

In humans, addition of myonuclei paralleled with an increase in global transcription rates from existing myonuclei are likely both contributing to accumulation of total RNA seen after resistance training.
Myonuclear accretion does not, however, seem to be a limiting factor for muscle growth
[@RN2055;<!-- Mobley evidence for myonuclei addition, not different between responder groups  -->
@RN2864].<!-- MacCarthy not a limiting factor for increased transcription in the initial phase of resistance training -->
Existing myonuclei are thus able to maintain transcriptional requirements during hypertrophy, possibly relating to a large degree of transcriptional reserve capacity in muscle fibers
[@RN2618].<!-- Newlands 1998 Transcriptional activity of myonuclei are not maximised, increase in transcription factors increase the chance of a loci being transcribed... -->
Indeed, this reserve capacity of existing myonuclei seems sufficient to induce a global shift in transcription in response to mechanical loading, including both mRNA and rRNA transcription
[@RN2616].<!-- Kirby existing nuclei can increase transcription  -->
Such a shift in global transcription is indeed a determinant of muscle growth as it sets the limit of ribosomal biogenesis and thus the muscle cells translational capacity (as reviewed above).

A change in transcription in response to mechanical loading is not only quantitative (RNA per unit of tissue mass) but also qualitative (changed profiles among e.g. mRNA) in nature with multiple transcriptional programs enabling coordinated adaptation to the specific stimuli.
Such "programming" in muscle tissue affects e.g. muscle fiber type composition where resistance training leads to "shut-down" of *MYH1*, coding for the myosin isoform represented in type IIX fibers
[@RN2056].<!-- Anderssen  -->
The changed, relative proportions of transcripts coding for myosin composition will after remodeling reflect the resulting phenotype
[@RN1489].<!-- Ellefsen -->
Furthermore, transcriptional remodeling of the extracellular matrix is readily affected by resistance training
[@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN2298].<!-- Damas regulation in rested state pre post 10 w training and pre post ex before and after training n = 9-->
Revisiting the role of satellite cells, briefly touched upon in a previous paragraph, highlights that transcriptional regulation of muscle remodeling is a shared venture between multiple cell types when it comes to the extracellular matrix.
If satellite cells are depleted from skeletal muscle of mice subjected to long-term mechanical overload through synergist ablation, a fibrous muscle phenotype is developed with accumulated extracellular matrix components
[@RN1354].<!-- Fry depleting satellite cells results in ECM accumulation, lower specific force (but not single fiber)  -->
Balance between muscle satellite cells, their daughter cells and fibroblasts enables coordination between myofiber hypertrophy and extracellular matrix remodeling
[@RN1354;<!-- Fry depleting satellite cells results in ECM accumulation, lower specific force (but not single fiber)  -->
@RN1888;<!-- Fry muscle satellite cells -> myocytes express mir 206 which in turn downregulate collagen synthesis in fibroblasts -->
@RN2421].<!--  Kjaer fibroblast are the main source of collagen biosynthesis-->
Such coordinated remodeling occurs at least partly through direct exosome transfer of regulating RNA between differentiated satellite cells and fibroblasts
[@RN1888].<!-- Fry muscle satellite cells -> myocytes express mir 206 which in turn downregulate collagen synthesis in fibroblasts -->

In humans, previous studies relating to resistance training and muscle growth/function have investigated a single resistance-exercise session [@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN2298;<!-- Damas regulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> @RN2408], <!-- Hyldahl 2011 acute activation in reponse to 100 eccentric contractions-->
repeated bouts
[@RN2299]<!-- Murton Exercise induced genes with no response in control group-->
and chronic resistance training [@RN2400;<!-- Robinson 2017 differences on common regulation between exercise types in you and old--> 
@RN2298;<!-- Damas reghulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> 
@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN774]<!-- Raue 2012. Differential expression  -->
for changes in transcriptome ("qualitative") characteristics using large-scale techniques (micro-arrays or RNA-seq).
Generally, these studies show that a bout of resistance exercise in the untrained state results in a transcriptome profile related to structural damage, remodeling and inflammation
[@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN2298;<!-- Damas regulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> @RN2408
@RN2299].<!-- Murton Exercise induced genes with no response in control group-->
Long term adaptations generally involves extracellular matrix remodeling, changes expression of genes related to energy metabolism  
[@RN2400;<!-- Robinson 2017 differences on common regulation between exercise types in you and old--> 
@RN2298;<!-- Damas reghulation in rested state pre post 10 w training and pre post ex before and after training n = 9--> 
@RN1825;<!-- Gordon 2012 Diff expr after acute exercise in trained vs. untrained (identifies ECM as higher in trained)-->
@RN774].<!-- Raue 2012. Differential expression  -->
Attempts have also been made to associate transcriptome characteristics and degrees of muscle growth
[@RN774;<!--Raue 2012. 661 genes regulated by exercise and correlated with muscle size and strength--> 
@RN826;<!-- Thalacker-Mercer -->
@RN753],<!-- Phillips 2013 association with  -->
and muscle function
[@RN2402;<!-- Hangelbroek 2018 gene cluster predicts 1RM, training induces expression of ECM/collagen -->
@RN2398].

Although these studies have broaden our understanding of transcriptional regulation during adaptations to resistance training, they also highlight the intrinsic difficulties in studying a dynamic and stochastic process
[@RN2618].
From a methodological point of view, skeletal muscle subjected to e.g. resistance training, concomitant with muscle hypertrophy both exhibit large global scale changes to the RNA pool as well as important *qualitative* changes to RNA sub-populations (mRNA).
A common assumption in studies of gene expression is the relative stability of most transcripts, as well as stability between sub-populations of RNA (mRNA vs. rRNA)
[@RN2359;<!-- Loven 2012 et al  -->
@RN2414].<!-- Robinson 2010 TMM method normalization assumes most genes not affected by treatment-->
<!-- Fix this -->
Such assumptions may not be true in many situations, including during muscle hypertrophy
[@RN2616;<!-- Kirby 2016 nascent RNA is detected in myonuclei after SA (mice) 47S is detected in EU labeled RNA pool -->
@RN2430;<!-- Lin 2012 c-Myc induced gene amplification through -->
@RN1755;<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809]. <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
Depending on the technique used for evaluating RNA expression (e.g. qPCR, micro-array or RNA-seq), data normalization during analysis aims to make experimental conditions comparable using some common denominator.
In cell culture experiments, cell number was suggested to be this denominator for transcript abundances in order to account for global changes in transcription
[@RN2359].<!-- Loven 2012 et al  -->
Despite the fact that some previous investigations, specifically investigating resistance training, have acknowledged a changed total RNA abundance per muscle weight, indicating that a lower amount of tissue being used in analysis in different conditions (trained vs. untrained muscle)
[@RN2298],<!-- Damas regulation in rested state pre post 10 w training and pre post ex before and after training n = 9-->
transcript counts are commonly expressed as per transcriptome.

In summary, both quantitative and qualitative changes in transcriptional profiles determines the cellular phenotype. Skeletal muscle hypertrophy may represent a special case in studies of transcriptome characteristics as it is typically associated with global changes in transcription as well as cellular growth. To fully understand such dynamic systems, first steps should include explicit evaluation of underlying assumptions
[@RN2426;<!-- Conessa 2016 best practices fom RNA seq-->
@RN2359;<!-- Loven 2012 et al  -->
@RN2878]


## Effects of exercise volume on molecular determinants of muscle growth
Given that exercise-training variables can modify responses to resistance training, it is reasonable to assume that these effects are mediated through determinants of muscle growth.
Exercise intensity has been evaluated with respect to protein synthesis and activation of targets downstream of mTORC1 (4E-BP1 and S6K1).
Kumar *et al.* showed that maximal stimulation of fractional synthetic rate was achieved with intensities greater than 60\% of 1RM, coinciding with signaling events
[@RN2720].<!-- Kumar 2009 increasing exercise intensities maximises post ex prot synthesis at 75% in young (plateau at 60-90% of 1RM)  -->
The same group subsequently investigated the effect of training volume at an intensity presumably leading to maximal stimulation of protein synthesis (75% of 1RM).
This analysis revealed a volume-dependent dose-response as six-sets of leg extension led to greater protein synthesis one hour after exercise and sustained S6K1 phsophorylation over up to four hours after exercise
[@RN2716].<!-- Kumar 2012 higher protein synthesis after 6 sets compared to 3 sets coinciding with sustained S6K1 phosphorylation -->
Further extending the time frame, Burd and colleagues evaluated a single session consisting of either one or three sets with biopsies sampled 5, 24 and 29 hour after exercise. Both conditions led to increased myofibrillar protein synthesis five and 29 hours after exercise, but to a larger extent in response to three sets.
However, volume-dependent regulation of S6K1 was only seen at 29 hours after exercise with earlier events (< 5 hours) possibly missed due to the timing of biopsy sampling.
No clear volume-dependency was seen in p90RSK1 (downstream of ERK) or rpS6, however eukaryotic initiation factor 2B (eIF2B$\epsilon$) phosphorylation was reduced only in the three set condition at five hours post exercise [@RN791], presumably mediating translation initiation, although its exact role is still unclear
[@RN2853].<!-- Pavit eIF2B review -->
Volume dependent regulation of S6K1 at Thr389 and rpS6 at Ser235/236 were reported 30 minutes after exercise by Terzis *et al.* as six sets of 6RM bilateral legpress resulted in greater phosphorylation compared to three and one set
[@RN784].
No clear differences between volume conditions were seen in Akt at Ser473, mTOR at Ser2448, ERK 1/2 at Thr202/Tyr204, p38 ($\alpha$,$\alpha$ and $\delta$) at Thr180/Tyr182, p38$\gamma$ Thr180/Tyr182 or AMPK at Thr172
[@RN784].
Corroborating previous observations regarding exercise volume-dependence of the S6K1-rpS6-axis, Ahtiainen and colleagues also found greater phosphorylation of S6K1 at Thr389, rpS6 at Ser235/236 and Ser240/244 30 minutes after exercise with ten compared to five sets of 10RM legpress.
Furthermore, ERK1/2 at Thr202/Tyr204 and p38 at Thr180/Tyr182 phosphorylations were not found to be volume dependent
[@RN1837].
However, in contrast to Terzis *et al.*, they reported volume-dependence in exercise-induced phsophorylation of AMPK$\alpha$ at Thr172, this together with AS160 at Thr642, IRS-I Ser636/639 and Akt at Ser473 together with a tendency in LKB1 at Ser428, all related to cellular energy status, glucose uptake and metabolism
[@RN1837;
@RN2856]<!-- Sano 2003 when unable to phosphorylate AS160 glut4 translocation to membrane is inhibited in  
with relatively high exercise volumes.-->

Collectively, although limited in precision due to small sample sizes (n = 8-19) and temporal resolution and 
<!-- Mitchell n = 18, Burd n = 8, kumar n = 12, Ahtiainen n = 19, Terzis = 8 --> these studies points to volume dependency in exercise-induced S6K1\textsuperscript{Thr389} phosphorylation, potentially further augmented when increased number of sets are examined in the acute phase 
(> 3 sets, < 5 hours; [@RN834; @RN791] vs. [@RN1837; @RN784; @RN2716]).
Increased S6K1 phosophorylation coincides with rpS6\textsuperscript{Ser235/236} and \textsuperscript{Ser240/244} in the acute phase (< 5 hours) after exercise
[@RN784; @RN1837] but not in the late phase (29 hours) [@RN791].
Although rpS6\textsuperscript{Ser235/236} is a known substrate for both S6K1 and ERK1/2 and p90RSK1
[@RN2311],
ERK1/2\textsuperscript{Thr202/Tyr204} was not been found to be volume-sensitive in the acute phase
[@RN784; @RN1837],
nor was P90RSK1\textsuperscript{Thr573} in the late phase [@RN791]
suggesting that signaling along the mTORC1-S6K1-rpS6-axis is more sensitive to variable volume than the parallell MEK/ERK-axis.
The above signaling events suggest exercise volume-dependent activation of the translational machinery, and increased volume indeed leads to increased muscle protein synthesis
[@RN791; @RN2716].
However, relatively high exercise volumes may lead to signaling events inhibiting protein synthesis as AMPK\textsuperscript{Thr172} showed volume dependency, presumably relating to cellular energy stress and glucose metabolism
[@RN1837;
@RN2856;<!-- Sano 2003 when unable to phosphorylate AS160 glut4 translocation to membrane is inhibited in  
with relatively high exercise volumes.-->
@RN2858].<!-- Cork review of cross talk between AMPK and mTORC in nutrient signaling, AMPK can inhibit mTOR -->
Interestingly, an increased AMPK-related signaling may be related to an accumulated effect due to short inter-set rest periods, coinciding with reduced proteins synthesis, at least in the acute phase after exercise
[@RN2857].<!--McKendry 2016 short interset rest leads to increased AMPK and coincides with reduced protein synthesis 0-5 h post exercise -->

In addition to the signaling events described above, training-induced satellite cell activation has been shown to be volume dependent.
Hanssen *et al.* found that training with three sets compared to a single-set per exercise led to more pronounced increase in cells positive for the satellite cell marker CD56 in *m. vastus lateralis* at two and twelve weeks of resistance training [@RN796]. 
This coincided with a greater number of myonuclei after 12 weeks of training, however with no clear differences between volume conditions.
In the same muscle specimen, no clear volume-dependent effect was seen on muscle fiber CSA, however on the muscle level a positive effect of three- compared to one-set was seen for muscle hypertrophy.
[@RN796;
@RN776]







<!--chapter:end:01-Background.Rmd-->

# Aims
<!-- Aim = what you hope to achieve; Objectives the action you will take to achieve the aims --> 
The primary aim of this thesis was to relate the adaptive response to resistance training with low- and moderate-volume to skeletal-muscle characteristics in previously untrained individuals. The key question was whether manipulation of exercise-volume will have diverse effects in different individuals related to muscular intrinsic characteristics. A further aim was to characterize exercise-volume dependence in muscle molecular characteristics and determine a time course profile of markers of ribosomal biogenesis in response to resistance training. <!-- A final aim of the present thesis was to develop an implementation of unbiased reference gene selection to aid gene expression analysis in repeated measures designs   --> Based on these aims, the objectives of the present thesis were;

- to relate skeletal muscle and systemic characteristics to benefit of moderate- compared to low-volume resistance training;
- To determine volume-dependence in molecular networks related to muscle growth and remodeling in response to resistance training
- To determine a time course of markers related to ribosome biogenesis in the early phase of resistance training.

<!-- To create a software implementation of reference gene selection algorithms -->


<!--chapter:end:02-Aims.Rmd-->

---
output: html_document
bibliography: bib/complete-bib.bib
editor_options: 
  chunk_output_type: console
  markdown: 
    wrap: 72
---

# Methods

```{r libs, echo=FALSE, warning=FALSE, message=FALSE}
source("./R/libraries.R")
source("./R/themes.R")
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

## Study overview and participants

Study I was designed to examine effects of low- (single-set per exercise) and moderate-volume (three sets per exercise, multiple-sets) on
responses to acute exercise and long-term training within participants.
A schematic overview of Study I can be seen in Figure \@ref(fig:study1-overview).
Prior to the training intervention, participants reported to the laboratory for bilateral baseline muscle biopsies (*m. vastus lateralis*).
Bilateral muscle biopsies were additionally sampled prior to, and 60 minutes after the fifth training session as well as after the intervention (Figure \@ref(fig:study1-overview)).
Assessments of muscle strength (unilateral isokinetic and isometric knee extension torque as well as unilateral knee extension and leg press one repetition maximum) were performed twice at baseline and further measured in weeks 3, 5, 9 and after the intervention (Figure \@ref(fig:study1-overview)).
Body composition measurements (magnetic resonance imaging and dual-energy X-ray absorptiometry) were performed at baseline and after the intervention.

Study II was designed to study the effects of resistance training *per se* and effects of variable compared to constant inter-session volume on selected markers related to ribosome biogenesis.
Participants were therefore recruited to an experimental group (*n =* 11) and a non-training control group (*n =* 8).
An overview of Study II can be seen in Figure \@ref(fig:study2-overview).
Baseline muscle strength (unilateral isokinetic and isometric knee-extension torque) was assessed during three initial visits to the laboratory, with the last baseline-assessment performed at least seven days prior to the first biopsy sampling. Follow-up measures of muscle strength in the experimental group were completed three and nine days after the last training session. 
Muscle thickness (*m. vastus lateralis*) was measured bilaterally prior to the study as well as two and eight days after the last training session in the experimental group.
Muscle biopsies were sampled bilaterally prior to any training, 48 hours after the first, fourth, fifth, eighth, ninth and twelfth session as well as eight days after the twelfth session (Figure \@ref(fig:study2-overview)).
The control group in Study II performed the same initial assessments as the experimental group. Muscle thickness was again assessed after a control period of 2-4 weeks.
Follow-up muscle biopsies were sampled from one leg 48 hours after the first biopsy as well as after the control period. Follow-up strength assessments were performed 24 hours after the last biopsy.


```{r study1-overview, fig.cap="Schematic representation of Study I, see text for details.  ", fig.scap="Study I, schematic overview", fig.height=2.8, fig.width=4.72, fig.align = "center"}


source("./R/themes.R")


data.frame(x = c(-8, 13), 
           y = c(-0.1, 0.6)) %>%
        ggplot(aes(x, y)) + 
        geom_blank() + 
        
        
        
        ## Horizontal text
        annotate("richtext", x = -1.5, 
                 y = c(0.52,
                       0.47, 
                       0.42,
                       0.3, 
                       0.15, 
                       0.06),
                 label = c("Biopsy sample", 
                           "Muscle mass assessment", 
                           "Strength assessment", 
                           "Training frequency<br>sessions week<sup>-1</sup>", 
                           "Training intensity", 
                           "Within-participant<br>training volume allocation"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) + 
        
        # Measurements
        annotate("richtext", 
                 y = c(rep(0.52, 4), 
                       rep(0.47, 2), 
                       rep(0.42, 6)), 
                 x = c(0, 1.9, 2, 12, 
                       -0.2, 12.5, 
                       -0.2, 0, 2.5, 4.5, 8.5, 12.5), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 4), 
                           rep("&Dagger;", 2), 
                           rep("*", 6))) +
        
        
        
        
        # Training frequency
        annotate("rect", 
                 xmin = seq(0, 11), 
                 xmax = seq(0.8, 11.8), 
                 ymin = rep(0.2, 12), 
                 ymax = c(0.3, 0.35, 0.3, 0.35, 0.3, rep(0.35, 3), 0.3, 0.35, 0.35, 0.3)) +
        
        # Training frequency y-axis
        
        annotate("segment", 
                 x =    c(-0.5, -0.5, -0.5, -0.5, -0.5), 
                 xend = c(-0.5, -0.7, -0.7, -0.7, -0.7), 
                 y =    c(0.2,   0.35,  0.3,  0.25,  0.2), 
                 yend = c(0.35,   0.35,  0.3,  0.25,  0.2)) +
        annotate("text", 
                 x = rep(-0.9, 4), 
                 y = c(0.2, 0.25, 0.3, 0.35), 
                 label = c(0, 1, 2, 3), 
                 size = text.size) +
        
        # Training intensity
        annotate("rect", 
                 fill = c("gray40", "gray60", "gray70"),
                 xmin = c(0, 2, 5), 
                 xmax = c(2, 5, 12), 
                 ymin = 0.12, 
                 ymax = 0.16) +
        
        annotate("text", 
                 x = c(0.1, 2.1, 5.1), 
                 hjust = 0,
                 y = 0.14, 
                 size = text.size,
                 label = c("10RM", "8RM", "7RM")) +
        
        # Training volume 
        
        
        annotate("rect", 
                 fill = c(group.study.color[1], group.study.color[2]),
                 xmin = 0, 
                 xmax = 12, 
                 ymin = c(0.06, 0.01), 
                 ymax = c(0.1, 0.05)) +
        
        annotate("text", 
                 x = 1, 
                 hjust = 0,
                 y = c(0.085, 0.035), 
                 size = text.size,
                 label = c("Moderate volume: Three sets per exercise", 
                           "Low volume: One set per exercise")) +
        
        # Time frame
        
        annotate("segment", 
                 x =    c(0, seq(0, 12)), 
                 xend = c(12, seq(0, 12)), 
                 y =    c(-0.025, rep(-0.025, 13)),
                 yend = c(-0.025, rep(-0.05, 13))) +        
        annotate("text", 
                 x = c(0, 2, 4, 6, 8, 10, 12, 6), 
                 y = c(rep(-0.07, 7), -0.1),  
                 label = c(0, 2, 4, 6, 8, 10, 12, "Week"), 
                 size = text.size) +
        
        
        theme_void()



```




```{r study2-overview, fig.cap="Schematic representation of Study II, see text for details.  ", fig.scap="Study II, schematic overview", fig.height=3.2, fig.width=4.72, fig.align = "center"}


data.frame(x = c(-8, 16), 
           y = c(-0.1, 0.6)) %>%
        ggplot(aes(x, y)) + 
        geom_blank() + 
        
        
     ## Group labels
        
        annotate("richtext", 
                 x = -2, 
                 y = c(0.6, 0.0125), 
                 label = c("<b>Experimental group</b>", 
                           "<b>Control group</b>"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size + 0.5, 
                 hjust = 1) +
        
        
           
        ## Horizontal text
        annotate("richtext", x = -1.5, 
                 y = c(0.52,
                       0.47, 
                       0.42,
                       0.3, 
                       0.15),
                 label = c("Biopsy sample", 
                           "Muscle thickness assessment", 
                           "Strength assessment", 
                           "Training volume<br>sets session<sup>-1</sup>", 
                           "Training intensity"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) + 
        
        # Measurements
        annotate("richtext", 
                 y = c(rep(0.52, 8), 
                       rep(0.47, 3), 
                       rep(0.42, 3)), 
                 x = c(0, 1.5, 4.5, 5.5, 8.5, 9.5, 12.5, 14.5, 
                       -0.2, 12.5, 14.5, 
                       -0.5, 12.9, 14.9), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 8), 
                           rep("&Dagger;", 3), 
                           rep("*", 3))) +
        
        
        # Training volume
        annotate("rect", 
                 fill = group.study.color[4],
                 xmin = seq(0.8, 11.8), 
                 xmax = seq(0.95, 11.95), 
                 ymin = rep(0.2, 12), 
                 ymax = rep(0.3, 12)) +
        
        annotate("rect", 
                 fill = group.study.color[5],
                 xmin = seq(1.05, 12.05), 
                 xmax = seq(1.2, 12.2), 
                 ymin = rep(0.2, 12), 
                 ymax = c(rep(0.3, 4), rep(0.25, 4), rep(0.35, 4))) +
        
        
        # Training frequency y-axis
        
        annotate("segment", 
                 x =    c(-0.5, -0.5, -0.5, -0.5, -0.5), 
                 xend = c(-0.5, -0.7, -0.7, -0.7, -0.7), 
                 y =    c(0.2,   0.35,  0.3,  0.25,  0.2), 
                 yend = c(0.35,   0.35,  0.3,  0.25,  0.2)) +
        
                annotate("text", 
                 x = rep(-0.9, 4), 
                 y = c(0.2, 0.25, 0.3, 0.35), 
                 label = c(0, 3, 6, 9), 
                 size = text.size) +
        
        # Training intensity
        annotate("rect", 
                 fill = c("gray40"),
                 xmin = c(0.8), 
                 xmax = c(12.2), 
                 ymin = 0.12, 
                 ymax = 0.16) +
        
        annotate("text", 
                 x = c(1.1), 
                 hjust = 0,
                 y = 0.14, 
                 size = text.size,
                 label = c("10RM")) +
        

        # Time frame
        
        annotate("segment", 
                 x =    c(1, seq(1, 12)), 
                 xend = c(12, seq(1, 12)), 
                 y =    c(0.1, rep(0.1, 12)),
                 yend = c(0.1, rep(0.09, 12))) +        
        annotate("text", 
                 x = c(1, 6, 12, 6), 
                 y = c(rep(0.07, 3), 0.025),  
                 label = c(1, 6, 12, "Session"), 
                 size = text.size) +
        
        
        # Control group
        
        
  annotate("richtext", x = -1.5, 
                 y = c(-0.075,
                       -0.125, 
                       - .175),
                 label = c("Biopsy sample", 
                           "Muscle thickness assessment", 
                           "Strength assessment"), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 hjust = 1) +
  # Annotations
     annotate("richtext", 
                 y = c(rep(-0.075, 3), 
                       rep(-0.125, 2), 
                       rep(-0.175, 2)), 
                 x = c(0, 1.5,  12.5,
                       -0.2, 12.5, 
                       -0.5, 12.9), 
                 
                 label.padding = grid::unit(0, "pt"),
                 fill = NA, 
                 label.color = NA,
                 label.margin = unit(0, "lines"),
                 label.size = 0,
                 size = text.size,
                 
                 label = c(rep("&dagger;", 3), 
                           rep("&Dagger;", 2), 
                           rep("\U2051", 2))) +
        
        annotate("segment", 
                 x =    c(1.5, 1.5, 12), 
                 xend = c(12, 1.5, 12), 
                 y =    c(-0.25, -0.25, -0.25),
                 yend = c(-0.25, -0.26, -0.26)) +        
        annotate("text", 
                 x = c(6.25), 
                 y = c(-0.34),  
                 label = c("2-4 weeks"), 
                 size = text.size) +
        
        
        
        theme_void()




```


```{r pwr_calc, echo=FALSE, warning=FALSE, message=FALSE}
library(pwr)
# Approximate power calculations that reproduce power calculations made in GPower

# To be able to detect a training-induced 5-unit change (...) given a within-group standard 
# deviation of 12, an alpha-level of 0.05 and a desired power of 0.8, 40 subjects needs to be
# recruited to the experiment in study B.


# Ronnestad 2007
# Isokinetic testing 
sd.iso <- 4 * sqrt(10)
diff.isok <- 6

d.ronnestad.isokinetic <- diff.isok/sd.iso

# pwr.t.test(d = d.ronnestad.isokinetic, power = 0.8, type = "paired")

### Knee extensor CSA
sd.csa <- 2 * sqrt(10)
diff.csa <- 3
d.ronnestad.csa <- diff.csa/sd.csa 

# pwr.t.test(d = d.ronnestad.csa, power = 0.8, type = "paired")

# Mitchell MVC
# This comparison is between 80%, 1-set and 80%1RM 3-set in MVC 
single.pre <- 192
single.post <- 248
  
multiple.pre <- 188
multiple.post <- 255

diff.mvc.mitchell <- (((multiple.post/multiple.pre)-1) - ((single.post / single.pre)-1))*100
d.mitchell.mvc <- diff.mvc.mitchell/sd.iso # Standard deviations borrowed from Rønnestad
# pwr.t.test(d = d.mitchell.mvc, power = 0.8, type = "paired")

# Mitchell CSA

diff.csa <- 3.5
sd.csa <- 2 * sqrt(12)
d.mitchell.csa <- diff.csa/sd.csa
# pwr.t.test(d = 0.47, power = 0.8, type = "paired")


es.min <- round(min(c(d.ronnestad.isokinetic, d.ronnestad.csa, d.mitchell.mvc, d.mitchell.csa)),2)
es.max <- round(max(c(d.ronnestad.isokinetic, d.ronnestad.csa, d.mitchell.mvc, d.mitchell.csa)),2)
```

Recruitment to both studies was done through advertising and word of mouth, primarily at Lillehammer University College/Inland University of Applied Sciences. Potential participants were interviewed and matched against inclusion and exclusion criteria. During initial interviews, participants were informed of the study design and time requirements. Participants were also informed about potential risks and sources of discomfort associated with the study prior to giving their informed consent.

To be eligible for participation in both studies, participants had to be young (Study I age 18-40; Study II 18-35) and non-smoking. Both men and women were considered for participation. Exclusion criteria included a training history of more than one weekly session during the last 12 (Study I) or six (Study II) months leading up to the study. Participants were also screened for intolerance to local anesthetic, current or previous injuries affecting their ability to perform resistance training, self-reported symptoms or history of disease, intake of medications or supplements with known effects on adaptations to training.
Participant characteristics for both studies are shown in Table
\@ref(tab:characteristics-table).

Forty-one healthy individuals were recruited and 34 of these completed at least 85% of the prescribed sessions and were thus included in subsequent data analyses. Reasons for not completing the trial included injury not related to the study (*n =* 1), pain or discomfort during exercises (*n =* 5) and non-adherence to the study protocol. There were no systematic differences in characteristics between participants included in or excluded from data analysis in Study I.

Prior to the study, all participants reported that they previously had been engaged in sporting activities. At enrollment, twenty participants reported regular physical activities ranging from once every other week to four times per week. Ten participants reported performing resistance-type exercise at the time of enrollment limited to no more than once a week.  


```{r characteristics-table, results = "asis", message=FALSE, warning=FALSE, echo=FALSE}

### Read data on included/excluded, body composition and cybex data
characteristics <- read_excel("./data/study-1/body-composition/bodycomp_DXA.xlsx") 
participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";")
load("./data/study-1/strength/cybex.rda")

library(knitr); library(kableExtra)

# Study 1 characteristics
cybex.temp <- data.frame(cybex) %>%
  filter(timepoint %in% c("pre", "session1"), type=="Isom")%>%
  group_by(subject)%>%
  summarise(trq=max(torque.max))

studyI <- participants %>%
  inner_join(characteristics) %>%
  filter(subject!="FP10") %>% # removes participant excluded prior to comecing the study
  filter(timepoint=="pre") %>%
  inner_join(cybex.temp) %>%
  dplyr::select(subject, sex, include, age, height, weight, fat.whole, lean.whole,BMC.whole, trq)%>%
  mutate(fat=((fat.whole/1000)/weight)*100,
         trq.lean=trq/(lean.whole/1000)) %>%
  inner_join(participants) %>%
        mutate(group = if_else(include == "incl", "Included", "Excluded"), 
               sex = factor(sex, levels = c("female", "male"), 
                            labels = c("Female", "Male")), 
               study = "Study I", 
               bmi = weight/(height/100)^2) %>%
        dplyr::select(participant = subject, 
                      study,
                      sex,group, age, height, weight,bmi,
                      fat.whole, 
                      lean.whole,
                      BMC.whole) 

# 
# age.m1 <- lme(age ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# weight <- lme(weight ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# height <- lme(height ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# fat    <- lme(fat ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# mvc    <- lme(trq.lean ~ include + sex, random = list(subject = ~1), data = characteristicsStats)
# 
# summary(age.m1)

# summary(weight)

# summary(height)

# summary(fat)

# summary(mvc)
  
### No differences between included not included.

# Study 2 characteristics 



dxa_whole <- read_excel("./data/study-2/body-composition/tr010_dxa_standarddefinitions.xlsx") %>%
  dplyr::select(Etternavn, Kjønn,Målingsdato,Alder, Vekt:`HeleVev%fett`) %>%
  pivot_longer(names_to = "variable", values_to = "value", cols = Vekt:`HeleVev%fett`) %>%
  mutate(value = gsub("kg", "", value), 
         value = gsub("[[:space:]]", "", value, fixed = FALSE), 
         value = as.numeric(value)) %>%

  separate(Etternavn, into = c("Study", "participant")) %>%
  dplyr::select(participant, Date = Målingsdato, variable, value) %>%
  filter(!(participant %in% c("P08", "P20", "P17"))) %>%
  group_by(participant) %>%
  mutate(time = if_else(Date == min(Date), "S0", 
                        if_else(Date == max(Date), "post1w", 
                                "S12")), 
         leg = if_else(variable == "BeinhøyreMagermasse", "R", "L")) %>%
  filter(variable %in% c("HeleBeinmasse", 
                         "HeleFettmasse", 
                         "HeleVevmasse", 
                         "HeleMagermasse" ,
                         "Vekt", 
                         "HeleTotalmasse")) %>%
  pivot_wider(names_from = variable, values_from = value) %>%
  mutate(fat_rel = 100*(HeleFettmasse/HeleTotalmasse)) %>%
          filter(time == "S0") 




studyII <- read_excel("./data/study-2/body-composition/tr010_age_participant.xlsx") %>%
  inner_join(dxa_whole) %>%
  mutate(age = as.numeric(difftime(recruited, dob)/365.2425), 
         bmi = weight/(height/100)^2, 
         study = "Study II") %>%
          dplyr::select(participant, 
                        study,
                        sex,
                        group,
                        age,
                        height, 
                weight = Vekt,
                bmi,

                fat.whole = HeleFettmasse,
                lean.whole = HeleMagermasse,
                BMC.whole = HeleBeinmasse) %>%
        mutate(sex = factor(sex, levels = c("F", "M"), 
                            labels = c("Female", "Male"))) 
        

# Combine both studies and make table
rbind(studyI, studyII) %>%

        mutate(rel_lean = ((lean.whole/1000) / weight) * 100, 
               rel_fat =  ((fat.whole/1000) / weight) * 100) %>%
        dplyr::select(participant:bmi, rel_lean:rel_fat) %>%
        pivot_longer(names_to = "variable", values_to = "values", cols = age:rel_fat ) %>%

        
        group_by(study, group, sex, variable) %>%
        summarise(m = mean(values), 
                  sd = sd(values)) %>% 
        ungroup() %>%
        mutate(m.sd = if_else(variable == "age", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                              if_else(variable == "bmi", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                      if_else(variable == "height", paste0(sprintf("%.0f", m), " (", sprintf("%.0f", sd), ")"), 
                                              if_else(variable == "rel_fat", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                      if_else(variable == "rel_lean", paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")"), 
                                                              paste0(sprintf("%.1f", m), " (", sprintf("%.1f", sd), ")")))))),
               group = factor(group, levels = c("Included", "Excluded", "experiment", "control"), 
                              labels = c("Included", "Excluded", "Training", "Control"))) %>%
        dplyr::select(study:variable, m.sd) %>%
        pivot_wider(names_from = variable, values_from = m.sd) %>%
        arrange(study, group) %>%
        dplyr::select(study, group, sex, age, height, weight, rel_fat, rel_lean) %>%

  kable(format = "latex", 
        booktabs = TRUE, 
        caption = "Participant characteristics", 
        escape = FALSE, 
        col.names = c(" ", " ", "Sex", "Age (years)", "Stature\n(cm)", "Mass (kg)", "Fat mass (\\%)", "Lean mass (\\%)")) %>%
        
        kable_styling(font_size = 7) %>%
        collapse_rows(1:2) %>%

  footnote(general = "Data are means and (SD)",
           general_title = "", 
           footnote_as_chunk = TRUE)


```



## Resistance training interventions

Studies were fully or partially performed as within-participant studies
as each participant had their legs assigned to different training
conditions (not including the control group in Study II). Allocation was
performed after enrollment where each participant had their legs
randomized to either low- or moderate volume (Study I, see Figure \@ref(fig:study1-overview)), or variable or constant volume (Study II).

Each training session started with a light, standardized warm-up (5 min
ergometer cycling and 10 repetitions each of push-ups, sit-ups,
back-extensions and squats). Before each exercise in the main program,
one set of 10 repetitions were performed in the specific exercise with
approximately 50% of 1RM.

In Study I, the low-volume protocol consisted of a single set of each
exercise and the moderate-volume consisted of three sets per exercise.
Three unilateral leg exercises were used (leg press, leg curl and knee
extension). The moderate volume-leg commenced all sessions and the low
volume-leg performed a single set of each exercise in the rest between
second and third set of the moderate volume training protocol.

In Study II, only unilateral knee-extension was performed. The constant-volume leg performed six sets of 10RM throughout the study and variable leg performed six sets in session one to four, three sets in session five to eight and nine sets in session nine to twelve with same relative intensity (see Figure \@ref(fig:study2-overview)).


## Muscle strength assessments

### Isokinetic and isometric maximal torque
Maximal isokinetic and isometric unilateral knee-extension strength was determined by use of a dynamometer (Study I: Cybex 6000, Cybex International, Medway USA; Study II: Humac Norm, CSMi, Stoughton, MA, USA).
After a brief warm-up (5 min ergometer cycling, RPE 12-14), participants were secured at the hip and shoulders in the dynamometer with the knee joint aligned to its rotation axis.
Individual settings were recorded and used in subsequent measurements.
Participants were familiarized with the test protocol by performing three sub-maximal efforts at each angular speed. 
In Study I, three angular velocities were used to determine isokinetic torque (60$^{\circ}$, 120$^{\circ}$ and 240$^{\circ} ~\times$ sec$^{-1}$), in Study II an angular velocity of 90$^{\circ} ~\times$ sec$^{-1}$ was used for this purpose. 
In Study I, participants performed two attempts at 60$^{\circ} ~\times$ sec$^{-1}$ and three attempts at 120 and 240$^{\circ}~\times$ sec$^{-1}$.
In Study II, three attempts were made at the designated angular velocity.
In both studies, attempts within each angular speed were performed in immediate succession.
After isokinetic testing, the lever was fixed at 30$^{\circ}$ (full extension $=90^{\circ}$), participants were instructed push with maximal effort for 5 seconds and the maximal isometric torque was recorded.
Two attempts were made for maximal isometric torque in Study I and a single attempt was made in Study II.
Sixty seconds of restitution was given between each measurement in both studies with the exception of between isometric contractions in Study I where a 30 second restitution period was used.     

In subsequent assessment sessions, the first measurement was performed on alternate legs.
In Study II, the dynamometer allowed for participants to remain seated for assessments of both legs. This was not possible in Study I as the measurement system required participants to be re-seated for assessment of the contra-lateral leg.

### One-repetition maximum
One repetition-maximum (1RM) was assessed in unilateral leg-press and knee-extension in Study I.
Each exercise was assessed after a specific warm-up (ten, six and three repetitions at $50$, 75 and $85\%$ of the anticipated maximum). Attempts were made with increasing resistance (four to six attempts) and one repetition maximum was defined as the highest resistance successfully lifted through the full range of motion.

### Strength assessment frequency (and statistics)

In Study I, maximal values from each assessment and time-point was used in statistical analyses including two separate assessments at baseline, separated by at least four days. Strength was determined 


In Study II, the maximal value from all successful attempts were used in statistical analyses with unsuccessful attempts identified based on obvious misinterpretation of instructions or failure to reach maximal subjective effort.  
Strength assessments were separated by at least 48 hours from preceding training sessions. 

At baseline, 1RM, isokinetic and isometric strength assessments were performed twice, separated by at least four days. The maximum value achieved for each of the tests was used in subsequent analysis. Strength tests were separated by at least 48 hours from preceeding training sessions. A combined measure of muscle strength was calculated as the average of all tests (1RM, isometric and isokinetic), wherein each test modality was given equal weight. A subset of the participants (n=18) performed strength assessment during the course of the study (at week two, five and nine). For the remaining participants, ordinary training sessions were prioritised when participants missed training or testing due to illness or scheduling difficulties. 

<!-- From TR010 -->

Muscle strength was assessed as maximal voluntary isokinetic (90&deg; sec<sup>-1</sup>) and isometric (60&deg; angle, fully extended leg 0&deg;) knee extension torque. After a brief warm-up (5-min cycling, RPE 12-14), participants were seated and secured in the individually adjusted dynamometer. Participants were instructed to gradually increase their effort during three warm-up repetitions (50, 60 and, 70% of subjective maximal effort). After a 30-sec restitution period participants were instructed to perform three repetitions with maximal effort in the concentric phase. Sixty seconds after the isokinetic test the lever automatically moved to a 60&deg; angle and participants were instructed to push against the lever enough to see feedback from the visual feedback system. After an additional 15-sec restitution period, participants were instructed to push against the lever with maximal effort. Within the same assessment session, participants remained seated in the dynamometer for measurement perfomed on both legs. The first measurement was alternated between legs every other session. For statistical treatment of the data, all successful attempts were used. The last strength assessment at baseline was performed at least seven days prior to the first biopsy sampling. At least one of the baseline strength tests was performed on separate day with two sessions allowed to be perform on the same day with a short rest between assessments. Post training assessments were performed 48 hours and eight days after the last session.


## Measures of muscle mass

In Study I muscle mass was measured by magnetic resonance imaging (MRI)
and dual energy X-ray absorptiometry (DXA) prior to and after the
intervention. Both MRI and DXA measurements were completed during the
same visit to the laboratory. Participants were instructed to refrain
from strenuous physical activity during the last 48 h leading up to the
measurements. The post-training measurements were completed at least 48
h after the last strength testing session. Participants were asked to
refrain from food consumption during 2 h leading up to the measurements.

MRI images were obtained from the mid-thigh and analyzed by the same
investigator blinded for time (pre- and post-training) and condition
(low- and moderate-volume). Multiple images were used to estimate the
cross-sectional area of the extensor muscles at the same distance from
the knee-joint.

See figure <!-- @ref(fig:body-comp-measures)-->

<!--Dallin et al. recently estimated the [@RN2541]
     MRI has shown better agreement with cadaver... see ref in Dallin 

    ## Muscle strength assessments

    Muscle strength was with  -->

In Study II, m. vastus lateralis muscle thickness was measured by B-mode ultrasonography (SmartUs EXT-1M, Telemed, Vilnius, Lithuania) using a 39 mm 12 MHz, linear array probe. At least three images were captured for each leg per time-point. Between each image acquisition the probe was relocated to the same position where on the skin and subsequently marked on a soft transparent plastic sheet superimposed on the thigh.  

Muscle thickness of m. vastus lateralis and m. rectus femoris were measured using  Transverse images were obtained $\sim$ 60 % distally from the trochanter major towards the femoral lateral epicondyle. Landmarks such as moles and scars were also marked on the plastic sheet for relocation of the scanned areas during post-training measurements. During analysis, pre and post images from the same participant were analyzed consecutively using the Fiji software,66 and by two independent researchers. The average muscle thickness of the three images captured per muscle was used for further analyses.

<!-- From tr010 -->
For determination of body composition participants were scanned using DXA (Lunar Prodigy densitometer, GE Healthcare, Madison, WI, USA) with the standard scanning mode (13-25 cm). Participants were lying supine within the scanning bed reference lines, with a strap secured around the ankles to ensure a standardized body position in each scan. The scans were conducted with participants in a fasted state between 07.00-10.00 AM, with empty bladder and wearing only under-wear. Prior to each scan, a phantom scan was run to prevent baseline drifting from affecting analyses. The same technician was used at each time point. Analyses was performed using GE enCORE version 17.0 software (GE Healthcare). Region of interest was customized for covering upper thigh, marked with a sqaure from pubic symphysis to lateral part of tuberculum major, and distal to art. genu.

Muscle thickness (MT) was measured using a B-mode ultra sound unit (SmartUS EXT-1M, Telemed, Vilnius, Lithuania). Participants lay supine in a relaxed position for 20 min before assessments, with their feet strapped in a standardized position. A mark was set on the line 60% of the distance between Spinia Iliac Anterior Superior and the lateral femur condyle. MT of *m.vastus lateralis* was measured applying a water-soluble transmission gel (Aquasonic 100 Ultrasound Transmission Gel; Parker Laboratories Inc., Fairfield, NJ, USA), and a 39 mm 12 MHz ultrasound probe was placed perpendicular to the site of interest without pressing the skin. When the quality of the image was satisfactory, evident as distinct upper and lower muscle fascia, three images were captured, where the probe was relocated to the same position between each image. Position of the probe was marked on the skin and subsequently marked on a transparent paper to ensure similar probe placement for both the right and left *m.vastus lateralis* at subsequent assessments. Analyses were done in ImageJ Fiji [@RN2561] with images cropped and coded to ensure blinding of the assessor.



## Muscle tissue sampling and preparations for downstream analyses

Muscle samples were obtained under local anesthesia (Study I, Xylocaine,
\SI{10}{\mg\per\ml} with adrenalin \SI{5}{\micro\gram\per\ml},
AstraZeneca, Oslo, Norway; Study II, Lidocaine Mylan,
\SI{10}{\mg\per\ml}, Mylan Ireland Ltd, Ireland) with a fine needle
(12-14 gauge; Universal-plus, Medax, Italy) operated with a
spring-loaded instrument (Bard Magnum, Bard Norway AS, Norway). Sampling
was performed as previously described
[@RN824],<!-- Hayot 2005 relible micriboiopsy technique --> with
modifications. Anesthesia was injected in the subcutaneous tissue with
care taken not to inject anesthesia into the muscle itself. Following
pilot experiments we decided not to use an insertion cannula as
described in [@RN824] as the biopsy needle itself could be used to
puncture the skin and muscle fascia. This also resulted in less
discomfort. Several passes through the same skin puncture was made to
obtain sufficient material for downstream analyses. A smaller needle (14
vs. 12 gauge) was used to further minimize discomfort in Study II where
more biopsies were sampled over a shorter time span, with exception from
when material was used for immunohistochemistry. The first biopsy was
sampled at one third of the distance between the patella to the
*anterior superior iliac spinae* with subsequent biopsies sampled
$\sim$\SI{2}{cm} proximal to previous samples. In Study II, samples
obtained more than one week apart were sampled with closer proximity and
distally from previous samples but never at previous sampling sites.

The micro biopsy technique produces smaller samples compared to other
biopsy techniques
[@RN2549]<!--Ekblom describes the Bergström and conchotome -->, and thus
requires several passes to produce sufficient material for multiple
downstream experiments. However, reports confirms that the micro biopsy
technique is comparable to the traditionally used Bergström technique in
several measures of muscle characteristics at the same time as being
well tolerated [@RN824; @RN2553]. Any reported differences in fiber type
distributions between sampling techniques have been suggested relating
to differences in sampling depth [@RN2553; @RN2552].<!-- Hayot -->

For determination of fiber type distributions, a threshold of 200-300
fibers has been suggested as a suitable sample size per specimen as more
fibers does not reduce the variation between dupliacte samples
[@RN874].<!--Blomstrand suggests duplicate samples and at least 200 fibres for determination of fiber distributions -->
In Study I, one or several pieces of muscle (total weight
$\sim$\SI{15}{mg}) were chosen per sampling for analysis of fiber type
distributions (described in detail below). The total number of fibers
were counted from these specimens (Figure ref fig). Using an average of
fibers from the first sampling time point the between leg coefficient of
variation was determined to 14% for Type I fibers and 11.3 for type II
fibers. The between leg variation in Type I fibers is similar to what
has been previously reported...

<!-- Latest paper on variability between samples from the same leg and due to number of fibers counted
Appl Physiol Nutr Metab
. 2020 Apr;45(4):368-375. doi: 10.1139/apnm-2019-0263. Epub 2020 Mar 24.

The within-leg was similar to between leg comparison in our samples. This might highlight sampling depth variation in microbiopsy technique in immunohistochemistry ?

To calculate variation in proportions

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3875499/
-->




#### Total RNA extraction
```{r rna-muscle-weight}

# Descriptive data on total RNA extractions from both studies

study1_sample <- read_excel("./data/study-1/sample_weight.xlsx") %>%
  mutate(rna = (conc1.5 * 5) * elution.volume, 
         sample = paste(subject, leg, timepoint, sep = "_")) %>%
  dplyr::select(sample, weight, rna) %>%
    mutate(study = "Study1") 


complete_sample <- readRDS("./data/derivedData/study-2/tot-rna.RDS") %>%
  filter(outlier != "out") %>%
  mutate(sample = paste(participant, leg, time, sep = "_")) %>%
  group_by(sample) %>%
  summarise(weight = mean(tissue_weight, na.rm = TRUE), 
            rna = mean(rna, na.rm = TRUE)) %>%
  mutate(study = "Study2") %>%
  rbind(study1_sample) 

## Calculate mean weights and rna per weight ##

desc_to_rna <- complete_sample %>%
  group_by(study) %>%
  summarise(m.weight = mean(weight, na.rm = TRUE), 
            s.weight = sd(weight, na.rm = TRUE), 
            m.rna_weight = mean(rna/weight, na.rm = TRUE), 
            s.rna_weight = sd(rna/weight, na.rm = TRUE)) 




## Figure on RNA descriptives

rna_fig_a <- complete_sample %>%
  ggplot(aes(weight, rna, fill = study)) + 
  geom_point(shape = 21, alpha = 0.2) +
    scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
      scale_y_continuous(limits = c(0, 25000), 
                     expand = c(0,0), 
                     breaks = c(0, 5000, 10000, 15000, 20000, 25000)) +
  dissertation_theme() + 
  theme(axis.title.x = element_blank(), 
        axis.text.x = element_blank(), 
        axis.line.x = element_blank(), 
        axis.ticks.x = element_blank(), 
        legend.position = "none") + 
  labs(y = "RNA (ng)")


rna_fig_b <- complete_sample %>%
  ggplot(aes(weight, fill = study)) + geom_histogram() +
  facet_grid(study ~ .) +
  dissertation_theme() + 
  scale_x_continuous(limits = c(0, 80), 
                     expand = c(0,0), 
                     breaks = c(0, 20, 40, 60, 80)) +
  theme(legend.position = "none", 
        strip.background = element_blank(),
        strip.text = element_blank()) +

  labs(y = "Counts", 
       x = "Weight (mg)")

# plot_grid(rna_fig_a, rna_fig_b,
#           ncol = 1,
#          rel_heights = c(1, 0.75), 
#          align = "v", 
#          axis = "trbl")
# 


### Calculate RNA per tissue at baseline
# 
# complete_sample %>%
#   separate(sample, into = c("subject", "leg", "timepoint")) %>%
#   filter(timepoint %in% c("S0", "w0")) %>%
#   mutate(rna_corrected = if_else(study == "Study1", rna * 1.125, rna)) %>%
#   lmer(rna ~ study + weight + (1|subject),
#       
#       data = .) %>%
#   summary()
#   
#   ggplot(aes(rna_corrected / weight, fill = study)) + geom_histogram()
#   
#   
#   group_by(study) %>%
#   summarise(rna_weight = mean(rna/weight), 
#             rna_corrected = mean(rna_corrected/weight)) %>%
#   print()
# 
# 
# 

```

Total-RNA was extracted from frozen muscle samples with weights measured at collection using a protocol modified from
[@RN2672]<!-- Chomczynski and Sacchi -->
using Trizol reagent (Life Technologies).
Muscle tissue was homogenized in \SI{300}{ul} Trizol with mechanical disruption achieved by Zirconium Oxide Beads (0.5 mm, Next Advance, Inc., New York, USA) and a bead mill (Bullet blender, Next Advance). External, non-mammalian RNA (Lambda PolyA External Standard Kit, Takara Bio Europe, Saint-Germain-en-Laye, France) was added with the initial volume of Trizol to enable per-weight normalization in subsequent analyses. After homogenization, additional Trizol was added to a total volume of \SI{1}{ml}. Phase separation was achieved by centrifugation after addition of chloroform (\SI{200}{ul}). The upper phase (\SI{400}{ul} in Study I; \SI{450}{ul} in Study II) was transferred to a fresh tube and RNA was precipitated using isopropanol (\SI{500}{ul}). After incubation (10 min, room-temperature) and centrifugation (12000 g, 10 min at 4$^{\circ}$C) the resulting RNA pellet was washed three times in chilled 75% ethanol.

As previously mentioned, to minimize discomfort with a larger number of biopsies sampled over a short time, a smaller needle was used for most biopsies in Study II. This generally led to a less tissue used for RNA extractions (Figure XX).  


#### Protein extraction immunoblotting 

<!-- TRaining response -->

Aliquots of muscle-tissue (approximately 25 mg wet weight) were homogenised using a plastic pestle in ice-cold lysis buffer (2 mM HEPES pH 7.4, 1 mM EDTA, 5 mM EGTA, 10 mM MgCl~2~, 1$\%$ Triton X-100) spiked with protease and phosphatase inhibitors (Halt, Thermo Fischer Scientific, Life Technologies AS, Oslo Norway), incubated at $4^{\circ}$ for 1 hr and centrifuged for 10 min at 10 000 g and 4$^{\circ}$C, after which the supernatants were collected.
Total protein concentrations were determined on a 1:10 dilution (Pierce Detergent Compatible Bradford Assay Reagent, Thermo Fischer Scientific). The remaining supernatant was diluted to 1.5 $\mu g  \times \mu l^{-1}$ total protein in lysis buffer and 4X Laemmli sample buffer (Bio-Rad Laboratories AB, Oslo Norway) containing 2-Mercaptoethanol.
Samples were heated to 95$^{\circ}$C for 5 min and stored at -20$^{\circ}$C until further processing.
During analyses, protein samples (20 $\mu g$ of total protein) were separated at 300 V for 30 min using 4-20% gels (Criterion TGX, Bio-Rad), followed by wet transfer to PVDF membranes (0.2 $\mu m$ Immun-Blot, Bio-Rad) at 300 mA for 3 h.
Gel electrophoresis and protein transfer were performed at 4$^{\circ}$C.
Membranes were then stained using a reversible total protein stain (Pierce Reversible Protein Stain, ThermoFischer Scientific) to ensure appropriate protein transfer.
Primary antibodies were purchased from Cell Signaling Technology (Leiden, The Netherlands): mTOR (mTOR^Ser2448^: #5536; pan: #4517), S6 kinase 1 (p85 S6K1^Thr412^: #9206; p70 S6K1^Thr389^: #9234; pan: #2708), ribosomal protein S6 (rpS6^Ser235/236^: #4858; pan: #2317).
Membranes were blocked for 2 h in tris-buffered saline (TBS, 20 mM Tris, 150 mM NaCl) containing $3\%$ bovine serum albumin and $0.1\%$ Tween-20, followed by over-night incubation with primary antibodies targeting either the phosphorylated or non-phosphorylated epitope diluted in blocking buffer, followed by 2 h incubation with secondary horseradish peroxidase-conjugated antibodies diluted in TBS containing 0.1% Tween-20 and $5\%$ skimmed milk. Membranes were washed in TBS containing $0.1\%$ Tween-20 for 6 $\times$ 5 min after incubation with primary antibody, and for 8 $\times$ 5 min after incubation with secondary antibodies. For rpS6 and mTOR antibodies, following chemiluminescent detection (SuperSignal™ West Femto Maximum Sensitivity Substrate, ThermoFischer Scientific), membranes were incubated with hydrogen peroxide (15 min, 37$^{\circ}$C) to inactivate the horseradish peroxidase (HRP), as described by Sennepin *et al.* [-@RN1930], followed by over-night incubation with primary or secondary antibodies as described above. If the phosphorylated epitope was targeted during the first incubation, antibodies for the non-phosphorylated epitope were used in the second and vice versa. HRP inactivation did not affect the phosphospecific to non-phosphorylated signal ratios. Importantly, as this technique did not involve removing the first primary antibody, antibodies from different hosts (mouse or rabbit) were used for phosphorylated and non-phosphorylated epitopes respectively. As the antibody targeting p70 S6K1^Thr389^ had the same host as the pan-antibody, total-protein was used to normalise chemiluminescent signals.  All incubation and washing steps were performed at $4^{\circ}$C using an automated membrane processor (BlotCycler, Precision Biosystems, Mansfield, MA, USA), except for p70 S6K1 experiments, which were performed by hand at room temperature with incubations at $4^{\circ}$C. For mTOR and rpS6, total-protein and chemiluminescence quantification was calculated as the mean value of two separate experiments. S6K1 was quantified once for each phospho-specific antibody. Total-protein content was quantified using ImageJ [@RN2259], and was defined as the mean grey value of the whole well with between-well values subtracted as background. Chemiluminescence signals were quantified using Image Studio Lite (LI-COR Biotechnology, Lincoln, Nebraska USA). 

<!-- Tr010 -->

Protein was extracted from Trizol preparations according to manufacturers instructions and [@RN2050]<!-- Kopec trizol protocol --> with modifications. The remaining aqueous phase was removed and DNA was precipitated by the addition of \SI{300}{ul} of absolute ethanol followed by gentle centrifugation (2000 g, 5 min at room temperature). An aliquot of the phenol-ethanol phase, corresponding to ~1.75 mg of tissue, was transferred to to a fresh tube. After addition of at least two volumes of isopropanol and incubation (10 min at room temperature), samples were centrifuged (7500 g, 10 min 4$^{\circ}$C) and a pellet formed. The pellet was washed three times in 95% ethanol with each wash separated by centrifugation (5000g, 5 min at room temperature). After the last wash all liquid was removed and \SI{45}{ul} of Kopec buffer [@RN2050] was added (5% SDS, 10 mM Tris, 140 mM NaCl and 20 mM EDTA, pH 8; containing protease and phosphatase inhibitors). Pellets were incubated at 50$^{\circ}$C for three hours after which the majority of samples were dissolved. Any undissolved material was sedimented by centrifugation (10000 g, 10 min at room temperature). Protein concentrations were measured (Pierce Detergent Compatible Bradford Assay, ThermoFisher Scientific). Sample were normalized in 4X Laemmli buffer (Bio-Rad Norway AS, Oslo, Norway), boiled (95$^{\circ}$C, 5 min) and stored at -20$^{\circ}$C before later use.

## RNA analysis

<!-- TR010 -->
Complementary DNA (cDNA) was synthesized in technical duplicates from 500 ng of total RNA using random hexemer and anchored Oligo-dT primers (Thermo Fisher Scientific) together with Superscript IV (Thermo Fisher Scientific) according to manufacturer's instruction. qPCR reactions were performed with diluted cDNA (2 ul, 1:25 dilution), a SYBR-green based commercial master mix (PowerUp™ SYBR™ Green Master Mix, Thermo Fisher) and, target-specific primers (500 nM) in 10ul reaction volumes using a real-time detection system (QuantStudio 5 Real-Time PCR System, Thermo Fisher Scientific). Fast cycling was used (1 sec denaturing, 30 sec annealing) after UNG (2 min, 50$^{\circ}$C) and polymerase  (2 min, 95&deg;C) activation. Melt curves were collected from all reactions to confirm single product amplification. Primers were further evaluated by agarose gel electrophoresis which confirmed primer sizes and non-template control experiments confirming no amplification without template. Primer sequences and their respective average performances are shown in Table 2.

<!-- Training response -->

Approximately 25 $mg$ of wet muscle-tissue was homogenised in a total volume of 1 ml of TRIzol reagent (Invitrogen, Life technologies AS, Oslo, Norway) using 0.5 mm RNase-free Zirconium Oxide beads and a bead homogeniser (Bullet Blender, Next Advanced, Averill Park, NY, USA) according to the manufacturer’s instructions.
In order to enable analysis of target gene-expression per-unit tissue weight, an exogenous RNA control ($\lambda$ polyA External Standard Kit, Takara Bio Inc, Shiga, Japan) was added at a fixed amount ($0.04  ~ng  \times ml^{-1}$ of Trizol reagent) per extraction prior to homogenisation, as previously described
[@RN1162;<!-- Ellefsen Improved normalization using an external RNA control -->
@RN1512]<!-- Ellefsen Irisin and FNDC5 -->.
Following phase-separation, 400 $\mu l$ of the upper phase was transferred to a fresh tube and RNA was precipitated using isopropanol.
The resulting RNA pellet was washed three times with 70% EtOH and finally eluted in TE buffer.
RNA quantity and purity was evaluated using a spectrophotometer, all samples had a 260/280 $nm$ ratio > 1.95. RNA was stored at -80$^{\circ}$C until further processing. In the analysis of total RNA content per-unit tissue weight, one sample was excluded prior to analysis due to negative deviation from the expected value based on the relationship between sample weight and RNA content suggesting sample loss in washing steps. 
RNA integrity was assessed by capillary electrophoresis (Experion Automated Electrophoresis Station using RNA StdSens Assay, Bio-Rad) with average integrity scores (RQI) 
Five-hundred nanograms of RNA were reverse transcribed using anchored Oligo-dT, random hexamer primers (Thermo Scientific) and SuperScript IV Reverse Transcriptase (Invitrogen) according to manufacturer's instructions.
All samples were reverse transcribed in duplicates and diluted 1:50 prior to real-time polymerase chain reaction (qPCR).
qPCR reactions were run on a fast-cycling real-time detection system (Applied Biosystems 7500 fast Real-Time PCR Systems, Life technologies AS), with a total volume of 10 $\mu l$, containing 2 $\mu l$ of cDNA, specific primers (0.5 $\mu M$ final concentration) and a commercial master mix (2X SYBR Select Master Mix, Applied Biosystems, Life technologies AS). qPCR reactions consisted of 40 cycles (three seconds 95$^{\circ}$C denaturing and 30 seconds 60$^{\circ}$C annealing).
Melt-curve analyses were performed for all reactions to verify single-product amplification. Gene-specific primers were designed for all targets using Primer-BLAST [@RN1815] and Primer3Plus [@RN1816] and ordered from Thermo Scientific, except for the external RNA control, for which primers were supplied with the kit.
Raw fluorescence data was exported from the platform specific software and amplification curves were modelled with a best-fit sigmoidal model using the qpcR-package [@RN1768] written for R [@R-base]. Threshold cycles (Ct) were estimated from the models by the second-derivate maximum method with technical duplicates modelled independently.
Amplification efficiencies were estimated for every reaction [as described by @RN1934; implemented in @RN1768]. For every primer pair, mean amplification efficiencies ($E$) were utilised to transform data to the linear scale using $E^{-Ct}$. Primer sequences and primer characteristics (i.e. average primer efficiencies and Ct values) are presented in Table 2. Gene expression data was log-transformed prior to statistical analysis. As Ct-values, but not efficiencies are related to RNA integrity [@RN2248], RQI scores were used in the statistical treatment of qPCR data to control for potential degradation effects on a by target basis (see below). 




### Hormonal measurements
Hormone analyses were performed on blood samples collected at five time points: alongside muscle biopsies (Figure 1, four sampling events) and 10 minutes after completion of the fifth training session. Samples were drawn from the antecubital vein into serum-separating tubes and kept at room temperature for 30 min before centrifugation (1500 g, 10 min). Serum was immediately aliquoted and stored at -80$^{\circ}$C until further processing. Serum concentrations of total testosterone, cortisol, growth hormone and insulin-like growth-factor 1 (IGF-1) were measured on an Immulite 1000 analyzer, using kits from the Immulite Immunoassay System menu (Siemens Medical Solutions Diagnostics, NY, USA), performed according to manufacturer’s protocols. Serum Vitamin D (S-25-OH-D) levels were measured in samples collected before and after the intervention using a electrochemiluminescence immunoassay (Roche Cobas Vitamin D total assay, Roche Diagnostics GmbH., Mannheim, Germany) using automated instrumentation (Roche Cobas 6000’s module e601, Roche Diagnostics GmbH., Mannheim, Germany). 




## Statistics and data analysis



In Study I, an *a priori* sample-size calculations ($\beta=20\%$, $\alpha=5\%$) indicated that 40 participants was sufficient to detect $\sim$ 3\% and 5\%-point differences between volume conditions in primary outcomes, training induced changes in muscle cross-sectional area and maximal voluntary strength respectively. Sample-size calculations were based on assumed differences between volume condition corresponding to effect sizes of `r es.min`-`r es.max`, as estimated from previous studies [@RN834; @RN776].

In Study II, an initial sample size calculation was made based on a best case scenario. Data from Brook et al. [@RN1642] indicated that a within-participant differences in RNA per tissue weight of $15\%$ could be detected with seven participants in the experimental group (e.g. 2.79 (0.65) to 3.21 (0.74) \SI{}{\micro\gram\per\micro\litre}, $\alpha<0.05$, $\beta<0.2$, effect size=1.2). The control group was included in the study as a negative control primarily for the sake of systems validation of experimental procedures regarding measures of e.g. ribosomal biogenesis. With a balanced design, accounting for drop-outs eight participants were required in each group. After a preliminary analysis of the experiment (experimental group *n = *7, control group *n = *7)[@RN2822], additional participants (experimental group *n =* 4, control group *n = *1) were recruited to the study to increase the precision of estimates, primarily in analyses within the experimental group.




TO DO:

-   For methods discussion, compare product length, efficiencies and ct
    values in relation to RQI-values. See Fleige 2006 for reference.


### Normalization

-   An external reference gene was added at a constant amount in Trizol
    preps
-   A normalization factor was used to express relative target gene
    abundance per-weight tissue.
-   In qPCR the linearised expression (effectivety \^cq) was used to
    express the fraction of external reference per total RNA.
-   In RNA-seq the external reference gene was sequenced and counts were
    used to express external RNA as a fraction of total RNA.
-   In both cases the normalization factor was calculated as mw \*
    counts.

A simulation to see that this is equivalent to tissue used in prep when
no measurement errors exists.

```{r}

library(tidyverse)

expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 250, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) 



expand_grid(mg = seq(from = 5, to = 100, by = 5), 
            rna.mg = seq(from = 50, to = 600, by = 25),
            ext = 0.04) %>%
  mutate(tot.rna = mg * rna.mg, 
         ext.frac = ext / (ext + tot.rna), 
         mg.inprep = 1000 / ((ext + tot.rna) / mg), 
         nf = ext.frac * mg) %>%
         ggplot(aes(rna.mg, mg.inprep,  color = mg)) + geom_point(size = 2)




```

## Meta-analysis of within-session training volume

### Literature search, inclusion criteria and coding of studies

A first set of studies were coded based previously published
meta-analyses [@RN1767; @RN789]. For more recent studies, PubMed, Google
Scholar and SportDiscuss searches were made with search terms being
"training volume", "resistance training", "strength training", "set",
"muscle strength", "muscle hypertrophy" used in different combinations.
Studies examining the effect of within-session training volume on muscle
strength and mass, with all other training variables kept constant
between study groups were considered for inclusion in the meta-analysis.
Studies were further assessed for inclusion based on criteria being; (i)
participants described as healthy without medications affecting muscle
metabolism, (ii) interventions lasting at least 6 weeks and (iii) RT
performed without additional stimuli (e.g. blood flow restriction) at
intensities above 65% of 1RM or 20RM.

All available outcome measures of muscle mass and strength gains in
response to RT were extracted from each study with exception of outcomes
reported both as summaries and individual measures (e.g. muscle
thickness reported as individual muscles and summarized for the whole
muscle group). In such cases the summary was used as outcome. Weekly
training volume was calculated as product of weekly sessions, number of
sets and exercises for each muscle group assessed for muscle hypertrophy
or strength gains. An intervention average of weekly sessions was used
when the number of sessions per week differed over the course of the
intervention. An exercise was assumed to influence an outcome when it
targeted prime movers also assessed for strength or muscle hypertrophy
measures. Participant characteristics were coded based on sex (male,
female or mixed when a study failed to discern between male and
females), age (young, middle-aged, old or mixed), body-mass index (BMI,
calculated from average body mass and height when BMI values were not
available), training status (trained, \> 1 session per week during the
last 6 months leading up to the intervention; untrained \< 1 session).
Study groups were considered independent also in studies utilizing
within-participant models.

### Calculations of effect sizes and statistical analysis

Group-wise effect sizes were calculated for each outcome measure based
on the within-group change score pre- to post-training divided by the
pre-training standard deviation (SD). Pre-training SD's were calculated
as a pooled SD within outcome and study. Variances of the effect size
were calculated using an average effect size across all outcomes within
muscle strength or mass, and correlations specific to each measurement
type (isokinetic-, isometric- or repetition maximum strength tests;
muscle thickness, magnetic resonance imaging, dual energy X‐ray
absorptiometry) estimated from previous studies.\
A correction factor () was applied to both effect sizes and their
variances.

Mixed-effects meta-regression models were used to model the effect of
weekly number of sets on RT-induced muscle mass and strength gains.
Models were fitted in a Bayesian framework using the brms-package
[@RN2562].

<!--chapter:end:03-Methods.Rmd-->

# Results and Discussion

```{r settings, include=FALSE}

library(nlme); library(kableExtra)


```


## Within-session training volume affects training-induced changes in muscle mass and strength and molecular determinants of muscle hypertrophy 

```{r comparing-avg-strength-csa, echo=FALSE, warning=FALSE, message=FALSE}


source("./R/study-1/mr-and-dxa-data.R")


participants <- read.csv("./data/study-1/body-composition/oneThreeSetLeg.csv", sep=";") %>%
        pivot_longer(names_to = "sets", values_to = "leg", cols = multiple:single) 


### Overall % increase in CSA (not corrected for baseline values)
over.all.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(subject) %>%
  summarise(mean.csa = mean(change, na.rm = TRUE)) %>%
  ungroup() %>%
  summarise(m.csa = mean(mean.csa),
            s.csa = sd(mean.csa))

sets.csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>%
  group_by(sets) %>%
  summarise(m.csa = mean(change),
            s.csa = sd(change))



### Baseline corrected % increase
csa.corrected.data <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(change = ((post/pre)-1)*100) %>% # Calculates % change
  group_by(sex) %>%
  mutate(pre = pre-mean(pre)) %>% # Mean centered baseline 
  # (no need for sex-diff in model)
  group_by(subject, sex) %>% # Keeps sex for test in model
  summarise(mean.csa = mean(change, na.rm = TRUE),
            mean.baseline = mean(pre)) # Average over both conditions 



## Anocova with baseline values as co-variate
csa.corr.m1 <- lm(mean.csa ~  mean.baseline,
                  data = csa.corrected.data)

csa.m1 <- lm(mean.csa ~  1,
                  data = csa.corrected.data)



csa.corr.confint  <- round(confint(csa.corr.m1)[1,],1)
csa.corr.estimate <- round(summary(csa.corr.m1)$coef[1,1],1)




csa <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  group_by(timepoint, sets, sex) %>%
  summarise(m=sprintf("%.1f", round(mean(CSA.avg),1)),
            s=sprintf("%.1f", round(sd(CSA.avg),1)))


csa.r <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject, sex, sets, csa.change) %>%
  spread(sets, csa.change)

csa.r.model <- mr.results %>%

  inner_join(participants) %>%
  filter(include=="incl") %>%
  dplyr::select(subject,leg, timepoint, sex, sets, CSA.avg) %>%
  spread(timepoint, CSA.avg) %>%
  mutate(csa.change=((post/pre)-1)*100) %>%
  dplyr::select(subject,leg, sex, sets, pre, csa.change) %>%
  group_by(sex) %>%
  mutate(pre.m = pre-mean(pre, na.rm = TRUE), 
         sets = factor(sets, levels = c("single", "multiple"))) 

#### Change to ancova ###


csa.m0 <-nlme::lme(csa.change ~ sets +  pre.m,
                random = list(subject = ~1),
                na.action = na.omit, 
                data = csa.r.model, method = "ML", 
                control=list(msMaxIter=120,
                             opt = "nloptwrap", msVerbose = FALSE))




csa.estimate <- sprintf("%.2f",round(coef(summary(csa.m0))[2,1], 2))
csa.confint <- intervals(csa.m0)$fixed[2, c(1,3)]

### t-test Does not change the conclusion 
# csa.ttest <- with(csa.r, t.test(multiple,single, paired=T))
# csa.estimate <- sprintf("%.2f",round(csa.ttest$estimate, 2))
# csa.confint <- csa.ttest$conf.int


csa.corr <- cor.test(csa.r$multiple, csa.r$single)


female.t <- with(csa.r[csa.r$sex=="female",], t.test(multiple,single, paired=T))
female.estimate <- sprintf("%.2f",round(female.t$estimate, 2))
female.confint <- female.t$conf.int


male.t <- with(csa.r[csa.r$sex=="male",], t.test(multiple,single, paired=T))
male.estimate <- sprintf("%.2f",round(male.t$estimate, 2))
male.confint <- male.t$conf.int

#### Strength data for overall stats



rm_str <- read_excel("./data/study-1/strength/strengthTests.xlsx") %>%
        filter(!(exercise %in% c("benchpress", "legcurl"))) %>%
        inner_join(participants) %>%
        filter(include == "incl") %>%

  dplyr::select(subject, sex, sets, exercise, timepoint, load)%>%
  mutate(load = as.numeric(load),
         sets = factor(sets, levels = c("single", "multiple"))) %>%
  spread(timepoint, load) %>%
  mutate(baseline = pmax(pre, session1)) %>%
  dplyr::select(subject, sex, sets, exercise, baseline, week2, week5, week9, post) %>%
  gather(week, load, baseline:post) %>%
  mutate(week = factor(week, levels = c("baseline", "week2", "week5", "week9", "week12", "post"))) %>%
  group_by(exercise) %>%
  mutate(str = load / max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) 

# load cybex data 
load("./data/study-1/strength/cybex.rda")


cybex <- data.frame(cybex) %>%
        mutate(leg = if_else(leg == "Left", "L", "R")) %>%
        inner_join(participants) %>%
        filter(include == "incl", 
               timepoint %in% c("pre","session1", "post")) %>%

        filter(!(subject=="FP6" & timepoint == "post" & speed == 240))  # This removes one record, exclusion due to error in measurement, see protocol FP6


# Clean cybex data set and combine with 1RM 
total_strength <- cybex %>%
  ungroup() %>%
  dplyr::select(subject, timepoint, sets,speed, sex, torque.max) %>%
  spread(timepoint, torque.max) %>%
  rowwise()%>%
  mutate(baseline = max(pre, session1, na.rm=T)) %>%
  ungroup() %>%
  dplyr::select(subject, sex, sets, exercise = speed, baseline,  post) %>%
  gather(week, load, baseline:post) %>%
  group_by(exercise) %>%
  mutate(str = load/max(load, na.rm = TRUE)) %>%
  dplyr::select(-load) %>%
  filter(week %in% c("baseline", "post")) %>%
  rbind(rm_str) %>% # This includes all strength tests
  mutate(ex_fam = if_else(exercise %in% c("legext", "legpress"), 
                          "rm", if_else(exercise == 0, "isom", "isok"))) %>%
  group_by(subject, sex, sets, week, ex_fam) %>%
  summarise(str = mean(str, na.rm = TRUE)) %>%
  group_by(subject, sex, sets, week) %>%
  summarise(str = mean(str, na.rm = TRUE)) 


strength.stats <- total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  group_by(sex) %>%
  mutate(BASELINE = baseline - mean(baseline)) %>%
    group_by(sex, subject) %>%
    summarise(increase = mean(increase, na.rm = TRUE),
              BASELINE = mean(BASELINE, na.rm = TRUE)) 




## ANCOVA for the strength 1RM
strength.m1 <- lm(increase ~ sex + BASELINE,
                  data = strength.stats)

strength.corr.confint  <- round(confint(strength.m1)[1,],0)
strength.corr.estimate <- round(summary(strength.m1)$coef[1,1],0)



### Correlation in strength gains ###

str.corr.data <-  total_strength %>%
  spread(week, str) %>%
  mutate(increase = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject, sex, sets, increase) %>%
  spread(sets, increase) 

str.corr <- cor.test(str.corr.data$single,
                          str.corr.data$multiple)

#### Correlation strength and csa increases

# Average increases

csa.str.corr.data <- total_strength %>%
  spread(week, str) %>%
  mutate(str.rel.change = 100 * ((post/baseline)-1)) %>%
  dplyr::select(subject,  sets, sex, sets,baseline, str.rel.change) %>%

  inner_join(mr.results %>%
                mutate(condition=leg) %>%
                inner_join(participants) %>%
                filter(include=="incl") %>%
                dplyr::select(subject, timepoint, sex, sets, CSA.avg) %>%
                spread(timepoint, CSA.avg) %>%
                mutate(csa.rel.change = 100 * ((post/pre)-1)) %>%
                dplyr::select(subject, sets, csa.rel.change)) %>%
        ungroup() 





# The code below could possibly be used to calculate the relative contribution of 
# csa increase to muscle strength increases


# m <- lme4::lmer(str.rel.change ~ csa.rel.change + sets + (1|subject), 
#                 data = csa.str.corr.data)
# 
# 
# 
# mx.null <- lme4::lmer(str.rel.change ~ 1 + (1|subject), data = csa.str.corr.data)
# mx.csa <- lme4::lmer(str.rel.change ~ csa.rel.change  + (1|subject), data = csa.str.corr.data)
# mx.sets <- lme4::lmer(str.rel.change ~ sets + (1|subject), data = csa.str.corr.data)
# 
# 
# 
# boot.fun <- function(x) {
#         data.frame(nlme::VarCorr(x))[2, 4]
# }
# 
# 
# null.boot <- lme4::bootMer(mx.null, boot.fun, nsim = 500, parallel = "snow", cl = 12)
# csa.boot <- lme4::bootMer(mx.csa, boot.fun, nsim = 500)
# sets.boot <- lme4::bootMer(mx.sets, boot.fun, nsim = 500)
# 
# 
# 
# qt.null <- quantile(null.boot$t, c(0.025, 0.975))
# qt.csa <- quantile(csa.boot$t, c(0.025, 0.975))
# qt.sets <- quantile(sets.boot$t, c(0.025, 0.975))
# 
# 
# results <- data.frame(variable = c("sets", "csa"),
#                            m = c(1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4],
#                                  1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / # data.frame(nlme::VarCorr(mx.null))[2, 4]),
#                            lwr = c(1- qt.sets[1] / qt.null[1],
#                                    1- qt.csa[1] / qt.null[1]),
#                            upr = c(1- qt.sets[2] / qt.null[2],
#                                    1- qt.csa[2] / qt.null[2]))
# 
# 
# 
# 
# 
# csa <- 1-  data.frame(nlme::VarCorr(mx.csa))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# sets <- 1-  data.frame(nlme::VarCorr(mx.sets))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# exp <- 1 - data.frame(nlme::VarCorr(mx.exp))[2, 4] / data.frame(nlme::VarCorr(mx.null))[2, 4]
# 
# 

# Average csa and strength gains

avg.str.csa <- csa.str.corr.data %>%
        group_by(subject) %>%
        summarise(str = mean(str.rel.change), 
                  csa = mean(csa.rel.change))


csa.str_corr <- cor.test(avg.str.csa$csa, avg.str.csa$str)

## Comparisons with other published studies on per session gains in csa and strength

### Per session ##
per.session <- read_excel("./data/study-1/trainingdata.xlsx", sheet = 1) %>%
  filter(!is.na(date)) %>%
  group_by(subject) %>%
  summarise(sessions = n()) %>%
        
inner_join(csa.str.corr.data) %>%

  mutate(str.change.day = str.rel.change/((sessions/31) * 12*7),
         csa.change.day = csa.rel.change/((sessions/31) * 12*7), 
         str.session = str.rel.change/sessions,
          csa.session = csa.rel.change/sessions ) %>%
   dplyr::select(-baseline) %>%
  ungroup() %>%
        pivot_longer(names_to = "variable", values_to = "gain", cols = str.rel.change:csa.session) 
       

sum.stats <- cbind(per.session %>%
        group_by(sets, variable, sex) %>%
        summarise(mean = mean(gain), 
                  sd = sd(gain)) %>%
        arrange(variable) %>%
        data.frame(), 
        data.frame(reference.mean = c(0.11, 0.11, 0.11, 0.11,
                              NA, NA, NA, NA, 
                              0.0825, 0.1375,0.0825, 0.1375, 
                              NA, NA, NA, NA, 
                               NA, NA, NA, NA,
                              0.6675, 0.475, 0.6675, 0.475), 
           reference.range = c("[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "[0.04-0.26]a", 
                               "", "", "", "",
                               "(0.22)b", "(0.14)b",  "(0.22)b", "(0.14)b", 
                               "", "", "", "",
                              "", "", "", "",
                               "(0.35)b", "(0.22)b","(0.35)b", "(0.22)b" ))) %>%
        mutate(m.sd = if_else(mean > 6, paste0(sprintf("%.1f",mean), " (", sprintf("%.1f",sd), ")"), 
                              paste0(sprintf("%.2f", mean), " (", sprintf("%.2f",sd), ")")), 
               reference = if_else(is.na(reference.mean), "",
                                   paste0(sprintf("%.2f",reference.mean), " ", reference.range))) %>%
        filter(!(variable %in% c("str.change.day") )) %>%
                       dplyr::select(variable, sex, sets, m.sd, reference) %>%
        
        mutate(variable = factor(variable, levels = c("csa.rel.change", 
                                                      "csa.change.day", 
                                                      "csa.session", 
                                                      "str.rel.change", 
                                                      "str.session"), 
                                 labels = c("CSA %-change", 
                                            "CSA %-change per day", 
                                            "CSA %-change per session", 
                                            "Average strength %-change", 
                                            "Average strength %-change per session")), 
               sex = factor(sex, levels = c("female", 
                                            "male"), 
                            labels = c("Female", "Male")), 
               sets = factor(sets, levels = c("single", "multiple"), 
                             labels = c("LOW", "MOD"))) 

## Ahtiaianen on average  20 weeks with 2 sessions per week = 40 sessions

## Muscle hypertrophy
## In the younger cohorts, male average 5.5% (5.6) = 5.5/40 ~ 0.1375 per session
# in females 3.3 (8.6)% = 0.825
# 8.6/3.3 * 3.3/40
# 5.6/5.5 * 5.5/40
# Muscle strength in younger cohorts
#  male    19% (8.6)/40 sessions ~  0.5 % per session
#  female  26.7% (13.9)/40 ~ 
# 8.6/19 * (19/40)
# 13.9/26.7 * 26.7/40

# Wernbom increase per day in quadriceps CSA per day was 0.06%







```

In Study I, the average increases in muscle strength and mass in each volume condition corresponded to what could be expected based on previous research in young, healthy participants (Table \@ref(tab:csa-str-tab)) 
[@RN2007;<!-- Wernbom 2007 -->
@RN1741],<!-- Ahtiainen  -->
indicating the general efficacy of the training program.


(ref:wernbom-citation) @RN2007

(ref:ahtiainen-citation) @RN1741

```{r csa-str-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

sum.stats %>%
        arrange(variable, sex, sets) %>%
  dplyr::select(-variable) %>%
        kable(format = "latex", booktabs = TRUE, linesep = "",
              col.names = c("Sex", "Condition", "Mean (SD)", "Reference"), 
              caption = "Training induced changes in muscle CSA and average strength in Study I")  %>%
        collapse_rows(c(1, 4))  %>%
        kable_styling(font_size = 7) %>%
          
          pack_rows("CSA %-change", 1, 4) %>%
        pack_rows("CSA %-change per day", 5, 8) %>%
          pack_rows("CSA %-change per session", 9, 12) %>%
          pack_rows("Average strength %-change", 13, 16) %>%
          pack_rows("Average strength %-change per session", 17, 20) %>%
  
        add_footnote(label = c(paste0("Estimates from Wernbom et al. ", "(ref:wernbom-citation)"),
                               paste0("Estimates from Ahtiainen et al. ", "(ref:ahtiainen-citation)")), 
                     notation = "alphabet")

```

The moderate-volume condition consistently showed favorable adaptations when compared to the low-volume condition in measures of muscle hypertrophy and strength gains (Figure \@ref(fig:comb-fig-s1)a).
This was in general agreement with previous meta-analyses indicating that an increased training volume will lead to more favorable outcomes, both in terms of muscle mass
[@RN1767;<!-- Schenfeld  2016--> 
@RN789]<!-- Krieger 2010 -->
and strength
[@RN2492;<!-- Ralston --> 
@RN793].<!-- Krieger 2009 -->

```{r meta-analyses-descriptive, warning=FALSE, message=FALSE, echo=FALSE}

summary_table_meta <- readRDS( "./data/derivedData/meta/naive_models.RDS")


summary_table_meta <- summary_table_meta %>%
  mutate(stat = paste0(sprintf("%.3f", median), 
                       " [",
                       sprintf("%.3f", cri.lwr), 
                       ", ",
                       sprintf("%.3f", cri.upr), 
                       "]"))


```

Since publication of the most recent meta-analyses [@RN1767;@RN2492],<!-- Schenfeld  2016 / Ralston 2107 --> 
additional studies have been published investigating the effect of resistance training volume on muscle mass and strength. Based on studies included in the meta-analysis performed by Schoenfeld *et al.* [@RN1767], additional results were coded based on a systematic literature search among newly published studies (from January 2015 to June 2020). Compared to Schoenfeld *et al.* [@RN1767], an additional ten studies were included in the present analysis.
Subsequently, effect sizes were coded from 25 studies, both for muscle mass and strength and meta-regression models were used to investigate the effect of weekly number of sets on training-induced changes in these outcomes.
In "naive" models, with the weekly number of sets as the only explanatory variable, both muscle mass and strength were shown to be robustly affected by an increased training volume.
For every additional weekly set, the effect size measure (Hedges' *g*, [95% credible intervals] (CrI)), increased by
`r summary_table_meta[1, 5]` and `r summary_table_meta[2, 5]`
for muscle mass (Figure \@ref(fig:comb-fig-s1)b)
and strength (Figure \@ref(fig:comb-fig-s1)c),
respectively.
Together these data underlines the training volume dose-dependence of both muscle hypertrophy and strength gains.

\pagebreak

```{r comb-fig-s1, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences in training-induced relative changes in muscle-mass and strength (a). Estimates are derived from ANCOVA models controling for baseline values. Estimates from meta-regression models examining the effect of weekly number of sets per muscle group on muscle hypertrophy (b) and strength gains (c). Estimates are effect size measures (Hedges g) with 95\\% credible intervals (CrI).", fig.scap = "Differences in training induced changes to muscle mass and strength measures between volume conditions in Study I and weekly training volume meta-regression.", fig.height=6, fig.width=4.72, fig.align='center'}

library(ggplot2); library(ggtext)

# The figure and estimates are from ./R/study-1/muscle-strength-size.R
muscle_strength_fig <- readRDS("./data/derivedData/study1_muscle-strength-size/muscle-strength-fig.RDS")

muscle_strength_fig

```

\pagebreak

In an attempt to explain differences in training outcomes between volume-conditions, selected molecular markers, known to influence adaptations to resistance training were investigated for volume-dependency.
First, acute activation of signaling along the mechanosensitive mTORC1-pathway was measured in before and after the fifth training session in Study I.
A commonly used readout of mTORC1-signaling is the phosphorylation of S6-kinase (S6K1) at Thr\textsuperscript{389}/Thr\textsuperscript{412} which in turn precedes phosphorylation of ribosomal protein S6 (rpS6, see Figure \@ref(fig:mtor-fig)a).
In the present study, exercise-induced S6K1\textsuperscript{Thr\textsuperscript{389}/Thr\textsuperscript{412}} phosphorylation was indeed shown to be volume dependent along with phosphorylation of rpS6\textsuperscript{Ser\textsuperscript{235/236}} and mTOR\textsuperscript{Ser\textsuperscript{2448}} (Figure \@ref(fig:mtor-fig) b).
Together these observations indicates larger perturbations along the mTORC1 signaling axis which confirms previous studies showing exercise-volume dependency of mTORC1-related signaling in the acute phase (< 5 hours) after exercise, with the low-volume condition doing at least three sets activating the muscle under study
[@RN2716;<!-- Kumar 2012 -->
@RN1837;<!-- Ahtiainen 5 vs 10 sets p70 and s6 different -->
@RN784;<!-- Terzis volume affect activation -->
@RN791;
@RN834].
It is recognized that phosphorylation of mTOR itself at Ser\textsuperscript{2448} primarily should be regarded as indicative for negative feedback as this site is phosphorylated due to S6K1 activity
[@RN2309] (Figure \@ref(fig:mtor-fig) a).<!-- Figueriedo mTOR and ser2448 signalling review -->
It is also recognized that the phosphorylation status of rpS6 at Ser\textsuperscript{235/236} is not solely due to mTORC1 signalling as both mTORC1 and extracellular signal-regulated kinases (ERK)-signalling converges here
[@RN2311].<!-- Roux S6 phopshorylation at ser 235 236 promoted by ras/erk -->
Interestingly, Terzis *et al.* and Ahtiainen *et al.* did not report any clear volume dependency in exercise-induced activation of ERK 1/2
[@RN785;<!--Terzis et al., 2008-->
@RN1837], 
suggesting that any volume-dependent regulation of this pathway was outside the time-frame of these studies. However, it might also suggest that volume selectively modulates specific pathways in the acute phase after resistance exercise.
It should however be noted that the present, and previous studies are limited in their temporal resolution and different patterns over time in related to exercise-volume cannot be ruled out.

```{r p70-csa-correlation-data, warning=FALSE, message=FALSE, echo=FALSE}

### Check association with 

p70_csa <- readRDS("./data/derivedData/study1-western-blot/csa_p70.RDS")

### Calculate raw correlations
cor_est_p70 <- p70_csa %>%
  group_by(sets) %>%
   summarise(p70.fc.cor = cor.test(fc, change, method = "spearman")$estimate, 
            p85.fc.cor = cor.test(fc.p85, change, method = "spearman")$estimate, 
            p70.fc.cor.p = cor.test(fc, change, method = "spearman")$p.value, 
            p85.fc.cor.p = cor.test(fc.p85, change, method = "spearman")$p.value)  %>%
  data.frame()

p70.m <- lme(change ~ w2pre + scale(fc) + sets, 
             random = list(subject = ~ 1), 
             weights = varExp(),
             data = p70_csa)

p85.m <- lme(change ~  w2pre.p85 + scale(fc.p85)  + sets , 
             random = list(subject = ~ 1),
             weights = varExp(),
             data = p70_csa)

# plot(p70.m)
# plot(p85.m)


p70.mod <- paste0(sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p70.m)$fixed)[3, 3]), 
                  "]")

p85.mod <- paste0(sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 2]), 
                  " [", 
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 1]),
                  ", ",
                  sprintf("%.2f", data.frame(intervals(p85.m)$fixed)[3, 3]), 
                  "]")

corr_est_p70 <- paste0(sprintf("%.3f",min(c(cor_est_p70[,2],cor_est_p70[,3]))), 
       " - ", 
       sprintf("%.3f",max(c(cor_est_p70[,2],cor_est_p70[,3])))) 

```

The importance of mTORC1 signalling for protein synthesis in the acute phase after resistance exercise is well established.
In humans, administration of rapamycin, a selective inhibitor of mTORC1, prior to resistance exercise leads to delayed or blunted activation of mTORC1 effectors such as
S6K1 and rpS6 as well as unchanged levels of phopshorylation of 4E-BP1 and eEF2 concomitantly with abolished exercise-induced increase in protein synthesis
[@RN780].<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
Volume dependent increase in mTORC1 signalling also coincide with larger protein synthesis rates
[@RN791;<!-- Burd in single session single and multiple sets phosphorylation p70   -->
@RN2716]<!-- Kumar volume affects protein synthesis -->
As such, volume dependence of mTORC1-related signalling suggests that higher within-session volume can be regarded as leading to an increased potential for protein synthesis in the acute-phase after exercise.
Previous studies has indirectly linked mTORC1-related signaling to muscle growth as acute perturbation along this pathway has been correlated with muscle growth
[@RN785;<!--Terzis et al., 2008-->
@RN788].<!-- Mitchell et al., 2013-->
However, this is not a consistent findings in the literature 
[@RN834;<!-- Mitchell et al., 2012-->
@RN2171],<!-- Phillips et al., 2017-->
nor is it consistent with results in the present study as acute activation of S6K1 in response to the fifth session was not associated with training-induced changes in muscle mass.^[Raw Spearman's correlations between exercise induced change in S6K1 phosphorylation and change in muscle mass (measured with MRI) indicated weak (insignificant) associations ($\rho=$ `r corr_est_p70`) depedning on the isoform measured (p70 or p85) and number of sets. More elaborate modeling confirmed weak relationships between degree of phsophorylation and muscle mass change. When controlling for baseline values in degree of phsophorylation (as this influence the calculation of fold-change) and number of sets, standardized estimates for the effect of phosphorylation were (increase in %-muscle mass change for every 1SD change in fold-change phsophorylation, with [95% CI])  $\beta=$ `r p70.mod` for p70 S6K1-p70\textsuperscript{Thr389} and $\beta=$ `r p85.mod` for S6K1-p85\textsuperscript{Thr412}.].

```{r mtor-fig, warning=FALSE, message=FALSE, echo=FALSE, fig.cap="Measured phosphorylation sites in context (a) and differences between volume conditions in phosphorylation status of S6K1 at Thr \\textsuperscript{389} (p70) Thr \\textsuperscript{412} (p85), rpS6 at Ser \\textsuperscript{235 236} and mTOR at Ser \\textsuperscript{2448} induced by acute exercise and expressed as fold-changes (b). Estimates are derived from ANCOVA models controling for baseline values. Values in b are point estimates with 95\\% confidence intervals.", fig.scap = "Differences between volume conditions in exercise induced phosphorylation of proteins related to mTORC1 signaling", fig.height=3.5, fig.width=4.72, fig.align='center'}

western_fig <- readRDS("./figures/results/study1-western-blot.RDS")
western_fig

```

Although mTORC1 signaling most certainly contributes to protein synthesis in the acute recovery phase after resistance exercise [@RN780],<!-- Drummond rapamycin treatment and acute phase recovery (2 hours) -->
its use as a predictor of training induced muscle hypertrophy is complicated by a number of reasons.
First, measuring its activity is not straight forward for biological reasons including pathway cross-talk and negative feedback as well as technical aspects
[@RN1949;<!-- Figueriedo, mTOR feedback -->
@RN2311;<!-- Cross talk with erk -->
@RN1942].<!--  Bass technical considerations in western blotting -->
Secondly, mTORC1 signaling is not a stable phenomenon either between or within individuals as short- and long-term resistance training leads to diminished activity along its pathway
[@RN1809;<!-- mTOR signal lost after three weeks -->
@RN1871].
Thirdly, mTORC1 readily integrate multiple signals but also convey these to multiple downstream processes
[@RN2320].
As such, mTORC1 activity represent an early, transient response to resistance exercise with its signaling contributing to accumulative responses.
One such downstream target of mTORC1 is ribosomal biogenesis
[@RN1632;<!-- Nader mTOR regulates ribo biogenthrough cell cycle regulators -->
@RN1810;<!-- von Walden 2012 mTOR regulates myotube... -->  
@RN1828;<!-- von Walden 2016 -->
@RN2321],<!-- Chauvin S6K controls RiBi transcriptional program -->
ultimately leading to accumulation of ribosomal RNA, a response typically seen in connection with to resistance training
[@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].

```{r ribo-markers}

descriptive_ribo_markers <- readRDS("./data/derivedData/study1-ribo-biogenesis/descriptive_ribo_markers.RDS")

rrna <- descriptive_ribo_markers %>%
  filter(target != "totalrna") %>%
  summarise(min.w2pre = round(min(w2pre),0), 
            max.w2pre = round(max(w2pre),0), 
            min.w12 = round(min(w12),0), 
            max.w12 = round(max(w12),0)) 
```

As ribosomal RNA is the most abundant constituent of muscle RNA it provides an estimate of muscle ribosomal abundance when expressed per unit tissue weight
[@RN2054;<!-- Zak 1967 in heart muscle 85% of RNA is ribosomal RNA -->
@RN2145].<!-- Young -->
From baseline to prior to the fifth training session (Week 2), total RNA per mg tissue increased by $\sim$ `r round(descriptive_ribo_markers[9,2], 0)` and `r round(descriptive_ribo_markers[10,2], 0)`% in the single-set and multiple-sets condition, respectively. Total RNA levels were still elevated above baseline after the intervention (Week 12, single-set $\sim$ `r round(descriptive_ribo_markers[9,3], 0)`; multiple-sets $\sim$ `r round(descriptive_ribo_markers[10,3], 0)`%). Similar patterns were seen in target analysis of ribosomal RNA using qPCR (single-set increase from baseline to Week 2 $\sim$ `r rrna[1,2]`-`r rrna[1,3]`% and Week 12 $\sim$ `r rrna[1,4]`-`r rrna[1,5]`%; multiple-sets $\sim$ `r rrna[2,2]`-`r rrna[2,3]`% and Week 12 $\sim$ `r rrna[2,4]`-`r rrna[2,5]`%). Comparing volume conditions revealed higher levels of total RNA and mature ribosomal RNA species in the multiple-sets condition at Week 2 (Figure \@ref(fig:rrna-fig)a). At Week 12 differences between volume conditions were in total RNA less clear and ribosomal RNA 28S showing higher levels in the single-set leg (Figure \@ref(fig:rrna-fig)a).

```{r rrna-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Differences between volume conditions in total RNA and ribosomal RNA-species (pre-rRNA 45S, rRNA 18S, rRNA 5.8S and rRNA 28S) measured at rest during the course of Study I (a). Acute changes in abundance of c-Myc mRNA and pre-rRNA 45S in response to acute exercise in Week 2 and differences between volume conditions (b). Errorbars represents 95\\% confidence intervals, transparent points and errorbars signifies that the confidence interval contain 1.", fig.scap = "Differences between volume conditions total RNA and ribosomal RNA", fig.height=2.5, fig.width=4.72, fig.align='center'}

ribo_markers_fig <- readRDS("./figures/results/study1-ribo-markers.RDS")

ribo_markers_fig

```

Analysis of c-Myc mRNA abundance in response to the fifth training session also showed volume-dependent regulation with exercise-induced increases being $\sim$ 1.5-fold higher in response to the moderate- compared to the low-volume condition (Figure \@ref(fig:rrna-fig)b). c-Myc represents a rapamycin-insesitive signaling pathway, paralell to mTORC1, known to also stimulating ribosomal biogenesis 
[@RN1754;<!-- West -->
@RN1834;
@RN1832].<!-- cMyc driver of ribosomal biogenesis -->

Ribosomal biogenesis is an early adaptation to resistance training with initial increases in total RNA evident before any hypertrophy can be detected
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN1596;<!-- Defreitas 2011 approx Linear increase in muscle mass and strength in response to 8 wk str training -->
@RN2113].<!-- Stock 2017 Concentric training only for arms muscle thickness and lean mass gains after 6 and 7 sessions-->
The increase in ribosomal RNA represents an accumulation of ribosomes and an increased translational capacity. This improved translational capacity contributes to protein accreetion resulting in muscle hypertrophy, reflected in associations between measures of ribosome abundance (typically total RNA) and muscle hypertrophy
[@RN1644;<!-- Figueriedo 2015 correlation between delta csa and fold change of total RNA -->
@RN1755;<!-- Stec 2016 only high responders have increased total RNA and rRNA abundance after short training -->
@RN2055;<!-- Mobley weak correlation r = 0.28 p = 0.026  and no cluster * time interaction effect but increases in total RNA 34% in high responders vs 8% in low -->
@RN1897].<!-- Riedy 11% increase and correlation with muscle thickness delta 36 sessions young -->
Due to the sequential nature of these events (increased translational capacity before muscle hypertrophy), early increases in total RNA would possible better reflect training-induced changes in muscle hypertrophy.
Any increase in muscle mass would also contribute to a dilution of the RNA pool
[@RN2142]<!-- figueriedo description of dilution effect -->
possibly masking their causal relationship.
Comparing the relationships between RNA abundance and muscle hypertrophy at different time-points supported this view as the amount of RNA at Week 2 showed the strongest relationship with the resulting muscle growth (Figure \@ref(fig:rrna-csa-fig), \@table(rna-csa-tab)). This indicates that translational capacity, affected by exercise, early in the training program is determining training-induced changes in muscle mass.  

```{r rna-csa-tab, warning=FALSE, message=FALSE, echo=FALSE, results = "asis"}

rna_csa_change_models <- readRDS("./data/derivedData/study1-rna-csa-cor/regression.RDS")

rna_csa_change_models %>%
  mutate(Estimate = round(Estimate, 2), 
         SE = round(SE, 2), 
         df = df, 
         t.value = round(t.value, 2), 
         ci = paste0("[", sprintf('%#.2f', lower), ", ", sprintf('%#.2f', upper), "]"), 
         beta = paste0(sprintf('%#.2f',scaled.est), 
                       " [", 
                       sprintf('%#.2f', scaled.lower), 
                       ", ", 
                       round(scaled.upper, 2), "]"),
         beta = if_else(term %in% c("(Intercept)", "setsmultiple"), "", beta)) %>%
  dplyr::select(term, Estimate, SE, df, t.value, ci, beta) %>%
  mutate(term = factor(term, 
                       levels = c("(Intercept)",
                                  "setsmultiple", 
                                        "w0", 
                                        "w2pre", 
                                        "w12"), 
                       labels = c("(Intercept)", 
                                  "Moderate- vs. low-volume",
                                  "Week 0", 
                                  "Week 2", 
                                  "Week 12"))) %>%
  kable(format = "latex", 
        booktabs = TRUE, 
        escape = FALSE,
        caption = "Influence of RNA abundance on training-induced muscle growth measured with MRI", 
        col.names = c("", "Estimate", "SE", "df", "\\textit{t}-value", "95\\% CI", "Estimate [95\\% CI]")) %>%
  add_header_above(c(" " = 6, "Standardized coefficients" = 1), 
                   escape = FALSE) %>%
  pack_rows("RNA abundance (ng mg\\textsuperscript{-1})", 3, 5, escape = FALSE) %>%
  kable_styling(font_size = 8) %>%
  add_footnote("Standardized coefficients scaled by its SD")


```



```{r rrna-csa-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Relationship between total RNA and training induced change in thigh cross sectional area (CSA) as measured by MRI. Dashed line represent naive relationship, solid line represent adjusted reliationship as seen in \\@ref(table:rna-csa-tab).", fig.scap = "Relationship between total RNA and training induced muscle growth", fig.height=4.72/3, fig.width=4.72, fig.align='center'}

rna_csa_fig <- readRDS("./figures/results/study1-rna-csa.RDS")


rna_csa_fig

```

## Volume-dependent remodeling of muscle fiber-type composition

In muscle tissue, multiple-set training led to more pronounced conversion of Type IIX fibres into Type IIA fibres from week zero to twelve than single-set training, measured as both cell counts using immunohistochemistry . Surprisingly, at week two, the relationship between training volume and fibre conversion was the opposite, with single-set legs showing greater IIX to IIA transition. This volume-dependent effect was accompanied by a difference in the abundance of IIX/IIA hybrid fibres at week two, with multiple-set condition showing higher levels (Figure 3C). Notably, from baseline to week two, a pronounced decrease was seen in MYH1 gene expression (coding for the Type IIX myosin-heavy chain transcript) and more so in response to multiple-set training than to single-set training. This change was partly reversed in week twelve (Figure 3D).

To our knowledge, this is the first study to show that muscle fibre transitions from Type IIX to IIA depend on resistance training volume. Moderate volume resulted in %-point greater reductions in Type IIX fibre expression from baseline to post intervention compared to low volume, presumably driven by more pronounced reductions in mRNA expression of the MYH1 
Previous studies have not compared this transition directly between volume protocols. However, Pareja-Blanco *et al.* [-@RN2217] observed blunted IIX $\rightarrow$ IIA transitions in response to non-exhaustive high-load resistance training compared to load-matched training to volatile failure. Together with our data, this makes exercise volume and subsequent metabolic stress and dosage of neuromuscular activity plausible candidates for regulation of IIX $\rightarrow$ IIA reprogramming, as opposed to mechanical stimuli.
Indeed, in rodents, mechanical load does not affect fibre-type transitions
[@RN2104]<!-- Eftestol -->, which is instead linked to neural activation. Interestingly, after two weeks of training, the volume effect on IIX $\rightarrow$ IIA transitions was opposite to our main finding after twelve weeks, with low-volume resistance training resulting in more pronounced decreases in IIX on the cellular level, accompanied by lower abundances of IIX/IIA hybrid fibres. This seemingly early benefit of single-set training on overall IIX levels was not observed at the mRNA level, with MYH1 being more heavily suppressed in the moderate volume condition. Instead, at week two, there seemed to be a disconnection between MYH1 mRNA and IIX protein adaptations in the multiple-set leg compared to the single-set leg. Whether this phenomenon was caused by increased need for tissue-repair in the moderate-volume condition at this time-point
[@RN1521;<!--Kim 2005 -->
@RN2144]<!-- Damas 2016 -->
or other causalities, rather than myofibril-specific adaptations remains unclear. Regardless of causality, these data underline the importance of optimising exercise volume to achieve optimal training progression, such as making use of progressive volume protocols. Although such protocols remain largely unexplored, previously untrained individuals will likely benefit from careful calibration of training volume during early phases of resistance training. Too large or too small a training volume may lead to suboptimal adaptations.

\pagebreak

```{r myhc-fig, warning=FALSE,message=FALSE, echo=FALSE, fig.cap="Relationship between total RNA and training induced change in thigh cross sectional area (CSA) as measured by MRI. Dashed line represent naive relationship, solid line represent adjusted reliationship as seen in \\@ref(table:rna-csa-tab).", fig.scap = "Relationship between total RNA and training induced muscle growth", fig.height=7, fig.width=4.72, fig.align='center'}

# load fiber typing figure
img <- ggdraw() + draw_image("./data/study-1/immuno/immuno_control_image2.png")

# load figures
myhc_fig_comps <- readRDS( "./data/derivedData/study1-myhc-composition/myhc_fig.RDS")


## Combine figure directly in manuscript as saving the png takes some space 

myhc_fig <- plot_grid(plot_grid(img, myhc_fig_comps$overall_type2x, 
                    rel_widths = c(0.72, 0.25), 
                    ncol = 2),
               plot_grid(
                       plot_grid(myhc_fig_comps$myhc2x_mrna_means, 
                                 myhc_fig_comps$diff_2x_mRNA, 
                                 ncol = 1, 
                                 align = "v",
                                 rel_heights = c(0.4, 0.6)), 
                       myhc_fig_comps$fiber_type_insets, 
                       ncol = 2, 
                       rel_widths = c(0.5, 0.7)), 
          ncol = 1, 
          rel_heights = c(0.6, 0.4))
   
myhc_fig

```

\pagebreak






<!--chapter:end:04-results.Rmd-->

# Methodological considerations

## Feasibility of model-based normalization of RT-qPCR-based gene-expression data in a case of human muscle under hypertrophic stress

Quantitative reverse-transcription real-time polymerase chain-reaction (RT-qPCR) is the method of choice for conducting targeted gene-expression studies.
For accurate interpretation of such data the selection of normalization strategy needs to account for biological as well as non-biological variation.
Whereas analysis using endogenous control genes is the most commonly adopted approach, model-based strategies using estimates of technical variation from multiple genes could offer increased sensitivity and accuracy.  
Here, we compare the feasibility gene-specific and model-based normalization strategies for analyzing the effect of two resistance-training programs on abundance of IGF-1 and myostatin mRNA in humans. As for gene-specific normalization, neither of the eleven commonly utilized endogenous control genes investigated exhibited stability suggesting that the use of endogenous internal controls for normalization may affect interpretation. In contrast, using mixed-effects models utilized to estimate technical variation between samples without relying on endogenous controls proved to be feasible. 
Simulations further showed that model-based strategies were computationally robust and had higher power than traditional analysis.


## Increased relevance of RNA-seq data through data-driven selection of 



<!--chapter:end:05-methodological-considerations.Rmd-->

# Conclusion {-}

If we don't want Conclusion to have a chapter number next to it, we can add the `{-}` attribute.

**More info**

And here's some other random info: the first paragraph after a chapter title or section head _shouldn't be_ indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there.


<!--chapter:end:06-conclusion.Rmd-->

# Acknowledgements

<!-- -->

This thesis was primarily realized at Lillehammer University College/Inland University of Applied Sciences and Sykehuset innlandet. Without the help of  

Stian Ellefsen, my main supervisor during these years, I would not have been the same scientist without your enthusiasm. You have thought me that science is fun, and serious, bust mostly fun, but also serious business.  

Bent R. Rønnestad, my co-supervisor

Eva Blomstrand, my co-supervisor at GIH, thank you for taking me on board and for all your support. 

I am grateful for the support from The Swedish School of Sport and Health
Sciences, GIH, especially Kent Sahlin who created a spot for me in your doctorate program and Camilla Norrbin and Bim O’Reilly for keeping track on me during these years. I regret not spending more time at GIH during these years, I always felt at home Lidingövägen 1.

This thesis would not have been possible without the  Jon Elling Whist




Sjur J. Øfsteng was my second office mate at Lillehammer and 

Olav Vikmoen was my first office mate at Lillehammer, he speaks an peculiar dialect native to southern parts of Gudbrandsdalen. This was  



Anne Grete Mathisen, help with blood sampling during training study (knitting a shirt to grandkids between sampling)
Jostein, Caroline, Anne Cecilie  (study I)
to Johanne Seeberg, Stine Dahl, Marianne Bratlien, Martin Nordseth, Erlend Hakestad, Ole-Martin Hveem and Mari Skifjeld  (study II)
Jon Elling Whist, Marita Hanestadhaugen, Lise Koll
Bente Malerbakken, Jostein Flata (R-sykehuset)


Knut Sindre Mølmen 

Nicke W. Almquist

Joar Hansen
Eirik Grindaker
Håvard Nygaard


William Apro thaught everything I know about western blotting! Thank you for being a great mentor and meeting me at the Åstrand laboratory at 5:45 in the morning. 




While working with this thesis I have been fortunate to teach at our bachelor- and master-program in sport science. I would like to thank all devoted students trying to learn Swedish while I was talking about cross-country skiing, strength testing or data science. You have undoubtedly thought me more than I have managed to accomplish in the opposite direction.



Jostein Flata, Bente Malerbakken

During my years at Lillehammer university college (later Inland University of Applied Sciences)^[I do not know who came up with this name, the abbreviation does not read well in English] we planned three different labs and built one. 


Thanks to all my old colleagues at Dalarna University (Anders, Tomas, Magnus, Frej, Michail, ), I hope you have not forgotten about me! Thanks to Emma Hawke for advice on life in general, science and the English language in particular. 


Thanks to my Norwegian family, Torill, Eivind, Stina, Petter, Emil, Anton, Oskar, Ida, Bent, Elvira, Olava and Ingrid for supporting me and Marie during these years.

Marie, I'm sorry for all the late nights, early mornings and weekends i did not spend with you. You have supported me during this time, and we are still together, success! I love you! 





<!--chapter:end:08-acknowledgements.Rmd-->

# Svensk sammanfattning

<!--chapter:end:09-svensksammanfattning.Rmd-->

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->

\backmatter

<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->

# References {-}
<!--
This manually sets the header for this unnumbered chapter.
-->
\markboth{References}{References}
\twocolumn

<!--
To remove the indentation of the first entry.

\noindent
-->
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.


\setlength{\parindent}{-0.20in}


\setlength{\leftskip}{0.20in}
-->
\setlength{\parskip}{4pt}

\rightskip3em

\footnotesize

<!--chapter:end:99-references.Rmd-->

